---
title: "IGHV1-2"
output:
  html_document:
    toc: true
    toc_float: true
runtime: shiny
---









## Allele appearnce 

The group of IGHV1-2 includes 7 alleles, 7 out of the alleles are functional.

For each allele we counted the number of appearances across the population, any appearance was considered valid. 

![plot of chunk unnamed-chunk-4](figure/unnamed-chunk-4-1.png)

## Group alignment

Based on the viewd alleles, we calculated the distance between the germline sequences.

![plot of chunk unnamed-chunk-5](figure/unnamed-chunk-5-1.png)

## Sequence depth

To examine the potential cutoff we observed the sequence depth for each allele

![plot of chunk unnamed-chunk-6](figure/unnamed-chunk-6-1.png)

## 0.5% cutoff 

We set an initial cutoff of $0.5\%$ to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.




With the selected cutoff we saw that there are 4 zygousity states.

preserve4ce86163c6f6f68f

## Observations

This section is editable by clicking on the edit button below. To refresh the section click on the refresh button


```
## [1] TRUE
```

![plot of chunk unnamed-chunk-9](figure/unnamed-chunk-9-1.png)

## Conclusions

From the results we belive that the cutoff for this group should be closer to $2\%$, and for the adjusted states the allele combinations and the relations are stated in the table below.

![plot of chunk unnamed-chunk-10](figure/unnamed-chunk-10-1.png)

For interactive play on the threshold and relative allele fractions, please refer to [this site](https://peresay.shinyapps.io/cluster_app/?url=%22page_allele%22&func_group=%22IGHV1-2%22&allele_thresh=%222%22)
