<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>3 IGHV1-2 - G2 | Functional groups</title>
  <meta name="description" content="3 IGHV1-2 - G2 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="3 IGHV1-2 - G2 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="3 IGHV1-2 - G2 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="01-G1.html"/>
<link rel="next" href="03-G3.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="01-G1.html"><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03"><i class="fa fa-check"></i><b>2.6.1</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="04-G4.html"><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02"><i class="fa fa-check"></i><b>5.6.1</b> Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="11.6.1" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i><b>11.6.1</b> Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./"> Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv1-2---g2" class="section level1" number="3">
<h1><span class="header-section-number">3</span> IGHV1-2 - G2</h1>
<div id="allele-appearnce-1" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Allele appearnce</h2>
<p>The group of IGHV1-2 includes 7 alleles, 7 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-19-1.png" width="672" /></p>
</div>
<div id="group-alignment-1" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-20-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-1" class="section level2" number="3.3">
<h2><span class="header-section-number">3.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-960f01b2580f8371ec6b" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-960f01b2580f8371ec6b">{"x":{"data":[{"x":[2,2,3,1,2,2,1,2,1,2,1,2,1,2,1,2,1,2,2,4,2,1,2,1,2,4,2,1,2,1,2,1,5,4,1,2,4,2,4,2,3,1,2,4,3,1,3,4,4,2,1,2,4,2,2,1,4,1,4,1,4,1,2,1,2,2,1,2,1,4,1,1,2,4,1,2,1,2,1,2,1,4,4,1,1,2,4,2,1,1,1,2,1,4,5,1,1,1,3,4,1,4,1,2,1,2,4,1,4,1,2,4,2,1,2,1,1,1,4,2,3,1,2,1,2,2,3,4,2,1,2,4,2,1,2,4,4,2,2,3,1,2,2,1,2,1,2,2,4,1,1,2,1,2,1,1,3,1,4,2,2,4,2,1,1,2,1,2,2,1,2,1,3,1,2,1,2,4,4,2,3,1,2],"y":[100,213,16,259,48,512,673,139,820,178,117,42,120,42,416,74,93,29,120,373,180,289,77,475,151,46,21,950,212,449,199,692,611,1,783,200,2,187,576,207,1,79,25,500,27,396,21,1,344,229,530,156,11,1,32,474,409,103,78,270,1,241,102,327,272,283,840,286,409,326,887,319,122,1,152,40,413,45,80,11,418,292,190,155,1045,248,96,37,927,359,678,171,848,1,1,1114,985,693,17,1,126,95,1372,288,918,191,1,33,30,994,2,1,314,566,211,879,876,669,671,297,1,215,31,571,178,175,21,834,347,739,235,1,153,594,220,1,263,146,192,24,392,75,464,454,77,1570,633,242,372,314,348,141,623,187,565,336,11,1454,1,602,411,322,118,1503,774,177,267,58,250,332,72,250,14,32,6,1577,1,1,452,55,1,58,12],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[2.01860391739756,1.98269096454605,3.01784459776245,0.908178895525634,2.0602408288978,2.01695770076476,1.05617008092813,2.01645645620301,1.03382962350734,1.96513802395202,0.956803916208446,1.90628897617571,1.04602003600448,1.90550392880104,1.03869730075821,2.09806055398658,0.955419062962756,2.07109637521207,2.00324462167919,3.95217944015749,2.07429797635414,0.950956084532663,1.9066919837147,1.00605160333216,2.00308902813122,3.95392451677471,2.02077873833478,1.08768077604473,2.04694856586866,0.929778525233269,1.96137567013502,1.07353762700222,4.99448429858312,4.03104835203849,1.04269680320285,2.04323617843911,3.96938186767511,2.09889380680397,3.97861426309682,1.98221050561406,2.97918849829584,0.970677283778787,1.96637464421801,4.05897917244583,3.00658621625043,0.969366036960855,3.01566967251711,3.99682413502596,4.02970393360592,1.93082280494273,0.912351304478943,2.09776072325185,4.09334142371081,1.97564410506748,2.01723684612662,0.964723055949435,4.03483889098279,0.983528487943113,4.03255242626183,0.998027673782781,4.04815650843084,0.930756546417251,2.08892074264586,1.05824500499293,2.09558758358471,1.91449395567179,0.965218652645126,1.99692128892057,1.02522868504748,4.05687636374496,1.09928967407905,1.02739212806337,1.95022790227085,4.07129900264554,1.02556752944365,1.95411103270017,1.08240389325656,1.90700517240912,1.09419864155352,2.05649522556923,1.03735037343577,4.0660044267308,4.00165351037867,0.926525048539042,1.00748088192195,2.03135301158763,4.08600519141182,1.91174201052636,0.947264396864921,1.08654016708024,1.09494922426529,2.09286147211678,0.901013281010091,3.94445355255157,5.05969929303974,0.983067056257278,1.04800993553363,0.980357147566974,3.04848394831642,3.94791760216467,0.980573204997927,3.98210468380712,1.08613098929636,1.96484057898633,1.06553870914504,2.03481051037088,4.08508010338992,1.01322107492015,3.93919433760457,1.0601954640355,2.06536962673999,3.90683021945879,2.00600351579487,1.03304940583184,1.90323092192411,0.96610070890747,0.957284815469757,1.02589283231646,3.96111144958995,2.01033560996875,2.94101572078653,1.06139945434406,2.02425598241389,0.962249674741179,1.98986789449118,1.97904030103236,3.01189722148702,3.91097782854922,2.05148568223231,0.994669374683872,1.91768409679644,4.09743177010678,2.02988005802035,1.07597962529399,1.90126559352502,3.90882182177156,3.95534415962175,2.03678414491005,2.04125454481691,3.0854332418181,1.06826350954361,1.90001145065762,2.04542811303399,1.00044797561131,2.01340756439604,1.04854088239372,1.91908691432327,1.95961719346233,4.09844687664881,0.954876733664423,1.03226338168606,2.0494970574975,1.03088438534178,1.91218358841725,1.06367010716349,1.07868869425729,2.90717386375181,0.953584752231836,4.0492982944008,1.90504776737653,1.99283224474639,4.08259289148264,1.9278252478689,1.07416346711107,1.02014381354675,2.05365515924059,1.08284117057919,1.90906913965009,1.92603312605061,0.957435819506645,1.90909989350475,0.942727877805009,2.91199417631142,0.977471858123317,1.91395525080152,0.924201891990378,1.99995986572467,3.98699600617401,4.07954857209697,1.90422356631607,2.93785291407257,1.08787146178074,1.96579796201549],"y":[100.198566678353,213.089376855269,15.6417740084231,258.85035044495,48.2769138297066,512.035217499919,673.064153366163,138.729070637189,820.265879208967,177.629519580863,117.323740352504,42.2440936587751,120.043042799085,42.3394371293485,415.908734498918,73.8062028557062,93.0782274851576,29.2715763578191,119.826550518163,373.087181963585,180.17839627862,288.808953675069,77.1491533853114,475.183213094249,151.117610347085,45.7702445728704,21.016339885816,950.358552600257,211.943475353904,449.280527283438,198.958186144941,691.791265191324,610.862686315924,1.35310195423663,782.864925266802,199.932957414724,1.89686763640493,186.615037194453,576.299848158658,207.091278638504,0.788421110995114,79.1536527393386,25.3772156935185,500.133537935279,27.0597544744611,396.093751828931,20.7703883187845,1.00183993838727,343.716262093745,228.855695181154,529.895791067556,155.603793456405,10.7668823363259,1.27645106054842,31.6449134135619,474.392291360907,408.718429073319,103.038917380944,77.6934651339427,269.874745639041,1.06325086764991,241.194741190225,101.872623294219,326.839923483692,272.343750798888,283.376328726485,840.267746542953,285.696470913291,409.133392461017,325.808574387431,886.983326665685,318.837757635862,122.023245021515,1.09179010950029,151.809972480126,39.9591926453635,412.999574105255,44.9480301521719,80.1789302561432,11.041916616261,417.910915701278,292.063269695081,189.6306383349,154.705932153575,1045.31787452996,247.680588554405,96.0410196799785,36.7220293084159,926.835142769665,358.902000043541,678.249022690207,171.268861485086,848.08549154345,0.626234166882932,1.2632168572396,1114.0170895081,984.686580809206,693.23497852087,17.287702286616,0.750917424447834,126.37086171452,94.959907711111,1371.80448408462,288.12740948908,917.739504436031,191.028458077274,0.855821055360138,33.1979944879189,30.1436916181818,993.685027825087,1.78662541359663,1.37108969334513,313.68594638072,565.751718055271,210.832143733837,878.723543458618,876.310240794532,669.165199207515,671.078911933862,296.783015166409,0.783685102686286,214.763196449541,30.7066882530227,570.991036103666,178.26138746161,175.145610804856,21.3842136442661,834.34842036292,347.035890292376,739.334445701726,235.192027279735,1.1964449968189,152.944988106191,593.98118518088,220.035104868561,0.961135339550674,263.213188630342,145.61921865996,191.882597370259,24.1959645224735,392.161785082705,75.1440470112488,463.986140250415,453.666455459408,77.3326036872342,1570.38673006371,632.901129639894,242.062323754653,371.975250974856,314.22024783399,348.339293066226,141.157209607773,622.715880616195,187.115260132216,565.082501170971,335.885487185605,10.9511263387278,1454.20633809622,1.09754568524659,602.026709768735,411.189184542745,321.957749716192,117.912114072964,1502.97256942727,773.825744329579,177.235175994225,267.327405284345,58.2680449001491,250.022760401107,331.861602152325,71.7150975698605,250.353202143125,14.208175367862,31.8564796056598,5.70088651124388,1577.39531593863,1.15711226835847,1.10262819640338,452.014830986969,55.140058247745,0.667838842980564,58.2056756351143,11.7783838856965],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  100 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  213 <\/br>Relative freq.:  0.9301 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  05 <\/br># assignments:  16 <\/br>Relative freq.:  0.0699 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  02 <\/br># assignments:  259 <\/br>Relative freq.:  0.8436 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  04 <\/br># assignments:  48 <\/br>Relative freq.:  0.1564 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  512 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0215","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  673 <\/br>Relative freq.:  0.8288 <\/br>Relative Rep. freq.:  0.0282","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  139 <\/br>Relative freq.:  0.1712 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  820 <\/br>Relative freq.:  0.8216 <\/br>Relative Rep. freq.:  0.0334","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  178 <\/br>Relative freq.:  0.1784 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  117 <\/br>Relative freq.:  0.7358 <\/br>Relative Rep. freq.:  0.0194","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  42 <\/br>Relative freq.:  0.2642 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  120 <\/br>Relative freq.:  0.7407 <\/br>Relative Rep. freq.:  0.0193","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  42 <\/br>Relative freq.:  0.2593 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  416 <\/br>Relative freq.:  0.849 <\/br>Relative Rep. freq.:  0.0292","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  74 <\/br>Relative freq.:  0.151 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  93 <\/br>Relative freq.:  0.7623 <\/br>Relative Rep. freq.:  0.0165","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  29 <\/br>Relative freq.:  0.2377 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  120 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  06 <\/br># assignments:  373 <\/br>Relative freq.:  0.6745 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  180 <\/br>Relative freq.:  0.3255 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  289 <\/br>Relative freq.:  0.7896 <\/br>Relative Rep. freq.:  0.0175","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  77 <\/br>Relative freq.:  0.2104 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  475 <\/br>Relative freq.:  0.7588 <\/br>Relative Rep. freq.:  0.0346","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  151 <\/br>Relative freq.:  0.2412 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  06 <\/br># assignments:  46 <\/br>Relative freq.:  0.6866 <\/br>Relative Rep. freq.:  0.0192","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.3134 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  950 <\/br>Relative freq.:  0.8176 <\/br>Relative Rep. freq.:  0.0352","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  212 <\/br>Relative freq.:  0.1824 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  449 <\/br>Relative freq.:  0.6929 <\/br>Relative Rep. freq.:  0.0256","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  199 <\/br>Relative freq.:  0.3071 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  692 <\/br>Relative freq.:  0.5307 <\/br>Relative Rep. freq.:  0.0308","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  07 <\/br># assignments:  611 <\/br>Relative freq.:  0.4686 <\/br>Relative Rep. freq.:  0.0272","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  783 <\/br>Relative freq.:  0.7949 <\/br>Relative Rep. freq.:  0.0335","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  200 <\/br>Relative freq.:  0.203 <\/br>Relative Rep. freq.:  0.0086","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  187 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0178","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  06 <\/br># assignments:  576 <\/br>Relative freq.:  0.7347 <\/br>Relative Rep. freq.:  0.0223","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  207 <\/br>Relative freq.:  0.264 <\/br>Relative Rep. freq.:  0.008","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  79 <\/br>Relative freq.:  0.7596 <\/br>Relative Rep. freq.:  0.02","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.2404 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  06 <\/br># assignments:  500 <\/br>Relative freq.:  0.9488 <\/br>Relative Rep. freq.:  0.0222","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  05 <\/br># assignments:  27 <\/br>Relative freq.:  0.0512 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  396 <\/br>Relative freq.:  0.9474 <\/br>Relative Rep. freq.:  0.0316","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  05 <\/br># assignments:  21 <\/br>Relative freq.:  0.0502 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  06 <\/br># assignments:  344 <\/br>Relative freq.:  0.6003 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  229 <\/br>Relative freq.:  0.3997 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  530 <\/br>Relative freq.:  0.7726 <\/br>Relative Rep. freq.:  0.0212","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  156 <\/br>Relative freq.:  0.2274 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  06 <\/br># assignments:  11 <\/br>Relative freq.:  0.9167 <\/br>Relative Rep. freq.:  0.0313","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0833 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  04 <\/br># assignments:  32 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0179","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  02 <\/br># assignments:  474 <\/br>Relative freq.:  0.5368 <\/br>Relative Rep. freq.:  0.0227","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  06 <\/br># assignments:  409 <\/br>Relative freq.:  0.4632 <\/br>Relative Rep. freq.:  0.0196","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  02 <\/br># assignments:  103 <\/br>Relative freq.:  0.5691 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  06 <\/br># assignments:  78 <\/br>Relative freq.:  0.4309 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  270 <\/br>Relative freq.:  0.9963 <\/br>Relative Rep. freq.:  0.0356","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0037 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  241 <\/br>Relative freq.:  0.7026 <\/br>Relative Rep. freq.:  0.0212","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  04 <\/br># assignments:  102 <\/br>Relative freq.:  0.2974 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  327 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0399","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  272 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0148","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  04 <\/br># assignments:  283 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  840 <\/br>Relative freq.:  0.746 <\/br>Relative Rep. freq.:  0.0345","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  04 <\/br># assignments:  286 <\/br>Relative freq.:  0.254 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  02 <\/br># assignments:  409 <\/br>Relative freq.:  0.5565 <\/br>Relative Rep. freq.:  0.0234","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  06 <\/br># assignments:  326 <\/br>Relative freq.:  0.4435 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  02 <\/br># assignments:  887 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0451","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  02 <\/br># assignments:  319 <\/br>Relative freq.:  0.7217 <\/br>Relative Rep. freq.:  0.0274","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  04 <\/br># assignments:  122 <\/br>Relative freq.:  0.276 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  02 <\/br># assignments:  152 <\/br>Relative freq.:  0.7917 <\/br>Relative Rep. freq.:  0.0247","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  04 <\/br># assignments:  40 <\/br>Relative freq.:  0.2083 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  02 <\/br># assignments:  413 <\/br>Relative freq.:  0.9017 <\/br>Relative Rep. freq.:  0.0278","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  04 <\/br># assignments:  45 <\/br>Relative freq.:  0.0983 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  02 <\/br># assignments:  80 <\/br>Relative freq.:  0.8791 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  04 <\/br># assignments:  11 <\/br>Relative freq.:  0.1209 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  02 <\/br># assignments:  418 <\/br>Relative freq.:  0.5887 <\/br>Relative Rep. freq.:  0.019","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  06 <\/br># assignments:  292 <\/br>Relative freq.:  0.4113 <\/br>Relative Rep. freq.:  0.0133","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  06 <\/br># assignments:  190 <\/br>Relative freq.:  0.5507 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  02 <\/br># assignments:  155 <\/br>Relative freq.:  0.4493 <\/br>Relative Rep. freq.:  0.0153","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  1045 <\/br>Relative freq.:  0.8082 <\/br>Relative Rep. freq.:  0.0283","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  248 <\/br>Relative freq.:  0.1918 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  06 <\/br># assignments:  96 <\/br>Relative freq.:  0.7218 <\/br>Relative Rep. freq.:  0.0152","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  04 <\/br># assignments:  37 <\/br>Relative freq.:  0.2782 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  02 <\/br># assignments:  927 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0339","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  02 <\/br># assignments:  359 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0391","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  02 <\/br># assignments:  678 <\/br>Relative freq.:  0.7986 <\/br>Relative Rep. freq.:  0.0303","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  04 <\/br># assignments:  171 <\/br>Relative freq.:  0.2014 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  02 <\/br># assignments:  848 <\/br>Relative freq.:  0.9976 <\/br>Relative Rep. freq.:  0.0395","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  02 <\/br># assignments:  1114 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0604","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  02 <\/br># assignments:  985 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0501","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  02 <\/br># assignments:  693 <\/br>Relative freq.:  0.9747 <\/br>Relative Rep. freq.:  0.0286","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  05 <\/br># assignments:  17 <\/br>Relative freq.:  0.0239 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  02 <\/br># assignments:  126 <\/br>Relative freq.:  0.5701 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  06 <\/br># assignments:  95 <\/br>Relative freq.:  0.4299 <\/br>Relative Rep. freq.:  0.0125","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  02 <\/br># assignments:  1372 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0502","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  04 <\/br># assignments:  288 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  02 <\/br># assignments:  918 <\/br>Relative freq.:  0.827 <\/br>Relative Rep. freq.:  0.0269","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  04 <\/br># assignments:  191 <\/br>Relative freq.:  0.1721 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  02 <\/br># assignments:  33 <\/br>Relative freq.:  0.5238 <\/br>Relative Rep. freq.:  0.0245","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  06 <\/br># assignments:  30 <\/br>Relative freq.:  0.4762 <\/br>Relative Rep. freq.:  0.0222","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  02 <\/br># assignments:  994 <\/br>Relative freq.:  0.997 <\/br>Relative Rep. freq.:  0.0463","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  04 <\/br># assignments:  314 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0137","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  02 <\/br># assignments:  566 <\/br>Relative freq.:  0.7284 <\/br>Relative Rep. freq.:  0.0294","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  04 <\/br># assignments:  211 <\/br>Relative freq.:  0.2716 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  02 <\/br># assignments:  879 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0468","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  02 <\/br># assignments:  876 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0445","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  02 <\/br># assignments:  669 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0541","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  06 <\/br># assignments:  671 <\/br>Relative freq.:  0.6925 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  04 <\/br># assignments:  297 <\/br>Relative freq.:  0.3065 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  02 <\/br># assignments:  215 <\/br>Relative freq.:  0.874 <\/br>Relative Rep. freq.:  0.0282","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  04 <\/br># assignments:  31 <\/br>Relative freq.:  0.126 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  571 <\/br>Relative freq.:  0.7623 <\/br>Relative Rep. freq.:  0.0242","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  178 <\/br>Relative freq.:  0.2377 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  04 <\/br># assignments:  175 <\/br>Relative freq.:  0.8929 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  05 <\/br># assignments:  21 <\/br>Relative freq.:  0.1071 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  06 <\/br># assignments:  834 <\/br>Relative freq.:  0.7062 <\/br>Relative Rep. freq.:  0.0254","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  04 <\/br># assignments:  347 <\/br>Relative freq.:  0.2938 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  02 <\/br># assignments:  739 <\/br>Relative freq.:  0.7579 <\/br>Relative Rep. freq.:  0.023","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  04 <\/br># assignments:  235 <\/br>Relative freq.:  0.241 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  04 <\/br># assignments:  153 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0085","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  02 <\/br># assignments:  594 <\/br>Relative freq.:  0.7288 <\/br>Relative Rep. freq.:  0.0238","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  04 <\/br># assignments:  220 <\/br>Relative freq.:  0.2699 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  06 <\/br># assignments:  263 <\/br>Relative freq.:  0.643 <\/br>Relative Rep. freq.:  0.015","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  04 <\/br># assignments:  146 <\/br>Relative freq.:  0.357 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  04 <\/br># assignments:  192 <\/br>Relative freq.:  0.8889 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  05 <\/br># assignments:  24 <\/br>Relative freq.:  0.1111 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  02 <\/br># assignments:  392 <\/br>Relative freq.:  0.8394 <\/br>Relative Rep. freq.:  0.0211","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  04 <\/br># assignments:  75 <\/br>Relative freq.:  0.1606 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  04 <\/br># assignments:  464 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0214","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  02 <\/br># assignments:  454 <\/br>Relative freq.:  0.855 <\/br>Relative Rep. freq.:  0.0245","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  04 <\/br># assignments:  77 <\/br>Relative freq.:  0.145 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  1570 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0479","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  04 <\/br># assignments:  633 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0211","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  04 <\/br># assignments:  242 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  06 <\/br># assignments:  372 <\/br>Relative freq.:  0.5423 <\/br>Relative Rep. freq.:  0.0269","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  02 <\/br># assignments:  314 <\/br>Relative freq.:  0.4577 <\/br>Relative Rep. freq.:  0.0227","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  02 <\/br># assignments:  348 <\/br>Relative freq.:  0.7117 <\/br>Relative Rep. freq.:  0.024","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  04 <\/br># assignments:  141 <\/br>Relative freq.:  0.2883 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  02 <\/br># assignments:  623 <\/br>Relative freq.:  0.7691 <\/br>Relative Rep. freq.:  0.0261","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  04 <\/br># assignments:  187 <\/br>Relative freq.:  0.2309 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  02 <\/br># assignments:  565 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0608","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  02 <\/br># assignments:  336 <\/br>Relative freq.:  0.9683 <\/br>Relative Rep. freq.:  0.0179","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  05 <\/br># assignments:  11 <\/br>Relative freq.:  0.0317 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  02 <\/br># assignments:  1454 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.0509","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  04 <\/br># assignments:  602 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0184","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  04 <\/br># assignments:  411 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0175","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  06 <\/br># assignments:  322 <\/br>Relative freq.:  0.7318 <\/br>Relative Rep. freq.:  0.0201","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  118 <\/br>Relative freq.:  0.2682 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  02 <\/br># assignments:  1503 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0481","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  02 <\/br># assignments:  774 <\/br>Relative freq.:  0.8139 <\/br>Relative Rep. freq.:  0.0328","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  04 <\/br># assignments:  177 <\/br>Relative freq.:  0.1861 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  02 <\/br># assignments:  267 <\/br>Relative freq.:  0.8215 <\/br>Relative Rep. freq.:  0.0272","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  04 <\/br># assignments:  58 <\/br>Relative freq.:  0.1785 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  04 <\/br># assignments:  250 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  02 <\/br># assignments:  332 <\/br>Relative freq.:  0.8218 <\/br>Relative Rep. freq.:  0.0264","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  04 <\/br># assignments:  72 <\/br>Relative freq.:  0.1782 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  02 <\/br># assignments:  250 <\/br>Relative freq.:  0.947 <\/br>Relative Rep. freq.:  0.0197","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  05 <\/br># assignments:  14 <\/br>Relative freq.:  0.053 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  02 <\/br># assignments:  32 <\/br>Relative freq.:  0.8421 <\/br>Relative Rep. freq.:  0.0245","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  04 <\/br># assignments:  6 <\/br>Relative freq.:  0.1579 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  02 <\/br># assignments:  1577 <\/br>Relative freq.:  0.9987 <\/br>Relative Rep. freq.:  0.0668","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  06 <\/br># assignments:  452 <\/br>Relative freq.:  0.8898 <\/br>Relative Rep. freq.:  0.0262","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  04 <\/br># assignments:  55 <\/br>Relative freq.:  0.1083 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  02 <\/br># assignments:  58 <\/br>Relative freq.:  0.8286 <\/br>Relative Rep. freq.:  0.0298","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  04 <\/br># assignments:  12 <\/br>Relative freq.:  0.1714 <\/br>Relative Rep. freq.:  0.0062"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":78.9705271897053,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["V1-2*02","V1-2*04","V1-2*05","V1-2*06","V1-2*07"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["V1-2*02","V1-2*04","V1-2*05","V1-2*06","V1-2*07"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-78.2122199217044,1656.23377002722],"tickmode":"array","ticktext":["0","500","1000","1500"],"tickvals":[0,500,1000,1500],"categoryorder":"array","categoryarray":["0","500","1000","1500"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5c52b6387b96f":{"x":{},"y":{},"text":{},"type":"box"},"5c52b18d06f08":{"x":{},"y":{},"text":{}}},"cur_data":"5c52b6387b96f","visdat":{"5c52b6387b96f":["function (y) ","x"],"5c52b18d06f08":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-549e9cfd118fffd0c44c" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-549e9cfd118fffd0c44c">{"x":{"data":[{"x":[1,2,2,2,2,2,2,2,4,2,1,2,2,2,3,4,2,2,2,1,2,1,2,4,1,4,1,2,2,1,2,1,4,4,2,4,2,2,1,2,1,2,1,3,2,1,2,3,4,4,2,1,2,1,1,4,2,1,1,1,2,1,2,1,4,1,2,2,1,2,1,2,1],"y":[245,43,45,52,9,106,102,86,226,105,3,118,69,47,7,163,26,108,86,169,43,320,47,1,460,1,125,30,112,139,26,221,216,138,22,311,39,233,447,104,446,87,219,10,77,1,30,6,1,127,86,217,69,778,246,130,33,1,1405,198,49,39,8,108,80,286,58,85,2,121,192,28,638],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(76,138,126,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[0.964234853209928,1.94667529426515,2.01270262943581,1.93904240364209,1.93630268000998,1.95194772076793,2.07919883769937,2.0876863618847,3.92687213085592,1.93978219502605,0.984828312322497,2.06507756318897,2.01108132395893,2.05955324186943,2.98061339678243,4.06404944639653,2.00288942856714,1.99312449009158,2.07802085578442,0.95146349016577,2.05253062713891,0.984073205990717,2.06742852660827,4.01425048457459,1.07452023569494,4.0446821573656,1.02122498219833,2.03944013505243,2.06294398801401,0.9840444103349,2.07501361980103,0.912649680767208,4.07284390027635,3.96375984735787,2.0840711908415,3.94555081781,2.04049298157915,1.93261129166931,0.954527185484767,2.00927456878126,0.918470707722008,2.07296412852593,0.94127791877836,3.03911644066684,2.05486506479792,1.02285306011327,2.005234353384,2.94472946580499,3.92378287883475,4.01119705885649,1.98943843315355,1.01512643727474,2.08476600619033,1.00330548887141,0.912346082692966,3.94746035756543,2.00514885005541,1.02611965704709,0.957110044779256,1.07423454071395,2.03949642116204,0.924720158567652,2.0523898714222,0.924183213245124,3.99992209812626,0.940139943500981,1.93790702023543,1.90028697578236,0.942351038893685,2.0876101154834,1.08778582760133,2.05796199259348,0.955487902462482],"y":[244.827923038043,42.9362289944664,44.9673930805177,52.3580068036914,8.64880952015519,105.880907811224,102.003005205281,86.2233150072396,225.727190891653,105.300784884579,3.2820255657658,117.97182997223,68.9963069353253,46.7676881505176,7.37143694423139,162.960133985616,25.7482212338597,108.016584566981,85.9044969804585,169.099036503583,42.6802904093638,320.159062819369,46.923233474046,0.716354172304273,459.624812353589,1.30772664695978,124.906564232893,29.6852641208097,112.311933911406,138.944926012307,26.2957991123199,221.220550726168,215.651330063865,138.246025298722,22.0940827755257,311.300593606941,38.9145579189062,232.857217210717,447.334696835652,103.645953501761,445.828315100446,87.1951713992283,218.621633002907,9.76385830435902,77.0176663525403,1.01894690413028,30.2215198827907,6.14956137221307,1.08204758968204,127.190585102886,86.1180526915938,217.152857429907,69.3995828207582,778.096630593948,245.897576291114,129.979137043655,32.6348638612777,0.97779424097389,1405.11947542727,198.088065207563,49.0715136377141,39.1698136866093,7.66917119808495,108.032671396621,80.3398703770712,285.688867322728,57.681540566124,85.3794420108199,1.8514880426228,120.677408420481,192.347788889706,27.7362926663831,638.313280859776],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  245 <\/br>Relative freq.:  0.8507 <\/br>Relative Rep. freq.:  0.0418","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  43 <\/br>Relative freq.:  0.1493 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  45 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  52 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  9 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  106 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  102 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  86 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  06 <\/br># assignments:  226 <\/br>Relative freq.:  0.6766 <\/br>Relative Rep. freq.:  0.0272","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  105 <\/br>Relative freq.:  0.3144 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.009 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  118 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0207","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  69 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  47 <\/br>Relative freq.:  0.8704 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  05 <\/br># assignments:  7 <\/br>Relative freq.:  0.1296 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  06 <\/br># assignments:  163 <\/br>Relative freq.:  0.8624 <\/br>Relative Rep. freq.:  0.0319","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  26 <\/br>Relative freq.:  0.1376 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  108 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0178","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  86 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0217","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  169 <\/br>Relative freq.:  0.7972 <\/br>Relative Rep. freq.:  0.0389","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  43 <\/br>Relative freq.:  0.2028 <\/br>Relative Rep. freq.:  0.0099","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  320 <\/br>Relative freq.:  0.8696 <\/br>Relative Rep. freq.:  0.0406","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  47 <\/br>Relative freq.:  0.1277 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  460 <\/br>Relative freq.:  0.9978 <\/br>Relative Rep. freq.:  0.0905","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  125 <\/br>Relative freq.:  0.8065 <\/br>Relative Rep. freq.:  0.0552","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  30 <\/br>Relative freq.:  0.1935 <\/br>Relative Rep. freq.:  0.0133","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  112 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0158","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  139 <\/br>Relative freq.:  0.8424 <\/br>Relative Rep. freq.:  0.0387","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  26 <\/br>Relative freq.:  0.1576 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  221 <\/br>Relative freq.:  0.5057 <\/br>Relative Rep. freq.:  0.0317","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  06 <\/br># assignments:  216 <\/br>Relative freq.:  0.4943 <\/br>Relative Rep. freq.:  0.031","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  06 <\/br># assignments:  138 <\/br>Relative freq.:  0.8625 <\/br>Relative Rep. freq.:  0.0303","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  22 <\/br>Relative freq.:  0.1375 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  06 <\/br># assignments:  311 <\/br>Relative freq.:  0.8886 <\/br>Relative Rep. freq.:  0.0403","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  04 <\/br># assignments:  39 <\/br>Relative freq.:  0.1114 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  233 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0402","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  447 <\/br>Relative freq.:  0.8113 <\/br>Relative Rep. freq.:  0.0501","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  104 <\/br>Relative freq.:  0.1887 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  02 <\/br># assignments:  446 <\/br>Relative freq.:  0.8368 <\/br>Relative Rep. freq.:  0.042","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  04 <\/br># assignments:  87 <\/br>Relative freq.:  0.1632 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  02 <\/br># assignments:  219 <\/br>Relative freq.:  0.9563 <\/br>Relative Rep. freq.:  0.0393","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  05 <\/br># assignments:  10 <\/br>Relative freq.:  0.0437 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  04 <\/br># assignments:  77 <\/br>Relative freq.:  0.9872 <\/br>Relative Rep. freq.:  0.0161","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0128 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  04 <\/br># assignments:  30 <\/br>Relative freq.:  0.8108 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  05 <\/br># assignments:  6 <\/br>Relative freq.:  0.1622 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.027 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  06 <\/br># assignments:  127 <\/br>Relative freq.:  0.5962 <\/br>Relative Rep. freq.:  0.0237","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  04 <\/br># assignments:  86 <\/br>Relative freq.:  0.4038 <\/br>Relative Rep. freq.:  0.016","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  217 <\/br>Relative freq.:  0.7587 <\/br>Relative Rep. freq.:  0.0425","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  69 <\/br>Relative freq.:  0.2413 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  778 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.084","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  246 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0793","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  06 <\/br># assignments:  130 <\/br>Relative freq.:  0.7927 <\/br>Relative Rep. freq.:  0.0252","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  33 <\/br>Relative freq.:  0.2012 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0061 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  02 <\/br># assignments:  1405 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0911","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  198 <\/br>Relative freq.:  0.8016 <\/br>Relative Rep. freq.:  0.0453","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  04 <\/br># assignments:  49 <\/br>Relative freq.:  0.1984 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  02 <\/br># assignments:  39 <\/br>Relative freq.:  0.8298 <\/br>Relative Rep. freq.:  0.0346","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  04 <\/br># assignments:  8 <\/br>Relative freq.:  0.1702 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  02 <\/br># assignments:  108 <\/br>Relative freq.:  0.5745 <\/br>Relative Rep. freq.:  0.0349","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  06 <\/br># assignments:  80 <\/br>Relative freq.:  0.4255 <\/br>Relative Rep. freq.:  0.0258","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  286 <\/br>Relative freq.:  0.8314 <\/br>Relative Rep. freq.:  0.0338","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  58 <\/br>Relative freq.:  0.1686 <\/br>Relative Rep. freq.:  0.0069","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  04 <\/br># assignments:  85 <\/br>Relative freq.:  0.977 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.023 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  121 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  192 <\/br>Relative freq.:  0.8727 <\/br>Relative Rep. freq.:  0.0358","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  28 <\/br>Relative freq.:  0.1273 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  638 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0863"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(76,138,126,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(76,138,126,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":78.9705271897053,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,4.6],"tickmode":"array","ticktext":["V1-2*02","V1-2*04","V1-2*05","V1-2*06"],"tickvals":[1,2,3,4],"categoryorder":"array","categoryarray":["V1-2*02","V1-2*04","V1-2*05","V1-2*06"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-69.503801890444,1475.33963149002],"tickmode":"array","ticktext":["0","500","1000"],"tickvals":[0,500,1000],"categoryorder":"array","categoryarray":["0","500","1000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5c52b2b20f1fe":{"x":{},"y":{},"text":{},"type":"box"},"5c52b38edab36":{"x":{},"y":{},"text":{}}},"cur_data":"5c52b2b20f1fe","visdat":{"5c52b2b20f1fe":["function (y) ","x"],"5c52b38edab36":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-5dda39d243b1d66b7b29" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-5dda39d243b1d66b7b29">{"x":{"data":[{"x":[1,4,5,2,2,1,1,3,4,5,2,1,2,4,4,1,5,1,2,4,5,1,2,5,2,1,1,2,4,4,1,1,2,1,4,1,2,4,5,1,4,2,1,4,2,3,1,4,1,4,5,1,2,4,5,1,4,2,5,1,4,2,1,2,4,1,2,4,2,1,1,2,5,4,4,2,1,3,1,2,4,3,4,2,1,2,3,1,4],"y":[1309,11,7,1,33,1,438,13,4,3,1,450,130,4,247,136,2,799,93,9,4,398,82,4,171,8,206,29,19,849,6,147,19,519,8,1368,17,2,2,225,5,1,328,2,12,6,1,1,365,352,2,453,98,3,1,773,6,1,1,1299,4,2,670,117,6,315,47,16,159,4,206,19,9,4,217,23,7,1,509,101,6,1,222,39,6,61,8,5,5],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(86,63,80,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[0.94312926181592,4.09179865992628,5.01261039599776,2.04042135076597,2.02458830331452,0.969799033738673,1.03640874745324,3.08233301178552,4.01441765180789,4.9947087558452,1.97716978015378,1.00484890867956,1.93180786403827,3.92797079430893,4.06140539436601,0.977087138593197,5.0910036622081,0.982165490835905,1.99612968964502,3.99189766375348,4.95722269117832,1.08165136114694,1.96834435248747,5.02162985871546,1.94032584428787,1.00370332086459,0.911672004079446,2.08262805743143,4.02344173309393,3.91510212039575,0.950798395881429,0.990528152137995,2.00399002521299,0.998378865560517,3.94484898610972,1.01954731182195,1.9292403191328,3.91860808152705,5.0781325574033,1.02612820193172,3.94946363954805,2.0036127446685,0.931389275053516,4.07745392741635,2.03415999151766,3.00006761387922,1.08917626515031,4.06105660810135,0.952616264345124,4.01897996282205,4.98720078249462,0.97646442274563,1.93383617354557,3.91437019193545,5.08956313305534,1.04252131078392,3.979539151676,2.06925751660019,5.03248877427541,0.922842124383897,4.03645043913275,1.95594927985221,0.963192826323211,2.09368613474071,4.05402725110762,0.955710303643718,2.00121599733829,3.90432584090158,1.94207840673625,0.917262232815847,0.959478704072535,1.90518553298898,5.04641149407253,3.95043222834356,4.0518330427818,2.09975709593855,1.09731316459365,3.03992865611799,1.00229162983596,2.00405066870153,3.94296191646717,2.97984953350388,4.00941992420703,2.08567303065211,0.995683265570551,1.9755282686092,3.03153398958966,1.08312433389947,4.02883285623975],"y":[1308.73352124207,11.1657584866509,7.28949913475662,1.34418509025127,32.6707441091537,0.729696897789836,437.929357902519,12.8428008507937,3.9553272895515,3.09006747175008,1.22871716339141,450.26726460997,130.196144880168,3.61875054184347,246.835649957694,136.398531127907,2.37678114008158,798.950715819188,93.0905734043568,9.24033979661763,3.69185873176903,397.802764595672,81.8835888173431,4.0775691607967,171.365028527379,7.75008673127741,205.612688328512,29.2570210041478,19.177444087714,848.710701715946,6.23423655163497,146.891495628282,18.8112970229238,519.136326546967,7.63722583446652,1367.63494460732,17.3290374768898,1.86864237543195,2.19719189088792,225.103338325582,4.83707533441484,0.706458490528166,327.833208151907,1.68875010348856,11.979668719694,5.7281973676756,1.39297528155148,1.30409251023084,365.036038229615,352.040343910269,2.28685621302575,452.87680359371,97.950246767886,2.6656941358,1.30854991432279,772.825480417907,6.14258859418333,0.809416682086885,0.64297136105597,1298.8732378263,4.11787258032709,1.72332712765783,670.165988701768,116.663963301852,6.39402869660407,314.800192455575,47.0270687272772,15.9114122042432,158.786004189588,4.14704886320978,206.028222221136,19.1401635758579,9.0719449935481,4.10287069808692,216.66599615328,22.9483507476747,6.89468118194491,0.971395514719188,508.681340614147,100.889669678546,5.82860438991338,0.663591073080897,222.164375188015,39.0204939538613,6.17831299938261,61.0067045612261,8.24046382773668,5.39110070150346,5.24525646027178],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  1309 <\/br>Relative freq.:  0.9857 <\/br>Relative Rep. freq.:  0.1146","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  06 <\/br># assignments:  11 <\/br>Relative freq.:  0.0083 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  07 <\/br># assignments:  7 <\/br>Relative freq.:  0.0053 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  33 <\/br>Relative freq.:  0.9706 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0294 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  02 <\/br># assignments:  438 <\/br>Relative freq.:  0.9542 <\/br>Relative Rep. freq.:  0.039","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  05 <\/br># assignments:  13 <\/br>Relative freq.:  0.0283 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  06 <\/br># assignments:  4 <\/br>Relative freq.:  0.0087 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  07 <\/br># assignments:  3 <\/br>Relative freq.:  0.0065 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  450 <\/br>Relative freq.:  0.7705 <\/br>Relative Rep. freq.:  0.0297","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  130 <\/br>Relative freq.:  0.2226 <\/br>Relative Rep. freq.:  0.0086","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  06 <\/br># assignments:  4 <\/br>Relative freq.:  0.0068 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  06 <\/br># assignments:  247 <\/br>Relative freq.:  0.6416 <\/br>Relative Rep. freq.:  0.0471","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  136 <\/br>Relative freq.:  0.3532 <\/br>Relative Rep. freq.:  0.0259","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  07 <\/br># assignments:  2 <\/br>Relative freq.:  0.0052 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  799 <\/br>Relative freq.:  0.8829 <\/br>Relative Rep. freq.:  0.0467","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  93 <\/br>Relative freq.:  0.1028 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  06 <\/br># assignments:  9 <\/br>Relative freq.:  0.0099 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  07 <\/br># assignments:  4 <\/br>Relative freq.:  0.0044 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  398 <\/br>Relative freq.:  0.8223 <\/br>Relative Rep. freq.:  0.0335","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  82 <\/br>Relative freq.:  0.1694 <\/br>Relative Rep. freq.:  0.0069","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  07 <\/br># assignments:  4 <\/br>Relative freq.:  0.0083 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  171 <\/br>Relative freq.:  0.9553 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.0447 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  206 <\/br>Relative freq.:  0.811 <\/br>Relative Rep. freq.:  0.0288","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  29 <\/br>Relative freq.:  0.1142 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  06 <\/br># assignments:  19 <\/br>Relative freq.:  0.0748 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  06 <\/br># assignments:  849 <\/br>Relative freq.:  0.993 <\/br>Relative Rep. freq.:  0.051","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.007 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  147 <\/br>Relative freq.:  0.8855 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  19 <\/br>Relative freq.:  0.1145 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  519 <\/br>Relative freq.:  0.9848 <\/br>Relative Rep. freq.:  0.046","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  06 <\/br># assignments:  8 <\/br>Relative freq.:  0.0152 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  1368 <\/br>Relative freq.:  0.9849 <\/br>Relative Rep. freq.:  0.0767","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  17 <\/br>Relative freq.:  0.0122 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  07 <\/br># assignments:  2 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  225 <\/br>Relative freq.:  0.974 <\/br>Relative Rep. freq.:  0.0577","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  06 <\/br># assignments:  5 <\/br>Relative freq.:  0.0216 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0043 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  328 <\/br>Relative freq.:  0.9939 <\/br>Relative Rep. freq.:  0.0489","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.0061 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  12 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  05 <\/br># assignments:  6 <\/br>Relative freq.:  0.3 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.05 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.05 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  365 <\/br>Relative freq.:  0.5076 <\/br>Relative Rep. freq.:  0.0275","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  06 <\/br># assignments:  352 <\/br>Relative freq.:  0.4896 <\/br>Relative Rep. freq.:  0.0265","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  07 <\/br># assignments:  2 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  453 <\/br>Relative freq.:  0.8162 <\/br>Relative Rep. freq.:  0.0299","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  98 <\/br>Relative freq.:  0.1766 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.0054 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  773 <\/br>Relative freq.:  0.9898 <\/br>Relative Rep. freq.:  0.0412","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  06 <\/br># assignments:  6 <\/br>Relative freq.:  0.0077 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  1299 <\/br>Relative freq.:  0.9954 <\/br>Relative Rep. freq.:  0.0519","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  06 <\/br># assignments:  4 <\/br>Relative freq.:  0.0031 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  02 <\/br># assignments:  670 <\/br>Relative freq.:  0.8449 <\/br>Relative Rep. freq.:  0.03","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  117 <\/br>Relative freq.:  0.1475 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  06 <\/br># assignments:  6 <\/br>Relative freq.:  0.0076 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  02 <\/br># assignments:  315 <\/br>Relative freq.:  0.8333 <\/br>Relative Rep. freq.:  0.0202","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  47 <\/br>Relative freq.:  0.1243 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  06 <\/br># assignments:  16 <\/br>Relative freq.:  0.0423 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  159 <\/br>Relative freq.:  0.9755 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0245 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  206 <\/br>Relative freq.:  0.8655 <\/br>Relative Rep. freq.:  0.03","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  19 <\/br>Relative freq.:  0.0798 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  07 <\/br># assignments:  9 <\/br>Relative freq.:  0.0378 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  06 <\/br># assignments:  4 <\/br>Relative freq.:  0.0168 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  06 <\/br># assignments:  217 <\/br>Relative freq.:  0.875 <\/br>Relative Rep. freq.:  0.0252","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  23 <\/br>Relative freq.:  0.0927 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.0282 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  509 <\/br>Relative freq.:  0.825 <\/br>Relative Rep. freq.:  0.0345","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  101 <\/br>Relative freq.:  0.1637 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  06 <\/br># assignments:  6 <\/br>Relative freq.:  0.0097 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  06 <\/br># assignments:  222 <\/br>Relative freq.:  0.8315 <\/br>Relative Rep. freq.:  0.027","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  39 <\/br>Relative freq.:  0.1461 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0225 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  61 <\/br>Relative freq.:  0.7722 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  05 <\/br># assignments:  8 <\/br>Relative freq.:  0.1013 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0633 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  06 <\/br># assignments:  5 <\/br>Relative freq.:  0.0633 <\/br>Relative Rep. freq.:  4e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(86,63,80,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(86,63,80,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":78.9705271897053,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["V1-2*02","V1-2*04","V1-2*05","V1-2*06","V1-2*07"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["V1-2*02","V1-2*04","V1-2*05","V1-2*06","V1-2*07"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-67.7248800708912,1436.36785143195],"tickmode":"array","ticktext":["0","500","1000"],"tickvals":[0,500,1000],"categoryorder":"array","categoryarray":["0","500","1000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5c52b67db9257":{"x":{},"y":{},"text":{},"type":"box"},"5c52b22c2963a":{"x":{},"y":{},"text":{}}},"cur_data":"5c52b67db9257","visdat":{"5c52b67db9257":["function (y) ","x"],"5c52b22c2963a":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-1" class="section level2" number="3.4">
<h2><span class="header-section-number">3.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 4 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV1-2G2%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:900px;" cellspacing="0">
</iframe>
</div>
<div id="observations-1" class="section level2" number="3.5">
<h2><span class="header-section-number">3.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/lasdc2i84m8j4lohwgrpa/IGHV1-2G2.docx?dl=0&amp;rlkey=lrfhe80o9069hpn82tcxud8a7" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV1-2G2" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-1" class="section level2" number="3.6">
<h2><span class="header-section-number">3.6</span> Conclusions</h2>
<p>From the results we believe that the cutoff for this group should be closer to <span class="math inline">\(5\%\)</span> except allele <strong>05</strong> that should stay at <span class="math inline">\(0.5\%\)</span>, and for the adjusted states the allele combinations and the relations are stated in the table below.</p>
<div id="allele-specific-cutoff-5-for-all-except-0.5-for-allele-05" class="section level3" number="3.6.1">
<h3><span class="header-section-number">3.6.1</span> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></h3>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV1-2G2%22&amp;allele_thresh=%22V1-2*01q5bV1-2*02q5bV1-2*03q5bV1-2*04q5bV1-2*05q0.5bV1-2*06q5bV1-2*07q5%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:900px;" cellspacing="0">
</iframe>
<div id="htmlwidget-ecc5966c70530911a45c" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-ecc5966c70530911a45c">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16"],[1,1,1,2,2,2,2,2,2,2,2,2,3,3,4,4],["V1-2*02","V1-2*04","V1-2*06","V1-2*02;V1-2*02","V1-2*02;V1-2*04","V1-2*02;V1-2*05","V1-2*02;V1-2*06","V1-2*02;V1-2*07","V1-2*04;V1-2*04","V1-2*04;V1-2*05","V1-2*04;V1-2*06","V1-2*06;V1-2*06","V1-2*02;V1-2*02;V1-2*04","V1-2*04;V1-2*06;V1-2*06","V1-2*02;V1-2*04;V1-2*04;V1-2*05","V1-2*02;V1-2*04;V1-2*05;V1-2*06"],["1:0.729","1:0.771","0.993:0.993","0.844:0.7;0.283:0.157;0.111:0.041;0.802:0.466","0.844:0.7;0.283:0.157","0.844:0.7;0.111:0.041","0.844:0.7;0.802:0.466","0.844:0.7;0.626:0.521","0.283:0.157;0.111:0.041","0.283:0.157;0.111:0.041","0.283:0.157;0.802:0.466","0.283:0.157;0.111:0.041;0.802:0.466","0.685:0.34;0.34:0.109;0.579:0.079","0.685:0.34;0.34:0.109;0.579:0.079","0.124:0.057;0.756:0.67;0.25:0.151;0.06:0.053","0.124:0.057;0.756:0.67;0.25:0.151;0.06:0.053"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Zygousity state<\/th>\n      <th>V\nallele<\/th>\n      <th>IQR range<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"tipr","columnDefs":[{"className":"dt-right","targets":1},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="01-G1.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="03-G3.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/02-G2.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
