<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV1-2 - G2 | Functional groups</title>
  <meta name="description" content="IGHV1-2 - G2 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV1-2 - G2 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV1-2 - G2 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="01-G1.html"/>
<link rel="next" href="03-G3.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-checklist"><i class="fa fa-check"></i>Group Checklist</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv1-2---g2" class="section level1">
<h1>IGHV1-2 - G2</h1>
<div id="allele-appearnce-1" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of IGHV1-2 includes 7 alleles, 7 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-19-1.png" width="672" /></p>
</div>
<div id="group-alignment-1" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-20-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-1" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-322dc749a8a28398a46b" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-322dc749a8a28398a46b">{"x":{"data":[{"x":[2,2,3,1,2,2,1,2,1,2,1,2,1,2,1,2,1,2,2,4,2,1,2,1,2,4,2,1,2,1,2,1,5,4,1,2,2,4,2,3,1,2,4,3,1,3,4,2,1,2,4,2,1,4,1,4,1,1,2,1,2,2,1,2,1,4,1,1,2,4,1,2,1,2,1,2,1,4,4,1,1,2,4,2,1,1,1,2,1,4,1,1,1,3,1,4,1,2,1,2,4,1,4,1,2,2,1,2,1,1,1,4,2,3,1,2,1,2,2,3,4,2,1,2,2,1,2,4,4,2,2,3,1,2,2,1,2,1,2,2,4,1,1,2,1,2,1,1,3,1,4,2,2,4,2,1,1,2,1,2,2,1,2,1,3,1,2,1,4,2,3,1,2],"y":[94,207,16,250,48,508,652,137,809,177,112,42,111,41,404,72,90,29,117,368,179,254,72,470,147,46,21,914,209,442,197,680,606,1,768,198,184,562,203,1,76,25,486,26,343,17,341,228,520,156,11,32,467,401,97,73,252,240,100,322,264,281,826,283,399,311,872,312,119,1,147,37,412,45,74,11,415,284,139,115,1031,247,95,37,917,354,667,170,846,1,1104,953,667,17,126,93,1357,286,856,185,1,33,29,847,1,310,548,209,873,872,630,650,294,1,208,31,546,174,175,21,828,347,711,231,153,590,218,1,258,144,190,23,369,72,464,420,73,1552,625,238,344,270,331,139,616,186,554,332,11,1317,1,596,403,315,118,1483,738,172,253,56,242,326,71,240,13,23,6,1478,431,54,1,55,12],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[2.01986557599157,1.91473206109367,2.9856608025264,1.06742862639949,2.00635461993515,1.92660634419881,1.04323304351419,2.02832814343274,0.972557782428339,1.95080389478244,1.04090508380905,2.09842771370895,1.01576743116602,1.92091173101217,1.05716302022338,1.93318353043869,0.907793755549937,1.9196089359466,2.0864621182438,3.96238575004973,2.02750111832283,0.901093676965684,2.06421814784408,1.02256229971535,2.08575484422036,4.01918232487515,1.97326866327785,1.07880828445777,2.07600498381071,0.970875875838101,2.0339205596596,1.08802632498555,5.05150703079998,4.07955297757871,1.09324507429264,1.95228959317319,2.05043211793527,3.92834580498748,2.04210377116688,3.08618524600752,0.945532160811126,2.03402229901403,3.9677392387297,2.97764450688846,0.974828960653394,3.07576619996689,4.02308885143138,2.03029287490062,0.99432009216398,1.92255697692744,3.90357399745844,2.0694529092405,0.948267995659262,4.09815502874553,1.00510760084726,3.93430813746527,1.07578267664649,1.02434758814052,2.08045760318637,0.948343856073916,1.99560619830154,1.9542780670803,0.951370837492868,1.93851664382964,1.07264758646488,3.94984846878797,0.924050231603906,1.07569905845448,2.014531771373,3.99674859768711,0.901510721305385,2.02654539300129,0.923511986294761,2.08012669412419,0.911363894352689,2.03041392085142,0.981968929013237,4.07057277788408,3.97756623933092,1.038557843538,0.91199926417321,2.0106199296657,3.97687114859,2.06877509229817,0.919774366356432,0.98173110531643,1.0372699606698,1.93597019962035,0.940409526322037,3.96521435976028,1.07743202415295,1.07961638369597,0.913822754565626,2.95237041087821,1.06858363547362,3.96457489309832,0.987387134321034,1.9957804269623,1.02576972958632,1.96815466680564,4.05882306681015,0.943310063704848,3.99241280560382,1.06055492362939,2.0717285476625,1.9270688382443,0.968037639418617,1.9388252380304,0.907098672306165,1.07105238377117,1.09764417284168,4.0136136262212,2.00029089883901,2.9028170440346,1.03299155435525,1.98538654106669,1.04940475607291,2.08755841790698,1.99390619867481,3.0396547911223,3.98301563477144,1.93904744749889,1.02533462583087,1.93939144718461,1.94347297255881,1.04942818307318,2.07156597534195,3.97085510110483,3.92613480277359,1.9632483581081,1.92777688293718,2.96718727038242,0.995762097882107,1.94420937569812,1.92498649009503,0.920694413222373,2.09115733797662,1.05781390885822,1.91151924096048,1.9327313888818,3.97813387932256,0.982055274071172,1.04806957319379,1.93301399522461,0.933756697271019,1.9465982362628,1.0881658999715,1.09362498726696,2.92414130386896,1.06752318073995,3.97102486500517,1.9662072155159,2.07876410894096,3.97366339238361,2.0031149441842,1.09776940122247,0.921930690249428,2.01730099744163,0.948721571965143,2.00486759217456,2.0716169395484,1.04432880515233,1.96141777532175,0.939966559782624,2.95450513791293,0.922774050245062,1.95766887390055,1.00431202561595,3.940841601789,1.98456552238204,3.03026923462749,1.08202175423503,2.09890288063325],"y":[94.3482374904677,206.627202354185,16.3479899998754,249.614749459177,47.7653086276725,507.636385901272,652.108075173758,136.837264839001,809.363186858408,176.861617612652,111.907049763948,41.8622156407684,111.268022521771,40.6125979527831,404.070106356218,71.8230610782281,90.095370217599,29.3265646629035,117.101208846457,367.672312337346,178.952741346881,254.086208910495,71.8987502478063,470.08795792833,146.632375635765,45.6864264089614,21.1655465686694,914.058193202876,208.973266266473,442.25220806133,196.980663323961,680.249978052638,605.955635715462,0.960685123689473,767.976655557938,197.947327522561,184.068529896252,561.914972451143,203.290124758519,1.23282545916736,76.3851943960413,25.1946417329833,485.723997495323,26.1823872862384,343.066811553016,17.1684205830097,341.314779930003,228.088520028815,519.728526014276,156.042482143641,11.2598046759143,31.8019494153559,467.133148766123,401.391154664382,97.2798088576645,72.8439966417849,251.632281483151,240.392733685859,99.6813439942896,321.883079935238,263.833574315719,280.864531988278,825.904807186499,283.196018865705,398.626724445261,311.143156229518,872.283989086188,312.234710296616,119.146297316253,0.631858745589852,147.367236360349,37.1208086946979,411.762345950864,44.7122765852138,73.7731372708455,11.3478677423671,415.272056888044,283.825847285613,139.224332307465,115.311886152066,1031.25732148103,247.325803103857,94.8316754253581,37.2323141187429,916.758932536468,353.781734465249,667.195713693276,170.368389760517,846.339683637582,0.817974659614265,1104.23306354154,952.982903030328,667.053027006984,17.3369405156001,125.767875030637,92.8369347857311,1356.82979238424,286.054200498201,855.725516414084,184.69368609786,1.31815674901009,32.7160276442766,29.1493933044374,846.857076271251,1.29917984232306,309.987800228223,547.795987599157,208.787676369585,873.317249020748,871.829390942864,630.153926198557,649.949305731431,294.135142429546,0.756461246870458,207.98969776351,31.1654788220301,545.915337582305,173.934625691548,175.203906505369,20.9188871422783,827.777751664817,346.675055712275,711.046350528672,230.877500119619,152.782850048318,590.032791853696,218.091358487494,0.817401606403291,258.187541953661,143.702955461666,189.832100325264,22.7196329917759,369.07830184456,72.1514187576249,464.356452336907,420.080869442783,72.9985262012109,1552.32755928803,625.081679013558,238.179460811429,344.213023163006,269.875138520449,331.327048643306,139.159839964099,615.626215299778,186.153160784766,553.890900401957,331.769554274157,11.0461640851572,1317.26945766862,1.25716311838478,596.191926397197,402.870110106096,314.924868965149,118.02924619168,1482.63858772889,738.389925124682,172.154832522757,253.030278847367,55.7513331541792,242.177413408831,325.774311430194,70.6935695111752,239.990971893817,13.0371855344623,22.6473099460825,6.33817287888378,1478.10913950764,430.891625264473,54.1967549921945,0.916450801119208,55.3362558532506,11.6805947044864],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  94 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  207 <\/br>Relative freq.:  0.9283 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  05 <\/br># assignments:  16 <\/br>Relative freq.:  0.0717 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  02 <\/br># assignments:  250 <\/br>Relative freq.:  0.8389 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  04 <\/br># assignments:  48 <\/br>Relative freq.:  0.1611 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  508 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0216","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  652 <\/br>Relative freq.:  0.8264 <\/br>Relative Rep. freq.:  0.0279","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  137 <\/br>Relative freq.:  0.1736 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  809 <\/br>Relative freq.:  0.8205 <\/br>Relative Rep. freq.:  0.0336","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  177 <\/br>Relative freq.:  0.1795 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  112 <\/br>Relative freq.:  0.7273 <\/br>Relative Rep. freq.:  0.0191","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  42 <\/br>Relative freq.:  0.2727 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  111 <\/br>Relative freq.:  0.7303 <\/br>Relative Rep. freq.:  0.0191","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  41 <\/br>Relative freq.:  0.2697 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  404 <\/br>Relative freq.:  0.8487 <\/br>Relative Rep. freq.:  0.029","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  72 <\/br>Relative freq.:  0.1513 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  90 <\/br>Relative freq.:  0.7563 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  29 <\/br>Relative freq.:  0.2437 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  117 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  06 <\/br># assignments:  368 <\/br>Relative freq.:  0.6728 <\/br>Relative Rep. freq.:  0.0169","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  179 <\/br>Relative freq.:  0.3272 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  254 <\/br>Relative freq.:  0.7791 <\/br>Relative Rep. freq.:  0.0171","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  72 <\/br>Relative freq.:  0.2209 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  470 <\/br>Relative freq.:  0.7618 <\/br>Relative Rep. freq.:  0.035","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  147 <\/br>Relative freq.:  0.2382 <\/br>Relative Rep. freq.:  0.0109","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  06 <\/br># assignments:  46 <\/br>Relative freq.:  0.6866 <\/br>Relative Rep. freq.:  0.02","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.3134 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  914 <\/br>Relative freq.:  0.8139 <\/br>Relative Rep. freq.:  0.0349","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  209 <\/br>Relative freq.:  0.1861 <\/br>Relative Rep. freq.:  0.008","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  442 <\/br>Relative freq.:  0.6917 <\/br>Relative Rep. freq.:  0.0256","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  197 <\/br>Relative freq.:  0.3083 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  680 <\/br>Relative freq.:  0.5284 <\/br>Relative Rep. freq.:  0.0308","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  07 <\/br># assignments:  606 <\/br>Relative freq.:  0.4709 <\/br>Relative Rep. freq.:  0.0275","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  768 <\/br>Relative freq.:  0.795 <\/br>Relative Rep. freq.:  0.0338","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  198 <\/br>Relative freq.:  0.205 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  184 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0182","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  06 <\/br># assignments:  562 <\/br>Relative freq.:  0.7337 <\/br>Relative Rep. freq.:  0.0227","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  203 <\/br>Relative freq.:  0.265 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  76 <\/br>Relative freq.:  0.7525 <\/br>Relative Rep. freq.:  0.0198","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.2475 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  06 <\/br># assignments:  486 <\/br>Relative freq.:  0.9492 <\/br>Relative Rep. freq.:  0.0223","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  05 <\/br># assignments:  26 <\/br>Relative freq.:  0.0508 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  343 <\/br>Relative freq.:  0.9528 <\/br>Relative Rep. freq.:  0.033","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  05 <\/br># assignments:  17 <\/br>Relative freq.:  0.0472 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  06 <\/br># assignments:  341 <\/br>Relative freq.:  0.5993 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  228 <\/br>Relative freq.:  0.4007 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  520 <\/br>Relative freq.:  0.7692 <\/br>Relative Rep. freq.:  0.0214","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  156 <\/br>Relative freq.:  0.2308 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  06 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0331","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  04 <\/br># assignments:  32 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0183","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  02 <\/br># assignments:  467 <\/br>Relative freq.:  0.538 <\/br>Relative Rep. freq.:  0.0228","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  06 <\/br># assignments:  401 <\/br>Relative freq.:  0.462 <\/br>Relative Rep. freq.:  0.0196","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  02 <\/br># assignments:  97 <\/br>Relative freq.:  0.5706 <\/br>Relative Rep. freq.:  0.0191","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  06 <\/br># assignments:  73 <\/br>Relative freq.:  0.4294 <\/br>Relative Rep. freq.:  0.0144","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  252 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0354","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  240 <\/br>Relative freq.:  0.7059 <\/br>Relative Rep. freq.:  0.0214","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  04 <\/br># assignments:  100 <\/br>Relative freq.:  0.2941 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  322 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0402","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  264 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.015","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  04 <\/br># assignments:  281 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  826 <\/br>Relative freq.:  0.7448 <\/br>Relative Rep. freq.:  0.0345","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  04 <\/br># assignments:  283 <\/br>Relative freq.:  0.2552 <\/br>Relative Rep. freq.:  0.0118","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  02 <\/br># assignments:  399 <\/br>Relative freq.:  0.562 <\/br>Relative Rep. freq.:  0.0237","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  06 <\/br># assignments:  311 <\/br>Relative freq.:  0.438 <\/br>Relative Rep. freq.:  0.0184","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  02 <\/br># assignments:  872 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0452","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  02 <\/br># assignments:  312 <\/br>Relative freq.:  0.7222 <\/br>Relative Rep. freq.:  0.0277","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  04 <\/br># assignments:  119 <\/br>Relative freq.:  0.2755 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  02 <\/br># assignments:  147 <\/br>Relative freq.:  0.7989 <\/br>Relative Rep. freq.:  0.0246","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  04 <\/br># assignments:  37 <\/br>Relative freq.:  0.2011 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  02 <\/br># assignments:  412 <\/br>Relative freq.:  0.9015 <\/br>Relative Rep. freq.:  0.0282","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  04 <\/br># assignments:  45 <\/br>Relative freq.:  0.0985 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  02 <\/br># assignments:  74 <\/br>Relative freq.:  0.8706 <\/br>Relative Rep. freq.:  0.0234","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  04 <\/br># assignments:  11 <\/br>Relative freq.:  0.1294 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  02 <\/br># assignments:  415 <\/br>Relative freq.:  0.5937 <\/br>Relative Rep. freq.:  0.0192","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  06 <\/br># assignments:  284 <\/br>Relative freq.:  0.4063 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  06 <\/br># assignments:  139 <\/br>Relative freq.:  0.5472 <\/br>Relative Rep. freq.:  0.0191","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  02 <\/br># assignments:  115 <\/br>Relative freq.:  0.4528 <\/br>Relative Rep. freq.:  0.0158","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  1031 <\/br>Relative freq.:  0.8067 <\/br>Relative Rep. freq.:  0.0285","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  247 <\/br>Relative freq.:  0.1933 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  06 <\/br># assignments:  95 <\/br>Relative freq.:  0.7197 <\/br>Relative Rep. freq.:  0.0153","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  04 <\/br># assignments:  37 <\/br>Relative freq.:  0.2803 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  02 <\/br># assignments:  917 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.034","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  02 <\/br># assignments:  354 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0394","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  02 <\/br># assignments:  667 <\/br>Relative freq.:  0.7969 <\/br>Relative Rep. freq.:  0.0305","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  04 <\/br># assignments:  170 <\/br>Relative freq.:  0.2031 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  02 <\/br># assignments:  846 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0399","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  02 <\/br># assignments:  1104 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0609","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  02 <\/br># assignments:  953 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0499","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  02 <\/br># assignments:  667 <\/br>Relative freq.:  0.9751 <\/br>Relative Rep. freq.:  0.0291","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  05 <\/br># assignments:  17 <\/br>Relative freq.:  0.0249 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  02 <\/br># assignments:  126 <\/br>Relative freq.:  0.5753 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  06 <\/br># assignments:  93 <\/br>Relative freq.:  0.4247 <\/br>Relative Rep. freq.:  0.0126","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  02 <\/br># assignments:  1357 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0503","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  04 <\/br># assignments:  286 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  02 <\/br># assignments:  856 <\/br>Relative freq.:  0.8215 <\/br>Relative Rep. freq.:  0.0268","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  04 <\/br># assignments:  185 <\/br>Relative freq.:  0.1775 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  02 <\/br># assignments:  33 <\/br>Relative freq.:  0.5323 <\/br>Relative Rep. freq.:  0.0252","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  06 <\/br># assignments:  29 <\/br>Relative freq.:  0.4677 <\/br>Relative Rep. freq.:  0.0221","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  02 <\/br># assignments:  847 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0445","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  04 <\/br># assignments:  310 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  02 <\/br># assignments:  548 <\/br>Relative freq.:  0.7239 <\/br>Relative Rep. freq.:  0.0295","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  04 <\/br># assignments:  209 <\/br>Relative freq.:  0.2761 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  02 <\/br># assignments:  873 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0472","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  02 <\/br># assignments:  872 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0447","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  02 <\/br># assignments:  630 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0539","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  06 <\/br># assignments:  650 <\/br>Relative freq.:  0.6878 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  04 <\/br># assignments:  294 <\/br>Relative freq.:  0.3111 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  02 <\/br># assignments:  208 <\/br>Relative freq.:  0.8703 <\/br>Relative Rep. freq.:  0.028","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  04 <\/br># assignments:  31 <\/br>Relative freq.:  0.1297 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  546 <\/br>Relative freq.:  0.7583 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  174 <\/br>Relative freq.:  0.2417 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  04 <\/br># assignments:  175 <\/br>Relative freq.:  0.8929 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  05 <\/br># assignments:  21 <\/br>Relative freq.:  0.1071 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  06 <\/br># assignments:  828 <\/br>Relative freq.:  0.7047 <\/br>Relative Rep. freq.:  0.0256","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  04 <\/br># assignments:  347 <\/br>Relative freq.:  0.2953 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  02 <\/br># assignments:  711 <\/br>Relative freq.:  0.7548 <\/br>Relative Rep. freq.:  0.023","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  04 <\/br># assignments:  231 <\/br>Relative freq.:  0.2452 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  04 <\/br># assignments:  153 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0086","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  02 <\/br># assignments:  590 <\/br>Relative freq.:  0.7293 <\/br>Relative Rep. freq.:  0.0238","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  04 <\/br># assignments:  218 <\/br>Relative freq.:  0.2695 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  06 <\/br># assignments:  258 <\/br>Relative freq.:  0.6418 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  04 <\/br># assignments:  144 <\/br>Relative freq.:  0.3582 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  04 <\/br># assignments:  190 <\/br>Relative freq.:  0.892 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  05 <\/br># assignments:  23 <\/br>Relative freq.:  0.108 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  02 <\/br># assignments:  369 <\/br>Relative freq.:  0.8367 <\/br>Relative Rep. freq.:  0.0217","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  04 <\/br># assignments:  72 <\/br>Relative freq.:  0.1633 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  04 <\/br># assignments:  464 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0216","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  02 <\/br># assignments:  420 <\/br>Relative freq.:  0.8519 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  04 <\/br># assignments:  73 <\/br>Relative freq.:  0.1481 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  1552 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0482","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  04 <\/br># assignments:  625 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0214","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  04 <\/br># assignments:  238 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0108","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  06 <\/br># assignments:  344 <\/br>Relative freq.:  0.5603 <\/br>Relative Rep. freq.:  0.0281","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  02 <\/br># assignments:  270 <\/br>Relative freq.:  0.4397 <\/br>Relative Rep. freq.:  0.022","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  02 <\/br># assignments:  331 <\/br>Relative freq.:  0.7043 <\/br>Relative Rep. freq.:  0.0239","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  04 <\/br># assignments:  139 <\/br>Relative freq.:  0.2957 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  02 <\/br># assignments:  616 <\/br>Relative freq.:  0.7681 <\/br>Relative Rep. freq.:  0.0265","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  04 <\/br># assignments:  186 <\/br>Relative freq.:  0.2319 <\/br>Relative Rep. freq.:  0.008","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  02 <\/br># assignments:  554 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0614","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  02 <\/br># assignments:  332 <\/br>Relative freq.:  0.9679 <\/br>Relative Rep. freq.:  0.018","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  05 <\/br># assignments:  11 <\/br>Relative freq.:  0.0321 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  02 <\/br># assignments:  1317 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.0515","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  04 <\/br># assignments:  596 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  04 <\/br># assignments:  403 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0182","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  06 <\/br># assignments:  315 <\/br>Relative freq.:  0.7275 <\/br>Relative Rep. freq.:  0.02","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  118 <\/br>Relative freq.:  0.2725 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  02 <\/br># assignments:  1483 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0482","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  02 <\/br># assignments:  738 <\/br>Relative freq.:  0.811 <\/br>Relative Rep. freq.:  0.0327","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  04 <\/br># assignments:  172 <\/br>Relative freq.:  0.189 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  02 <\/br># assignments:  253 <\/br>Relative freq.:  0.8188 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  04 <\/br># assignments:  56 <\/br>Relative freq.:  0.1812 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  04 <\/br># assignments:  242 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  02 <\/br># assignments:  326 <\/br>Relative freq.:  0.8212 <\/br>Relative Rep. freq.:  0.0267","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  04 <\/br># assignments:  71 <\/br>Relative freq.:  0.1788 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  02 <\/br># assignments:  240 <\/br>Relative freq.:  0.9486 <\/br>Relative Rep. freq.:  0.0196","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  05 <\/br># assignments:  13 <\/br>Relative freq.:  0.0514 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  02 <\/br># assignments:  23 <\/br>Relative freq.:  0.7931 <\/br>Relative Rep. freq.:  0.0209","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  04 <\/br># assignments:  6 <\/br>Relative freq.:  0.2069 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  02 <\/br># assignments:  1478 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0679","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  06 <\/br># assignments:  431 <\/br>Relative freq.:  0.8868 <\/br>Relative Rep. freq.:  0.0264","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  04 <\/br># assignments:  54 <\/br>Relative freq.:  0.1111 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  02 <\/br># assignments:  55 <\/br>Relative freq.:  0.8209 <\/br>Relative Rep. freq.:  0.0298","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  04 <\/br># assignments:  12 <\/br>Relative freq.:  0.1791 <\/br>Relative Rep. freq.:  0.0065"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":78.9705271897053,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["V1-2*02","V1-2*04","V1-2*05","V1-2*06","V1-2*07"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["V1-2*02","V1-2*04","V1-2*05","V1-2*06","V1-2*07"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-76.9529262815323,1629.91234431515],"tickmode":"array","ticktext":["0","500","1000","1500"],"tickvals":[0,500,1000,1500],"categoryorder":"array","categoryarray":["0","500","1000","1500"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"7d6f71d765d1e":{"x":{},"y":{},"text":{},"type":"box"},"7d6f74b67d0c2":{"x":{},"y":{},"text":{}}},"cur_data":"7d6f71d765d1e","visdat":{"7d6f71d765d1e":["function (y) ","x"],"7d6f74b67d0c2":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-1d396b54bcc8b4f367f0" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-1d396b54bcc8b4f367f0">{"x":{"data":[{"x":[1,2,2,2,2,2,2,2,4,2,1,2,2,2,3,4,2,2,2,1,2,1,2,4,1,1,2,2,1,2,1,4,4,2,4,2,2,1,2,1,2,1,3,2,2,3,4,2,1,2,1,1,4,2,1,1,1,2,1,2,1,4,1,2,2,2,1,2,1],"y":[241,43,45,51,9,104,100,86,217,104,3,115,70,46,7,159,25,107,85,160,43,289,44,1,451,120,30,99,136,26,218,215,137,20,302,39,232,442,103,438,86,188,8,74,29,6,124,85,166,62,756,239,121,32,1,1398,196,49,39,8,104,78,273,59,82,116,177,27,624],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.00921831205487,2.09588677547872,2.01120641706511,1.9241777382791,1.94383296533488,1.98074643877335,1.94944466962479,2.06313744098879,3.94277753643692,2.05576359764673,0.915965082636103,1.93121326393448,2.00023762630299,2.09358983607963,2.9985382175073,3.98077271752991,2.01861002338119,1.9618261278607,1.96355755557306,0.937928947713226,2.00116059714928,1.06256867568009,1.98066828059964,4.03086401475593,1.0233592773322,0.958903802139685,2.00619677840732,1.9216566357296,1.09701762003824,1.97592789125629,0.935959179513156,3.980734635517,4.08799321753904,1.96658627404831,4.02669679410756,1.94452383155003,2.00359548772685,0.987205021549016,1.9526900480967,0.974059488158673,1.99509545606561,1.08608713983558,2.99619948039763,1.96393843712285,1.91296228244901,2.94826253075153,3.96816337336786,1.99699481488205,0.974724479811266,1.92006344259717,0.907273105205968,0.942745978757739,3.97401333404705,1.92761025629006,0.95130039001815,1.02453685072251,0.928745820978656,2.08228377150372,1.01767257405445,1.97278041052632,1.02800978859887,4.03238183241338,0.913149881688878,2.0112275743857,1.98088741581887,2.0017877118662,1.03566151116975,2.05380544853397,0.925522519275546],"y":[241.077444223501,43.274773972109,44.7166737180203,51.3374531878158,8.64305650796741,103.867687036097,99.8428867585957,86.3953197447583,216.776443116553,104.091003977507,3.26195291057229,114.83804880064,69.6560796149075,45.9416381603107,7.12469294015318,159.345834250003,25.3964577481151,106.688756914623,85.229992803745,159.869243293628,43.271479286626,288.668688572012,44.2465937966481,0.990624645538628,451.300587139092,119.987648714893,30.043983113952,99.0687921119854,135.715843953937,25.8995661344379,218.356702050194,214.895526473783,136.698177158088,19.8953533437103,302.379114304297,39.0101396379992,231.887894192524,441.990005331859,102.689618843608,438.38485256359,86.2795308053494,188.065471921302,8.1147583797574,73.624777531065,28.8972106626257,5.77075627669692,123.833556286804,84.9889923255891,165.764117901586,61.9526896141469,755.862015713938,238.883362673409,120.93126958143,32.3973791023716,1.00925006046891,1398.10147014316,195.696953740902,48.6048524904996,39.1889545137063,8.07755921296775,104.054483497329,77.6065380167216,272.878591293097,59.1560189075768,81.9500249683857,115.988488934375,176.856193917617,26.8954452468082,623.613215411454],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  241 <\/br>Relative freq.:  0.8486 <\/br>Relative Rep. freq.:  0.0425","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  43 <\/br>Relative freq.:  0.1514 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  45 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  51 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  9 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  104 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0144","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  100 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  86 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  06 <\/br># assignments:  217 <\/br>Relative freq.:  0.6698 <\/br>Relative Rep. freq.:  0.0275","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  104 <\/br>Relative freq.:  0.321 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0093 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  115 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0211","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  70 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  46 <\/br>Relative freq.:  0.8679 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  05 <\/br># assignments:  7 <\/br>Relative freq.:  0.1321 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  06 <\/br># assignments:  159 <\/br>Relative freq.:  0.8641 <\/br>Relative Rep. freq.:  0.0319","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.1359 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  107 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0183","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  85 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.022","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  160 <\/br>Relative freq.:  0.7882 <\/br>Relative Rep. freq.:  0.0384","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  43 <\/br>Relative freq.:  0.2118 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  289 <\/br>Relative freq.:  0.8653 <\/br>Relative Rep. freq.:  0.0392","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  44 <\/br>Relative freq.:  0.1317 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.003 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  451 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0907","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  120 <\/br>Relative freq.:  0.8 <\/br>Relative Rep. freq.:  0.0549","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  30 <\/br>Relative freq.:  0.2 <\/br>Relative Rep. freq.:  0.0137","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  99 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  136 <\/br>Relative freq.:  0.8395 <\/br>Relative Rep. freq.:  0.0394","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  26 <\/br>Relative freq.:  0.1605 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  218 <\/br>Relative freq.:  0.5035 <\/br>Relative Rep. freq.:  0.0318","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  06 <\/br># assignments:  215 <\/br>Relative freq.:  0.4965 <\/br>Relative Rep. freq.:  0.0314","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  06 <\/br># assignments:  137 <\/br>Relative freq.:  0.8726 <\/br>Relative Rep. freq.:  0.0311","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  20 <\/br>Relative freq.:  0.1274 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  06 <\/br># assignments:  302 <\/br>Relative freq.:  0.8856 <\/br>Relative Rep. freq.:  0.0405","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  04 <\/br># assignments:  39 <\/br>Relative freq.:  0.1144 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  232 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0407","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  442 <\/br>Relative freq.:  0.811 <\/br>Relative Rep. freq.:  0.0509","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  103 <\/br>Relative freq.:  0.189 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  02 <\/br># assignments:  438 <\/br>Relative freq.:  0.8359 <\/br>Relative Rep. freq.:  0.042","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  04 <\/br># assignments:  86 <\/br>Relative freq.:  0.1641 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  02 <\/br># assignments:  188 <\/br>Relative freq.:  0.9592 <\/br>Relative Rep. freq.:  0.0367","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  05 <\/br># assignments:  8 <\/br>Relative freq.:  0.0408 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  04 <\/br># assignments:  74 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  04 <\/br># assignments:  29 <\/br>Relative freq.:  0.8286 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  05 <\/br># assignments:  6 <\/br>Relative freq.:  0.1714 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  06 <\/br># assignments:  124 <\/br>Relative freq.:  0.5933 <\/br>Relative Rep. freq.:  0.0238","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  04 <\/br># assignments:  85 <\/br>Relative freq.:  0.4067 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  166 <\/br>Relative freq.:  0.7281 <\/br>Relative Rep. freq.:  0.0431","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  62 <\/br>Relative freq.:  0.2719 <\/br>Relative Rep. freq.:  0.0161","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  756 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0831","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  239 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0784","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  06 <\/br># assignments:  121 <\/br>Relative freq.:  0.7857 <\/br>Relative Rep. freq.:  0.0244","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  32 <\/br>Relative freq.:  0.2078 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0065 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  02 <\/br># assignments:  1398 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0912","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  196 <\/br>Relative freq.:  0.8 <\/br>Relative Rep. freq.:  0.046","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  04 <\/br># assignments:  49 <\/br>Relative freq.:  0.2 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  02 <\/br># assignments:  39 <\/br>Relative freq.:  0.8298 <\/br>Relative Rep. freq.:  0.0349","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  04 <\/br># assignments:  8 <\/br>Relative freq.:  0.1702 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  02 <\/br># assignments:  104 <\/br>Relative freq.:  0.5714 <\/br>Relative Rep. freq.:  0.035","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  06 <\/br># assignments:  78 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.0263","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  273 <\/br>Relative freq.:  0.8223 <\/br>Relative Rep. freq.:  0.0339","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  59 <\/br>Relative freq.:  0.1777 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  04 <\/br># assignments:  82 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  116 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0207","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  177 <\/br>Relative freq.:  0.8676 <\/br>Relative Rep. freq.:  0.0346","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  27 <\/br>Relative freq.:  0.1324 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  624 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0859"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":78.9705271897053,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,4.6],"tickmode":"array","ticktext":["V1-2*02","V1-2*04","V1-2*05","V1-2*06"],"tickvals":[1,2,3,4],"categoryorder":"array","categoryarray":["V1-2*02","V1-2*04","V1-2*05","V1-2*06"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-68.8649176293425,1467.95701241804],"tickmode":"array","ticktext":["0","500","1000"],"tickvals":[0,500,1000],"categoryorder":"array","categoryarray":["0","500","1000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"7d6f72eb88169":{"x":{},"y":{},"text":{},"type":"box"},"7d6f77256504e":{"x":{},"y":{},"text":{}}},"cur_data":"7d6f72eb88169","visdat":{"7d6f72eb88169":["function (y) ","x"],"7d6f77256504e":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-1" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 3 zygousity states.</p>
<style>
  
#scroll-box {
  overflow-y: scroll;
  overflow-x: scroll !important;
}
</style>
<div class="container">
<iframe id="scroll-box" src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV1-2G2%22" scrolling="yes" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:800px;overflow-x: scroll !important;overflow: scroll;" cellspacing="0">
</iframe>
</div>
</div>
<div id="observations-1" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/lasdc2i84m8j4lohwgrpa/IGHV1-2G2.docx?dl=0&amp;rlkey=lrfhe80o9069hpn82tcxud8a7" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV1-2G2" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-1" class="section level2">
<h2>Conclusions</h2>
<p>From the results we believe that the cutoff for this group should be closer to <span class="math inline">\(5\%\)</span> except allele <strong>05</strong> that should stay at <span class="math inline">\(0.5\%\)</span>, and for the adjusted states the allele combinations and the relations are stated in the table below.</p>
<div id="allele-specific-cutoff-5-for-all-except-0.5-for-allele-05" class="section level3">
<h3>Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></h3>
<style>
  
#scroll-box {
  overflow-y: scroll;
  overflow-x: scroll !important;
}
</style>
<div class="container">
<iframe id="scroll-box" src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV1-2G2%22&amp;allele_thresh=%22V1-2*01q5bV1-2*02q5bV1-2*03q5bV1-2*04q5bV1-2*05q0.5bV1-2*06q5bV1-2*07q5%22" scrolling="yes" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:800px;overflow-x: scroll !important;overflow: scroll;" cellspacing="0">
</iframe>
</div>
<div id="htmlwidget-cbabd15bb5bb680cef66" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-cbabd15bb5bb680cef66">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7"],[1,1,1,2,2,2,2],["V1-2*02","V1-2*04","V1-2*06","V1-2*02;V1-2*02","V1-2*04;V1-2*04","V1-2*06;V1-2*06","V1-2*07;V1-2*07"],["1:0.743","1:0.771","1:1","0.839:0.704;0.308:0.163;0.114:0.041;0.754:0.466","0.308:0.163;0.114:0.041","0.839:0.704;0.308:0.163;0.114:0.041;0.754:0.466","0.839:0.704;0.68:0.471"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Zygousity state<\/th>\n      <th>V\nallele<\/th>\n      <th>IQR range<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"tipr","columnDefs":[{"className":"dt-right","targets":1},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="01-G1.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="03-G3.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/02-G2.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
