<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV1-58 - G7 | Functional groups</title>
  <meta name="description" content="IGHV1-58 - G7 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV1-58 - G7 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV1-58 - G7 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-15" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="06-G6.html"/>
<link rel="next" href="08-G8.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-checklist"><i class="fa fa-check"></i>Group Checklist</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv1-58---g7" class="section level1">
<h1>IGHV1-58 - G7</h1>
<div id="allele-appearnce-6" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of IGHV1-58 includes 3 alleles, 3 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-67-1.png" width="672" /></p>
</div>
<div id="group-alignment-5" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-68-1.png" width="1728" /><img src="reference-book_files/figure-html/unnamed-chunk-68-2.png" width="1728" /></p>
</div>
<div id="sequence-depth-6" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-e476142b60e5f19e39e2" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-e476142b60e5f19e39e2">{"x":{"visdat":{"9c99e4f725123":["function () ","plotlyVisDat"]},"cur_data":"9c99e4f725123","attrs":{"9c99e4f725123":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatter","x":{},"y":{},"text":{},"marker":{"color":"#FAAB18","size":8,"line":{"width":1,"color":"gray"}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":"text","inherit":true},"9c99e4f725123.1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"marker":{"color":"#FAAB18"},"type":"box","hoverinfo":"none","showlegend":false,"fillcolor":"transparent","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","xaxis":{"domain":[0,1],"automargin":true,"title":"Alleles","autotick":false,"tickmode":"array","tickvals":[1,2,3],"ticktext":["V1-58*01","V1-58*02","V1-58*03"]},"yaxis":{"domain":[0,1],"automargin":true,"title":"# Sequences"},"annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}],"showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"type":"scatter","x":[1.12470122352242,2.09958501672372,1.19859409276396,1.98919108482078,1.87831897307187,0.913445879612118,2.09317165678367,1.14298039609566,1.89363323012367,1.07582994056866,2.07004526033998,1.16267537632957,1.11751388153061,1.87215053793043,2.14000772433355,1.11148567283526,2.16554921437055,0.855418387055397,0.879768485110253,1.86310438532382,0.893437514547259,0.816893730126321,1.06365938736126,3.19516874980181,1.9399620658718,1.07595320576802,0.865019357576966,2.06795989088714,1.03301196256652,2.00752700874582,3.04802159331739,1.18395529584959,2.02576124612242,0.950891193654388,1.95618857303634,1.84335832241923,0.953235523588955,2.1830924930051,1.90734978243709,0.955941919516772,1.14314883053303,1.95473410151899,2.05991372922435,1.04006749382243,2.0834401121363,0.953359258174896,1.92534972056746,0.939550508745015,2.07688222676516,1.00196810765192,2.19257431775331,0.821542466152459,1.86235431581736,1.09275818429887,0.888286061026156,1.85235306946561,1.87499070195481,1.05119473878294,1.96536304829642,1.19971098052338,1.15093114348128,2.11860106047243,1.0850401898846,2.04364420035854,0.994448989909142,2.16608292795718,1.00093973064795,2.15613870853558,0.914708437770605,1.90364991175011,2.09224997237325,0.875057159829885,0.839298684056848,3.13239274369553,0.893892415519804,1.85822669537738,0.850985670089722,1.08925101449713,2.17049543848261,1.10396053278819,2.135257334169,1.16248949794099,1.96850719703361,1.02212078487501,2.11508817393333,1.17463011164218,1.92055974313989,2.17881757840514,1.12752058012411,2.13378989668563,1.16474704872817,2.17645782558247,1.85972021892667,0.979180397745222,1.19016202520579,0.964028993155807,0.91331809842959,1.86371950041503,0.976587674114853,1.85335739757866,0.85497372597456,1.89801485957578,0.99426018409431,2.04939378276467,1.00425140652806,0.955071074515581,2.05375871676952,0.994237128365785,1.85864428449422,2.06050948612392,1.0378500287421,0.976825221814215,2.03110886253417,2.10178533103317,0.831877912208438,0.849357604514807,0.842140633706003,0.806190146040171,2.00246751094237,3.08707519033924,0.871235696785152,1.0871080879122,2.04738530181348,1.07153931241482,2.14665633318946,1.9313013214618,0.826821838039905,1.87084223469719,1.13503210153431,2.12857962492853,1.95826561730355,1.01817636182532,2.09209384163842,3.10554489484057,1.80817979546264,0.819460540637374,0.912965758889914,2.19931229026988,1.09514256007969,2.16837430177256,1.18587762080133,1.8570571209304,0.907239707559347,1.99896283596754,0.916709155589342,1.96730847703293,1.92608950203285,0.899310035351664,1.95306424722075,1.01377606010064,1.18443816984072,1.86216833861545],"y":[54,71,20,11,115,61,41,52,26,24,18,40,10,2,18,93,53,89,29,19,4,144,92,1,40,36,41,38,37,13,1,65,9,16,6,68,32,18,104,117,1,1,57,12,4,11,3,34,12,32,2,79,58,137,107,76,39,43,19,55,14,3,29,12,3,2,89,11,199,13,30,25,40,1,60,30,120,38,36,40,17,87,15,202,30,67,57,4,43,28,52,41,41,86,99,40,93,36,12,6,89,85,73,47,134,55,21,66,37,49,35,158,46,118,78,92,205,105,52,1,78,43,27,29,13,79,26,11,40,16,65,75,65,1,69,68,56,17,43,28,20,19,75,26,24,16,27,2,54,64,2,2],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  54 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  71 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  20 <\/br>Relative freq.:  0.6452 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  11 <\/br>Relative freq.:  0.3548 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  115 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  61 <\/br>Relative freq.:  0.598 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  41 <\/br>Relative freq.:  0.402 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  52 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  26 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  24 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  18 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  40 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  10 <\/br>Relative freq.:  0.8333 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  2 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  18 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  93 <\/br>Relative freq.:  0.637 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  53 <\/br>Relative freq.:  0.363 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  89 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  29 <\/br>Relative freq.:  0.6042 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  19 <\/br>Relative freq.:  0.3958 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  144 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  92 <\/br>Relative freq.:  0.9892 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  V1-58*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0108 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  40 <\/br>Relative freq.:  0.5263 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  36 <\/br>Relative freq.:  0.4737 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  41 <\/br>Relative freq.:  0.519 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  38 <\/br>Relative freq.:  0.481 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  37 <\/br>Relative freq.:  0.7255 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  13 <\/br>Relative freq.:  0.2549 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  V1-58*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0196 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  65 <\/br>Relative freq.:  0.8784 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  9 <\/br>Relative freq.:  0.1216 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  16 <\/br>Relative freq.:  0.7273 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  6 <\/br>Relative freq.:  0.2727 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  68 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  32 <\/br>Relative freq.:  0.64 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  18 <\/br>Relative freq.:  0.36 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  104 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  117 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  1 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  1 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  57 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  12 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  4 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  11 <\/br>Relative freq.:  0.7857 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  3 <\/br>Relative freq.:  0.2143 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  34 <\/br>Relative freq.:  0.7391 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  12 <\/br>Relative freq.:  0.2609 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  32 <\/br>Relative freq.:  0.9412 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0588 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  79 <\/br>Relative freq.:  0.5766 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  58 <\/br>Relative freq.:  0.4234 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  137 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  107 <\/br>Relative freq.:  0.5847 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  76 <\/br>Relative freq.:  0.4153 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  39 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  43 <\/br>Relative freq.:  0.6935 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  19 <\/br>Relative freq.:  0.3065 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  55 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  14 <\/br>Relative freq.:  0.8235 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  3 <\/br>Relative freq.:  0.1765 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  29 <\/br>Relative freq.:  0.7073 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  12 <\/br>Relative freq.:  0.2927 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  3 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  2 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  89 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  199 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  30 <\/br>Relative freq.:  0.5455 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  25 <\/br>Relative freq.:  0.4545 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  40 <\/br>Relative freq.:  0.9756 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  V1-58*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0244 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  60 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  30 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  120 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  38 <\/br>Relative freq.:  0.5135 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  36 <\/br>Relative freq.:  0.4865 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  40 <\/br>Relative freq.:  0.7018 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  17 <\/br>Relative freq.:  0.2982 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  87 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  15 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  202 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  30 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  67 <\/br>Relative freq.:  0.5403 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  57 <\/br>Relative freq.:  0.4597 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  43 <\/br>Relative freq.:  0.6056 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  28 <\/br>Relative freq.:  0.3944 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  52 <\/br>Relative freq.:  0.5591 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  41 <\/br>Relative freq.:  0.4409 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  41 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  86 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  99 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  40 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  93 <\/br>Relative freq.:  0.7209 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  36 <\/br>Relative freq.:  0.2791 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  12 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  6 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  89 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  85 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  73 <\/br>Relative freq.:  0.6083 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  47 <\/br>Relative freq.:  0.3917 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  134 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  55 <\/br>Relative freq.:  0.7237 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  21 <\/br>Relative freq.:  0.2763 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  66 <\/br>Relative freq.:  0.6408 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  37 <\/br>Relative freq.:  0.3592 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  49 <\/br>Relative freq.:  0.5833 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  35 <\/br>Relative freq.:  0.4167 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  158 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  46 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  118 <\/br>Relative freq.:  0.602 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  78 <\/br>Relative freq.:  0.398 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  92 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  205 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  105 <\/br>Relative freq.:  0.6646 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  52 <\/br>Relative freq.:  0.3291 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  V1-58*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0063 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  78 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  43 <\/br>Relative freq.:  0.6143 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  27 <\/br>Relative freq.:  0.3857 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  29 <\/br>Relative freq.:  0.6905 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  13 <\/br>Relative freq.:  0.3095 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  79 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  26 <\/br>Relative freq.:  0.7027 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  11 <\/br>Relative freq.:  0.2973 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  40 <\/br>Relative freq.:  0.7143 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  16 <\/br>Relative freq.:  0.2857 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  65 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  75 <\/br>Relative freq.:  0.5319 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  65 <\/br>Relative freq.:  0.461 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V1-58*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0071 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  69 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  68 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  56 <\/br>Relative freq.:  0.7671 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  17 <\/br>Relative freq.:  0.2329 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  43 <\/br>Relative freq.:  0.6056 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  28 <\/br>Relative freq.:  0.3944 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  20 <\/br>Relative freq.:  0.5128 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  19 <\/br>Relative freq.:  0.4872 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  75 <\/br>Relative freq.:  0.7426 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  26 <\/br>Relative freq.:  0.2574 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  24 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  16 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  27 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  54 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  64 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  2 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  2 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0011"],"marker":{"color":"#FAAB18","size":8,"line":{"color":"gray","width":1}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"transparent","x":[1,2,1,2,2,1,2,1,2,1,2,1,1,2,2,1,2,1,1,2,1,1,1,3,2,1,1,2,1,2,3,1,2,1,2,2,1,2,2,1,1,2,2,1,2,1,2,1,2,1,2,1,2,1,1,2,2,1,2,1,1,2,1,2,1,2,1,2,1,2,2,1,1,3,1,2,1,1,2,1,2,1,2,1,2,1,2,2,1,2,1,2,2,1,1,1,1,2,1,2,1,2,1,2,1,1,2,1,2,2,1,1,2,2,1,1,1,1,2,3,1,1,2,1,2,2,1,2,1,2,2,1,2,3,2,1,1,2,1,2,1,2,1,2,1,2,2,1,2,1,1,2],"y":[54,71,20,11,115,61,41,52,26,24,18,40,10,2,18,93,53,89,29,19,4,144,92,1,40,36,41,38,37,13,1,65,9,16,6,68,32,18,104,117,1,1,57,12,4,11,3,34,12,32,2,79,58,137,107,76,39,43,19,55,14,3,29,12,3,2,89,11,199,13,30,25,40,1,60,30,120,38,36,40,17,87,15,202,30,67,57,4,43,28,52,41,41,86,99,40,93,36,12,6,89,85,73,47,134,55,21,66,37,49,35,158,46,118,78,92,205,105,52,1,78,43,27,29,13,79,26,11,40,16,65,75,65,1,69,68,56,17,43,28,20,19,75,26,24,16,27,2,54,64,2,2],"marker":{"color":"#FAAB18","line":{"color":"rgba(255,127,14,1)"}},"type":"box","hoverinfo":["none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none"],"showlegend":false,"name":"transparent","line":{"color":"rgba(255,127,14,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-bcd7fc34a6192cd88763" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-bcd7fc34a6192cd88763">{"x":{"visdat":{"9c99e56170b25":["function () ","plotlyVisDat"]},"cur_data":"9c99e56170b25","attrs":{"9c99e56170b25":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatter","x":{},"y":{},"text":{},"marker":{"color":"#588300","size":8,"line":{"width":1,"color":"gray"}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":"text","inherit":true},"9c99e56170b25.1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"marker":{"color":"#588300"},"type":"box","hoverinfo":"none","showlegend":false,"fillcolor":"transparent","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","xaxis":{"domain":[0,1],"automargin":true,"title":"Alleles","autotick":false,"tickmode":"array","tickvals":[1,2,3],"ticktext":["V1-58*01","V1-58*02","V1-58*03"]},"yaxis":{"domain":[0,1],"automargin":true,"title":"# Sequences"},"annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}],"showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"type":"scatter","x":[2.05957321384922,0.856226634047925,0.922199081815779,0.873536295443773,0.86660867119208,1.07960773240775,0.833680503442883,1.10172381578013,2.12085446724668,0.913180327042937,1.97677622400224,0.947628172021359,2.0080854495056,0.906283249240369,0.962437234725803,1.82555332286283,0.934178067930043,2.07752664061263,1.1737708450295,2.13790699364617,2.16150830890983,0.92721867095679,0.808595746848732,0.966551457345486,1.80064124045894,0.907098388578743,2.0816097214818,1.16772978408262,2.08369792476296,1.94276060508564,1.19455736009404,1.94751911349595,0.926082996930927,1.9033905592747,1.88713442180306,1.1952594040893,1.93831762224436,1.14804964456707,1.8972803203389,1.14534695940092,0.913097622990608,1.80518339630216,2.14655098672956,2.03846338801086,0.841444617509842,1.87689336966723,1.03683640053496,2.07565727839246,0.808020710293204,2.05485567655414,1.15188401257619,2.14514244236052,0.821028502378613,1.10221580294892,1.1791669853963,1.88310976270586,0.919147143419832,0.933027968369424],"y":[13,10,17,4,42,24,37,17,10,14,5,5,4,39,8,5,12,8,6,4,19,17,11,4,2,9,8,8,5,11,9,8,26,15,17,19,13,15,2,29,14,21,18,27,16,10,12,29,28,4,4,2,12,45,33,13,32,34],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  10 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  17 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  24 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  37 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  17 <\/br>Relative freq.:  0.6296 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  10 <\/br>Relative freq.:  0.3704 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  14 <\/br>Relative freq.:  0.7368 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  5 <\/br>Relative freq.:  0.2632 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  5 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  4 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  39 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  8 <\/br>Relative freq.:  0.6154 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  5 <\/br>Relative freq.:  0.3846 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  12 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  8 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  6 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  4 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  19 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  17 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  4 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  2 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  9 <\/br>Relative freq.:  0.5294 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  8 <\/br>Relative freq.:  0.4706 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  8 <\/br>Relative freq.:  0.6154 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  5 <\/br>Relative freq.:  0.3846 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  11 <\/br>Relative freq.:  0.55 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  9 <\/br>Relative freq.:  0.45 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  8 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  26 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  15 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  17 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  19 <\/br>Relative freq.:  0.5938 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  13 <\/br>Relative freq.:  0.4062 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  15 <\/br>Relative freq.:  0.8824 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  2 <\/br>Relative freq.:  0.1176 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  29 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  14 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  21 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  18 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  27 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  16 <\/br>Relative freq.:  0.6154 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  10 <\/br>Relative freq.:  0.3846 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  29 <\/br>Relative freq.:  0.5088 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  28 <\/br>Relative freq.:  0.4912 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  4 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  2 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  45 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  33 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  V1-58*02 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  32 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  V1-58*01 <\/br># assignments:  34 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0047"],"marker":{"color":"#588300","size":8,"line":{"color":"gray","width":1}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"transparent","x":[2,1,1,1,1,1,1,1,2,1,2,1,2,1,1,2,1,2,1,2,2,1,1,1,2,1,2,1,2,2,1,2,1,2,2,1,2,1,2,1,1,2,2,2,1,2,1,2,1,2,1,2,1,1,1,2,1,1],"y":[13,10,17,4,42,24,37,17,10,14,5,5,4,39,8,5,12,8,6,4,19,17,11,4,2,9,8,8,5,11,9,8,26,15,17,19,13,15,2,29,14,21,18,27,16,10,12,29,28,4,4,2,12,45,33,13,32,34],"marker":{"color":"#588300","line":{"color":"rgba(255,127,14,1)"}},"type":"box","hoverinfo":["none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none"],"showlegend":false,"name":"transparent","line":{"color":"rgba(255,127,14,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-5" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 5 zygousity states.</p>
<style>
  
#scroll-box {
  overflow-y: scroll;
  overflow-x: scroll !important;
}
</style>
<div class="container">
<iframe id="scroll-box" src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV1-58G7%22" scrolling="yes" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:800px;overflow-x: scroll !important;overflow: scroll;" cellspacing="0">
</iframe>
</div>
</div>
<div id="observations-6" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/451zbu4vc3azmn10g7i3r/IGHV1-58G7.docx?dl=0&amp;rlkey=q4wythpidgi1tzi4x7eqtv5cv" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV1-58G7" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-6" class="section level2">
<h2>Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="06-G6.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="08-G8.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/07-G7.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
