<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>8 IGHV1-58 - G7 | Functional groups</title>
  <meta name="description" content="8 IGHV1-58 - G7 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="8 IGHV1-58 - G7 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="8 IGHV1-58 - G7 | Functional groups" />
  
  
  



<meta name="date" content="2021-11-30" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="06-G6.html"/>
<link rel="next" href="08-G8.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv1-58---g7" class="section level1" number="8">
<h1><span class="header-section-number">8</span> IGHV1-58 - G7</h1>
<div id="allele-appearnce-6" class="section level2" number="8.1">
<h2><span class="header-section-number">8.1</span> Allele appearnce</h2>
<p>The group of IGHV1-58 includes 3 alleles, 3 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-64-1.png" width="672" /></p>
</div>
<div id="group-alignment-5" class="section level2" number="8.2">
<h2><span class="header-section-number">8.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-65-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-6" class="section level2" number="8.3">
<h2><span class="header-section-number">8.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-38863e1a1382c0ea86e3" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-38863e1a1382c0ea86e3">{"x":{"data":[{"x":[1,2,1,2,2,1,2,1,2,1,2,1,1,2,2,1,2,1,1,2,1,1,1,3,2,1,1,2,1,2,3,1,2,1,2,2,1,2,2,1,1,2,2,1,2,1,2,1,2,1,2,1,2,1,1,2,2,1,2,1,1,2,1,2,1,2,1,2,1,2,2,1,1,3,1,2,1,1,2,1,2,1,2,1,2,1,2,2,1,2,3,1,2,2,1,1,1,1,2,1,2,1,2,1,2,1,1,2,1,2,2,1,1,2,2,1,1,1,1,2,3,1,1,2,1,2,2,1,2,1,2,2,1,3,1,2,3,2,1,1,2,1,2,1,2,1,2,1,2,2,1,2,1,1,2],"y":[63,72,20,11,117,65,41,53,26,24,18,40,10,2,18,93,53,93,31,19,4,150,92,1,40,36,41,39,40,14,1,67,10,16,7,69,38,22,106,119,2,1,57,12,4,11,3,34,12,32,2,84,64,139,107,76,42,44,19,55,14,3,34,12,3,2,91,19,199,13,30,25,40,1,63,30,123,39,36,41,17,93,16,203,32,71,60,4,51,29,1,56,42,42,87,100,41,94,39,12,6,92,85,74,47,140,55,21,66,37,50,35,160,50,119,80,95,207,107,54,1,87,50,32,30,13,82,27,12,41,16,72,1,1,76,66,1,71,68,56,17,44,29,22,20,81,27,26,16,27,2,60,69,2,2],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.04607822643593,2.04619396119379,0.974582083197311,1.91513511231169,1.92032252466306,0.959061182895675,2.01668544244021,0.98702409863472,1.91838420298882,0.947710392717272,1.98983747740276,1.07514547798783,1.03908417508937,1.97670897198841,2.01760569340549,0.94601869196631,1.95969485668466,0.921117123542354,0.917676954017952,1.97066570064053,0.980281154345721,0.963825826765969,0.987038842728362,3.05666811801493,1.92202112437226,1.06052223984152,0.954098905948922,2.0165931112133,1.09593310998753,2.00212111272849,3.05430188947357,1.09258596180007,1.99254588205367,0.977667892538011,2.09737399988808,2.09043645551428,0.971929432498291,2.04001061534509,1.97924349214882,0.945575846778229,0.928352606436238,2.08938462343067,2.04933014819399,0.987060328572989,2.07913110158406,0.923640708858147,1.9769090639893,0.940962261054665,2.05562953269109,1.07411330156028,1.96576302247122,1.04302308675833,1.99476781282574,0.964025911409408,1.02283165119588,1.91113311364315,1.92146126753651,0.968235962325707,1.98610373986885,1.03117906963453,0.982400250108913,2.05192664708011,1.05743391239084,1.92770725763403,0.93323618946597,2.07970019038767,0.916779675986618,2.04772576242685,1.06291791717522,2.09435844831169,2.02854346297681,1.08363340022042,1.09795398912393,2.94982634480111,0.901722575072199,1.93732677935623,1.00659216623753,0.903223240980878,2.0555684753228,1.05607429044321,2.06922355308197,1.08525453754701,1.98944277833216,1.01511442069896,2.00934711950831,0.910360934212804,2.09134772801772,2.02881949339062,0.973358318815008,2.05595472017303,3.01741322753951,1.03425867361948,2.0114956762176,1.9369444002863,0.930794965848327,0.954103619093075,0.914077296108007,1.08460946981795,1.93456188854761,1.04499805793166,1.9101085006725,0.936204582825303,2.04277733126655,0.994675092864782,2.03012350592762,1.00508633689024,1.04225255479105,2.05040080226026,1.06000610804185,2.04898505001329,1.96114568160847,1.09632616317831,1.02368882470764,2.03583158152178,1.91172888600267,1.06313239508308,0.910755904344842,0.941636303951964,0.989040732430294,2.06789259719662,2.98751048650593,1.06606402248144,1.00403398890048,1.96857751794159,0.953328999690711,1.96432783948258,2.0875691564288,0.990496931504458,1.91249869205058,0.940616883104667,2.07807888030075,2.04303367496468,0.968718351796269,2.96281379661523,1.05725659932941,1.99522648351267,3.00116799185052,1.90860662767664,1.04992280607112,0.948482847772539,1.91628361875191,0.949370207125321,1.90764609049074,1.0047196900472,1.99928946509026,0.985570092173293,1.93334801835008,0.953235794510692,1.94467297196388,2.00515510588884,1.07190078711137,2.08946914388798,1.08158434848301,1.08613827307709,2.0432416123338],"y":[63.1487486029044,71.7500980226323,19.8108759528026,11.1130073808134,116.915697490424,65.3536745879799,40.6205109508708,53.272508292459,25.800722672604,23.7462626706809,17.7922868160531,39.9448710430413,10.1070276262239,1.63611621335149,17.7677649831399,92.9731966037303,53.0262364083901,92.7323411036283,30.6890438454226,19.3008472120389,3.73508516792208,149.945773172937,91.8957125974819,1.3416580112651,39.6559028185904,35.9937965573743,41.2711009090766,38.9596699584275,39.7967552935705,14.048322815448,1.03546655979007,67.2599582737312,9.63274457491934,15.771921488829,6.77323626987636,68.9920516204089,37.6808869505301,21.9930083232,106.245480256528,119.018043917231,2.36251341663301,1.25253734923899,57.2644743137062,11.8896257141605,4.3230998609215,10.8687908219174,3.24730886444449,33.6809936607257,11.6027441628277,32.340513048321,1.77994371280074,84.3455983757973,63.8342394100502,138.776951492019,106.784573275596,75.8428902583197,42.2445269711316,44.2374894907698,19.2971440276131,55.2138574138284,13.7108041893691,2.69649720601738,34.3509146407247,11.9443828143179,2.97890572156757,1.91722362842411,90.7900025231764,18.8199901251122,199.233967305347,13.3294370342046,29.7488257968798,24.8302442759275,40.251278854534,1.26354502290487,62.7435086891055,29.7344490427524,123.113523943536,38.6239467665553,35.6509275911376,40.630702101253,17.3716477362439,93.3311969691888,16.0063844686374,202.736710623279,32.1575583186001,70.6927585126832,60.158859949559,3.82647140156478,51.0717408871278,28.7543627154082,0.797670298814774,55.8536792054772,42.3528821971267,41.9728599699214,86.700148913078,99.7156820241362,41.0699818149209,93.749026873894,38.6832985294983,11.7881978530437,5.68797377254814,91.7679162915796,85.1768771152943,73.9994217172265,47.3987621665001,139.600406890735,54.8612736059353,20.6275264868513,66.2254701690748,36.7439052779227,50.1986986510456,34.7352216379717,159.697247008979,50.237047485821,119.346737865359,79.919156037271,94.6365633020177,206.892016410269,106.750470080227,53.6323254499584,1.10472250021994,87.1379957539961,50.2951214715838,32.1414227092639,30.3919878423214,12.8306318467483,82.2485006585717,27.2892636287957,12.3017585171387,41.1311545019969,16.1311711827293,72.3102529350668,0.768813383020461,0.719897210225463,75.8192367242649,66.2390805210918,0.926858011074364,71.2477556584403,67.6954866735265,56.0204046145082,16.9423084467649,43.8158295983449,28.6206689562649,21.9516419060528,19.9280941372737,81.3390812113881,27.3470819562674,26.1203718969598,16.0817854464054,26.6633744189516,1.99990498255938,60.2040834831074,69.2630148956552,1.93710643090308,2.38493076618761],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  63 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  02 <\/br># assignments:  72 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.6452 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  02 <\/br># assignments:  11 <\/br>Relative freq.:  0.3548 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  02 <\/br># assignments:  117 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  65 <\/br>Relative freq.:  0.6132 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  41 <\/br>Relative freq.:  0.3868 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  53 <\/br>Relative freq.:  0.6709 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  26 <\/br>Relative freq.:  0.3291 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  18 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  40 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  10 <\/br>Relative freq.:  0.8333 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  02 <\/br># assignments:  18 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  93 <\/br>Relative freq.:  0.637 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  53 <\/br>Relative freq.:  0.363 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  93 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  31 <\/br>Relative freq.:  0.62 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  19 <\/br>Relative freq.:  0.38 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  150 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  92 <\/br>Relative freq.:  0.9892 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0108 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  40 <\/br>Relative freq.:  0.5263 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  0.4737 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  0.5125 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  39 <\/br>Relative freq.:  0.4875 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  40 <\/br>Relative freq.:  0.7273 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  14 <\/br>Relative freq.:  0.2545 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0182 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  67 <\/br>Relative freq.:  0.8701 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.1299 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  0.6957 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.3043 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  02 <\/br># assignments:  69 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  38 <\/br>Relative freq.:  0.6333 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  22 <\/br>Relative freq.:  0.3667 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  02 <\/br># assignments:  106 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  119 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  02 <\/br># assignments:  57 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.7857 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.2143 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  0.7391 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  0.2609 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  32 <\/br>Relative freq.:  0.9412 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0588 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  84 <\/br>Relative freq.:  0.5676 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  64 <\/br>Relative freq.:  0.4324 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  139 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  107 <\/br>Relative freq.:  0.5847 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  76 <\/br>Relative freq.:  0.4153 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  02 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  44 <\/br>Relative freq.:  0.6984 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  02 <\/br># assignments:  19 <\/br>Relative freq.:  0.3016 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  55 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  0.8235 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.1765 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  0.7391 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  0.2609 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  91 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  02 <\/br># assignments:  19 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  199 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  02 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  02 <\/br># assignments:  30 <\/br>Relative freq.:  0.5455 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  25 <\/br>Relative freq.:  0.4545 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  40 <\/br>Relative freq.:  0.9756 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0244 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  63 <\/br>Relative freq.:  0.6774 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  02 <\/br># assignments:  30 <\/br>Relative freq.:  0.3226 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  123 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  0.52 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  02 <\/br># assignments:  36 <\/br>Relative freq.:  0.48 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  0.7069 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  02 <\/br># assignments:  17 <\/br>Relative freq.:  0.2931 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  93 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  02 <\/br># assignments:  16 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  203 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  02 <\/br># assignments:  32 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.542 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  02 <\/br># assignments:  60 <\/br>Relative freq.:  0.458 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  0.6296 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  02 <\/br># assignments:  29 <\/br>Relative freq.:  0.358 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0123 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  56 <\/br>Relative freq.:  0.5714 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  02 <\/br># assignments:  42 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  02 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  87 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  100 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  94 <\/br>Relative freq.:  0.7068 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  02 <\/br># assignments:  39 <\/br>Relative freq.:  0.2932 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  92 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  02 <\/br># assignments:  85 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  74 <\/br>Relative freq.:  0.6116 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  02 <\/br># assignments:  47 <\/br>Relative freq.:  0.3884 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  140 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  55 <\/br>Relative freq.:  0.7237 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  02 <\/br># assignments:  21 <\/br>Relative freq.:  0.2763 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  66 <\/br>Relative freq.:  0.6408 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  02 <\/br># assignments:  37 <\/br>Relative freq.:  0.3592 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  02 <\/br># assignments:  50 <\/br>Relative freq.:  0.5882 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  35 <\/br>Relative freq.:  0.4118 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  01 <\/br># assignments:  160 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  02 <\/br># assignments:  50 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  02 <\/br># assignments:  119 <\/br>Relative freq.:  0.598 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  80 <\/br>Relative freq.:  0.402 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  95 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  207 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  01 <\/br># assignments:  107 <\/br>Relative freq.:  0.6605 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  02 <\/br># assignments:  54 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0062 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  87 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  0.6098 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  02 <\/br># assignments:  32 <\/br>Relative freq.:  0.3902 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  0.6977 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  02 <\/br># assignments:  13 <\/br>Relative freq.:  0.3023 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  02 <\/br># assignments:  82 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.6923 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  0.3077 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  0.7193 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  02 <\/br># assignments:  16 <\/br>Relative freq.:  0.2807 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  02 <\/br># assignments:  72 <\/br>Relative freq.:  0.973 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0135 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0135 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  76 <\/br>Relative freq.:  0.5315 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  02 <\/br># assignments:  66 <\/br>Relative freq.:  0.4615 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.007 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  02 <\/br># assignments:  71 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  68 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  56 <\/br>Relative freq.:  0.7671 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  02 <\/br># assignments:  17 <\/br>Relative freq.:  0.2329 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  44 <\/br>Relative freq.:  0.6027 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  02 <\/br># assignments:  29 <\/br>Relative freq.:  0.3973 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  22 <\/br>Relative freq.:  0.5238 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  02 <\/br># assignments:  20 <\/br>Relative freq.:  0.4762 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  81 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  02 <\/br># assignments:  27 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  26 <\/br>Relative freq.:  0.619 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  02 <\/br># assignments:  16 <\/br>Relative freq.:  0.381 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  02 <\/br># assignments:  27 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  02 <\/br># assignments:  60 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  69 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.001"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V1-58*01","V1-58*02","V1-58*03"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V1-58*01","V1-58*02","V1-58*03"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-9.59410792926326,217.314005139489],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"3c32a652cef41":{"x":{},"y":{},"text":{},"type":"box"},"3c32abfdb7a9":{"x":{},"y":{},"text":{}}},"cur_data":"3c32a652cef41","visdat":{"3c32a652cef41":["function (y) ","x"],"3c32abfdb7a9":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-618393221cedab882911" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-618393221cedab882911">{"x":{"data":[{"x":[2,1,1,1,1,3,1,3,1,1,2,1,2,1,2,1,1,2,1,2,1,2,2,1,1,1,2,2,1,1,2,2,1,2,1,2,2,1,2,1,2,3,1,1,2,2,2,2,1,2,1,2,1,2,1,2,1,1,1,2,1,1],"y":[14,11,19,4,43,1,24,1,37,18,10,15,5,6,4,39,8,5,12,8,6,4,21,19,11,4,2,13,9,8,5,12,10,10,28,16,17,19,13,20,2,2,34,15,1,21,21,28,16,11,12,30,28,4,4,2,13,50,36,15,33,35],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(86,63,80,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[2.0977873433847,0.912266118079424,0.901180636929348,0.94594545098953,1.05242194803432,3.00048729013652,1.05185699714348,3.05541702667251,1.0989597084932,0.953352717170492,2.08701709895395,1.0623054554686,1.99583172388375,0.910586609039456,1.91408459926024,1.070811249502,1.00969647513703,1.93700225334615,0.9099419712089,1.95149028575979,1.02191524887457,1.95333566009067,2.09194019944407,1.06153777344152,1.03989951643161,1.06693669357337,2.01204477678984,2.04358979896642,0.979059491259977,1.07134495112114,2.00763784823939,1.9318489744328,1.05549917505123,1.93251434252597,1.05094335628673,2.09462322527543,2.00814097644761,1.00886734584346,2.0406004420016,0.978227831842378,2.05491761891171,3.00659029935487,0.913088382128626,0.949858774058521,2.01549072884954,1.91216569854878,1.95139916529879,2.04586797468364,0.946094293193892,2.05571918501519,0.948898996599019,2.05891531915404,0.924022640381008,1.97430243417621,1.00191954239272,1.94934746650979,1.00549891134724,1.08770083598793,0.986073743738234,1.93561430103146,1.09319177083671,0.908606607653201],"y":[14.3463281830773,11.0389711342752,18.9778025202453,3.73707512393594,43.2282618235797,0.741994382627308,24.2350726531819,1.23805252164602,36.8871551943943,17.8563031189144,10.125032036379,14.7752695962787,5.05235721766949,5.9483454529196,3.61739717870951,39.0076599936932,7.61380806528032,5.24447445403784,11.7725845769048,8.36774816345423,5.97004977688193,3.8912108367309,21.3868976783007,18.6077564049512,11.1051239926368,3.6859292620793,1.95830608606338,13.3696094442159,8.73745747171342,8.12891581617296,5.28918229099363,11.6579653639346,9.65717090871185,10.0932161387056,27.6146923964843,15.7100755088031,16.969428480044,18.6366636684164,13.0723676228896,19.8263078963384,1.70551690552384,1.86520733386278,34.2498468227684,15.3241357086226,0.970549228973687,20.6593253932893,21.0419977240264,28.0606795907021,15.9576564848423,11.3677759775892,12.2322943978012,29.9602971937507,28.3119739796966,4.37671040557325,3.75051239430904,2.39310946818441,12.6192564655095,49.7099750647321,35.6317062975839,15.1718122858554,32.7074044849724,34.6254544392228],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  14 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  43 <\/br>Relative freq.:  0.9773 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0227 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  0.96 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.04 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  37 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  18 <\/br>Relative freq.:  0.6429 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.3571 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  15 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.6154 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.3846 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  21 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  13 <\/br>Relative freq.:  0.5909 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  0.4091 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.6154 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.3846 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  0.5455 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  10 <\/br>Relative freq.:  0.4545 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  02 <\/br># assignments:  16 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  17 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  0.5938 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  02 <\/br># assignments:  13 <\/br>Relative freq.:  0.4062 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.8333 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0833 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0833 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  15 <\/br>Relative freq.:  0.9375 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0625 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  02 <\/br># assignments:  21 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  21 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  28 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  0.5926 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  11 <\/br>Relative freq.:  0.4074 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  02 <\/br># assignments:  30 <\/br>Relative freq.:  0.5172 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.4828 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  15 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  35 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0047"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(86,63,80,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(86,63,80,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":34.8692403486924},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V1-58*01","V1-58*02","V1-58*03"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V1-58*01","V1-58*02","V1-58*03"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1.72090589824133,52.4629002808686],"tickmode":"array","ticktext":["0","10","20","30","40","50"],"tickvals":[2.22044604925031e-16,10,20,30,40,50],"categoryorder":"array","categoryarray":["0","10","20","30","40","50"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"3c32a261b7c06":{"x":{},"y":{},"text":{},"type":"box"},"3c32a5e306a8f":{"x":{},"y":{},"text":{}}},"cur_data":"3c32a261b7c06","visdat":{"3c32a261b7c06":["function (y) ","x"],"3c32a5e306a8f":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-12bb90cf6ca1c4af68e5" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-12bb90cf6ca1c4af68e5">{"x":{"data":[{"x":[1,2,3,1,1,2,1,2,1,2,2,1,2,1,1,2,1,3,1,1,1,1,2,3,2,2,2,1,3,1,3,2,1,3,2,2,1,2,3,1,3,2,2,1,1,2,1,2,3,2,2,1,1,2,2,1],"y":[64,22,4,16,37,27,39,28,10,4,47,8,93,57,52,18,44,1,55,36,50,43,17,3,12,9,68,33,2,21,9,2,38,2,1,30,57,39,1,95,3,2,30,5,20,14,23,1,1,9,63,3,31,9,34,1],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.02629824937321,1.99631311995909,2.90225354460999,0.919091952545568,0.950989067927003,1.98972662733868,1.08478456865996,2.0323967625387,0.999728434719145,1.98684729742818,2.09570276904851,0.966250860830769,2.02643648046069,0.929889999749139,1.06870156307705,1.93680059323087,1.0620527701918,3.02994660427794,0.918925208318979,0.943035074882209,0.995231674658135,1.0902534567751,1.99293099041097,2.97740816432051,1.97635352103971,1.95653245490976,1.96988773308694,1.08212744742632,3.00191415431909,0.9637420729734,3.07836882076226,2.03149584089406,0.94457493131049,3.03017164031044,1.97539529977366,2.09294652747922,1.02317164051346,2.02802029424347,2.91824741484597,1.09774664272554,2.97009876300581,1.96341638248414,2.00111498120241,1.06820658645593,0.964172090822831,1.99219136517495,1.07232254450209,2.00078301392496,3.00066724224016,2.08659391435795,2.04138100389391,1.03761873338372,1.09952961835079,1.98962775520049,2.06772783617489,0.950372210610658],"y":[64.2450902996585,21.8379675455391,3.86226944606751,16.3843581430614,37.1560460304841,26.6093069573864,38.7722970504314,27.8168217798695,9.63813715055585,3.91119124852121,46.9097951188684,7.86168346889317,93.2052930166945,56.6453716576099,52.1816862126812,18.3186384066939,44.2273432418704,1.37874447144568,54.7189439052716,36.3189523510635,49.6393943754956,42.7746488342062,17.2266804002225,2.64658052511513,11.6888216691092,9.24106652140617,67.6420324170962,32.9400604886934,1.77358587812632,20.9854492146522,8.95441572163254,2.22767207939178,38.0778067963198,2.18159869499505,1.05407810546458,30.0996329065412,56.6648663282394,39.1305336991325,0.749339386075735,94.7083487510681,3.3828353099525,2.28056809771806,30.257415683195,4.71430862601846,20.1359078167006,14.0082226583734,22.6369031319395,1.04338946714997,1.3528619132936,8.99956093449146,63.3540763920173,3.20379799362272,31.2543781748042,9.35455689113587,34.2207231406122,1.35748715698719],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  64 <\/br>Relative freq.:  0.7111 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  22 <\/br>Relative freq.:  0.2444 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  03 <\/br># assignments:  4 <\/br>Relative freq.:  0.0444 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  37 <\/br>Relative freq.:  0.5781 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  02 <\/br># assignments:  27 <\/br>Relative freq.:  0.4219 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  0.5821 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  28 <\/br>Relative freq.:  0.4179 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  10 <\/br>Relative freq.:  0.7143 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.2857 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  47 <\/br>Relative freq.:  0.8545 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.1455 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  93 <\/br>Relative freq.:  0.62 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  0.38 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  52 <\/br>Relative freq.:  0.7429 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  18 <\/br>Relative freq.:  0.2571 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  44 <\/br>Relative freq.:  0.9778 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0222 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  55 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  43 <\/br>Relative freq.:  0.6825 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  17 <\/br>Relative freq.:  0.2698 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.0476 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  68 <\/br>Relative freq.:  0.6602 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  0.3204 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0194 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  21 <\/br>Relative freq.:  0.6562 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  03 <\/br># assignments:  9 <\/br>Relative freq.:  0.2812 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0625 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  38 <\/br>Relative freq.:  0.9268 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0488 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0244 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  30 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  0.5876 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  39 <\/br>Relative freq.:  0.4021 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0103 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  95 <\/br>Relative freq.:  0.95 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.03 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.02 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  02 <\/br># assignments:  30 <\/br>Relative freq.:  0.8571 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  0.1429 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.5882 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  14 <\/br>Relative freq.:  0.4118 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  0.92 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  63 <\/br>Relative freq.:  0.9545 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  0.0455 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  31 <\/br>Relative freq.:  0.775 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  0.225 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  34 <\/br>Relative freq.:  0.9714 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0286 <\/br>Relative Rep. freq.:  1e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":34.8692403486924},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V1-58*01","V1-58*02","V1-58*03"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V1-58*01","V1-58*02","V1-58*03"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-3.96319364462048,99.7125330306962],"tickmode":"array","ticktext":["0","25","50","75"],"tickvals":[0,25,50,75],"categoryorder":"array","categoryarray":["0","25","50","75"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"3c32a35ed3edd":{"x":{},"y":{},"text":{},"type":"box"},"3c32a16cf77b2":{"x":{},"y":{},"text":{}}},"cur_data":"3c32a35ed3edd","visdat":{"3c32a35ed3edd":["function (y) ","x"],"3c32a16cf77b2":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-5" class="section level2" number="8.4">
<h2><span class="header-section-number">8.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 6 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV1-58G7%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:900px;" cellspacing="0">
</iframe>
</div>
<div id="observations-6" class="section level2" number="8.5">
<h2><span class="header-section-number">8.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/451zbu4vc3azmn10g7i3r/IGHV1-58G7.docx?dl=0&amp;rlkey=q4wythpidgi1tzi4x7eqtv5cv">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV1-58G7" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-6" class="section level2" number="8.6">
<h2><span class="header-section-number">8.6</span> Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="06-G6.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="08-G8.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/07-G7.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
