<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV2-70/IGHV2-70D - G13 | Functional groups</title>
  <meta name="description" content="IGHV2-70/IGHV2-70D - G13 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV2-70/IGHV2-70D - G13 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV2-70/IGHV2-70D - G13 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="12-G12.html"/>
<link rel="next" href="14-G14.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i>Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv2-70ighv2-70d---g13" class="section level1">
<h1>IGHV2-70/IGHV2-70D - G13</h1>
<div id="allele-appearnce-12" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of IGHV2-70/IGHV2-70D includes 20 alleles, 20 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-121-1.png" width="672" /></p>
</div>
<div id="group-alignment-10" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-122-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-12" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-5d90f2d4bf3f84aa4b44" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-5d90f2d4bf3f84aa4b44">{"x":{"data":[{"x":[1,6,3,1,3,3,1,6,3,1,6,1,6,3,1,6,1,6,3,3,1,6,1,2,1,6,1,1,1,6,3,1,6,1,6,1,3,1,6,3,1,6,6,3,4,1,6,3,1,6,3,6,1,3,3,1,1,6,1,6,1,3,1,4,1,3,1,3,1,1,3,1,6,3,1,3,6,3,1,6,1,6,2,3,1,3,6,3,6,1,6,1,6,3,1,6,3,1,3,6,3,6,1,3,1,5,1,1,3,1,6,2,4,1,6,1,6,1,6,1,6,1,6,3,1,4,1,6,1,6,2,3,1,1,1,6,1,3,6,3,1,6,1,6,3,1,2,1,1,6,1,1,6,3,1,6,3,1,3,1,1,6,3,1,3,4,3,1,6,1,6,1,6,3,3,1,6,3,1,3,6,1,3,3,6,1,4,3,1,6,3,6],"y":[69,46,139,24,171,114,68,67,75,63,25,17,3,23,46,9,11,2,22,78,74,45,112,1,34,1,7,116,79,46,42,29,21,37,1,29,29,60,25,8,4,2,42,165,1,128,46,2,1,1,5,3,1,155,20,7,7,5,54,29,36,182,102,1,110,134,75,65,49,82,66,60,23,11,9,79,11,8,2,1,85,49,1,28,147,8,1,52,35,59,16,102,81,1,175,63,326,71,63,22,127,76,62,16,292,1,42,128,8,135,112,1,1,52,40,60,2,115,49,146,90,48,17,174,126,1,12,8,87,86,1,152,82,143,88,2,90,73,50,78,52,27,142,44,241,104,1,122,198,69,182,121,75,100,36,21,174,31,89,35,89,4,224,109,183,1,73,54,25,64,52,48,39,54,97,74,54,40,21,34,3,2,1,84,41,39,1,71,30,21,6,1],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[0.908728073257953,5.99360990235582,3.02421100283973,0.952270324761048,2.90666401903145,2.91860602172092,1.06350227282383,5.94225184144452,2.9810329509899,1.01691585672088,5.95291813141666,0.905167104676366,6.07016729121096,3.01396975000389,0.934321318380535,6.09365431573242,1.01570990621112,5.98388945842162,3.02133504953235,3.0335097250063,1.00130067388527,5.9866762206424,1.03288901336491,1.9059896315448,0.902338770916685,6.03807731447741,1.06087201181799,0.937110715033487,0.994235254684463,6.03585347407497,2.97183885462582,1.01617104327306,6.04001863533631,0.990211840951815,5.90706468867138,1.02344267978333,2.95631295619532,0.996030087769032,5.97823354252614,2.98896639989689,0.92285186056979,6.02910053469241,5.94376230598427,3.08298974330537,3.98543058815412,1.06730913873762,6.09601646834053,2.98074775701389,1.0079356606584,5.94677668232471,3.08166784550995,6.04098903448321,0.967739985650405,2.94279605527408,3.06602423479781,0.973485457058996,1.04590062764473,5.96370524386875,0.9008360770531,5.96990584502928,0.930091459490359,2.91497762417421,0.949704098980874,3.91325880889781,0.963763373577967,3.09402957060374,1.07476800680161,2.94744812552817,1.04699982749298,0.905604932270944,3.07867046459578,1.02786740255542,6.0787290723063,2.94953694925643,0.977355921966955,2.91631727670319,6.02311962605454,2.9448463623412,1.05187118016183,6.05846371930093,0.952376918820664,6.068700165255,1.93081585778855,3.01087032728829,0.937645353702828,2.94609471736476,6.02246375759132,2.92072698171251,5.9858382825274,1.01290332791395,5.91953606405295,1.012687798962,5.96498568295501,3.0918370758649,0.996251632319763,5.90373159553856,3.07688058721833,1.00055734398775,3.00532878926024,5.90397948818281,2.98106904653832,5.98025080687366,1.00777760329656,3.02486398406327,1.0810725745745,4.94593353192322,1.05201974185184,0.952842939458787,2.9805124070961,1.00071299481206,6.06361977201886,2.06025905557908,4.06110245538875,0.952382306661457,6.03000157717615,1.06280299457721,6.03354645185173,0.934657613560557,6.04170060898177,0.981740351114422,6.07140603372827,1.03403245755471,6.01046094344929,2.90866859792732,1.05140018807724,4.0354236044921,1.01449530846439,6.05393810016103,0.986716485070065,6.01580779370852,2.05415682028979,2.99767728969455,1.00999887380749,1.05426774211228,1.01993414540775,5.92308259201236,0.996434405585751,3.03243629718199,6.08727989750914,2.95178718692623,0.963634808175266,5.92977743549272,0.992568143829703,6.03390262140892,3.08070836896077,0.926302879676223,1.96811791155487,0.960189083265141,1.09725443120114,5.94846779732034,0.92339558978565,1.06974215353839,6.05598925580271,2.99816763876006,1.0875044613611,5.99990485613234,2.9352967812214,0.97001919709146,3.04549520206638,0.968914727354422,1.06753336698748,5.9617823401466,2.95861190319993,1.07482553343289,2.96302697979845,3.97658997676335,3.0384925047867,0.981963929161429,6.04752773484215,0.953707844344899,6.04001716636121,0.962321803811938,5.90678004752845,3.08821711963974,2.90153239192441,0.997149976110086,6.05644352720119,2.97624417738989,0.946159250941128,2.95483531607315,5.96832455047406,1.00385391209275,3.09144205688499,3.03441163743846,5.98654369004071,1.09303621193394,4.07131209350191,2.96572223333642,0.981880359863862,6.01262531792745,3.06718042530119,6.07157933795825],"y":[68.8996782809496,46.0138266758993,138.882150956802,24.3692333694547,171.376557531394,113.776873637363,68.0629686539993,67.1814947137609,74.6779205905273,62.8843179980293,25.0235260790214,17.2872745582834,2.89844235535711,22.6821387145668,45.6998849228024,9.09031347986311,10.8551850307733,2.126055515185,21.7552305642515,78.2245353408158,74.3554241191596,45.2878689618781,111.800844808854,0.911575665324926,33.6713276594877,0.925948002934456,7.08894266784191,115.661388386227,78.9733489574865,46.3389336274937,42.0665135296062,28.6862626532093,20.8945097221062,36.7807681586593,1.03278724513948,29.1655669569969,28.9204104391858,59.8528770186007,25.0442186515778,7.94469633102417,4.14016050528735,2.26441350858659,42.1980006581172,165.105510529317,0.885599519498646,128.022487963736,45.6200747070834,2.32542543672025,1.33193483091891,1.00536275785416,4.83266242202371,2.96837014127523,0.782161455415189,155.35321233999,19.6701941961423,6.73732825145125,7.25647681131959,5.22817693073303,53.7954309290275,28.6820872528479,36.2455169556662,182.388170552999,101.845166117512,1.19073616378009,109.833309198543,134.167869712412,75.1242179580033,65.2855415698141,49.0375544887036,82.1585944162682,65.7063568918034,59.9625327056274,22.8189066717401,10.791326601617,9.05287444647402,79.2435515439138,10.807837998867,8.24704076685011,2.33333180397749,1.12256554421037,85.199826397188,49.2513800511137,0.614243520610035,27.9032509688288,146.89319871366,7.8319321218878,0.83354730065912,51.9109661966562,35.2798558864743,58.6655993163586,15.9646607674658,101.935048947111,81.2032099157572,1.03406067285687,175.192969318852,63.3783551214263,326.213048408367,70.6949602937326,62.7374936971813,21.9081229940057,126.873945682682,76.0937320433557,62.3152377927676,16.1601042302325,292.381797470152,1.20031555145979,41.6362442197278,128.151525779627,7.8397418897599,135.240860106982,111.867620601133,0.771513158641756,1.25248147249222,52.1221276126802,40.3108403820545,60.0214284131303,2.01930103525519,115.227218282036,48.9045955734327,146.395314182341,89.9211768230423,47.7466967839748,16.7857982210815,174.27095189169,126.216404071637,1.26112429481,11.9786135483533,8.10668333694339,87.2573379175738,86.0100451994687,0.717603592760861,151.639503199607,81.714881448634,143.142459483258,87.7305419431999,1.75105702988803,89.7859620546922,73.3073007358238,49.6258355936036,77.7630268497392,52.2330487921834,26.9117449024692,142.176368875802,43.717948159948,240.665323393419,104.305329298228,0.831584195047617,121.891941899993,198.278076617233,68.9660157095641,181.89895434659,121.002558836155,74.8033042069524,99.8836264325306,35.7554748311639,21.0555137936026,174.320167770423,31.0305792529136,89.2918099731207,35.3365249041468,89.3423706199974,3.88575307615101,224.201606559008,109.247242619097,183.114267336205,0.770702878199518,73.2991549339145,54.3657703733072,25.0116517623886,63.9657311700284,52.1668136600405,47.8058386335149,38.6580619294196,54.0717534091324,96.7656857009977,74.3610667753965,53.8908063700423,40.2899948097765,21.1054418545216,33.7918901704252,2.88510846551508,2.29829628784209,0.674877252057195,83.8447883341461,40.9135789457709,38.8564492912963,1.28669774308801,71.0304600110278,30.2643690086901,21.0664467560127,5.96463346518576,0.774177101999521],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  69 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  46 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  13 <\/br># assignments:  139 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  13 <\/br># assignments:  171 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  13 <\/br># assignments:  114 <\/br>Relative freq.:  0.4578 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  68 <\/br>Relative freq.:  0.2731 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  67 <\/br>Relative freq.:  0.2691 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  13 <\/br># assignments:  75 <\/br>Relative freq.:  0.4601 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  63 <\/br>Relative freq.:  0.3865 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.1534 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  17 <\/br>Relative freq.:  0.85 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.15 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  13 <\/br># assignments:  23 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.8364 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  9 <\/br>Relative freq.:  0.1636 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.8462 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.1538 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  13 <\/br># assignments:  22 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  13 <\/br># assignments:  78 <\/br>Relative freq.:  0.3959 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  74 <\/br>Relative freq.:  0.3756 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  45 <\/br>Relative freq.:  0.2284 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  112 <\/br>Relative freq.:  0.9912 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0088 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  0.9714 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0286 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  116 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  79 <\/br>Relative freq.:  0.632 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  46 <\/br>Relative freq.:  0.368 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  13 <\/br># assignments:  42 <\/br>Relative freq.:  0.4565 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.3152 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.2283 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  37 <\/br>Relative freq.:  0.9737 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0263 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  13 <\/br># assignments:  29 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.7059 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.2941 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  04 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  13 <\/br># assignments:  165 <\/br>Relative freq.:  0.994 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.006 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  128 <\/br>Relative freq.:  0.7356 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  46 <\/br>Relative freq.:  0.2644 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  13 <\/br># assignments:  2 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  13 <\/br># assignments:  5 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.1111 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  13 <\/br># assignments:  155 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  13 <\/br># assignments:  20 <\/br>Relative freq.:  0.7407 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.2593 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.5833 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  5 <\/br>Relative freq.:  0.4167 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  54 <\/br>Relative freq.:  0.6506 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  04 <\/br># assignments:  29 <\/br>Relative freq.:  0.3494 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  13 <\/br># assignments:  182 <\/br>Relative freq.:  0.6386 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  102 <\/br>Relative freq.:  0.3579 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0035 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  110 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  13 <\/br># assignments:  134 <\/br>Relative freq.:  0.6411 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  75 <\/br>Relative freq.:  0.3589 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  13 <\/br># assignments:  65 <\/br>Relative freq.:  0.5702 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.4298 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  82 <\/br>Relative freq.:  0.5541 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  13 <\/br># assignments:  66 <\/br>Relative freq.:  0.4459 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.7229 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  04 <\/br># assignments:  23 <\/br>Relative freq.:  0.2771 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  13 <\/br># assignments:  11 <\/br>Relative freq.:  0.55 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  0.45 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  13 <\/br># assignments:  79 <\/br>Relative freq.:  0.8778 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  04 <\/br># assignments:  11 <\/br>Relative freq.:  0.1222 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  0.7273 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.1818 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0909 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  85 <\/br>Relative freq.:  0.6296 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  04 <\/br># assignments:  49 <\/br>Relative freq.:  0.363 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0074 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  13 <\/br># assignments:  28 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  147 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  0.8889 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.1111 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  13 <\/br># assignments:  52 <\/br>Relative freq.:  0.5977 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  04 <\/br># assignments:  35 <\/br>Relative freq.:  0.4023 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  59 <\/br>Relative freq.:  0.7867 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  04 <\/br># assignments:  16 <\/br>Relative freq.:  0.2133 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  102 <\/br>Relative freq.:  0.5543 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  04 <\/br># assignments:  81 <\/br>Relative freq.:  0.4402 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  13 <\/br># assignments:  1 <\/br>Relative freq.:  0.0054 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  175 <\/br>Relative freq.:  0.7353 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  04 <\/br># assignments:  63 <\/br>Relative freq.:  0.2647 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  13 <\/br># assignments:  326 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.018","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.4551 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  13 <\/br># assignments:  63 <\/br>Relative freq.:  0.4038 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  04 <\/br># assignments:  22 <\/br>Relative freq.:  0.141 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  13 <\/br># assignments:  127 <\/br>Relative freq.:  0.4792 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  04 <\/br># assignments:  76 <\/br>Relative freq.:  0.2868 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  62 <\/br>Relative freq.:  0.234 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  13 <\/br># assignments:  16 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  292 <\/br>Relative freq.:  0.9966 <\/br>Relative Rep. freq.:  0.0108","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0034 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  128 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  135 <\/br>Relative freq.:  0.5422 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  04 <\/br># assignments:  112 <\/br>Relative freq.:  0.4498 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  52 <\/br>Relative freq.:  0.5652 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  04 <\/br># assignments:  40 <\/br>Relative freq.:  0.4348 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.9677 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0323 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  115 <\/br>Relative freq.:  0.7012 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  04 <\/br># assignments:  49 <\/br>Relative freq.:  0.2988 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  146 <\/br>Relative freq.:  0.6186 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  04 <\/br># assignments:  90 <\/br>Relative freq.:  0.3814 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  48 <\/br>Relative freq.:  0.7385 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  04 <\/br># assignments:  17 <\/br>Relative freq.:  0.2615 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  13 <\/br># assignments:  174 <\/br>Relative freq.:  0.5781 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  126 <\/br>Relative freq.:  0.4186 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  04 <\/br># assignments:  8 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  87 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  86 <\/br>Relative freq.:  0.4943 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0057 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  13 <\/br># assignments:  152 <\/br>Relative freq.:  0.6496 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  82 <\/br>Relative freq.:  0.3504 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  143 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  88 <\/br>Relative freq.:  0.9778 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0222 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  90 <\/br>Relative freq.:  0.4225 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  13 <\/br># assignments:  73 <\/br>Relative freq.:  0.3427 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  04 <\/br># assignments:  50 <\/br>Relative freq.:  0.2347 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  13 <\/br># assignments:  78 <\/br>Relative freq.:  0.4968 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  52 <\/br>Relative freq.:  0.3312 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  04 <\/br># assignments:  27 <\/br>Relative freq.:  0.172 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  01 <\/br># assignments:  142 <\/br>Relative freq.:  0.7634 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  04 <\/br># assignments:  44 <\/br>Relative freq.:  0.2366 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  13 <\/br># assignments:  241 <\/br>Relative freq.:  0.6965 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  104 <\/br>Relative freq.:  0.3006 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0029 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  122 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  198 <\/br>Relative freq.:  0.7416 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  69 <\/br>Relative freq.:  0.2584 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  01 <\/br># assignments:  182 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  121 <\/br>Relative freq.:  0.6173 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  04 <\/br># assignments:  75 <\/br>Relative freq.:  0.3827 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  13 <\/br># assignments:  100 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  0.6316 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.3684 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  13 <\/br># assignments:  174 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  31 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  13 <\/br># assignments:  89 <\/br>Relative freq.:  0.7177 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  35 <\/br>Relative freq.:  0.2823 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  89 <\/br>Relative freq.:  0.957 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.043 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  13 <\/br># assignments:  224 <\/br>Relative freq.:  0.6727 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  109 <\/br>Relative freq.:  0.3273 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  13 <\/br># assignments:  183 <\/br>Relative freq.:  0.9946 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0054 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  13 <\/br># assignments:  73 <\/br>Relative freq.:  0.4803 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  54 <\/br>Relative freq.:  0.3553 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.1645 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  64 <\/br>Relative freq.:  0.5517 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  04 <\/br># assignments:  52 <\/br>Relative freq.:  0.4483 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  48 <\/br>Relative freq.:  0.5517 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  04 <\/br># assignments:  39 <\/br>Relative freq.:  0.4483 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  13 <\/br># assignments:  54 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  13 <\/br># assignments:  97 <\/br>Relative freq.:  0.4311 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  74 <\/br>Relative freq.:  0.3289 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  04 <\/br># assignments:  54 <\/br>Relative freq.:  0.24 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  13 <\/br># assignments:  40 <\/br>Relative freq.:  0.6557 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  21 <\/br>Relative freq.:  0.3443 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  13 <\/br># assignments:  34 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  13 <\/br># assignments:  1 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  13 <\/br># assignments:  84 <\/br>Relative freq.:  0.5091 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  04 <\/br># assignments:  41 <\/br>Relative freq.:  0.2485 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  0.2364 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0061 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  13 <\/br># assignments:  71 <\/br>Relative freq.:  0.582 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  0.2459 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.1721 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  13 <\/br># assignments:  6 <\/br>Relative freq.:  0.8571 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.1429 <\/br>Relative Rep. freq.:  5e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":181.253632212536,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["V2-70*01","V2-70*13","V2-70*15","V2-70*17","V2-70*20","V2-70D*04/V2-70*04"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["V2-70*01","V2-70*13","V2-70*15","V2-70*17","V2-70*20","V2-70D*04/V2-70*04"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-15.6656967237778,342.492988652755],"tickmode":"array","ticktext":["0","100","200","300"],"tickvals":[0,100,200,300],"categoryorder":"array","categoryarray":["0","100","200","300"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"7540573a8b1a4":{"x":{},"y":{},"text":{},"type":"box"},"75405b354afa":{"x":{},"y":{},"text":{}}},"cur_data":"7540573a8b1a4","visdat":{"7540573a8b1a4":["function (y) ","x"],"75405b354afa":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-9cc6adafc5926e6632e8" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-9cc6adafc5926e6632e8">{"x":{"data":[{"x":[3,3,4,1,1,4,4,1,1,4,2,1,4,1,4,1,3,4,1,3,3,1,3,1,4,3,1,4,3,1,4,3,4,1,3,1,4,1,4,1,3,4,3,1,4,1,3,1,4,3,1,4,3,3,1,3,4,1,3,4,1,1,4,3,4,3,4,3,3,1,1,4,3,1,3,3,1,4,1,4,1,4,1,4,3,1,4,3,1],"y":[117,66,39,24,94,45,3,2,75,24,1,49,14,81,27,45,31,16,26,14,29,24,47,35,19,30,24,13,24,18,17,13,8,6,36,48,28,71,48,7,7,1,30,13,7,12,44,32,21,36,44,18,22,66,62,60,28,27,76,44,26,68,9,25,2,25,1,139,38,29,80,55,144,98,12,29,18,10,36,25,76,20,98,38,75,71,18,57,23],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[3.00550333647989,2.94972817781381,4.08130882657133,1.06979781929404,0.925923411734402,4.08667031032965,3.99569181604311,0.96125113829039,1.05577095621265,4.08957713870332,2.05656605800614,0.921738781873137,4.03783810585737,1.0894221752882,3.97562674521469,0.97824023431167,2.96381431235932,3.94204210015014,1.02420387337916,3.09794925423339,3.00108244074509,0.953309359401464,2.9768863693811,1.00518495668657,4.02403542068787,2.98770723124035,0.960408805636689,4.044075921271,2.9060020740144,1.01130378129892,3.92588395001367,2.90292443819344,4.01486739581451,1.09009136846289,2.9105135998223,1.08826670041308,4.06779833124019,0.912949977116659,4.07256934680045,1.05769191379659,2.95882184375077,4.06623618463054,3.0114136970602,1.037652951153,4.03456800822169,0.999030543724075,2.93792694793083,0.966770568536594,4.09562621805817,3.03306434159167,0.952854261407629,4.07752005904913,3.06953025730327,2.97374489763752,0.984371108235791,3.03380471011624,4.08193973009475,0.904962670337409,2.95564299938269,4.09695723056793,0.904478012956679,0.942668409179896,4.05554151972756,2.98066138625145,4.05092335282825,2.97033298490569,4.09601053074002,3.04414053200744,3.06022087312303,0.942241351492703,1.06325723454356,3.95468552107923,3.04274445176125,1.09262710548937,3.09858271642588,2.91618822696619,1.04025675556622,4.03727808529511,1.09886984839104,3.99197485987097,0.98920167805627,4.00092941871844,0.917600628780201,4.09905974403955,2.91815589102916,0.959621586557478,3.96667767283507,2.96546856355853,1.05458063958213],"y":[116.958758197725,65.6692131740972,39.1773662211373,23.6170209046453,93.6218015251681,44.8962751317769,2.76329135857522,1.90919296778738,75.3216324053705,24.1758082281798,1.36438228525221,48.8948780607432,14.3242843631655,81.3971778564155,26.6382474994287,45.1880876496434,30.7368011107668,16.2910329179838,26.3743354024366,13.8178322199732,29.1237377412617,24.0922215824947,47.3941451421008,34.9005457475781,19.1071618340909,29.9405739590526,24.29245228488,13.0885279947892,23.884508889541,18.3683588728309,16.6867070658132,12.7537548299879,8.18187232259661,6.29097311515361,35.6004954740405,47.8311631087214,28.133709628135,70.6164840472862,47.71155850254,7.21422991845757,7.28532881252468,1.33109841048718,30.146977888234,12.6020899519324,7.38987098168582,12.1998273644596,44.1423844112083,32.0588117562234,21.1705291328952,35.7776844782755,43.9738014291972,17.6441187297925,21.9933420043439,66.3374088389799,62.0027701247484,59.6600695809349,28.0220912873745,27.2973170006648,76.2040682030842,44.3808679873124,26.013763104938,67.6930205686018,9.05105264950544,24.6632328603417,1.87109769452363,24.705764705874,1.16416186597198,139.200016446784,38.2392289936543,29.3341188905761,79.8140455272049,54.8006461052224,143.920917793922,98.3030339958146,11.6554657807574,28.8892676914111,18.3818584147841,10.0168357983232,35.7551102817059,25.1269039228559,76.1799150640145,20.3718885241076,98.2053895432502,38.2119051255286,74.9329435003921,70.7697873102501,17.7384036241099,57.3461702428758,23.0224366344512],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  13 <\/br># assignments:  117 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0206","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  13 <\/br># assignments:  66 <\/br>Relative freq.:  0.5116 <\/br>Relative Rep. freq.:  0.0174","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  39 <\/br>Relative freq.:  0.3023 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  0.186 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  94 <\/br>Relative freq.:  0.6763 <\/br>Relative Rep. freq.:  0.0211","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  45 <\/br>Relative freq.:  0.3237 <\/br>Relative Rep. freq.:  0.0101","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  75 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0104","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  24 <\/br>Relative freq.:  0.24 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.01 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.7778 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  14 <\/br>Relative freq.:  0.2222 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  81 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.016","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  27 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  45 <\/br>Relative freq.:  0.4891 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  13 <\/br># assignments:  31 <\/br>Relative freq.:  0.337 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  16 <\/br>Relative freq.:  0.1739 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  26 <\/br>Relative freq.:  0.65 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  13 <\/br># assignments:  14 <\/br>Relative freq.:  0.35 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  13 <\/br># assignments:  29 <\/br>Relative freq.:  0.5472 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  0.4528 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  13 <\/br># assignments:  47 <\/br>Relative freq.:  0.4653 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  35 <\/br>Relative freq.:  0.3465 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  19 <\/br>Relative freq.:  0.1881 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  13 <\/br># assignments:  30 <\/br>Relative freq.:  0.4478 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  0.3582 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  13 <\/br>Relative freq.:  0.194 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  13 <\/br># assignments:  24 <\/br>Relative freq.:  0.4068 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  18 <\/br>Relative freq.:  0.3051 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  17 <\/br>Relative freq.:  0.2881 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  13 <\/br># assignments:  13 <\/br>Relative freq.:  0.4815 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  8 <\/br>Relative freq.:  0.2963 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  0.2222 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  13 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0086","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  48 <\/br>Relative freq.:  0.6316 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  28 <\/br>Relative freq.:  0.3684 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.5966 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  48 <\/br>Relative freq.:  0.4034 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.4667 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  13 <\/br># assignments:  7 <\/br>Relative freq.:  0.4667 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0667 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  13 <\/br># assignments:  30 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  0.26 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  7 <\/br>Relative freq.:  0.14 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  13 <\/br># assignments:  44 <\/br>Relative freq.:  0.4536 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  32 <\/br>Relative freq.:  0.3299 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.2165 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  13 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  44 <\/br>Relative freq.:  0.7097 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  04 <\/br># assignments:  18 <\/br>Relative freq.:  0.2903 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  13 <\/br># assignments:  22 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  13 <\/br># assignments:  66 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  62 <\/br>Relative freq.:  0.4133 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  13 <\/br># assignments:  60 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  04 <\/br># assignments:  28 <\/br>Relative freq.:  0.1867 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  13 <\/br># assignments:  76 <\/br>Relative freq.:  0.5205 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  04 <\/br># assignments:  44 <\/br>Relative freq.:  0.3014 <\/br>Relative Rep. freq.:  0.0099","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  26 <\/br>Relative freq.:  0.1781 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  68 <\/br>Relative freq.:  0.8831 <\/br>Relative Rep. freq.:  0.0124","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  04 <\/br># assignments:  9 <\/br>Relative freq.:  0.1169 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  13 <\/br># assignments:  25 <\/br>Relative freq.:  0.9259 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0741 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  13 <\/br># assignments:  25 <\/br>Relative freq.:  0.9615 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0385 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  13 <\/br># assignments:  139 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0153","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  13 <\/br># assignments:  38 <\/br>Relative freq.:  0.5672 <\/br>Relative Rep. freq.:  0.0125","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.4328 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  80 <\/br>Relative freq.:  0.5926 <\/br>Relative Rep. freq.:  0.0161","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  55 <\/br>Relative freq.:  0.4074 <\/br>Relative Rep. freq.:  0.0111","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  13 <\/br># assignments:  144 <\/br>Relative freq.:  0.595 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  98 <\/br>Relative freq.:  0.405 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  13 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  13 <\/br># assignments:  29 <\/br>Relative freq.:  0.5088 <\/br>Relative Rep. freq.:  0.0259","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  18 <\/br>Relative freq.:  0.3158 <\/br>Relative Rep. freq.:  0.0161","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  04 <\/br># assignments:  10 <\/br>Relative freq.:  0.1754 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  0.5902 <\/br>Relative Rep. freq.:  0.0121","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.4098 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  76 <\/br>Relative freq.:  0.7917 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  20 <\/br>Relative freq.:  0.2083 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  98 <\/br>Relative freq.:  0.7206 <\/br>Relative Rep. freq.:  0.0181","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  04 <\/br># assignments:  38 <\/br>Relative freq.:  0.2794 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  13 <\/br># assignments:  75 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0134","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.7978 <\/br>Relative Rep. freq.:  0.0139","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  18 <\/br>Relative freq.:  0.2022 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  13 <\/br># assignments:  57 <\/br>Relative freq.:  0.7125 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  0.2875 <\/br>Relative Rep. freq.:  0.0032"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":181.253632212536,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,4.6],"tickmode":"array","ticktext":["V2-70*01","V2-70*13","V2-70*15","V2-70D*04/V2-70*04"],"tickvals":[1,2,3,4],"categoryorder":"array","categoryarray":["V2-70*01","V2-70*13","V2-70*15","V2-70D*04/V2-70*04"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-6.15,151.15],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[8.88178419700125e-16,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"7540522701068":{"x":{},"y":{},"text":{},"type":"box"},"754054b7eb7a5":{"x":{},"y":{},"text":{}}},"cur_data":"7540522701068","visdat":{"7540522701068":["function (y) ","x"],"754054b7eb7a5":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-10" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 4 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV2-70G13%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:800px;" cellspacing="0">
</iframe>
</div>
<div id="observations-12" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/dx0odrmkc0rmdhd5t2enk/IGHV2-70G13.docx?dl=0&amp;rlkey=wdypp2b015xm6q32k3jvyhdty" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV2-70G13" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-12" class="section level2">
<h2>Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="12-G12.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="14-G14.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/13-G13.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
