<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>14 IGHV2-70/IGHV2-70D - G13 | Functional groups</title>
  <meta name="description" content="14 IGHV2-70/IGHV2-70D - G13 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="14 IGHV2-70/IGHV2-70D - G13 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="14 IGHV2-70/IGHV2-70D - G13 | Functional groups" />
  
  
  



<meta name="date" content="2021-11-29" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="12-G12.html"/>
<link rel="next" href="14-G14.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-2-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 2% for all except 0.5% for allele <strong>05</strong></a></li>
<li class="chapter" data-level="3.6.2" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.2</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./"> Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv2-70ighv2-70d---g13" class="section level1" number="14">
<h1><span class="header-section-number">14</span> IGHV2-70/IGHV2-70D - G13</h1>
<div id="allele-appearnce-12" class="section level2" number="14.1">
<h2><span class="header-section-number">14.1</span> Allele appearnce</h2>
<p>The group of IGHV2-70/IGHV2-70D includes 21 alleles, 20 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-118-1.png" width="672" /></p>
</div>
<div id="group-alignment-10" class="section level2" number="14.2">
<h2><span class="header-section-number">14.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-119-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-12" class="section level2" number="14.3">
<h2><span class="header-section-number">14.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-234344fc9c8503a6689e" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-234344fc9c8503a6689e">{"x":{"data":[{"x":[1,8,5,1,5,5,1,8,5,1,8,1,8,5,1,8,1,8,5,5,1,8,1,4,7,1,8,1,1,1,8,5,1,8,1,8,1,5,1,8,4,5,1,8,8,5,6,1,8,5,1,8,5,8,1,5,5,1,1,8,1,8,1,5,1,6,1,5,1,5,1,1,5,1,8,5,1,5,8,5,1,8,1,8,4,5,1,4,5,8,5,8,1,8,1,8,4,5,1,8,5,1,5,8,5,8,1,2,5,1,7,1,1,8,2,4,5,1,8,4,6,1,8,1,8,1,8,1,8,1,8,4,5,1,6,1,8,8,1,4,3,5,1,6,1,7,1,8,1,5,8,5,1,8,1,8,4,5,1,4,6,1,1,8,1,8,1,8,5,6,1,8,5,1,5,1,4,1,8,5,5,1,5,6,5,1,8,1,8,1,8,4,5,6,5,1,8,5,1,5,8,1,5,5,8,1,4,6,5,1,8,5,8],"y":[76,60,140,24,171,114,69,68,76,63,26,17,4,23,46,10,11,2,23,83,75,45,116,1,1,35,1,7,120,79,47,42,29,22,38,1,29,29,60,31,1,8,4,2,54,166,1,130,49,2,1,1,5,3,1,156,21,7,8,6,55,32,36,187,106,2,110,140,75,68,49,82,68,60,23,12,9,79,11,8,2,1,85,50,1,30,148,1,8,1,52,35,59,17,103,82,1,1,175,63,331,71,63,22,132,81,65,1,16,294,1,45,131,1,1,1,8,154,128,3,1,54,43,60,2,116,51,146,89,49,18,1,179,126,1,13,8,90,87,1,1,154,81,1,147,1,88,2,91,73,50,79,52,27,144,44,1,242,104,1,1,126,200,69,185,1,122,82,108,1,36,21,177,32,90,35,1,95,5,1,229,112,188,2,73,54,25,65,55,49,43,1,57,1,102,79,61,40,21,34,3,2,1,88,44,42,1,1,73,31,22,6,1],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.00987594393082,8.08070757039823,4.90458851247095,0.937012534309179,4.90354171027429,4.94578279727139,1.04385134628974,7.90470132706687,5.06114149480127,1.08510337951593,7.94362044399604,0.925804015062749,8.04244454940781,5.03770690145902,1.02155017247424,8.00938861798495,0.916113801766187,7.91803192235529,4.98631614083424,4.91270657503046,0.968734799651429,8.06476880917326,1.01662450926378,4.06208847528324,7.08678455404006,0.98647850966081,7.91740692448802,1.08708606762812,0.903919554594904,1.06850789706223,8.01473358841613,4.933260462014,0.953950431849808,8.03503699447028,0.948612436745316,8.04485378549434,0.941897038510069,4.9466990982648,0.958061912935227,8.00166305359453,3.97681265515275,5.09836673149839,0.944089372782037,7.93950316896662,7.90561297680251,4.90614215130918,6.07046219655313,0.943076936295256,8.03376645552926,4.92560293874703,1.08856301954947,7.94823887324892,4.90365337389521,7.90591600690968,0.949247062020004,5.07380006238818,4.95989495869726,0.99557467806153,0.905758443335071,7.97686164029874,0.966611087787896,8.05172222391702,1.06035039341077,5.02296456261538,1.0762102631852,5.95690532154404,1.06283489838243,4.91363037615083,0.998619227437302,5.09178687310778,0.906357144983485,1.05394245591015,4.93419497301802,1.04723703451455,8.04766076793894,4.95297122616321,0.961621757084504,5.03052123510279,8.03459078990854,5.01526132142171,0.996865016315132,7.96226739948615,0.915770908817649,7.91205007098615,3.92651202832349,4.93041758332402,0.993233990250155,4.06413617101498,5.03863700763322,7.97955068256706,4.986603223579,8.00086271972395,0.939232328673825,7.92152991839685,0.990887764329091,8.04979361062869,3.9044454798568,5.0824379066471,1.02097984044813,7.975019805599,4.92508595958352,0.997419666266069,4.93827506345697,8.04771162043326,5.06991797019728,8.08957495600916,1.06235307878815,2.01967358859256,4.93700158428401,0.989734182972461,7.09619995383546,0.912913832534105,1.09155836226419,8.07556608342566,2.0967712554615,3.9943940710742,5.05999938664027,1.05117681976408,7.96799213425256,3.9730906563811,6.04038520632312,0.966717490553856,8.09552897503599,0.94605395141989,7.92916827369481,0.922155111841857,8.09086795868352,0.916845142142847,8.0545782412868,0.958689764048904,8.04858719077893,3.90086559699848,4.94553167433478,0.901226380141452,5.94432876976207,1.01886493475176,7.92862795358524,7.9929804909043,0.945158966770396,4.00063291480765,2.92091899430379,5.04098430755548,0.972010176302865,6.07152760187164,1.04881926188245,6.92280321791768,1.06386535023339,7.90801784191281,1.01961877201684,5.09200165104121,7.95467920904048,4.93459879676811,1.05592547035776,7.94033855376765,0.935390501189977,7.93050091951154,4.08412695145234,4.96323478156701,0.999688152549788,3.97593448893167,6.04059622595087,0.986412002565339,1.04815215063281,8.05827043973841,0.923971709422767,7.96559645351954,0.925506832776591,8.06931175608188,4.98487864877097,5.96480546044186,0.912703496776521,7.92033542245626,5.01251828921959,0.96375826606527,5.0392605505418,0.983937064697966,3.98498694943264,0.906323986174539,7.90949073657393,5.08597318530083,4.97386747063138,1.00235708793625,5.05381076745689,6.05439915228635,5.05692254863679,0.968260311568156,8.05353271183558,1.0209934564773,8.01664328626357,0.96720994845964,7.97747239111923,4.00379613153636,5.0687535550911,6.01076459204778,4.93410695386119,0.952247059764341,7.90152581888251,4.95943050766364,1.01848718584515,5.00555143509991,7.90621942421421,1.07320712693036,5.06410111361183,5.08300303071737,7.96471815370023,0.918659864692017,3.94696353357285,5.93288267045282,4.90064195450395,1.05411294535734,8.07560578095727,4.93679286697879,7.97888606833294],"y":[75.6183236068115,59.8302659822628,140.06346609965,23.6432276571169,170.74833033029,114.116543805599,68.7186635077,68.0474458640441,75.9058087132871,63.3960722722113,25.7233386525884,16.9948712073266,3.66654408890754,23.0788162944838,45.7019263073802,9.93100286573172,11.376708512567,2.31390118226409,22.9556879809126,82.8465995643288,75.2519440043718,44.8716898856685,115.86067458801,0.898873704113066,0.85153867546469,34.9382828010246,1.07208254747093,6.96696640979499,119.725451172329,78.9858422596008,46.7822027312592,42.069271537289,28.7846944592893,22.1987068226561,38.3029533727095,0.759291684068739,28.6863615753129,29.024326559715,60.076752204448,31.1891068598256,0.944069505855441,7.70646054465324,4.3068438116461,2.15073663089424,53.8424221122637,165.768925266527,0.906290029361844,129.950748924725,49.3707073498517,1.72955120094121,1.02216028049588,1.16325380820781,5.39956325925887,3.03500835932791,1.32696842998266,155.672450536303,20.7457186697051,7.10684700775892,8.32859936226159,6.04037915244699,54.9851953553036,32.2796899862587,36.3203615181148,186.982569369301,106.151286798529,1.77234055493027,109.885151027143,139.891397494823,75.2396435823292,67.9548716524616,48.829155455716,82.3144710026681,67.6361335359514,59.9631508143619,22.7489403925836,11.8921015031636,9.23053985051811,79.101631822437,10.7032924968749,7.72430493365973,2.37421904560179,0.735950943082571,84.913010247983,49.6844607228413,0.73156416323036,29.6482936138287,148.390997704305,1.3898704206571,7.7580531520769,1.30209560375661,52.2617921281606,35.0942830290645,58.6173847699538,17.0470202766359,103.149263162725,81.9527015114203,1.34038549270481,0.863473211601377,175.288294878788,62.6847552403808,331.244512637705,71.1718066550791,63.1874150753021,21.8912716714665,132.334396963194,81.0617454910651,65.3116703784093,1.07388252336532,16.2646329810843,294.178390369751,0.68073610868305,45.3066420439631,130.618811359815,1.19984930455685,0.725166973099112,0.811380665004253,7.64433688670397,154.160069840029,128.055674555525,3.138472292386,0.849239547550678,53.9133347744122,43.0521318227053,60.1560475572944,2.11472079735249,116.365727797709,51.3902085710317,146.326308411546,88.6025657033548,49.2316046584398,18.2449820907786,0.837018684670329,178.698383845016,126.065349817835,0.86772621050477,12.6439451631159,7.71004798598588,89.7222397027537,87.3618616424501,0.636724531464279,1.15386131498963,154.141974725947,80.7454922957346,1.37418714724481,147.213961657695,0.958724102936685,87.8732127716765,1.82729676403105,90.7523930007592,73.2359577583149,50.2889431605116,79.0964110150933,51.7086218664423,27.093187202327,144.117805493437,43.9458129009232,0.873461800254881,241.970146381296,104.044457411766,0.820823699235916,0.73117988910526,125.684004140645,199.712150758132,69.2348518203944,185.254633685015,0.625202126801014,122.24983925391,81.8178988579661,108.195637403801,1.21145036295056,35.9913311049342,21.2443605998531,177.390922317654,32.211008890532,90.0191946845502,34.6408787894994,1.01258347257972,94.8142219997942,4.95846560671926,1.29073260482401,229.106053164415,111.834948350675,188.237252864987,2.39674732834101,73.2865956982598,53.6983662381768,24.8705351062119,65.3180643912405,54.6100312735885,49.3064333762974,43.0441973930225,0.842071465030312,57.1337647264823,0.675472589954734,101.673614302278,78.6984942426905,60.9262283304706,39.6899890802801,21.1118180477992,34.1396106837317,2.70467368736863,2.32394705396146,0.867771800793707,87.7539736466482,43.8566228386015,42.3358279166743,1.19188061766326,0.703921459615231,72.6538305981085,30.8122104125097,22.1499526951462,5.7708561912179,0.890133883990347],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  76 <\/br>Relative freq.:  0.5588 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  60 <\/br>Relative freq.:  0.4412 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  13 <\/br># assignments:  140 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  13 <\/br># assignments:  171 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  13 <\/br># assignments:  114 <\/br>Relative freq.:  0.4542 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  69 <\/br>Relative freq.:  0.2749 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  68 <\/br>Relative freq.:  0.2709 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  13 <\/br># assignments:  76 <\/br>Relative freq.:  0.4606 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  63 <\/br>Relative freq.:  0.3818 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  26 <\/br>Relative freq.:  0.1576 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  17 <\/br>Relative freq.:  0.8095 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.1905 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  13 <\/br># assignments:  23 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.8214 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  10 <\/br>Relative freq.:  0.1786 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.8462 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.1538 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  13 <\/br># assignments:  23 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  13 <\/br># assignments:  83 <\/br>Relative freq.:  0.4089 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  75 <\/br>Relative freq.:  0.3695 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  45 <\/br>Relative freq.:  0.2217 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  116 <\/br>Relative freq.:  0.9831 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0085 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0085 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  35 <\/br>Relative freq.:  0.9722 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0278 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  120 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  79 <\/br>Relative freq.:  0.627 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  47 <\/br>Relative freq.:  0.373 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  13 <\/br># assignments:  42 <\/br>Relative freq.:  0.4516 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.3118 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  22 <\/br>Relative freq.:  0.2366 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  38 <\/br>Relative freq.:  0.9744 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0256 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  13 <\/br># assignments:  29 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.6522 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  31 <\/br>Relative freq.:  0.337 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0109 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  04 <\/br># assignments:  54 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  13 <\/br># assignments:  166 <\/br>Relative freq.:  0.994 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.006 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  130 <\/br>Relative freq.:  0.7263 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  49 <\/br>Relative freq.:  0.2737 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  13 <\/br># assignments:  2 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  13 <\/br># assignments:  5 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.1111 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  13 <\/br># assignments:  156 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  13 <\/br># assignments:  21 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.5714 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  6 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  55 <\/br>Relative freq.:  0.6322 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  04 <\/br># assignments:  32 <\/br>Relative freq.:  0.3678 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  13 <\/br># assignments:  187 <\/br>Relative freq.:  0.6339 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  106 <\/br>Relative freq.:  0.3593 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  15 <\/br># assignments:  2 <\/br>Relative freq.:  0.0068 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  110 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  13 <\/br># assignments:  140 <\/br>Relative freq.:  0.6512 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  75 <\/br>Relative freq.:  0.3488 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  13 <\/br># assignments:  68 <\/br>Relative freq.:  0.5812 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.4188 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  82 <\/br>Relative freq.:  0.5467 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  13 <\/br># assignments:  68 <\/br>Relative freq.:  0.4533 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.7229 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  04 <\/br># assignments:  23 <\/br>Relative freq.:  0.2771 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  13 <\/br># assignments:  12 <\/br>Relative freq.:  0.5714 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  13 <\/br># assignments:  79 <\/br>Relative freq.:  0.8778 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  04 <\/br># assignments:  11 <\/br>Relative freq.:  0.1222 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  0.7273 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.1818 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0909 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  85 <\/br>Relative freq.:  0.625 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  04 <\/br># assignments:  50 <\/br>Relative freq.:  0.3676 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0074 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  13 <\/br># assignments:  30 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  148 <\/br>Relative freq.:  0.9933 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0067 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  0.8889 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.1111 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  13 <\/br># assignments:  52 <\/br>Relative freq.:  0.5977 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  04 <\/br># assignments:  35 <\/br>Relative freq.:  0.4023 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  59 <\/br>Relative freq.:  0.7763 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  04 <\/br># assignments:  17 <\/br>Relative freq.:  0.2237 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  103 <\/br>Relative freq.:  0.5508 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  04 <\/br># assignments:  82 <\/br>Relative freq.:  0.4385 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0053 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  13 <\/br># assignments:  1 <\/br>Relative freq.:  0.0053 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  175 <\/br>Relative freq.:  0.7353 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  04 <\/br># assignments:  63 <\/br>Relative freq.:  0.2647 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  13 <\/br># assignments:  331 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0179","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.4551 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  13 <\/br># assignments:  63 <\/br>Relative freq.:  0.4038 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  04 <\/br># assignments:  22 <\/br>Relative freq.:  0.141 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  13 <\/br># assignments:  132 <\/br>Relative freq.:  0.4731 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  04 <\/br># assignments:  81 <\/br>Relative freq.:  0.2903 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  65 <\/br>Relative freq.:  0.233 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  10 <\/br># assignments:  1 <\/br>Relative freq.:  0.0036 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  13 <\/br># assignments:  16 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  294 <\/br>Relative freq.:  0.9966 <\/br>Relative Rep. freq.:  0.0108","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0034 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  45 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  131 <\/br>Relative freq.:  0.9776 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0075 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  10 <\/br># assignments:  1 <\/br>Relative freq.:  0.0075 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0075 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  154 <\/br>Relative freq.:  0.5385 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  04 <\/br># assignments:  128 <\/br>Relative freq.:  0.4476 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  12 <\/br># assignments:  3 <\/br>Relative freq.:  0.0105 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0035 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  54 <\/br>Relative freq.:  0.5567 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  04 <\/br># assignments:  43 <\/br>Relative freq.:  0.4433 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.9677 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0323 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  116 <\/br>Relative freq.:  0.6946 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  04 <\/br># assignments:  51 <\/br>Relative freq.:  0.3054 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  146 <\/br>Relative freq.:  0.6213 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  04 <\/br># assignments:  89 <\/br>Relative freq.:  0.3787 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.7206 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  04 <\/br># assignments:  18 <\/br>Relative freq.:  0.2647 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0147 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  13 <\/br># assignments:  179 <\/br>Relative freq.:  0.585 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  126 <\/br>Relative freq.:  0.4118 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  0.619 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  04 <\/br># assignments:  8 <\/br>Relative freq.:  0.381 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  90 <\/br>Relative freq.:  0.5056 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  87 <\/br>Relative freq.:  0.4888 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0056 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  11 <\/br># assignments:  1 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  13 <\/br># assignments:  154 <\/br>Relative freq.:  0.6525 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  81 <\/br>Relative freq.:  0.3432 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0042 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  147 <\/br>Relative freq.:  0.9932 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0068 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  88 <\/br>Relative freq.:  0.9778 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0222 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  91 <\/br>Relative freq.:  0.4252 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  13 <\/br># assignments:  73 <\/br>Relative freq.:  0.3411 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  04 <\/br># assignments:  50 <\/br>Relative freq.:  0.2336 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  13 <\/br># assignments:  79 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  52 <\/br>Relative freq.:  0.3291 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  04 <\/br># assignments:  27 <\/br>Relative freq.:  0.1709 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  01 <\/br># assignments:  144 <\/br>Relative freq.:  0.7619 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  04 <\/br># assignments:  44 <\/br>Relative freq.:  0.2328 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0053 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  13 <\/br># assignments:  242 <\/br>Relative freq.:  0.6954 <\/br>Relative Rep. freq.:  0.0111","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  104 <\/br>Relative freq.:  0.2989 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0029 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0029 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  126 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  200 <\/br>Relative freq.:  0.7435 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  69 <\/br>Relative freq.:  0.2565 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  01 <\/br># assignments:  185 <\/br>Relative freq.:  0.9946 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0054 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  122 <\/br>Relative freq.:  0.598 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  04 <\/br># assignments:  82 <\/br>Relative freq.:  0.402 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  13 <\/br># assignments:  108 <\/br>Relative freq.:  0.9908 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0092 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  0.6316 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.3684 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  13 <\/br># assignments:  177 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  32 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  13 <\/br># assignments:  90 <\/br>Relative freq.:  0.7143 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  35 <\/br>Relative freq.:  0.2778 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0079 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  95 <\/br>Relative freq.:  0.9406 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  04 <\/br># assignments:  5 <\/br>Relative freq.:  0.0495 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  13 <\/br># assignments:  1 <\/br>Relative freq.:  0.0099 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  13 <\/br># assignments:  229 <\/br>Relative freq.:  0.6716 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  112 <\/br>Relative freq.:  0.3284 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  13 <\/br># assignments:  188 <\/br>Relative freq.:  0.9895 <\/br>Relative Rep. freq.:  0.008","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  15 <\/br># assignments:  2 <\/br>Relative freq.:  0.0105 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  13 <\/br># assignments:  73 <\/br>Relative freq.:  0.4803 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  54 <\/br>Relative freq.:  0.3553 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.1645 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  65 <\/br>Relative freq.:  0.5417 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  04 <\/br># assignments:  55 <\/br>Relative freq.:  0.4583 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.5269 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  04 <\/br># assignments:  43 <\/br>Relative freq.:  0.4624 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0108 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  13 <\/br># assignments:  57 <\/br>Relative freq.:  0.9828 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0172 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  13 <\/br># assignments:  102 <\/br>Relative freq.:  0.4215 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  79 <\/br>Relative freq.:  0.3264 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  04 <\/br># assignments:  61 <\/br>Relative freq.:  0.2521 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  13 <\/br># assignments:  40 <\/br>Relative freq.:  0.6557 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  21 <\/br>Relative freq.:  0.3443 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  13 <\/br># assignments:  34 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  13 <\/br># assignments:  1 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  13 <\/br># assignments:  88 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  04 <\/br># assignments:  44 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  42 <\/br>Relative freq.:  0.2386 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0057 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0057 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  13 <\/br># assignments:  73 <\/br>Relative freq.:  0.5794 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  31 <\/br>Relative freq.:  0.246 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  04 <\/br># assignments:  22 <\/br>Relative freq.:  0.1746 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  13 <\/br># assignments:  6 <\/br>Relative freq.:  0.8571 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.1429 <\/br>Relative Rep. freq.:  5e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":181.253632212536,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,8.6],"tickmode":"array","ticktext":["V2-70*01","V2-70*11","V2-70*12","V2-70*13","V2-70*15","V2-70*17","V2-70*20","V2-70D*04/V2-70*04"],"tickvals":[1,2,3,4,5,6,7,8],"categoryorder":"array","categoryarray":["V2-70*01","V2-70*11","V2-70*12","V2-70*13","V2-70*15","V2-70*17","V2-70*20","V2-70D*04/V2-70*04"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-15.9057633987442,347.77547816325],"tickmode":"array","ticktext":["0","100","200","300"],"tickvals":[-1.77635683940025e-15,100,200,300],"categoryorder":"array","categoryarray":["0","100","200","300"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"258e72fa38f0d":{"x":{},"y":{},"text":{},"type":"box"},"258e765ba790b":{"x":{},"y":{},"text":{}}},"cur_data":"258e72fa38f0d","visdat":{"258e72fa38f0d":["function (y) ","x"],"258e765ba790b":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-6451c7c821dd31608169" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-6451c7c821dd31608169">{"x":{"data":[{"x":[4,4,6,1,1,6,6,1,1,6,3,1,6,1,6,1,4,6,3,1,4,4,1,4,1,6,4,1,6,4,1,6,4,6,1,4,1,6,1,6,4,1,3,6,4,1,6,1,4,1,6,4,1,6,4,5,4,1,4,6,1,4,6,1,1,6,4,6,4,6,5,4,4,1,1,6,2,4,1,4,4,1,6,1,6,1,6,3,1,6,4,1,6,4,1],"y":[119,70,40,24,93,44,3,2,76,25,1,49,14,83,29,46,31,18,1,27,16,28,24,51,37,22,30,25,13,25,18,17,13,9,6,38,50,33,72,50,8,7,2,1,36,16,9,12,46,32,22,36,44,18,22,1,66,63,60,30,29,81,47,27,68,11,25,2,30,1,1,141,40,28,81,59,1,145,99,12,29,18,10,40,29,76,20,1,99,38,77,71,19,57,24],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(187,159,61,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[3.98409296232276,4.0696416515857,5.95384641410783,0.911707286396995,1.0372697851155,5.97014111890458,5.98043124307878,1.09952930561267,1.02486788206734,6.0892845062539,3.07989432038739,0.983185690641403,5.97997453827411,1.09537884378806,5.90382296955213,0.908716659154743,3.93690395602025,5.95084993098862,2.94555876562372,0.975049771787599,3.9876895475667,3.987272950029,0.911719313962385,4.03757439218461,1.0638316730503,5.94132336382754,3.90280885868706,1.03069951771759,6.05712520782836,3.9838896300178,1.02014715378173,6.01924850139767,4.00698904218152,6.00389583362266,1.03323914129287,4.01116127357818,0.950780601240695,5.93698498136364,1.0510318542365,6.03041864121333,4.06001681699418,0.930372136039659,3.08588393935934,5.91067724544555,4.05388173651882,0.922063602693379,5.90334553145803,0.969314570212737,4.07507843766362,1.07178649893031,5.94617092953995,3.973802574398,1.03768316409551,6.09936422267929,4.02537667928264,5.07214206233621,3.93352297795936,1.04476937283762,4.06421856237575,5.98291864711791,1.03919758321717,4.06957783428952,5.99835127056576,1.0007314409595,1.08975922423415,5.96691064233892,3.92771750013344,6.06916994256899,4.03523644031957,5.901340375375,5.09888219423592,3.95051388191059,4.05426743202843,0.917359340051189,1.03561327983625,6.07368565676734,2.02977137090638,3.9845777164679,0.944119976321235,4.0784381696023,4.00604061200283,1.00545303802937,5.95843745106831,0.905195052456111,5.97886363626458,1.01268408303149,5.90430511538871,3.00485716820695,0.941059788130224,6.05267222486436,4.02842151816003,0.943302474403754,5.97007277132943,4.02791052618995,1.04085380164906],"y":[118.827148673683,70.0786621754989,39.6244582053274,23.7953571911901,93.3139002455398,44.1254380783066,3.16106939744204,1.92773609329015,76.3449076691642,24.9434523871168,0.858681284822524,48.6082451628521,14.2713976379484,83.1831828415394,29.2744568677619,46.2385736001655,30.7962420701981,17.6840635430068,0.615892279706895,26.6834568491206,15.8900201974437,28.1025512129068,23.6516408972442,51.3687072323635,37.2149297729135,22.0673682298511,29.6786096367985,24.8163179237396,12.8309025829658,24.7118677228689,18.0492458667606,16.9539379052818,12.830873949267,8.8094395333901,6.01383183933794,38.2850752463564,49.9360131537542,33.0759284650907,72.1727296324447,50.0925773160532,8.3454681430012,6.65172936730087,2.19029060602188,1.23540678843856,35.8893923437223,16.1305402537808,9.0756779205054,11.8946838259697,46.1318858645856,32.018954664655,21.7182458432391,35.9591499259695,43.7665525641292,18.1269123757258,22.0145009849221,0.808862619288266,65.67555343844,63.1314166437835,60.1837635852396,29.8532515276223,28.9923046747223,80.7396459963173,47.3640050483868,26.73985284511,68.0146917326376,11.2081163290888,25.1609411315992,1.76632053349167,29.8066337531433,0.616653485409915,1.01659975256771,141.040081094578,39.9427061265335,28.0445692017674,80.8942763987929,59.3720669014379,0.693063044175506,145.259079010412,99.035807919316,11.6720334513113,29.1458100665361,17.8356689490378,10.347566832602,40.0757054451853,29.2358027525246,76.2459967987612,19.9040981125087,0.86388159301132,98.9001327196136,38.178531804122,77.1232944227755,70.7970869597048,18.6759263068438,57.1465505450964,24.1097533896565],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  13 <\/br># assignments:  119 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0203","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  13 <\/br># assignments:  70 <\/br>Relative freq.:  0.5224 <\/br>Relative Rep. freq.:  0.0177","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  40 <\/br>Relative freq.:  0.2985 <\/br>Relative Rep. freq.:  0.0101","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  0.1791 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  93 <\/br>Relative freq.:  0.6788 <\/br>Relative Rep. freq.:  0.0204","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  44 <\/br>Relative freq.:  0.3212 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  76 <\/br>Relative freq.:  0.7451 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  25 <\/br>Relative freq.:  0.2451 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0098 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.7778 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  14 <\/br>Relative freq.:  0.2222 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  83 <\/br>Relative freq.:  0.7411 <\/br>Relative Rep. freq.:  0.016","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  29 <\/br>Relative freq.:  0.2589 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.4792 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  13 <\/br># assignments:  31 <\/br>Relative freq.:  0.3229 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  18 <\/br>Relative freq.:  0.1875 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0104 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.6279 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  13 <\/br># assignments:  16 <\/br>Relative freq.:  0.3721 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  13 <\/br># assignments:  28 <\/br>Relative freq.:  0.5385 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  0.4615 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  13 <\/br># assignments:  51 <\/br>Relative freq.:  0.4636 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  37 <\/br>Relative freq.:  0.3364 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  22 <\/br>Relative freq.:  0.2 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  13 <\/br># assignments:  30 <\/br>Relative freq.:  0.4412 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  25 <\/br>Relative freq.:  0.3676 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  13 <\/br>Relative freq.:  0.1912 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  13 <\/br># assignments:  25 <\/br>Relative freq.:  0.4167 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  18 <\/br>Relative freq.:  0.3 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  17 <\/br>Relative freq.:  0.2833 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  13 <\/br># assignments:  13 <\/br>Relative freq.:  0.4643 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  9 <\/br>Relative freq.:  0.3214 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  0.2143 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  13 <\/br># assignments:  38 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  0.6024 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  33 <\/br>Relative freq.:  0.3976 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  72 <\/br>Relative freq.:  0.5902 <\/br>Relative Rep. freq.:  0.0142","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  50 <\/br>Relative freq.:  0.4098 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  13 <\/br># assignments:  8 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.3889 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  12 <\/br># assignments:  2 <\/br>Relative freq.:  0.1111 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0556 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  13 <\/br># assignments:  36 <\/br>Relative freq.:  0.5902 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  0.2623 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  9 <\/br>Relative freq.:  0.1475 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  13 <\/br># assignments:  46 <\/br>Relative freq.:  0.46 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  32 <\/br>Relative freq.:  0.32 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  22 <\/br>Relative freq.:  0.22 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  13 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  44 <\/br>Relative freq.:  0.7097 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  04 <\/br># assignments:  18 <\/br>Relative freq.:  0.2903 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  13 <\/br># assignments:  22 <\/br>Relative freq.:  0.9565 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0435 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  13 <\/br># assignments:  66 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  63 <\/br>Relative freq.:  0.4118 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  13 <\/br># assignments:  60 <\/br>Relative freq.:  0.3922 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  04 <\/br># assignments:  30 <\/br>Relative freq.:  0.1961 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  13 <\/br># assignments:  81 <\/br>Relative freq.:  0.5226 <\/br>Relative Rep. freq.:  0.0169","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  04 <\/br># assignments:  47 <\/br>Relative freq.:  0.3032 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.1742 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  68 <\/br>Relative freq.:  0.8608 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  04 <\/br># assignments:  11 <\/br>Relative freq.:  0.1392 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  13 <\/br># assignments:  25 <\/br>Relative freq.:  0.9259 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0741 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  13 <\/br># assignments:  30 <\/br>Relative freq.:  0.9375 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0312 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0312 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  13 <\/br># assignments:  141 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0152","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  13 <\/br># assignments:  40 <\/br>Relative freq.:  0.5882 <\/br>Relative Rep. freq.:  0.0129","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.4118 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  81 <\/br>Relative freq.:  0.5745 <\/br>Relative Rep. freq.:  0.0157","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  59 <\/br>Relative freq.:  0.4184 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  11 <\/br># assignments:  1 <\/br>Relative freq.:  0.0071 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  13 <\/br># assignments:  145 <\/br>Relative freq.:  0.5943 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  99 <\/br>Relative freq.:  0.4057 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  13 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  13 <\/br># assignments:  29 <\/br>Relative freq.:  0.5088 <\/br>Relative Rep. freq.:  0.0258","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  18 <\/br>Relative freq.:  0.3158 <\/br>Relative Rep. freq.:  0.016","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  04 <\/br># assignments:  10 <\/br>Relative freq.:  0.1754 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  40 <\/br>Relative freq.:  0.5797 <\/br>Relative Rep. freq.:  0.0129","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  04 <\/br># assignments:  29 <\/br>Relative freq.:  0.4203 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  76 <\/br>Relative freq.:  0.7835 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  20 <\/br>Relative freq.:  0.2062 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0103 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  99 <\/br>Relative freq.:  0.7226 <\/br>Relative Rep. freq.:  0.0176","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  04 <\/br># assignments:  38 <\/br>Relative freq.:  0.2774 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  13 <\/br># assignments:  77 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0126","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.7889 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  19 <\/br>Relative freq.:  0.2111 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  13 <\/br># assignments:  57 <\/br>Relative freq.:  0.7037 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  0.2963 <\/br>Relative Rep. freq.:  0.0032"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(187,159,61,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(187,159,61,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":181.253632212536,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["V2-70*01","V2-70*12","V2-70*13","V2-70*15","V2-70*17","V2-70D*04/V2-70*04"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["V2-70*01","V2-70*12","V2-70*13","V2-70*15","V2-70*17","V2-70D*04/V2-70*04"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-6.61626705682837,152.491238346947],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[0,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"258e7ce4f52":{"x":{},"y":{},"text":{},"type":"box"},"258e77dcf0bba":{"x":{},"y":{},"text":{}}},"cur_data":"258e7ce4f52","visdat":{"258e7ce4f52":["function (y) ","x"],"258e77dcf0bba":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-5cb077c86131e1b3fea0" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-5cb077c86131e1b3fea0">{"x":{"data":[{"x":[5,1,7,4,9,3,10,2,1,11,10,4,5,11,1,7,9,4,5,1,7,4,10,1,11,4,10,12,5,7,11,4,1,2,1,11,4,1,11,10,4,1,11,10,4,8,2,9,11,1,4,10,1,4,10,11,1,4,10,6,11,1,5,5,1,7,4,10,9,1,4,5,7,11,11,1,1,11,4,5,10,7,5,11,7,10,5,1,7,4,11,5,1,7,4,12,10,5,11,7,5,1,7,10,8,4,1,11,4,5,10,5,7,1,5,7,4,5,11,1,7,4,9,10,5,7,4,10,9],"y":[132,60,22,8,7,4,4,1,23,20,2,1,212,88,70,16,4,3,229,110,53,3,1,51,41,9,4,2,128,19,6,3,2,1,51,4,2,45,4,3,1,52,45,4,2,2,1,1,98,85,1,1,49,1,1,156,143,7,2,1,2,1,1,116,79,4,3,2,1,2,1,31,4,4,60,21,125,120,14,4,2,1,112,9,7,2,236,118,21,2,186,185,121,10,5,2,1,44,2,1,88,58,7,5,2,1,368,332,15,1,1,189,19,2,234,11,1,51,32,27,9,4,2,1,126,14,7,3,1],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(123,147,98,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[4.92579480512068,1.00224980805069,6.9050591932144,4.0188177424483,8.93458560924046,3.04294960126281,9.94032041993924,2.0081175441388,0.973170075472444,11.0432195310015,10.082942597894,4.00936076086946,4.96449195938185,10.9375941429287,0.917627316294238,7.03999188970774,8.97832239237614,3.99384048446082,4.92226731232367,0.996477598370984,6.9434921334032,4.0324425406754,10.0193631934468,0.946457546995953,10.9905175369699,4.04513865588233,9.9438181581907,11.9975695656147,5.07442131033167,7.00456172893755,11.0542417126242,4.09574023387395,1.0597899528686,1.97235033824109,1.00599151402712,10.9647878810298,3.95948899174109,0.915305625973269,11.0636171801016,10.069019884197,3.98639823738486,0.996030080597848,11.0044943132438,10.0428553777281,4.02481853244826,8.07580996989273,1.94487435184419,9.07157901250757,10.9802288515959,1.02721769013442,4.06527646780014,9.942217751313,1.03432616936043,4.00263808500022,9.97268990571611,11.0129000361077,1.01767764701508,3.99269084911793,9.93535047066398,6.01518857716583,10.9161039006896,1.07258770861663,4.90353065375239,4.9711618647445,1.03574883961119,6.90277798380703,4.04473120998591,9.93661603857763,8.91997309783474,1.07376137673855,3.90210232702084,4.9152471258305,7.09874125435017,10.9241091053002,10.9447401469108,1.01651551942341,0.926973156677559,11.0814896339085,4.02793216491118,4.98489494514652,10.0387501005549,7.01167796701193,5.09355742889456,10.9854825819843,7.05043417387642,10.0948866425082,4.98725697989576,1.02969609037973,6.98103191759437,3.90050409929827,10.9233060137369,5.04414686216041,0.940928142145276,7.04545516604558,4.01748102959245,11.9461542621255,9.9839351549279,5.08532033935189,11.0716237816028,7.03463565250859,5.09706936273724,1.03481152299792,7.02080606976524,10.0011679836549,7.97813435308635,3.9695145658683,1.08454170576297,11.0320910501294,3.94178041946143,5.03623553290963,10.0435492862016,4.95838390886784,6.91981133292429,0.970854178583249,4.91460780263878,7.00048938570544,4.02461594231427,4.92404844113626,10.9597399781924,0.969464715197682,7.06945930472575,4.09555182568729,9.03465018793941,9.99197662705556,4.92938219844364,6.97947782506235,4.05514380023815,9.97193571054377,9.04709582808428],"y":[131.808611889556,59.6812420517206,22.2401988167316,7.62970484681427,6.67868450079113,4.21201786436141,4.20386417340487,0.901204594783485,23.332129544206,19.6697600340471,1.96703100632876,1.02603646684438,212.125853218138,87.6798887405544,69.7689126949757,15.6766557075083,4.39350085686892,2.66899590082467,229.32167388089,109.977081026509,52.823196589388,2.8417061785236,0.614145277068019,51.3168513245881,41.1602505108342,8.73478401787579,3.7576303947717,1.73485961984843,128.041426183283,18.9269294599071,5.65035274960101,2.75180942062289,1.84187825899571,1.28750154394656,51.3319933479652,4.36158102862537,2.38368151299655,44.805676978454,3.87233186531812,2.99705283325166,0.69823169708252,52.3317450083792,45.3988274699077,4.33424233216792,2.39437607433647,2.08583935331553,0.604221811704338,1.12169056106359,97.8842086588964,85.1381937988102,0.779445900954306,1.17712336778641,48.9865356357768,0.7814063873142,1.33131454642862,155.913447163813,143.239814587496,6.91198732536286,1.77586061190814,0.706416928209364,2.34838949535042,1.39630325809121,0.720954649336636,115.935916659236,79.3976141193882,4.29075122904032,2.75246165953577,1.60798627156764,1.35698581114411,2.37598737590015,1.16211446914822,30.8149154255167,3.80650916583836,3.95869087465107,60.196010569483,20.916712818481,125.018115032837,119.904376783967,13.806108256802,4.25019655730575,1.92813093345612,1.14861436169595,111.900018876977,8.94261011499912,7.14503007624298,2.24022006541491,236.226416090317,118.371864788607,21.2785790033638,2.28142957538366,185.795804763399,184.820811061934,120.889868702926,10.130138175562,4.88664298467338,2.15013737566769,1.22227568011731,43.7481976913288,1.73628666140139,1.15677597485483,88.129180477187,58.2437278609723,6.94019354954362,4.89330816753209,1.99244302362204,0.722677478194237,367.967498180456,331.767555698752,15.1002595037222,1.21014515124261,0.744051932916045,188.626523727737,18.9341812061146,1.63037017416209,234.15314153377,10.8232201002538,1.14999812655151,51.2839456822723,32.3744652716443,27.3652910398319,9.04677108954638,3.84875364731997,1.98438322581351,0.6927011763677,126.041083825938,14.1191433647647,6.91957766413689,3.01713058818132,1.32762576937675],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  13 <\/br># assignments:  132 <\/br>Relative freq.:  0.5546 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.2521 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  15 <\/br># assignments:  22 <\/br>Relative freq.:  0.0924 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  12 <\/br># assignments:  8 <\/br>Relative freq.:  0.0336 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  17 <\/br># assignments:  7 <\/br>Relative freq.:  0.0294 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  11 <\/br># assignments:  4 <\/br>Relative freq.:  0.0168 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  18 <\/br># assignments:  4 <\/br>Relative freq.:  0.0168 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  10 <\/br># assignments:  1 <\/br>Relative freq.:  0.0042 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  20 <\/br>Relative freq.:  0.4348 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  18 <\/br># assignments:  2 <\/br>Relative freq.:  0.0435 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0217 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  13 <\/br># assignments:  212 <\/br>Relative freq.:  0.5394 <\/br>Relative Rep. freq.:  0.0189","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  88 <\/br>Relative freq.:  0.2239 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  70 <\/br>Relative freq.:  0.1781 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  15 <\/br># assignments:  16 <\/br>Relative freq.:  0.0407 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  17 <\/br># assignments:  4 <\/br>Relative freq.:  0.0102 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  12 <\/br># assignments:  3 <\/br>Relative freq.:  0.0076 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  13 <\/br># assignments:  229 <\/br>Relative freq.:  0.5783 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  110 <\/br>Relative freq.:  0.2778 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  15 <\/br># assignments:  53 <\/br>Relative freq.:  0.1338 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  12 <\/br># assignments:  3 <\/br>Relative freq.:  0.0076 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0025 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  0.4766 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  41 <\/br>Relative freq.:  0.3832 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  12 <\/br># assignments:  9 <\/br>Relative freq.:  0.0841 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  18 <\/br># assignments:  4 <\/br>Relative freq.:  0.0374 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  19 <\/br># assignments:  2 <\/br>Relative freq.:  0.0187 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  13 <\/br># assignments:  128 <\/br>Relative freq.:  0.805 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  15 <\/br># assignments:  19 <\/br>Relative freq.:  0.1195 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  6 <\/br>Relative freq.:  0.0377 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  12 <\/br># assignments:  3 <\/br>Relative freq.:  0.0189 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.0126 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  10 <\/br># assignments:  1 <\/br>Relative freq.:  0.0063 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  0.8947 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.0702 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  12 <\/br># assignments:  2 <\/br>Relative freq.:  0.0351 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  45 <\/br>Relative freq.:  0.8491 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.0755 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  18 <\/br># assignments:  3 <\/br>Relative freq.:  0.0566 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0189 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  52 <\/br>Relative freq.:  0.486 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  45 <\/br>Relative freq.:  0.4206 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  18 <\/br># assignments:  4 <\/br>Relative freq.:  0.0374 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  12 <\/br># assignments:  2 <\/br>Relative freq.:  0.0187 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  16 <\/br># assignments:  2 <\/br>Relative freq.:  0.0187 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  10 <\/br># assignments:  1 <\/br>Relative freq.:  0.0093 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  17 <\/br># assignments:  1 <\/br>Relative freq.:  0.0093 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  98 <\/br>Relative freq.:  0.5297 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  85 <\/br>Relative freq.:  0.4595 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0054 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0054 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.9608 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0196 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0196 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  156 <\/br>Relative freq.:  0.5049 <\/br>Relative Rep. freq.:  0.0138","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  143 <\/br>Relative freq.:  0.4628 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  12 <\/br># assignments:  7 <\/br>Relative freq.:  0.0227 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  18 <\/br># assignments:  2 <\/br>Relative freq.:  0.0065 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  14 <\/br># assignments:  1 <\/br>Relative freq.:  0.0032 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.2222","<\/br>Project:  P4 <\/br>Subject:  P4_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.1111","<\/br>Project:  P4 <\/br>Subject:  P4_I20_S1 <\/br>Alleles:  13 <\/br># assignments:  1 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.1111","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  13 <\/br># assignments:  116 <\/br>Relative freq.:  0.5659 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  79 <\/br>Relative freq.:  0.3854 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  15 <\/br># assignments:  4 <\/br>Relative freq.:  0.0195 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  12 <\/br># assignments:  3 <\/br>Relative freq.:  0.0146 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  18 <\/br># assignments:  2 <\/br>Relative freq.:  0.0098 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  17 <\/br># assignments:  1 <\/br>Relative freq.:  0.0049 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  13 <\/br># assignments:  31 <\/br>Relative freq.:  0.8857 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  15 <\/br># assignments:  4 <\/br>Relative freq.:  0.1143 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  60 <\/br>Relative freq.:  0.7407 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  21 <\/br>Relative freq.:  0.2593 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  125 <\/br>Relative freq.:  0.4699 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  120 <\/br>Relative freq.:  0.4511 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  12 <\/br># assignments:  14 <\/br>Relative freq.:  0.0526 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  13 <\/br># assignments:  4 <\/br>Relative freq.:  0.015 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  18 <\/br># assignments:  2 <\/br>Relative freq.:  0.0075 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0038 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  13 <\/br># assignments:  112 <\/br>Relative freq.:  0.8615 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  9 <\/br>Relative freq.:  0.0692 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  15 <\/br># assignments:  7 <\/br>Relative freq.:  0.0538 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  18 <\/br># assignments:  2 <\/br>Relative freq.:  0.0154 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  13 <\/br># assignments:  236 <\/br>Relative freq.:  0.626 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  118 <\/br>Relative freq.:  0.313 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  15 <\/br># assignments:  21 <\/br>Relative freq.:  0.0557 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  12 <\/br># assignments:  2 <\/br>Relative freq.:  0.0053 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  186 <\/br>Relative freq.:  0.3647 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  13 <\/br># assignments:  185 <\/br>Relative freq.:  0.3627 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  121 <\/br>Relative freq.:  0.2373 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  15 <\/br># assignments:  10 <\/br>Relative freq.:  0.0196 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  12 <\/br># assignments:  5 <\/br>Relative freq.:  0.0098 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  19 <\/br># assignments:  2 <\/br>Relative freq.:  0.0039 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  13 <\/br># assignments:  44 <\/br>Relative freq.:  0.9362 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0426 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  15 <\/br># assignments:  1 <\/br>Relative freq.:  0.0213 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  13 <\/br># assignments:  88 <\/br>Relative freq.:  0.5466 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  58 <\/br>Relative freq.:  0.3602 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  15 <\/br># assignments:  7 <\/br>Relative freq.:  0.0435 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  18 <\/br># assignments:  5 <\/br>Relative freq.:  0.0311 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  16 <\/br># assignments:  2 <\/br>Relative freq.:  0.0124 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0062 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  368 <\/br>Relative freq.:  0.5132 <\/br>Relative Rep. freq.:  0.0535","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  332 <\/br>Relative freq.:  0.463 <\/br>Relative Rep. freq.:  0.0483","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  12 <\/br># assignments:  15 <\/br>Relative freq.:  0.0209 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  13 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  13 <\/br># assignments:  189 <\/br>Relative freq.:  0.9 <\/br>Relative Rep. freq.:  0.0219","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  15 <\/br># assignments:  19 <\/br>Relative freq.:  0.0905 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.0095 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  13 <\/br># assignments:  234 <\/br>Relative freq.:  0.9512 <\/br>Relative Rep. freq.:  0.0158","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  15 <\/br># assignments:  11 <\/br>Relative freq.:  0.0447 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0041 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  13 <\/br># assignments:  51 <\/br>Relative freq.:  0.4048 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  32 <\/br>Relative freq.:  0.254 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.2143 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  15 <\/br># assignments:  9 <\/br>Relative freq.:  0.0714 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  12 <\/br># assignments:  4 <\/br>Relative freq.:  0.0317 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  17 <\/br># assignments:  2 <\/br>Relative freq.:  0.0159 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0079 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  13 <\/br># assignments:  126 <\/br>Relative freq.:  0.8344 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  15 <\/br># assignments:  14 <\/br>Relative freq.:  0.0927 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  12 <\/br># assignments:  7 <\/br>Relative freq.:  0.0464 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  18 <\/br># assignments:  3 <\/br>Relative freq.:  0.0199 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  17 <\/br># assignments:  1 <\/br>Relative freq.:  0.0066 <\/br>Relative Rep. freq.:  1e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(123,147,98,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(123,147,98,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":181.253632212536,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,12.6],"tickmode":"array","ticktext":["V2-70*01","V2-70*11","V2-70*12","V2-70*13","V2-70*15","V2-70*16","V2-70*17","V2-70*18","V2-70*19","V2-70*20","V2-70D*04/V2-70*04","V2-70D*14"],"tickvals":[1,2,3,4,5,6,7,8,9,10,11,12],"categoryorder":"array","categoryarray":["V2-70*01","V2-70*11","V2-70*12","V2-70*13","V2-70*15","V2-70*16","V2-70*17","V2-70*18","V2-70*19","V2-70*20","V2-70D*04/V2-70*04","V2-70D*14"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-17.7655670977104,386.369788909415],"tickmode":"array","ticktext":["0","100","200","300"],"tickvals":[0,100,200,300],"categoryorder":"array","categoryarray":["0","100","200","300"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"258e710368e54":{"x":{},"y":{},"text":{},"type":"box"},"258e74439f10d":{"x":{},"y":{},"text":{}}},"cur_data":"258e710368e54","visdat":{"258e710368e54":["function (y) ","x"],"258e74439f10d":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-10" class="section level2" number="14.4">
<h2><span class="header-section-number">14.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 10 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV2-70G13%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2);width:800px;height:900px;" cellspacing="0">
</iframe>
</div>
<div id="observations-12" class="section level2" number="14.5">
<h2><span class="header-section-number">14.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/dx0odrmkc0rmdhd5t2enk/IGHV2-70G13.docx?dl=0&amp;rlkey=wdypp2b015xm6q32k3jvyhdty">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV2-70G13" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-12" class="section level2" number="14.6">
<h2><span class="header-section-number">14.6</span> Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="12-G12.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="14-G14.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/13-G13.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
