<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV3-11 - G14 | Functional groups</title>
  <meta name="description" content="IGHV3-11 - G14 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV3-11 - G14 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV3-11 - G14 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="13-G13.html"/>
<link rel="next" href="15-G15.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-checklist"><i class="fa fa-check"></i>Group Checklist</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./"> Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv3-11---g14" class="section level1">
<h1>IGHV3-11 - G14</h1>
<div id="allele-appearnce-13" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of IGHV3-11 includes 5 alleles, 5 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-130-1.png" width="672" /></p>
</div>
<div id="group-alignment-11" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-131-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-13" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-3ae5e418f410469e9a26" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-3ae5e418f410469e9a26">{"x":{"data":[{"x":[4,1,4,1,4,1,4,1,3,1,4,3,1,1,1,4,1,3,1,4,1,4,1,1,4,1,1,1,1,3,4,3,1,3,3,1,1,1,4,1,4,3,4,3,4,1,4,1,1,4,1,3,4,1,4,2,4,1,1,1,4,1,1,4,1,4,1,1,1,4,1,4,3,1,3,1,4,1,4,3,4,1,4,1,4,3,1,1,4,1,3,1,1,2,1,4,2,4,1,1,4,1,4,1,4,1,1,4,1,4,1,1,4,4,1,4,1,4,4,1,4,4,1,4,1,3,4,1,4,2,1,4,1,4,3,4,1,1,1,3,4,1,1,4,1,4,1,4,3,1,3,4,1,3,1,4,1,4,3,1,1,4,1],"y":[48,36,114,288,70,306,136,336,322,71,24,102,60,363,86,48,220,353,253,1,64,41,343,39,12,512,330,483,578,167,347,264,54,41,488,78,208,262,165,5,5,35,13,317,63,78,30,143,179,40,238,94,11,317,219,1,489,418,438,107,72,164,237,70,49,48,579,143,459,274,58,58,298,296,160,158,170,383,191,228,112,310,171,94,82,235,168,315,164,609,20,16,236,148,250,47,118,100,417,181,94,223,58,650,75,1,297,50,241,165,514,303,157,178,313,176,194,113,487,231,124,127,167,122,534,471,355,208,151,1,155,94,153,54,330,109,111,673,609,466,204,469,216,36,343,237,214,151,122,117,281,79,165,122,165,94,14,10,252,203,182,80,47],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[4.02442346634343,1.03075037477538,3.91835479815491,1.01063391505741,4.01182630364783,0.931135901529342,3.91417780900374,0.915463987318799,2.97547507146373,0.980598937580362,4.06690349448472,3.02212892021053,1.01640315297991,0.939250420732424,1.07290016501211,4.0495829815045,1.03790626381524,3.0242107859347,0.99303296180442,3.95971681182273,0.903677296685055,4.03569801384583,0.930364172207192,1.01547820619307,3.97908418257721,1.05216259136796,0.96743635465391,1.08614750937559,0.980445738090202,2.98043035347946,3.92610698114149,2.92552427304909,1.04961301777512,2.92310764072463,2.98057271512225,0.981589902797714,0.962533381721005,1.02523890906014,4.01427333694883,0.933772612409666,3.9366619864013,3.09077168703079,3.90034737540409,2.98587044561282,3.93517644316889,0.979983739368617,4.0613016305957,0.912199656479061,0.962398088397458,4.08846886297688,0.988973592966795,3.0541957131587,4.00557839646935,1.08481869515963,3.93033592752181,1.95037704664282,3.94603602327406,0.994324589893222,1.01652608322911,1.07624372961,3.95120932143182,1.09172563767061,0.91953949755989,3.9642809653189,1.02119220783934,4.05831587850116,0.969527081027627,1.09984769388102,0.966430755984038,3.91422305922024,1.06512543391436,3.95181344016455,2.97164286826737,1.01171257216483,3.0639581119176,1.02254872405902,4.03791514770128,0.973669599741697,3.94594363062643,2.99307555039413,4.04553882325999,1.07889267317951,4.03396151331253,1.00649110800587,4.09508838909678,3.03674409929663,0.910395997529849,0.966082172188908,4.0979924284853,1.09886904270388,3.08863163650967,1.0176052138675,1.0939044107683,2.06124115115963,1.03761450285092,3.93359103975818,2.00829582940787,4.01628726040944,1.05765779395588,0.980428778193891,4.04156038882211,0.930584184080362,3.90521231335588,0.972687435010448,4.04835239294916,0.95226928954944,0.959553197119385,3.95037432680838,1.0995788901113,3.93088979111053,0.94361754162237,1.01217445321381,4.05564984381199,4.08725684685633,0.982498833257705,4.00719543714076,0.995317199639976,4.0854653798975,4.07055117138661,1.02394414995797,3.90105792116374,3.9022880172357,1.09835953726433,3.96462146826088,1.09660831443034,2.994926376082,4.05342229781672,0.968838280579075,3.99338728925213,1.99042000477202,0.992217515362427,3.91019524242729,0.906637099571526,3.91054007369094,3.00819642371498,4.08437612121925,1.06082211844623,0.92938741366379,1.06043019467033,2.92918420634232,4.00927613298409,0.974779935833067,1.08730723191984,3.99318706858903,0.949906370090321,3.99595014597289,0.994337842706591,3.98886093604378,2.93208878426813,0.955647080624476,2.99400952919386,3.94144026823342,0.934158014506102,2.90649900524877,1.00321564255282,3.92974661095068,1.0784383954946,4.00527842142619,3.07452453407459,0.970190728828311,0.944740041531622,4.02242235685699,1.08684620358981],"y":[48.3476768406108,35.6904663374647,114.21458783187,288.054695886746,70.3921320224181,305.627736386098,136.25454132501,335.960904779471,321.88348974809,71.2455330746248,24.1016241259873,102.341038193181,60.3966285223141,362.852137807757,85.7062207497656,47.9060680376366,220.186470008641,353.290466193855,252.645039122738,1.21989104561508,63.8409368174151,40.944507695362,343.339160091616,39.2734996313229,11.8799002846703,512.302164323442,330.215492742881,482.602310031466,577.829885431938,166.987447044067,346.758790461905,263.697056569904,54.1024048771709,40.6327018393204,488.159036628902,77.7256253369153,207.899677392282,261.711345056631,165.188537459634,4.94514253661036,4.78744477275759,35.2426254818216,12.6380126381293,316.965654962696,63.1670688018203,78.3741797346622,29.6519071526825,143.086733058281,178.948103380948,39.7958331717178,238.281507750973,93.9725820865482,11.2146587841213,317.334160021693,218.828434674814,0.702082995325327,488.871855612099,418.340724691004,437.72260767296,106.64542194251,72.3081829937175,163.778554942459,237.283100961708,69.7895681563765,48.9155256113037,47.877112891525,579.136309548095,142.985410443135,459.301675444096,274.317047129758,57.7070726504549,58.0005594955757,298.31628036052,296.163944262452,159.70372062698,157.623309919983,169.868738848343,383.372412690893,191.12994727958,228.389777440764,111.862273687497,310.19850142505,171.396570189483,93.876550125517,81.9586477614939,235.308040218614,167.793736307323,315.251372878812,163.865223965608,609.176951674931,20.2409162865952,16.115436370112,235.93703615889,148.305927144736,250.153383164853,47.174159729667,117.753287229873,100.320621237531,417.139860904031,181.396002736874,94.187832439132,222.665228994004,58.20684902668,650.142776442692,75.1408790228888,1.29155445806682,296.88217067346,49.9746016511694,241.276458648778,164.810554447398,513.751371464506,303.207613055781,156.641984565929,178.138419667818,313.382871697098,175.799300680682,193.652619036473,113.296731764264,487.052588390373,231.362741468102,124.021734306961,126.994091896154,167.234671523608,122.197422972694,533.638663773611,471.018879823759,354.855998048186,207.908771778829,150.725032360479,0.715548899397254,155.238967655413,93.6589190742001,153.200699626654,54.3270259266719,329.984932808392,108.881772783399,111.114216836356,672.65848212596,608.649162294902,466.366850892641,204.123633295111,468.810395623557,215.737529699318,35.7441183945164,343.219736604951,236.797135746852,214.105166449212,151.034967455827,121.616960455664,116.994670475647,281.090269043855,79.3188304023817,165.219116291963,122.308876328915,164.867317090929,94.1304849946871,14.0282170347869,10.1712480776012,251.694566158392,202.749410981499,182.194927198999,79.7551843704656,47.1033133110031],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  06 <\/br># assignments:  48 <\/br>Relative freq.:  0.5714 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  06 <\/br># assignments:  114 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  288 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0277","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  06 <\/br># assignments:  70 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  306 <\/br>Relative freq.:  0.6923 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  06 <\/br># assignments:  136 <\/br>Relative freq.:  0.3077 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  336 <\/br>Relative freq.:  0.5106 <\/br>Relative Rep. freq.:  0.014","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  05 <\/br># assignments:  322 <\/br>Relative freq.:  0.4894 <\/br>Relative Rep. freq.:  0.0134","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.7474 <\/br>Relative Rep. freq.:  0.0121","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  06 <\/br># assignments:  24 <\/br>Relative freq.:  0.2526 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  05 <\/br># assignments:  102 <\/br>Relative freq.:  0.6296 <\/br>Relative Rep. freq.:  0.0175","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.3704 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  363 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0261","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  86 <\/br>Relative freq.:  0.6418 <\/br>Relative Rep. freq.:  0.0156","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  06 <\/br># assignments:  48 <\/br>Relative freq.:  0.3582 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  220 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0226","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  05 <\/br># assignments:  353 <\/br>Relative freq.:  0.5815 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  253 <\/br>Relative freq.:  0.4168 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  64 <\/br>Relative freq.:  0.6095 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  06 <\/br># assignments:  41 <\/br>Relative freq.:  0.3905 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  343 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0255","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  0.7647 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  06 <\/br># assignments:  12 <\/br>Relative freq.:  0.2353 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  512 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0196","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  330 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0191","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  483 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0219","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  578 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0254","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  167 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0165","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  06 <\/br># assignments:  347 <\/br>Relative freq.:  0.5679 <\/br>Relative Rep. freq.:  0.014","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  05 <\/br># assignments:  264 <\/br>Relative freq.:  0.4321 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  54 <\/br>Relative freq.:  0.5684 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  05 <\/br># assignments:  41 <\/br>Relative freq.:  0.4316 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  05 <\/br># assignments:  488 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0224","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  78 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  208 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0104","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  262 <\/br>Relative freq.:  0.6136 <\/br>Relative Rep. freq.:  0.0108","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  06 <\/br># assignments:  165 <\/br>Relative freq.:  0.3864 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  06 <\/br># assignments:  5 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  05 <\/br># assignments:  35 <\/br>Relative freq.:  0.7292 <\/br>Relative Rep. freq.:  0.02","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  06 <\/br># assignments:  13 <\/br>Relative freq.:  0.2708 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  05 <\/br># assignments:  317 <\/br>Relative freq.:  0.8342 <\/br>Relative Rep. freq.:  0.0155","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  06 <\/br># assignments:  63 <\/br>Relative freq.:  0.1658 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  78 <\/br>Relative freq.:  0.7222 <\/br>Relative Rep. freq.:  0.0154","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  06 <\/br># assignments:  30 <\/br>Relative freq.:  0.2778 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  143 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0201","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  179 <\/br>Relative freq.:  0.8174 <\/br>Relative Rep. freq.:  0.016","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  06 <\/br># assignments:  40 <\/br>Relative freq.:  0.1826 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  238 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0297","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  05 <\/br># assignments:  94 <\/br>Relative freq.:  0.8952 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  06 <\/br># assignments:  11 <\/br>Relative freq.:  0.1048 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  317 <\/br>Relative freq.:  0.5903 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  06 <\/br># assignments:  219 <\/br>Relative freq.:  0.4078 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  06 <\/br># assignments:  489 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0204","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  418 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0248","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  438 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0227","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  107 <\/br>Relative freq.:  0.5978 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  06 <\/br># assignments:  72 <\/br>Relative freq.:  0.4022 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  164 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0274","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  237 <\/br>Relative freq.:  0.772 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  06 <\/br># assignments:  70 <\/br>Relative freq.:  0.228 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.5052 <\/br>Relative Rep. freq.:  0.0155","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  06 <\/br># assignments:  48 <\/br>Relative freq.:  0.4948 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  579 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0267","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  143 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0197","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  459 <\/br>Relative freq.:  0.6262 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  06 <\/br># assignments:  274 <\/br>Relative freq.:  0.3738 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  58 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  06 <\/br># assignments:  58 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  05 <\/br># assignments:  298 <\/br>Relative freq.:  0.5017 <\/br>Relative Rep. freq.:  0.0111","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  296 <\/br>Relative freq.:  0.4983 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  05 <\/br># assignments:  160 <\/br>Relative freq.:  0.5031 <\/br>Relative Rep. freq.:  0.0178","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  158 <\/br>Relative freq.:  0.4969 <\/br>Relative Rep. freq.:  0.0176","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  06 <\/br># assignments:  170 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  383 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0181","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  06 <\/br># assignments:  191 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  05 <\/br># assignments:  228 <\/br>Relative freq.:  0.6706 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  06 <\/br># assignments:  112 <\/br>Relative freq.:  0.3294 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  310 <\/br>Relative freq.:  0.6445 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  06 <\/br># assignments:  171 <\/br>Relative freq.:  0.3555 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  94 <\/br>Relative freq.:  0.5341 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  06 <\/br># assignments:  82 <\/br>Relative freq.:  0.4659 <\/br>Relative Rep. freq.:  0.0111","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  05 <\/br># assignments:  235 <\/br>Relative freq.:  0.5831 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  168 <\/br>Relative freq.:  0.4169 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  315 <\/br>Relative freq.:  0.6576 <\/br>Relative Rep. freq.:  0.0101","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  06 <\/br># assignments:  164 <\/br>Relative freq.:  0.3424 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  609 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.019","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  05 <\/br># assignments:  20 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.0153","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  0.0122","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  236 <\/br>Relative freq.:  0.6146 <\/br>Relative Rep. freq.:  0.0124","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  04 <\/br># assignments:  148 <\/br>Relative freq.:  0.3854 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  250 <\/br>Relative freq.:  0.8418 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  06 <\/br># assignments:  47 <\/br>Relative freq.:  0.1582 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  04 <\/br># assignments:  118 <\/br>Relative freq.:  0.5413 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  06 <\/br># assignments:  100 <\/br>Relative freq.:  0.4587 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  417 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0225","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  181 <\/br>Relative freq.:  0.6582 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  06 <\/br># assignments:  94 <\/br>Relative freq.:  0.3418 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  223 <\/br>Relative freq.:  0.7936 <\/br>Relative Rep. freq.:  0.0191","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  06 <\/br># assignments:  58 <\/br>Relative freq.:  0.2064 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  650 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  06 <\/br># assignments:  75 <\/br>Relative freq.:  0.9868 <\/br>Relative Rep. freq.:  0.0101","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0132 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  297 <\/br>Relative freq.:  0.8559 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  06 <\/br># assignments:  50 <\/br>Relative freq.:  0.1441 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  241 <\/br>Relative freq.:  0.5936 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  06 <\/br># assignments:  165 <\/br>Relative freq.:  0.4064 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  514 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0159","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  303 <\/br>Relative freq.:  0.6587 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  06 <\/br># assignments:  157 <\/br>Relative freq.:  0.3413 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  06 <\/br># assignments:  178 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  313 <\/br>Relative freq.:  0.6401 <\/br>Relative Rep. freq.:  0.0126","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  06 <\/br># assignments:  176 <\/br>Relative freq.:  0.3599 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  194 <\/br>Relative freq.:  0.6319 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  06 <\/br># assignments:  113 <\/br>Relative freq.:  0.3681 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  06 <\/br># assignments:  487 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0209","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  231 <\/br>Relative freq.:  0.6507 <\/br>Relative Rep. freq.:  0.0136","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  06 <\/br># assignments:  124 <\/br>Relative freq.:  0.3493 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  06 <\/br># assignments:  127 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  167 <\/br>Relative freq.:  0.5779 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  06 <\/br># assignments:  122 <\/br>Relative freq.:  0.4221 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  534 <\/br>Relative freq.:  0.5313 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  05 <\/br># assignments:  471 <\/br>Relative freq.:  0.4687 <\/br>Relative Rep. freq.:  0.0146","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  06 <\/br># assignments:  355 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0122","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  208 <\/br>Relative freq.:  0.5778 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  06 <\/br># assignments:  151 <\/br>Relative freq.:  0.4194 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  155 <\/br>Relative freq.:  0.6225 <\/br>Relative Rep. freq.:  0.0126","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  06 <\/br># assignments:  94 <\/br>Relative freq.:  0.3775 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  153 <\/br>Relative freq.:  0.7391 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  06 <\/br># assignments:  54 <\/br>Relative freq.:  0.2609 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  05 <\/br># assignments:  330 <\/br>Relative freq.:  0.7517 <\/br>Relative Rep. freq.:  0.0142","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  06 <\/br># assignments:  109 <\/br>Relative freq.:  0.2483 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  111 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0123","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  673 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0364","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  609 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0238","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  05 <\/br># assignments:  466 <\/br>Relative freq.:  0.6955 <\/br>Relative Rep. freq.:  0.0146","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  06 <\/br># assignments:  204 <\/br>Relative freq.:  0.3045 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  01 <\/br># assignments:  469 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0211","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  216 <\/br>Relative freq.:  0.8571 <\/br>Relative Rep. freq.:  0.0137","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  06 <\/br># assignments:  36 <\/br>Relative freq.:  0.1429 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  343 <\/br>Relative freq.:  0.5914 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  06 <\/br># assignments:  237 <\/br>Relative freq.:  0.4086 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  214 <\/br>Relative freq.:  0.5863 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  06 <\/br># assignments:  151 <\/br>Relative freq.:  0.4137 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  05 <\/br># assignments:  122 <\/br>Relative freq.:  0.5105 <\/br>Relative Rep. freq.:  0.0133","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  117 <\/br>Relative freq.:  0.4895 <\/br>Relative Rep. freq.:  0.0128","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  05 <\/br># assignments:  281 <\/br>Relative freq.:  0.7806 <\/br>Relative Rep. freq.:  0.0128","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  06 <\/br># assignments:  79 <\/br>Relative freq.:  0.2194 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  165 <\/br>Relative freq.:  0.5749 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  05 <\/br># assignments:  122 <\/br>Relative freq.:  0.4251 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  165 <\/br>Relative freq.:  0.6371 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  06 <\/br># assignments:  94 <\/br>Relative freq.:  0.3629 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  0.5833 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  06 <\/br># assignments:  10 <\/br>Relative freq.:  0.4167 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  05 <\/br># assignments:  252 <\/br>Relative freq.:  0.5538 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  203 <\/br>Relative freq.:  0.4462 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  182 <\/br>Relative freq.:  0.6947 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  06 <\/br># assignments:  80 <\/br>Relative freq.:  0.3053 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  47 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0255"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,4.6],"tickmode":"array","ticktext":["V3-11*01","V3-11*04","V3-11*05","V3-11*06"],"tickvals":[1,2,3,4],"categoryorder":"array","categoryarray":["V3-11*01","V3-11*04","V3-11*05","V3-11*06"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-32.9128128549084,706.614895850234],"tickmode":"array","ticktext":["0","200","400","600"],"tickvals":[0,200,400,600],"categoryorder":"array","categoryarray":["0","200","400","600"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"7d6f766445edb":{"x":{},"y":{},"text":{},"type":"box"},"7d6f779438e79":{"x":{},"y":{},"text":{}}},"cur_data":"7d6f766445edb","visdat":{"7d6f766445edb":["function (y) ","x"],"7d6f779438e79":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-d1f89ac1e4e322144137" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-d1f89ac1e4e322144137">{"x":{"data":[{"x":[1,4,4,4,4,4,1,4,1,4,1,4,1,4,1,4,1,4,1,4,3,4,1,4,1,1,1,4,3,4,1,4,1,4,1,4,4,1,4,2,1,4,1,2,4,1,4,1,4,1,2,1,2,1,2,4,2,1,4,4,1,4,1,1,4,2,4,3,1,4,1],"y":[33,9,4,19,12,23,46,21,11,8,73,49,12,7,57,8,30,17,64,23,45,17,20,15,129,41,29,7,73,29,28,14,16,8,51,67,61,50,25,1,63,27,50,1,4,46,6,58,20,23,10,79,1,12,8,20,64,2,50,3,1,11,8,60,24,1,18,89,21,8,39],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.07860012697056,4.01794614568353,4.01410347851925,3.98249107571319,4.08196487240493,4.01318598319776,0.908980259625241,4.01216389164329,0.916245085559785,3.99551494708285,1.09232192593627,4.01551219257526,0.906473360024393,3.95407835277729,1.00687343040481,3.92287000850774,0.905366457719356,4.01312763211317,1.00395954335108,3.97428299305029,3.00967524363659,3.98378454162739,0.982518015056849,4.09155226899311,0.900937044247985,1.02861061072908,1.01828134832904,4.03862218335271,3.03302630884573,3.94363481020555,1.03352800174616,3.935948176682,0.957154251215979,4.00326424636878,0.927559999004006,4.07789928913116,3.94842726774514,1.07162444698624,3.93270029989071,1.96262131002732,0.943292293604463,4.07265676818788,0.936759551335126,2.03058422035538,3.96003168737516,1.04532638969831,4.01479529477656,0.99908562945202,3.98708823388442,0.903880572738126,2.02242618962191,1.04432086735033,1.93986846124753,1.02832289510407,2.0534200495109,4.01825085626915,2.0952571245376,1.0696786083281,4.01283817184158,3.98847682294436,1.02908698269166,4.04458922571503,0.94974748683162,1.01175301140174,3.9220000714995,2.09906461415812,4.0709596187342,2.95955337677151,0.939690436888486,4.02239868273027,1.06109642637894],"y":[32.9272789400071,8.88858594223857,3.60699049532413,19.0806508043781,11.8147507639602,22.8077809419483,45.9340275002643,20.9726949220523,10.7860636826605,7.75770260300487,72.6119126671925,48.6502794977278,12.2125109367073,7.31239948831499,56.768019670248,7.78432799447328,30.2844985770062,16.9152506766841,64.1797914523631,23.0581461954862,45.2191797414795,17.1597920309752,20.1159003365785,15.1059283349663,129.278182233311,40.6276859765872,29.0855575893074,6.61871482655406,73.0135309740901,29.3110101921484,28.0333005156368,14.3813044127077,15.888082401827,7.92112623043358,51.3827171929181,67.0038638722151,61.0716283032671,50.1326702130958,25.1400152068585,1.17434454988688,62.7996254535392,27.3940285138786,49.7296316539869,0.773020467720926,3.72145569827408,46.0786238618195,5.62211783807725,57.7405174588785,20.0615440793335,22.8168556172401,9.7525496782735,78.8090422337875,1.21273448243737,12.1629604805261,7.62061241809279,19.9451563853771,64.3433733511716,1.67174450717866,49.9853658264503,3.13576098736376,0.787432724609971,11.1950873628259,8.35991907641292,59.915468962118,23.9346134599298,0.685651643201709,18.3395469333976,89.375519573316,20.6947270104662,8.00086630750448,39.0445146573707],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  0.7857 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  06 <\/br># assignments:  9 <\/br>Relative freq.:  0.2143 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  06 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  06 <\/br># assignments:  19 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  06 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  06 <\/br># assignments:  23 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.6866 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  06 <\/br># assignments:  21 <\/br>Relative freq.:  0.3134 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.5789 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  06 <\/br># assignments:  8 <\/br>Relative freq.:  0.4211 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  73 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  06 <\/br># assignments:  49 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.6316 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  06 <\/br># assignments:  7 <\/br>Relative freq.:  0.3684 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  0.8769 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  06 <\/br># assignments:  8 <\/br>Relative freq.:  0.1231 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  0.6383 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  06 <\/br># assignments:  17 <\/br>Relative freq.:  0.3617 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  64 <\/br>Relative freq.:  0.7356 <\/br>Relative Rep. freq.:  0.0109","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  06 <\/br># assignments:  23 <\/br>Relative freq.:  0.2644 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  05 <\/br># assignments:  45 <\/br>Relative freq.:  0.7258 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  06 <\/br># assignments:  17 <\/br>Relative freq.:  0.2742 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.5714 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  06 <\/br># assignments:  15 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  129 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0175","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.8056 <\/br>Relative Rep. freq.:  0.0133","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  06 <\/br># assignments:  7 <\/br>Relative freq.:  0.1944 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  73 <\/br>Relative freq.:  0.7157 <\/br>Relative Rep. freq.:  0.0123","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  06 <\/br># assignments:  29 <\/br>Relative freq.:  0.2843 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  06 <\/br># assignments:  14 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  06 <\/br># assignments:  8 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  06 <\/br># assignments:  67 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  06 <\/br># assignments:  61 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  0.6579 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  06 <\/br># assignments:  25 <\/br>Relative freq.:  0.3289 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0132 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  63 <\/br>Relative freq.:  0.7 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  06 <\/br># assignments:  27 <\/br>Relative freq.:  0.3 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  0.9804 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0196 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  06 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.8846 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  06 <\/br># assignments:  6 <\/br>Relative freq.:  0.1154 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  01 <\/br># assignments:  58 <\/br>Relative freq.:  0.7436 <\/br>Relative Rep. freq.:  0.0111","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  06 <\/br># assignments:  20 <\/br>Relative freq.:  0.2564 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  0.697 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  10 <\/br>Relative freq.:  0.303 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  79 <\/br>Relative freq.:  0.9875 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0125 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  04 <\/br># assignments:  8 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  06 <\/br># assignments:  20 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  04 <\/br># assignments:  64 <\/br>Relative freq.:  0.9697 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.0303 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  06 <\/br># assignments:  50 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  06 <\/br># assignments:  11 <\/br>Relative freq.:  0.5789 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.4211 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.7059 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  06 <\/br># assignments:  24 <\/br>Relative freq.:  0.2824 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0118 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  06 <\/br># assignments:  18 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  05 <\/br># assignments:  89 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0159","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  21 <\/br>Relative freq.:  0.7241 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  06 <\/br># assignments:  8 <\/br>Relative freq.:  0.2759 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0054"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,4.6],"tickmode":"array","ticktext":["V3-11*01","V3-11*04","V3-11*05","V3-11*06"],"tickvals":[1,2,3,4],"categoryorder":"array","categoryarray":["V3-11*01","V3-11*04","V3-11*05","V3-11*06"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-5.74397488630377,135.707808762817],"tickmode":"array","ticktext":["0","50","100"],"tickvals":[0,50,100],"categoryorder":"array","categoryarray":["0","50","100"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"7d6f79bb2707":{"x":{},"y":{},"text":{},"type":"box"},"7d6f72ca2bf4b":{"x":{},"y":{},"text":{}}},"cur_data":"7d6f79bb2707","visdat":{"7d6f79bb2707":["function (y) ","x"],"7d6f72ca2bf4b":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-11" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 3 zygousity states.</p>
<style>
  
#scroll-box {
  overflow-y: scroll;
  overflow-x: scroll !important;
}
</style>
<div class="container">
<iframe id="scroll-box" src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV3-11G14%22" scrolling="yes" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:800px;overflow-x: scroll !important;overflow: scroll;" cellspacing="0">
</iframe>
</div>
</div>
<div id="observations-13" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/p2mofpyiwe7vd7gob1z1x/IGHV3-11G14.docx?dl=0&amp;rlkey=6u1i4saep6p1e8149fbj6xmq6" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV3-11G14" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-13" class="section level2">
<h2>Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="13-G13.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="15-G15.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/14-G14.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
