<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>17 IGHV3-15 - G16 | Functional groups</title>
  <meta name="description" content="17 IGHV3-15 - G16 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="17 IGHV3-15 - G16 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="17 IGHV3-15 - G16 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="15-G15.html"/>
<link rel="next" href="17-G17.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="04-G4.html"><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>5.6.1</b> Allele specific cutoff: 5% for alleles <strong>01,05</strong> except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="11.6.1" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i><b>11.6.1</b> Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv3-15---g16" class="section level1" number="17">
<h1><span class="header-section-number">17</span> IGHV3-15 - G16</h1>
<div id="allele-appearnce-15" class="section level2" number="17.1">
<h2><span class="header-section-number">17.1</span> Allele appearnce</h2>
<p>The group of IGHV3-15 includes 8 alleles, 8 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-148-1.png" width="672" /></p>
</div>
<div id="group-alignment-13" class="section level2" number="17.2">
<h2><span class="header-section-number">17.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-149-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-15" class="section level2" number="17.3">
<h2><span class="header-section-number">17.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-545c0dfefba5e85f273b" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-545c0dfefba5e85f273b">{"x":{"data":[{"x":[6,1,6,1,6,1,1,6,1,1,1,3,1,6,1,1,3,4,1,2,1,6,1,1,6,1,1,1,6,3,4,1,1,2,1,4,1,3,1,2,4,1,1,6,1,1,1,6,1,1,6,1,1,1,6,1,6,1,1,1,3,4,1,3,1,1,1,1,2,1,2,4,3,1,4,1,6,6,1,1,6,1,2,3,1,6,1,1,4,1,4,1,1,1,4,6,1,2,1,1,4,2,1,6,3,4,1,4,1,1,6,1,1,2,3,4,6,1,6,1,6,1,1,6,1,2,4,1,1,3,1,3,1,4,1,1,2,1,2,1,4,1,2,1,1,4,3,6,1,2,1,2,1,3,1,1,4,3,5,1,6,1,6,1,4,1,4,1,1,2,6,1,1,1,1,2,1,4,1,4,1],"y":[88,70,426,337,302,670,705,97,85,155,434,1,230,193,158,725,1,1,227,1,338,34,24,504,232,179,541,352,324,1,1,129,822,1,136,1,768,1,178,1,1,292,611,1,10,58,290,285,124,129,148,107,227,191,138,555,496,617,353,575,1,1,354,1,174,538,85,598,1,222,2,2,1,843,1,168,1,613,373,335,266,264,1,1,644,343,244,950,1,736,1,238,666,740,1,566,432,1,45,449,6,1,236,216,1,1,313,1,368,457,229,215,761,2,1,1,129,84,205,166,329,308,652,1,882,2,2,685,761,1,658,1,1099,2,537,293,1,553,1,813,1,739,2,497,455,4,1,292,132,2,760,1,213,1,431,844,4,2,1,1231,376,267,319,207,1,890,1,577,290,1,509,310,178,331,32,1,501,2,413,1,36],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[5.92273546410725,0.98600710388273,6.01151001290418,0.934726703213528,6.0586452789139,1.04427691921592,0.955602341890335,5.96574804610573,1.03453136272728,1.01749442717992,0.943972393963486,2.93342377613299,1.01624455722049,6.01907649645582,1.01637809206732,0.910977626219392,2.95968818976544,3.97995820841752,0.925086071807891,2.08643905362114,1.01611886261962,6.00744610526599,0.957004989590496,1.04225657139905,6.0492526280228,0.986294187558815,0.91823982889764,0.963643110357225,5.95956522854976,2.9218796081841,4.07849367042072,0.938499170960858,1.05257957861759,2.05762443114072,0.926234854664653,4.06776593006216,1.01719550676644,3.01791907381266,0.931937513966113,1.91403902401216,4.06211979915388,1.01520738624968,1.09649920077063,5.91879974738695,0.922302002413198,0.927306678937748,1.01955367410555,6.00929978061467,0.944971585599706,0.992968326527625,5.93692159298807,1.07582134120166,1.04450278244913,1.08677297946997,6.08903360441327,0.972868107166141,5.99646843657829,0.964043550891802,1.07519809524529,0.917163243610412,3.02913391618058,4.08655364671722,1.03861679411493,3.02554823057726,0.936723562981933,1.01827770858072,1.0608201559633,0.9232119995635,2.04381877565756,1.05410240874626,2.07803615247831,3.90676169111393,2.9759250943549,0.902989974757656,4.09091381407343,1.03063522661105,6.01441852934659,5.96134977941401,0.938696240261197,1.02688210927881,5.92686263634823,0.901344450609759,2.08413921473548,3.00849736300297,1.0905618282035,6.06487104524858,1.08877005577087,0.943404854414985,3.97419910239987,0.902846497530118,3.9587308682967,0.925442184368148,0.948481859033927,1.06504370537587,4.03119352869689,5.94765158006921,0.935110948886722,1.90192920728587,0.908746988978237,1.0236295230221,4.08733445974067,2.01025352962315,0.96068294136785,6.02165206181817,2.98353456286713,4.06089052041061,0.99386322577484,4.0723263238091,1.00570591441356,1.04575828802772,6.03604494901374,1.03740089652129,0.965641499264166,2.0893596206326,2.94656692454591,4.01705551436171,6.02124455333687,0.992822163319215,5.98139310604893,1.07314740438014,5.93616965110414,0.904362874664366,1.015420601191,6.07602898646146,0.91387020079419,1.94048975924961,3.97397125056013,1.07737144851126,1.08589110877365,3.02835786323994,0.911357419472188,2.97481722924858,0.997189603978768,3.98971852245741,0.980853305058554,0.930073233181611,1.99919695556164,0.906399937160313,1.93465700224042,0.928973031416535,4.06693148934282,1.02977912421338,1.92176840025932,1.03389737564139,1.09000413441099,3.93359204409644,2.99758923677728,5.94715398824774,1.08521799240261,2.01123809576966,0.930007887631655,1.94517521648668,0.939646543934941,2.96656101709232,0.944849458709359,1.039716202626,4.09846454178914,2.90753900166601,5.0991526377853,1.05110384877771,5.93778191851452,0.938992454297841,6.07523175445385,1.09143887721002,4.00708059980534,0.976882004039362,3.98963226545602,0.954388839285821,0.984046510746703,2.09065260365605,6.02450318508781,1.01928299772553,1.08988209329545,1.01946259415708,0.930684363050386,2.07365032010712,1.07122352947481,4.0208006309811,1.05884347017854,4.01921972106211,0.953086571302265],"y":[87.9851467074826,70.0113825049251,425.791942647845,337.19913687706,301.953237772174,670.038728818484,704.985412715003,97.0377274123952,84.6960999710485,155.147709653154,433.883466352522,1.03881021589041,230.120356343873,192.690629779547,157.711744209379,724.971455836482,1.05403540898114,0.799882843717933,226.810915095732,1.33264665231109,337.771565283462,33.9992273038253,24.3937871975824,503.640526897088,231.800152792409,179.135477731749,540.952069213986,351.877324750833,323.925238773786,0.973223520070314,0.682708768174052,128.811166697554,822.051096813381,1.1920185090974,135.693107665889,0.894713507406414,767.82893115636,1.35126984771341,177.616970130801,0.901396979391575,1.13458705842495,291.768053341657,610.723318339698,1.36269332934171,10.2962396325544,58.3147684736177,289.966913702339,284.726109982468,124.372539370693,129.207429204881,147.917798146419,106.834053039365,227.000827628933,191.242824549414,138.137900419906,554.861465010419,495.883936277404,617.005989593454,353.211435288005,575.282781559415,1.21649184785783,0.710130187869072,353.957124601118,0.734741124510765,174.033780782856,538.315717391111,85.2102509139106,598.322769104131,0.600122758932412,222.395319304988,1.95095043517649,2.1523497402668,0.952919592708349,843.018786261603,1.24043036662042,168.277945558168,0.613214615732431,612.659663999826,373.139219835214,335.281040606275,266.243676454015,263.858857252635,1.38244992904365,0.847496627271175,643.762616182864,343.369403541647,244.146630371362,950.226982503571,1.00594644416124,736.007802217826,1.34634487684816,237.611430933326,666.341084702686,739.939624588191,1.23974288832396,566.018501328677,431.808658285066,0.784390072338283,44.8061872124672,448.90684508048,6.39292367398739,1.34599261526018,236.398029549979,216.010715930164,1.18791697528213,1.22013738360256,313.222381461784,1.13606846891344,367.86584576387,456.929108554684,229.205099247769,214.603099630773,761.359823573753,2.08857979141176,1.02764388415962,1.16400690935552,129.339023653977,83.7947831355035,205.223764190637,166.37666366417,329.062765790336,307.976141104288,651.747561358288,1.37381722740829,881.809873151593,2.13984898552299,2.03096351362765,685.281931977533,760.776079602353,1.20216409582645,658.190200047754,1.22588319648057,1098.90735522248,2.11991606280208,536.811867186241,293.231071040034,1.24425920881331,553.028514348529,0.6930789032951,813.246192610078,1.24885316304863,739.03151916787,2.33965334147215,497.24848988466,454.975361710973,3.89140113592148,1.39246732462198,292.028244305402,132.278112416528,2.00714413691312,759.683828978613,1.05552491769195,213.057837379538,1.03889958057553,430.695316314884,843.605556271784,4.10325424503535,2.03947543594986,1.18713988866657,1231.2717366131,375.614789927006,267.14821533896,319.135582382046,207.281948620081,0.839429159834981,890.12056352701,0.807549179717898,577.249879108183,289.604292446747,0.844434792548418,509.244717331603,309.826901639067,178.1711984016,331.177879151516,32.3391347825527,0.995164176821709,501.173263781145,2.25381864737719,413.284108683839,0.97097816132009,35.713330286555],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  07 <\/br># assignments:  88 <\/br>Relative freq.:  0.557 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  70 <\/br>Relative freq.:  0.443 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  07 <\/br># assignments:  426 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0181","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  337 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0314","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  07 <\/br># assignments:  302 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  670 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.028","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  705 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0287","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  07 <\/br># assignments:  97 <\/br>Relative freq.:  0.533 <\/br>Relative Rep. freq.:  0.0161","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  85 <\/br>Relative freq.:  0.467 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  155 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0249","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  434 <\/br>Relative freq.:  0.9977 <\/br>Relative Rep. freq.:  0.0305","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  230 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0408","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  07 <\/br># assignments:  193 <\/br>Relative freq.:  0.5499 <\/br>Relative Rep. freq.:  0.0191","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  158 <\/br>Relative freq.:  0.4501 <\/br>Relative Rep. freq.:  0.0157","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  725 <\/br>Relative freq.:  0.9972 <\/br>Relative Rep. freq.:  0.0326","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  227 <\/br>Relative freq.:  0.9956 <\/br>Relative Rep. freq.:  0.0138","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0044 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  338 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0246","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  07 <\/br># assignments:  34 <\/br>Relative freq.:  0.5862 <\/br>Relative Rep. freq.:  0.0142","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  24 <\/br>Relative freq.:  0.4138 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  504 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  07 <\/br># assignments:  232 <\/br>Relative freq.:  0.5645 <\/br>Relative Rep. freq.:  0.0133","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  179 <\/br>Relative freq.:  0.4355 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  541 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  352 <\/br>Relative freq.:  0.5192 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  07 <\/br># assignments:  324 <\/br>Relative freq.:  0.4779 <\/br>Relative Rep. freq.:  0.0139","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  129 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0123","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  822 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0319","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  136 <\/br>Relative freq.:  0.9927 <\/br>Relative Rep. freq.:  0.0344","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0073 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  768 <\/br>Relative freq.:  0.9987 <\/br>Relative Rep. freq.:  0.0341","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  178 <\/br>Relative freq.:  0.9889 <\/br>Relative Rep. freq.:  0.0142","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0056 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0056 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  292 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  611 <\/br>Relative freq.:  0.9984 <\/br>Relative Rep. freq.:  0.0245","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  10 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0285","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  58 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0324","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  290 <\/br>Relative freq.:  0.5043 <\/br>Relative Rep. freq.:  0.0139","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  07 <\/br># assignments:  285 <\/br>Relative freq.:  0.4957 <\/br>Relative Rep. freq.:  0.0136","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  124 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0225","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  129 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  07 <\/br># assignments:  148 <\/br>Relative freq.:  0.5804 <\/br>Relative Rep. freq.:  0.013","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  107 <\/br>Relative freq.:  0.4196 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  227 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0277","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  191 <\/br>Relative freq.:  0.5805 <\/br>Relative Rep. freq.:  0.0104","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  07 <\/br># assignments:  138 <\/br>Relative freq.:  0.4195 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  555 <\/br>Relative freq.:  0.5281 <\/br>Relative Rep. freq.:  0.0195","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  07 <\/br># assignments:  496 <\/br>Relative freq.:  0.4719 <\/br>Relative Rep. freq.:  0.0175","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  617 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0253","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  353 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0202","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  575 <\/br>Relative freq.:  0.9965 <\/br>Relative Rep. freq.:  0.0292","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  354 <\/br>Relative freq.:  0.9972 <\/br>Relative Rep. freq.:  0.0304","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  174 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0283","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  538 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0362","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  85 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0257","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  598 <\/br>Relative freq.:  0.9983 <\/br>Relative Rep. freq.:  0.0272","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  222 <\/br>Relative freq.:  0.978 <\/br>Relative Rep. freq.:  0.0219","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0088 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0088 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0044 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  843 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0228","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  168 <\/br>Relative freq.:  0.9941 <\/br>Relative Rep. freq.:  0.0265","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0059 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  07 <\/br># assignments:  613 <\/br>Relative freq.:  0.6217 <\/br>Relative Rep. freq.:  0.0224","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  373 <\/br>Relative freq.:  0.3783 <\/br>Relative Rep. freq.:  0.0137","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  335 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0365","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  07 <\/br># assignments:  266 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  264 <\/br>Relative freq.:  0.4962 <\/br>Relative Rep. freq.:  0.0118","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  644 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.03","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  07 <\/br># assignments:  343 <\/br>Relative freq.:  0.5843 <\/br>Relative Rep. freq.:  0.0186","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  01 <\/br># assignments:  244 <\/br>Relative freq.:  0.4157 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  950 <\/br>Relative freq.:  0.9989 <\/br>Relative Rep. freq.:  0.0484","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  736 <\/br>Relative freq.:  0.9986 <\/br>Relative Rep. freq.:  0.0304","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  238 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0314","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  666 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0244","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  740 <\/br>Relative freq.:  0.9987 <\/br>Relative Rep. freq.:  0.0232","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  07 <\/br># assignments:  566 <\/br>Relative freq.:  0.5666 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  432 <\/br>Relative freq.:  0.4324 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  45 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0334","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  449 <\/br>Relative freq.:  0.9846 <\/br>Relative Rep. freq.:  0.0209","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  05 <\/br># assignments:  6 <\/br>Relative freq.:  0.0132 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  236 <\/br>Relative freq.:  0.5198 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  07 <\/br># assignments:  216 <\/br>Relative freq.:  0.4758 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  313 <\/br>Relative freq.:  0.9968 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0032 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  368 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0196","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  457 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0232","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  07 <\/br># assignments:  229 <\/br>Relative freq.:  0.5158 <\/br>Relative Rep. freq.:  0.0185","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  215 <\/br>Relative freq.:  0.4842 <\/br>Relative Rep. freq.:  0.0174","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  761 <\/br>Relative freq.:  0.9948 <\/br>Relative Rep. freq.:  0.0225","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0026 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  07 <\/br># assignments:  129 <\/br>Relative freq.:  0.6056 <\/br>Relative Rep. freq.:  0.0169","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  84 <\/br>Relative freq.:  0.3944 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  07 <\/br># assignments:  205 <\/br>Relative freq.:  0.5526 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  166 <\/br>Relative freq.:  0.4474 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  07 <\/br># assignments:  329 <\/br>Relative freq.:  0.5165 <\/br>Relative Rep. freq.:  0.0121","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  308 <\/br>Relative freq.:  0.4835 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  652 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  882 <\/br>Relative freq.:  0.9955 <\/br>Relative Rep. freq.:  0.0275","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  685 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0381","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  761 <\/br>Relative freq.:  0.9987 <\/br>Relative Rep. freq.:  0.0305","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  658 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0376","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  01 <\/br># assignments:  1099 <\/br>Relative freq.:  0.9982 <\/br>Relative Rep. freq.:  0.0461","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  537 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0289","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  293 <\/br>Relative freq.:  0.9966 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0034 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  553 <\/br>Relative freq.:  0.9982 <\/br>Relative Rep. freq.:  0.0298","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  813 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0248","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  01 <\/br># assignments:  739 <\/br>Relative freq.:  0.9973 <\/br>Relative Rep. freq.:  0.0247","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  497 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0216","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  455 <\/br>Relative freq.:  0.9891 <\/br>Relative Rep. freq.:  0.0329","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  05 <\/br># assignments:  4 <\/br>Relative freq.:  0.0087 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  07 <\/br># assignments:  292 <\/br>Relative freq.:  0.6854 <\/br>Relative Rep. freq.:  0.0202","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  132 <\/br>Relative freq.:  0.3099 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  01 <\/br># assignments:  760 <\/br>Relative freq.:  0.9987 <\/br>Relative Rep. freq.:  0.0319","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  213 <\/br>Relative freq.:  0.9953 <\/br>Relative Rep. freq.:  0.0229","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  431 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0229","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  844 <\/br>Relative freq.:  0.9918 <\/br>Relative Rep. freq.:  0.0296","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  05 <\/br># assignments:  4 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  1231 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0377","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  07 <\/br># assignments:  376 <\/br>Relative freq.:  0.5848 <\/br>Relative Rep. freq.:  0.016","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  01 <\/br># assignments:  267 <\/br>Relative freq.:  0.4152 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  07 <\/br># assignments:  319 <\/br>Relative freq.:  0.6053 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  207 <\/br>Relative freq.:  0.3928 <\/br>Relative Rep. freq.:  0.0129","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  890 <\/br>Relative freq.:  0.9989 <\/br>Relative Rep. freq.:  0.0285","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  577 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0244","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  290 <\/br>Relative freq.:  0.9966 <\/br>Relative Rep. freq.:  0.0296","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0034 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  07 <\/br># assignments:  509 <\/br>Relative freq.:  0.6215 <\/br>Relative Rep. freq.:  0.0217","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  310 <\/br>Relative freq.:  0.3785 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  178 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  331 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0261","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  32 <\/br>Relative freq.:  0.9697 <\/br>Relative Rep. freq.:  0.0245","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0303 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  501 <\/br>Relative freq.:  0.996 <\/br>Relative Rep. freq.:  0.0212","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  413 <\/br>Relative freq.:  0.9976 <\/br>Relative Rep. freq.:  0.0239","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0185"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["V3-15*01","V3-15*02","V3-15*04","V3-15*05","V3-15*06","V3-15*07"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["V3-15*01","V3-15*02","V3-15*04","V3-15*05","V3-15*06","V3-15*07"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-60.9334579337761,1292.80531730581],"tickmode":"array","ticktext":["0","250","500","750","1000","1250"],"tickvals":[-7.105427357601e-15,250,500,750,1000,1250],"categoryorder":"array","categoryarray":["0","250","500","750","1000","1250"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"56a766787bc79":{"x":{},"y":{},"text":{},"type":"box"},"56a764fedab9a":{"x":{},"y":{},"text":{}}},"cur_data":"56a766787bc79","visdat":{"56a766787bc79":["function (y) ","x"],"56a764fedab9a":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-9a52504a6ca8b9e6c733" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-9a52504a6ca8b9e6c733">{"x":{"data":[{"x":[1,1,5,4,1,1,1,1,3,1,1,1,1,5,1,1,3,1,1,1,1,1,5,1,1,3,4,1,5,1,2,1,5,1,4,1,1,1,1,5,5,1,1,1,1,1,1,1,1,1,1,1,4,1,1,1,1],"y":[50,9,9,1,22,15,69,103,1,37,68,115,32,65,56,57,1,150,143,72,109,12,9,40,138,2,2,67,26,16,1,23,15,130,1,41,88,109,75,80,44,43,47,56,108,19,22,181,79,3,23,117,1,51,77,59,41],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(111,145,106,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.06802194034681,0.923874489078298,5.03449091934599,3.91667656330392,0.926695103291422,0.935161189967766,0.98354051830247,1.0844089739956,3.04076484846883,0.953072289703414,1.06045366744511,0.948029123432934,1.09187832889147,4.99865330336615,0.979618663573638,0.996308440249413,2.9180311458651,0.948908410267904,1.07273161839694,1.05336736612953,1.03244338021614,1.02707267315127,4.90196547303349,1.08886284222826,0.927795859752223,2.91855648588389,4.06422386337072,0.928714788239449,4.90058256057091,1.02133038658649,2.06323903920129,0.935608756542206,5.03778836382553,0.99495853674598,3.91500305114314,0.992276958515868,0.965218020835891,0.972811286430806,1.03245147392154,5.03257924141362,5.07931702989154,0.940418373048306,1.09582039969973,0.904317045165226,0.99320606472902,1.04062289139256,1.02561828764156,1.08470871858299,1.09924865057692,1.01050408501178,0.988392568798736,0.90302423350513,4.0600692180451,1.08372904248536,0.913457340979949,1.07572598871775,1.00551210022531],"y":[49.7986301075667,9.1520782308653,9.22700793407857,0.838213842362165,21.6265237024054,15.0281029133126,69.1315283011645,103.216609416343,1.07173078041524,37.1382486144081,67.6324867075309,115.063847549818,32.3689062135294,65.2444058122113,56.062026132457,57.2587420320138,0.733678610250354,150.207630840875,142.809955725633,71.8192590171471,109.145657351986,12.0631918558851,9.08917006291449,39.7849007476121,137.77927671317,1.96667259670794,1.65020614527166,67.2148966440931,26.2861302768812,15.6489788005129,0.67284912597388,22.7297570837662,15.2513867938891,129.654574109241,1.03099721912295,41.0692589454353,87.7822631528601,109.380842088349,75.1606724658981,79.7010354815051,43.996049856022,42.6713844589889,46.7175007702783,55.9418622581288,108.271415050514,19.3484373509884,22.2172282515094,181.274322261475,79.0375009702519,2.97680818587542,23.289475629665,117.347423992492,1.15792648587376,50.9162260543555,77.1539329364896,59.3189761977643,41.378457121551],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0085","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  0.4737 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  07 <\/br># assignments:  9 <\/br>Relative freq.:  0.4737 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0526 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  22 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  15 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  69 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  103 <\/br>Relative freq.:  0.9904 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0096 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  37 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  68 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  115 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0202","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  32 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  07 <\/br># assignments:  65 <\/br>Relative freq.:  0.5372 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  56 <\/br>Relative freq.:  0.4628 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  0.9828 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0172 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  150 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0248","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  143 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.036","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  72 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  109 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0138","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.5714 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  07 <\/br># assignments:  9 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  40 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0177","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  138 <\/br>Relative freq.:  0.9718 <\/br>Relative Rep. freq.:  0.0195","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0141 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0141 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  67 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  07 <\/br># assignments:  26 <\/br>Relative freq.:  0.6047 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  0.3721 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0233 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  0.6053 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  07 <\/br># assignments:  15 <\/br>Relative freq.:  0.3947 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  130 <\/br>Relative freq.:  0.9924 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0076 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  88 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0099","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  109 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  75 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0134","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  07 <\/br># assignments:  80 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  07 <\/br># assignments:  44 <\/br>Relative freq.:  0.5057 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  43 <\/br>Relative freq.:  0.4943 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  01 <\/br># assignments:  47 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  56 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  108 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  22 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  181 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  79 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0181","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  117 <\/br>Relative freq.:  0.9915 <\/br>Relative Rep. freq.:  0.0138","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0085 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  77 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0126","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  59 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0055"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(111,145,106,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(111,145,106,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["V3-15*01","V3-15*02","V3-15*04","V3-15*05","V3-15*07"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["V3-15*01","V3-15*02","V3-15*04","V3-15*05","V3-15*07"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-8.35722453080118,190.30439591825],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[0,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"56a7617f0841f":{"x":{},"y":{},"text":{},"type":"box"},"56a7638236684":{"x":{},"y":{},"text":{}}},"cur_data":"56a7617f0841f","visdat":{"56a7617f0841f":["function (y) ","x"],"56a7638236684":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-2e4da2645950e46c3e23" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-2e4da2645950e46c3e23">{"x":{"data":[{"x":[1,2,4,3,6,6,1,4,2,3,6,6,1,3,2,4,1,3,6,2,4,1,4,2,3,6,1,2,4,3,6,1,4,2,3,1,2,4,3,5,1,4,2,3,6,5,1,4,2,3,6,1,4,2,3,6,6,1,2,4,1,4,6,1,3,4,1,4,2,3,1,4,3,2,5,6,1,2,6,3,4,1,4,3,2,6,1,4,3,2,1,6,2,4,1,4,2,1,4,2,3,5,6,1,2,4,1,4,6,2,1,2,3,4,1,2,4,1,4,2,3],"y":[177,4,4,3,3,270,202,7,2,2,2,276,217,8,3,1,206,3,3,2,1,283,9,3,1,1,221,9,9,1,1,313,13,6,2,101,5,4,1,1,336,16,6,6,2,1,140,27,6,3,1,146,15,9,6,2,341,183,13,6,52,2,184,67,2,1,3378,42,9,6,222,25,12,6,6,1,279,4,4,2,2,446,14,7,1,1,501,11,5,1,361,359,1,1,475,190,79,300,8,5,2,2,1,46,3,3,82,10,4,3,270,6,2,2,184,11,6,173,10,6,2],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(45,102,128,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.02599039352499,1.96451426455751,3.93589935516939,2.96836525653489,5.93393293023109,6.05468436800875,0.936910318210721,4.04361144606955,1.95952906850725,3.07523253643885,6.01703014150262,6.0443847423885,0.945417388482019,3.08739821561612,1.92954060495831,3.94211511425674,0.921622756123543,2.99389419248328,6.0732927557081,1.97463551345281,4.06109959129244,1.02231770469807,3.90226451824419,1.9621891812887,3.02415532176383,5.97276565628126,1.01353264045902,2.01310089170001,3.96074410625733,3.02848852872849,5.98802452445962,0.992860393133014,3.97773152515292,2.01119191669859,3.01996755944565,1.04104732074775,2.06539636082016,3.92208401774988,2.94659477556124,4.91407583910041,0.924112865421921,4.08632933069021,2.07690618159249,3.06693469765596,6.02715343968011,4.97355952444486,0.946058736043051,4.01563074178994,1.90381613350473,2.93548318427056,5.90926296617836,0.990612048050389,4.08318612035364,1.9094515749719,3.0829379520379,5.98513316684403,5.93589555365033,1.05380837656558,2.09557943358086,3.95123210344464,1.01390766948462,4.05714834523387,6.06650387910195,0.913352360157296,2.91661964342929,4.05994590530172,0.958961720531806,4.00969908214174,2.02287587062456,2.97677307943813,1.07913660928607,3.98272004635073,3.05433703567833,1.96531127695926,5.05769775244407,5.96559757883661,1.06082292464562,2.05121530271135,6.07338744057342,3.05600140714087,3.95016933595762,1.04915096429177,3.93089664443396,3.00025436552241,1.90417414880358,6.06073960848153,0.972458734130487,4.02077330769971,3.06570211062208,1.97723961533047,0.922715851990506,6.07469502999447,2.08123367149383,3.982509472128,1.09729726631194,3.99259981680661,2.01225657858886,0.909144739480689,4.05136253871024,1.94388103103265,3.0210494840052,5.00947510232218,5.90190171007998,1.08535481016152,1.97564515336417,4.07088088751771,1.05982726137154,3.94259112603031,6.02069942820817,1.97774979099631,1.01388559234329,2.02277912572026,2.91442481377162,4.03067681859247,0.903203850146383,2.08135373792611,3.92632226510905,1.08441770877689,4.03606162825599,2.01325369076803,3.05262855608016],"y":[177.361627036333,4.26194470729679,4.23238683082163,2.66266429852694,3.3691629813984,270.35159975253,202.016195777431,7.06189933363348,1.64023336879909,1.81473809927702,1.64822618179023,275.729812141322,217.265167245828,8.38016107305884,2.95888083912432,1.19419929068536,205.969780927896,3.23486003614962,2.76868602428585,1.83129814323038,0.903190930746496,283.335800504684,9.07428797036409,2.85385414659977,0.641166186146438,0.960548048280179,220.616219008714,9.29605414029211,9.26935617290437,0.61218055896461,1.31956931892782,313.064877481386,12.9951630908996,6.23000932503492,1.99883874859661,101.114171512611,5.04474640116096,4.04119633939117,1.07414598893374,1.36796826310456,336.269333904982,16.3421478902921,6.0129953302443,6.26879514697939,1.98554106801748,0.930169674009085,139.796551520191,27.07219681032,5.89939852133393,3.32360677253455,0.790355476178229,145.813659900799,15.322767210193,9.0443740343675,6.13692267183214,2.06086768433452,340.641501916945,182.967185176536,13.1918353848159,6.08263310659677,52.2255625931546,2.12459061276168,184.333627751283,67.2613963363692,2.06406976152211,1.28297971505672,3378.13251350597,41.7592255137861,9.04377101603895,6.00326715707779,222.287385393307,24.8693205382675,11.9288498053327,6.30926567818969,5.73725756369531,0.711637242138386,278.7365777798,3.68513910304755,3.63301866929978,1.80431350357831,1.84354186877608,445.886519281939,13.8149130264297,6.94959290400147,0.972724422253668,0.626652819477022,501.358333706669,10.665546426177,5.37195377852768,0.639023997634649,361.014432387613,359.36156673003,0.718495850078761,1.2033093046397,474.668068420328,189.771416987851,78.9365315014497,299.707793309167,7.62067531961948,5.12537402883172,1.65360053367913,1.71127828210592,1.15890414901078,45.8354047821835,2.6853863729164,3.24985190723091,81.6090419154614,9.74343745540827,4.06961937174201,3.26340674608946,269.604439117573,6.10785215198994,2.00457378122956,2.09341680891812,183.887459560297,11.1755701841786,5.8727708177641,172.850070361048,10.2170190921053,5.63663350287825,2.23684439938515],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  177 <\/br>Relative freq.:  0.9267 <\/br>Relative Rep. freq.:  0.0155","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0209 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  05 <\/br># assignments:  4 <\/br>Relative freq.:  0.0209 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.0157 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  07 <\/br># assignments:  3 <\/br>Relative freq.:  0.0157 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  07 <\/br># assignments:  270 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0718","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  202 <\/br>Relative freq.:  0.9395 <\/br>Relative Rep. freq.:  0.018","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  05 <\/br># assignments:  7 <\/br>Relative freq.:  0.0326 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0093 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0093 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  07 <\/br># assignments:  2 <\/br>Relative freq.:  0.0093 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  07 <\/br># assignments:  276 <\/br>Relative freq.:  0.5465 <\/br>Relative Rep. freq.:  0.0182","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  217 <\/br>Relative freq.:  0.4297 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  8 <\/br>Relative freq.:  0.0158 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0059 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  206 <\/br>Relative freq.:  0.9581 <\/br>Relative Rep. freq.:  0.0393","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.014 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  07 <\/br># assignments:  3 <\/br>Relative freq.:  0.014 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0093 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  283 <\/br>Relative freq.:  0.9529 <\/br>Relative Rep. freq.:  0.0165","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  05 <\/br># assignments:  9 <\/br>Relative freq.:  0.0303 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0101 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0034 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0034 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  221 <\/br>Relative freq.:  0.917 <\/br>Relative Rep. freq.:  0.0186","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  0.0373 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  05 <\/br># assignments:  9 <\/br>Relative freq.:  0.0373 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0041 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0041 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  313 <\/br>Relative freq.:  0.9371 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  05 <\/br># assignments:  13 <\/br>Relative freq.:  0.0389 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.018 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.006 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  101 <\/br>Relative freq.:  0.9018 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0446 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  05 <\/br># assignments:  4 <\/br>Relative freq.:  0.0357 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0089 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0089 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  336 <\/br>Relative freq.:  0.9155 <\/br>Relative Rep. freq.:  0.0202","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  05 <\/br># assignments:  16 <\/br>Relative freq.:  0.0436 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0163 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  6 <\/br>Relative freq.:  0.0163 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  07 <\/br># assignments:  2 <\/br>Relative freq.:  0.0054 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  140 <\/br>Relative freq.:  0.791 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  05 <\/br># assignments:  27 <\/br>Relative freq.:  0.1525 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0339 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.0169 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0056 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  146 <\/br>Relative freq.:  0.8202 <\/br>Relative Rep. freq.:  0.0129","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  05 <\/br># assignments:  15 <\/br>Relative freq.:  0.0843 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  0.0506 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  6 <\/br>Relative freq.:  0.0337 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  07 <\/br># assignments:  2 <\/br>Relative freq.:  0.0112 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  07 <\/br># assignments:  341 <\/br>Relative freq.:  0.628 <\/br>Relative Rep. freq.:  0.0191","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  183 <\/br>Relative freq.:  0.337 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  13 <\/br>Relative freq.:  0.0239 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  05 <\/br># assignments:  6 <\/br>Relative freq.:  0.011 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  52 <\/br>Relative freq.:  0.963 <\/br>Relative Rep. freq.:  0.0133","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.037 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  07 <\/br># assignments:  184 <\/br>Relative freq.:  0.7244 <\/br>Relative Rep. freq.:  0.0274","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  67 <\/br>Relative freq.:  0.2638 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0079 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0039 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  3378 <\/br>Relative freq.:  0.9834 <\/br>Relative Rep. freq.:  0.3675","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  05 <\/br># assignments:  42 <\/br>Relative freq.:  0.0122 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  0.0026 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  6 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  222 <\/br>Relative freq.:  0.8162 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  05 <\/br># assignments:  25 <\/br>Relative freq.:  0.0919 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  12 <\/br>Relative freq.:  0.0441 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0221 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  06 <\/br># assignments:  6 <\/br>Relative freq.:  0.0221 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0037 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  279 <\/br>Relative freq.:  0.9588 <\/br>Relative Rep. freq.:  0.0184","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0137 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  07 <\/br># assignments:  4 <\/br>Relative freq.:  0.0137 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0069 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0069 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  446 <\/br>Relative freq.:  0.951 <\/br>Relative Rep. freq.:  0.0238","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  05 <\/br># assignments:  14 <\/br>Relative freq.:  0.0299 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  7 <\/br>Relative freq.:  0.0149 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  501 <\/br>Relative freq.:  0.9672 <\/br>Relative Rep. freq.:  0.02","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  05 <\/br># assignments:  11 <\/br>Relative freq.:  0.0212 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  5 <\/br>Relative freq.:  0.0097 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  361 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  07 <\/br># assignments:  359 <\/br>Relative freq.:  0.4972 <\/br>Relative Rep. freq.:  0.0161","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  475 <\/br>Relative freq.:  0.6384 <\/br>Relative Rep. freq.:  0.0305","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  05 <\/br># assignments:  190 <\/br>Relative freq.:  0.2554 <\/br>Relative Rep. freq.:  0.0122","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  02 <\/br># assignments:  79 <\/br>Relative freq.:  0.1062 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  300 <\/br>Relative freq.:  0.9434 <\/br>Relative Rep. freq.:  0.0255","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  05 <\/br># assignments:  8 <\/br>Relative freq.:  0.0252 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0157 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0063 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.0063 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0031 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.8846 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0577 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  05 <\/br># assignments:  3 <\/br>Relative freq.:  0.0577 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  82 <\/br>Relative freq.:  0.8283 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  05 <\/br># assignments:  10 <\/br>Relative freq.:  0.101 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  07 <\/br># assignments:  4 <\/br>Relative freq.:  0.0404 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0303 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  270 <\/br>Relative freq.:  0.9643 <\/br>Relative Rep. freq.:  0.0183","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0214 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0071 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0071 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  184 <\/br>Relative freq.:  0.9154 <\/br>Relative Rep. freq.:  0.0224","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  11 <\/br>Relative freq.:  0.0547 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  05 <\/br># assignments:  6 <\/br>Relative freq.:  0.0299 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  173 <\/br>Relative freq.:  0.9058 <\/br>Relative Rep. freq.:  0.0155","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  05 <\/br># assignments:  10 <\/br>Relative freq.:  0.0524 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0314 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0105 <\/br>Relative Rep. freq.:  2e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(45,102,128,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(45,102,128,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["V3-15*01","V3-15*02","V3-15*04","V3-15*05","V3-15*06","V3-15*07"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["V3-15*01","V3-15*02","V3-15*04","V3-15*05","V3-15*06","V3-15*07"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-168.263836088385,3547.00853015332],"tickmode":"array","ticktext":["0","1000","2000","3000"],"tickvals":[0,1000,2000,3000],"categoryorder":"array","categoryarray":["0","1000","2000","3000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"56a767a56a0e3":{"x":{},"y":{},"text":{},"type":"box"},"56a769ca12cd":{"x":{},"y":{},"text":{}}},"cur_data":"56a767a56a0e3","visdat":{"56a767a56a0e3":["function (y) ","x"],"56a769ca12cd":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-13" class="section level2" number="17.4">
<h2><span class="header-section-number">17.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 9 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV3-15G16%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:900px;" cellspacing="0">
</iframe>
</div>
<div id="observations-15" class="section level2" number="17.5">
<h2><span class="header-section-number">17.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/pq392rgqrsu6e5h659cna/IGHV3-15G16.docx?dl=0&amp;rlkey=l45an65j4b4k3w3rdsherchty" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV3-15G16" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-15" class="section level2" number="17.6">
<h2><span class="header-section-number">17.6</span> Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="15-G15.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="17-G17.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/16-G16.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
