<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>20 IGHV3-23/IGHV3-23D - G19 | Functional groups</title>
  <meta name="description" content="20 IGHV3-23/IGHV3-23D - G19 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="20 IGHV3-23/IGHV3-23D - G19 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="20 IGHV3-23/IGHV3-23D - G19 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="18-G18.html"/>
<link rel="next" href="20-G20.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="04-G4.html"><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>5.6.1</b> Allele specific cutoff: 5% for alleles <strong>01,05</strong> except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="11.6.1" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i><b>11.6.1</b> Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv3-23ighv3-23d---g19" class="section level1" number="20">
<h1><span class="header-section-number">20</span> IGHV3-23/IGHV3-23D - G19</h1>
<div id="allele-appearnce-18" class="section level2" number="20.1">
<h2><span class="header-section-number">20.1</span> Allele appearnce</h2>
<p>The group of IGHV3-23/IGHV3-23D includes 6 alleles, 6 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-175-1.png" width="672" /></p>
</div>
<div id="group-alignment-16" class="section level2" number="20.2">
<h2><span class="header-section-number">20.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-176-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-18" class="section level2" number="20.3">
<h2><span class="header-section-number">20.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-5aa55d1a25da401151e7" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-5aa55d1a25da401151e7">{"x":{"data":[{"x":[4,2,3,4,4,2,4,4,4,4,4,4,4,2,4,2,4,4,2,4,4,4,2,4,4,4,4,4,4,2,4,2,4,3,4,4,4,4,2,4,3,4,2,4,4,4,3,4,4,4,2,4,4,4,2,2,4,4,2,4,2,4,2,4,3,4,4,2,4,4,4,2,4,2,2,4,2,4,4,4,4,4,4,2,4,4,1,2,4,2,3,4,2,4,4,4,4,2,4,4,2,4,2,4,4,4,2,4,4,3,4,4,4,4,2,4,2,4,2,4,4,4,4,4,4,2,4,2,4,2,4,4,4,2,4,4,2,4,4,4,4,2,4,3,4],"y":[1481,405,1,1079,1304,1,1381,2472,3223,1119,616,1861,473,317,1273,1,2667,2262,1,1391,353,2577,1,1702,1622,2629,821,3292,349,213,2366,908,880,1,1309,2980,28,254,1,2450,1,847,1,985,882,1187,1,1530,3936,1720,676,672,2292,975,496,327,264,749,592,423,2,3314,1,1231,3,3927,1100,3,4090,657,1241,719,1171,647,692,545,785,599,2457,1523,1544,5130,3341,1,166,1910,2,1,3457,2,1,1209,467,1725,1394,1413,2673,2,1026,2235,1,1668,1122,2509,5013,969,603,3116,1256,1,2052,2920,608,1124,671,3009,1769,2576,2,1427,1885,2501,896,1970,1085,854,4129,1,2762,1,1499,2325,1378,826,1516,1762,628,1766,2072,164,2851,1,2292,2,260],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[4.0568166592624,1.91991002210416,2.91712635578588,4.0070362876635,3.96512606940232,1.90061157243326,4.09662380651571,3.95186951728538,4.08380457013845,3.98390394612215,3.90205548265949,3.9428724156227,4.07164208195172,1.91418892354704,4.06777501385659,2.02408022768795,4.00358948907815,4.06672013290226,1.99086166713387,4.06696711727418,3.91716757882386,4.0941004535649,2.01067145769484,3.95216777250171,4.09478742973879,4.01261465330608,3.91415337417275,4.01577072157525,3.92830823902041,2.02788271554746,4.07922722683288,2.03683000775054,3.9582223592326,2.91040274435654,3.90068345563486,3.99848529240116,3.96194565529004,3.91685076453723,2.07204455793835,4.03990787719376,2.96488559879363,3.96735641616397,1.93871782859787,4.09319772403687,3.91921104546636,4.074176575616,3.008823887771,4.06347918985412,4.05245168353431,3.94565594219603,2.08422759119421,4.02194524290971,4.07710314677097,4.01811195504852,2.05085907666944,2.03936872989871,4.00196711486205,3.95442604590207,1.98617922589183,4.01707486300729,2.02228705403395,4.04465921246447,2.08838147632778,4.03300512772985,3.09970796857961,4.02828636574559,3.96578090535477,1.90736339669675,4.08663360518403,3.93669199105352,4.00427079480141,2.07958628972992,3.92414219090715,1.9594744425267,2.08499296936207,4.02145242448896,1.98987398035824,4.07919691763818,3.90696034864523,4.01763702882454,3.93455692110583,4.04093801607378,3.9657422198914,1.99748201649636,3.99224661942571,3.92179056592286,0.910509972367436,1.95332682840526,4.05495722549967,1.98947177585214,2.95905856960453,3.97348115248606,1.95392155898735,3.91744874720462,3.94658692018129,3.91989386603236,4.02101978980936,2.0420287149027,3.92443055864423,3.98835520693101,2.01922994521447,4.0613543251995,1.92924372600392,4.05739167057909,3.90435447595082,4.08208935977891,2.05422453023493,4.05885738935322,4.06034909766167,2.95138371540234,3.93719116179273,4.03927998831496,4.0219290623907,3.9419449146837,1.98349765008315,4.02056879680604,2.01631294256076,3.93161998121068,2.0522942266427,4.03202048442327,3.90121072768234,4.03951853010803,3.99650473459624,4.092753161164,3.99116552937776,1.91767441728152,3.99602794810198,1.97090338836424,3.94224101584405,1.99221926503815,3.92934494852088,4.04662293386646,4.02625021045096,2.08636733596213,4.09833929454908,4.06028710496612,2.08457139832899,4.02018622215837,4.05741063007154,4.08543393681757,3.99884383752942,2.00048792292364,4.01041977494024,3.06461729565635,4.05436125374399],"y":[1480.72849280443,404.769815614074,1.16532825566828,1078.79387210608,1303.85305711813,0.924284147657454,1380.62446182482,2472.12428350896,3223.0367895443,1119.14648086727,615.623647757992,1861.38850803655,472.934065151587,316.702888208628,1272.85597678348,0.723529130034149,2667.13585377969,2261.83165066689,0.656346861459315,1390.68414766416,352.777403289266,2576.98680147734,0.644267258606851,1701.64002228621,1621.91376477834,2629.1696634125,820.937982217781,3291.86747398879,349.290912231617,212.841787444241,2366.05895225592,908.250498106703,879.752227460966,1.11047143861651,1309.26812508628,2980.28250194769,28.3991716958582,254.261011675186,0.899475596472621,2450.00249895044,1.20233062021434,846.602821207978,1.36268453486264,984.651733994856,882.194696383365,1187.19002030753,0.767773798108101,1530.02398517262,3936.2608912712,1720.3677300524,675.748485149816,672.169967706129,2292.20898511335,975.11386105381,496.061855224334,326.949164293148,264.351151792333,749.318064120971,592.273309351131,423.362132886983,2.0439940655604,3313.60466005895,1.23082529529929,1230.81962892208,3.06042360533029,3927.22162251379,1100.09163961746,3.2256465325132,4090.3345234042,657.296047128923,1240.98419416491,719.000750114024,1170.99364233334,646.893658280559,692.036801110953,545.327259825543,784.988563963771,598.839643936977,2457.35171111897,1522.91850712858,1544.07338144034,5129.81076340005,3341.30129744876,1.34928371999413,165.827299300022,1909.88836956602,1.89812361821532,1.20162641480565,3457.24108951595,1.76385904382914,1.03631757553667,1209.17343301531,466.611870403588,1725.0656715719,1393.77783892807,1412.86618202068,2672.94143103305,2.00928773656487,1026.27253115568,2234.75696285684,0.902053902484477,1668.21975651048,1121.95171867181,2508.77704245821,5012.62705535106,968.931297731586,602.77636438217,3116.34489777051,1256.19189264216,1.36691433768719,2051.87645923328,2920.07539918479,607.60195665285,1124.34559138101,670.88782955762,3008.69067772441,1769.2453111928,2575.9894742934,2.0150015918538,1427.33944720794,1885.19439618755,2500.76163701601,896.217922988348,1970.27313074525,1085.2307434611,853.69670314379,4128.67398210373,0.957353856600821,2762.08935787529,1.23088137283921,1498.93846496399,2325.11076254789,1378.38322472516,825.97926991079,1516.1110632807,1761.90350278709,628.298609042913,1765.7215455059,2072.1342993537,164.221287247352,2851.00850293152,0.949194914102554,2291.94717679229,1.90963189154863,259.928415234573],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  1481 <\/br>Relative freq.:  0.7848 <\/br>Relative Rep. freq.:  0.1083","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  405 <\/br>Relative freq.:  0.2146 <\/br>Relative Rep. freq.:  0.0296","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  5e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  1079 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0458","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  1304 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.1215","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  1381 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0579","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  2472 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1035","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  3223 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1311","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  1119 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1855","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  616 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0991","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  1861 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1308","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  473 <\/br>Relative freq.:  0.5987 <\/br>Relative Rep. freq.:  0.0838","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  317 <\/br>Relative freq.:  0.4013 <\/br>Relative Rep. freq.:  0.0562","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  1273 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.1261","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  2667 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1201","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  2262 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.1372","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  1391 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1013","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  353 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1475","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  2577 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.0956","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  1702 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0972","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  1622 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0723","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  2629 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1125","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  821 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0783","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  3292 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1277","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  349 <\/br>Relative freq.:  0.621 <\/br>Relative Rep. freq.:  0.0882","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  213 <\/br>Relative freq.:  0.379 <\/br>Relative Rep. freq.:  0.0538","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  2366 <\/br>Relative freq.:  0.7227 <\/br>Relative Rep. freq.:  0.1051","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  908 <\/br>Relative freq.:  0.2773 <\/br>Relative Rep. freq.:  0.0404","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  880 <\/br>Relative freq.:  0.9989 <\/br>Relative Rep. freq.:  0.0703","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  1309 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0639","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  2980 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1193","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0798","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  254 <\/br>Relative freq.:  0.9961 <\/br>Relative Rep. freq.:  0.1419","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0039 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  2450 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.1172","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  847 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.1535","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  985 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1299","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  882 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0774","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  1187 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.1447","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  1530 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0833","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  3936 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1385","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  1720 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0706","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  04 <\/br># assignments:  676 <\/br>Relative freq.:  0.5015 <\/br>Relative Rep. freq.:  0.0387","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  672 <\/br>Relative freq.:  0.4985 <\/br>Relative Rep. freq.:  0.0385","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  2292 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1164","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  975 <\/br>Relative freq.:  0.6628 <\/br>Relative Rep. freq.:  0.0838","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  04 <\/br># assignments:  496 <\/br>Relative freq.:  0.3372 <\/br>Relative Rep. freq.:  0.0426","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  04 <\/br># assignments:  327 <\/br>Relative freq.:  0.5533 <\/br>Relative Rep. freq.:  0.0532","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  264 <\/br>Relative freq.:  0.4467 <\/br>Relative Rep. freq.:  0.043","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  749 <\/br>Relative freq.:  0.5585 <\/br>Relative Rep. freq.:  0.0504","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  04 <\/br># assignments:  592 <\/br>Relative freq.:  0.4415 <\/br>Relative Rep. freq.:  0.0398","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  423 <\/br>Relative freq.:  0.9953 <\/br>Relative Rep. freq.:  0.1277","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  3314 <\/br>Relative freq.:  0.9997 <\/br>Relative Rep. freq.:  0.1505","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  3e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  1231 <\/br>Relative freq.:  0.9976 <\/br>Relative Rep. freq.:  0.1212","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  05 <\/br># assignments:  3 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  3927 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1064","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  1100 <\/br>Relative freq.:  0.9973 <\/br>Relative Rep. freq.:  0.1737","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  4090 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1498","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  657 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0716","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  1241 <\/br>Relative freq.:  0.6332 <\/br>Relative Rep. freq.:  0.0554","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  04 <\/br># assignments:  719 <\/br>Relative freq.:  0.3668 <\/br>Relative Rep. freq.:  0.0321","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  1171 <\/br>Relative freq.:  0.6441 <\/br>Relative Rep. freq.:  0.0546","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  04 <\/br># assignments:  647 <\/br>Relative freq.:  0.3559 <\/br>Relative Rep. freq.:  0.0302","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  04 <\/br># assignments:  692 <\/br>Relative freq.:  0.5594 <\/br>Relative Rep. freq.:  0.0375","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  01 <\/br># assignments:  545 <\/br>Relative freq.:  0.4406 <\/br>Relative Rep. freq.:  0.0295","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  04 <\/br># assignments:  785 <\/br>Relative freq.:  0.5672 <\/br>Relative Rep. freq.:  0.04","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  599 <\/br>Relative freq.:  0.4328 <\/br>Relative Rep. freq.:  0.0305","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  2457 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1014","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  1523 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.2011","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  1544 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0565","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  5130 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1607","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  3341 <\/br>Relative freq.:  0.9997 <\/br>Relative Rep. freq.:  0.098","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  3e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  166 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1231","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  1910 <\/br>Relative freq.:  0.9984 <\/br>Relative Rep. freq.:  0.089","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  5e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  3457 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.1509","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  3e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  1209 <\/br>Relative freq.:  0.7214 <\/br>Relative Rep. freq.:  0.0628","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  04 <\/br># assignments:  467 <\/br>Relative freq.:  0.2786 <\/br>Relative Rep. freq.:  0.0243","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  1725 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0919","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  1394 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0708","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  1413 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1143","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  2673 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.0792","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  1026 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1345","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  2235 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.0948","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  1668 <\/br>Relative freq.:  0.5978 <\/br>Relative Rep. freq.:  0.0616","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  04 <\/br># assignments:  1122 <\/br>Relative freq.:  0.4022 <\/br>Relative Rep. freq.:  0.0414","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  2509 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0765","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  5013 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1563","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  969 <\/br>Relative freq.:  0.6164 <\/br>Relative Rep. freq.:  0.0538","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  04 <\/br># assignments:  603 <\/br>Relative freq.:  0.3836 <\/br>Relative Rep. freq.:  0.0335","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  3116 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1247","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  1256 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.0718","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  01 <\/br># assignments:  2052 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0861","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  2920 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.157","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  608 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.028","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  1124 <\/br>Relative freq.:  0.6262 <\/br>Relative Rep. freq.:  0.0606","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  04 <\/br># assignments:  671 <\/br>Relative freq.:  0.3738 <\/br>Relative Rep. freq.:  0.0362","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  3009 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0919","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  04 <\/br># assignments:  1769 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0591","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  2576 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.112","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  1427 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1032","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  1885 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1301","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  01 <\/br># assignments:  2501 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1049","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  896 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0964","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  1970 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1049","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  1085 <\/br>Relative freq.:  0.5596 <\/br>Relative Rep. freq.:  0.038","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  04 <\/br># assignments:  854 <\/br>Relative freq.:  0.4404 <\/br>Relative Rep. freq.:  0.0299","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  4129 <\/br>Relative freq.:  0.9998 <\/br>Relative Rep. freq.:  0.1264","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  2e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  01 <\/br># assignments:  2762 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.1178","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  1499 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0934","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  2325 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0744","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  1378 <\/br>Relative freq.:  0.6252 <\/br>Relative Rep. freq.:  0.0583","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  04 <\/br># assignments:  826 <\/br>Relative freq.:  0.3748 <\/br>Relative Rep. freq.:  0.035","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  1516 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1546","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  1762 <\/br>Relative freq.:  0.7372 <\/br>Relative Rep. freq.:  0.0753","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  04 <\/br># assignments:  628 <\/br>Relative freq.:  0.2628 <\/br>Relative Rep. freq.:  0.0268","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  1766 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1402","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  2072 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1633","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  164 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1256","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  2851 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.1207","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  2292 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.1328","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  260 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1334"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":181.253632212536,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,4.6],"tickmode":"array","ticktext":["V3-23*03","V3-23*04","V3-23*05","V3-23D*01/V3-23*01"],"tickvals":[1,2,3,4],"categoryorder":"array","categoryarray":["V3-23*03","V3-23*04","V3-23*05","V3-23D*01/V3-23*01"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-255.823519378463,5386.46778663707],"tickmode":"array","ticktext":["0","1000","2000","3000","4000","5000"],"tickvals":[0,1000,2000,3000,4000,5000],"categoryorder":"array","categoryarray":["0","1000","2000","3000","4000","5000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"56a761a3f3b4c":{"x":{},"y":{},"text":{},"type":"box"},"56a7651bd43fc":{"x":{},"y":{},"text":{}}},"cur_data":"56a761a3f3b4c","visdat":{"56a761a3f3b4c":["function (y) ","x"],"56a7651bd43fc":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-2d8249602fcff9fe75a3" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-2d8249602fcff9fe75a3">{"x":{"data":[{"x":[3,3,3,3,3,3,3,1,3,3,1,1,3,3,3,1,3,3,3,3,3,3,2,3,3,3,1,3,2,3,1,3,3,3,3,3,3,3,3,3,3,3,1,3,3,3,1,3,1,3,1,3,3,3,1],"y":[230,68,100,76,186,224,86,1,241,204,119,83,284,141,176,88,200,155,274,58,120,467,1,274,178,212,1,300,1,144,76,346,428,152,115,297,255,180,206,64,101,427,1,424,17,79,1,414,1,105,1,187,131,92,62],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(76,138,126,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[2.99168467195705,3.01334956581704,2.97841197378002,2.94026909652166,2.92463561124168,2.97607933310792,2.95826391060837,0.931809790898114,3.05924172527157,2.93193371612579,1.00832466762513,1.06192252379842,3.03797612958588,3.05790068325587,2.92742501930334,1.09642667896114,3.00067619974725,2.99302338710986,3.01672110105865,2.93978929505683,2.94423154480755,2.924637673283,1.90680029420182,2.98763833213598,2.98475243886933,3.05182481980883,0.96450997935608,3.00591244427487,1.93059737118892,2.91977064642124,0.910554970474914,2.95720906611532,2.91373016359285,2.94154511797242,3.08471480743028,3.04440051107667,2.98572481642477,2.91938754594885,3.07998460298404,2.99771013101563,3.0443349617999,2.95758295776322,0.987282757041976,3.04375396776013,3.09068968733773,2.94875058550388,1.01355573940091,3.08173502734862,0.97288824878633,2.91817132695578,0.939952221699059,2.91222470598295,3.03817539191805,2.95662714587525,1.05800002776086],"y":[229.778137215041,68.084479018487,100.008419276029,75.7193118624389,185.672020313703,224.186458414979,85.8856168949977,1.36826676297933,240.679214340448,203.830547762103,118.751890044287,82.8188403561711,283.720617552288,141.361258095503,175.760113200359,87.8543213557452,199.994585474953,155.112286368012,273.819001439214,57.6729268413037,120.237360304967,466.713700138591,0.895171654969454,273.686246052198,178.172278193571,211.961548264697,1.04174750428647,299.629182734154,1.2023459887132,144.069811571389,76.0577128360048,346.176808659732,428.354835839383,152.237312507071,114.724389311858,297.052240778692,254.698416462913,180.219281383976,206.057595926896,63.6847217889503,100.793842305429,426.688422067091,1.15752862207592,424.22855070848,16.7341572044417,78.9846754904836,1.2074786124751,414.125096913613,1.18282814268023,105.134813177586,0.977911949343979,187.2153532397,131.077422968112,91.6901661781594,61.8592670315877],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  230 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0392","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  68 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  100 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0219","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  76 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0863","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  186 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0251","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  224 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0373","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  86 <\/br>Relative freq.:  0.9885 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0115 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  241 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0291","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  204 <\/br>Relative freq.:  0.6316 <\/br>Relative Rep. freq.:  0.0358","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  119 <\/br>Relative freq.:  0.3684 <\/br>Relative Rep. freq.:  0.0209","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  83 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  284 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0332","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  141 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  176 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0291","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  88 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  200 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0504","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  155 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0357","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  274 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0348","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  58 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  120 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.053","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  467 <\/br>Relative freq.:  0.9979 <\/br>Relative Rep. freq.:  0.066","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  274 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0763","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  178 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0255","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  212 <\/br>Relative freq.:  0.9953 <\/br>Relative Rep. freq.:  0.0465","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  300 <\/br>Relative freq.:  0.9967 <\/br>Relative Rep. freq.:  0.0389","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  144 <\/br>Relative freq.:  0.6545 <\/br>Relative Rep. freq.:  0.0248","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  76 <\/br>Relative freq.:  0.3455 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  346 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0388","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  428 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0403","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  152 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0273","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  115 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.024","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  297 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0509","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  01 <\/br># assignments:  255 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0475","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  180 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0353","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  206 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0222","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  64 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0206","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  101 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0195","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  427 <\/br>Relative freq.:  0.9977 <\/br>Relative Rep. freq.:  0.0277","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  424 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0969","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  17 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  79 <\/br>Relative freq.:  0.9875 <\/br>Relative Rep. freq.:  0.0255","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0125 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  414 <\/br>Relative freq.:  0.9976 <\/br>Relative Rep. freq.:  0.0489","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  105 <\/br>Relative freq.:  0.9906 <\/br>Relative Rep. freq.:  0.0186","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0094 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  187 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0307","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  131 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0244","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  92 <\/br>Relative freq.:  0.5974 <\/br>Relative Rep. freq.:  0.0124","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  62 <\/br>Relative freq.:  0.4026 <\/br>Relative Rep. freq.:  0.0084"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(76,138,126,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(76,138,126,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":181.253632212536,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V3-23*04","V3-23*05","V3-23D*01/V3-23*01"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V3-23*04","V3-23*05","V3-23D*01/V3-23*01"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-22.4100697622821,490.305241417252],"tickmode":"array","ticktext":["0","100","200","300","400"],"tickvals":[0,100,200,300,400],"categoryorder":"array","categoryarray":["0","100","200","300","400"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"56a7626a0090f":{"x":{},"y":{},"text":{},"type":"box"},"56a763578f8af":{"x":{},"y":{},"text":{}}},"cur_data":"56a7626a0090f","visdat":{"56a7626a0090f":["function (y) ","x"],"56a763578f8af":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-efbc9aaab50a5d5ef9f4" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-efbc9aaab50a5d5ef9f4">{"x":{"data":[{"x":[5,3,1,2,4,5,3,5,3,1,2,4,5,4,2,3,5,4,3,1,2,5,3,2,5,3,4,2,5,3,1,2,4,5,3,2,4,5,3,4,5,3,4,5,3,4,1,5,3,4,5,3,5,4,3,5,3,4,3,5,2,4,5,3,4,1,2,5,3,4,5,4,3,2,5,4,3,2,5,3,2,4,1,5,3,4,5,3,2,4,5,3,4,5,3,4,2,1,5,3,4,2,5,3,4,1],"y":[236,193,3,2,2,268,114,1069,19,2,1,1,1303,5,3,1,241,3,2,1,1,306,185,123,879,8,8,3,1573,14,3,2,1,484,5,2,1,838,592,7,833,5,2,451,195,4,1,513,447,15,327,4,642,5,1,421,2,1,396,393,124,3,1236,13,5,1,1,2627,9,5,2288,5,2,1,2178,8,7,3,1676,17,8,5,2,1287,10,4,346,4,1,1,205,148,1,752,491,6,3,1,543,7,7,1,473,264,7,3],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(86,63,80,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[4.91064994027838,2.90531597086228,1.05085642063059,2.0722522854805,4.02211555922404,4.95547289759852,3.03601382472552,5.08888095482253,3.01652762391605,1.03134758747183,1.93290945854969,3.93764650914818,4.9571061857976,3.92054948313162,1.92678056107834,2.95165718044154,5.03841030611657,3.91679132576101,3.07379458909854,0.91457916367799,2.07860317891464,5.08730209879577,2.92371163470671,1.94112845114432,4.95943447300233,2.91309970081784,4.07709006010555,1.91494466797449,4.97033793819137,2.91499357861467,0.966114465426654,1.9630996035412,4.04989800662734,4.96487656864338,2.97933895112947,1.98623896595091,3.96681168833747,5.00890854676254,3.06116107814014,4.08332103574648,4.94292518352158,3.00582902333699,4.05044358419254,4.97123585492372,3.06569036361761,3.90651564374566,1.04653075924143,5.02735545835458,3.06097466954961,3.98072290774435,5.06441920180805,2.96347847892903,5.02687063692138,4.0809242955409,2.9326737084426,5.0648069978226,2.90342585616745,3.90387312564999,2.95004432983696,5.04058269909583,1.98027640162036,3.96437835986726,4.93190440023318,3.0066365938168,3.9116056276951,0.917659363010898,2.02857744642533,4.92521793292835,3.05747045795433,4.08794416184537,4.93238231986761,3.96256604464725,3.01784277125262,2.09133111229166,5.06897672908381,3.93386092437431,2.9371167124249,2.09724240996875,4.95819056793116,2.91472851871513,1.97021053242497,3.97443038136698,1.06656533367932,5.05985762216151,3.01675291093998,4.02593727023341,5.00570616549812,2.98799661109224,1.95636964412406,4.07724979245104,4.90963192633353,2.93224213784561,3.97581043634564,5.04039485710673,3.01418421678245,3.98062377744354,2.09841864467599,1.07916365456767,5.09043755796738,2.96454741111957,4.03870567493141,1.99787877714261,4.94072355367243,3.05856868275441,4.04885478522629,1.06355079752393],"y":[236.370876062103,193.171836115792,2.77892693132162,2.34166069105268,2.25774842724204,268.319512966462,114.24515285939,1068.61056693252,19.1155825611204,1.94811299797148,0.632471192441881,1.34403672050685,1303.30278815404,5.26211096104234,2.80599473435432,0.704668114706874,240.706945359148,2.96577550005168,1.79247320815921,0.856758575327694,1.30630102138966,306.31705907248,184.664474212006,123.02673575189,878.750399645418,8.35755246113986,7.61729811597615,3.0454543158412,1572.89270116948,13.7207292359322,3.20595833528787,2.24541478548199,0.762653587572277,483.929522654228,5.29024464711547,1.92637535948306,0.684933751262724,837.695792260394,591.934282114357,6.69297509435564,832.986551551148,4.6642706438899,2.05218527261168,451.362757029198,194.811372105032,3.67627538628876,0.826610802486539,513.031351575069,447.166565608047,15.2314467832446,326.914095482416,3.81533944699913,641.706797121465,4.60191914364696,1.06651773825288,421.133744481765,2.34225828368217,0.82551312930882,396.178458633088,393.263236022554,124.267829970643,2.79733638390899,1235.68093069755,13.303771657683,4.78355927374214,1.13201112151146,1.07218902669847,2627.14367541801,8.96805194951594,5.30453850366175,2288.1842338549,4.70657422840595,1.99971938189119,1.01290032826364,2177.74230392985,8.31266347803176,6.73157219458371,3.24009323138744,1675.68647290636,16.9938860679045,7.90840498898178,4.6271067449823,1.70102987680584,1287.04313997794,10.1618188370019,4.0241651352495,345.969413816184,4.37048832569271,1.15272612348199,0.928004955686629,204.804613899253,148.130801782571,0.917770693451166,751.888551239669,490.887141243368,5.96927225012332,2.85903167165816,1.15017465706915,543.137435617298,6.91378933433443,7.06865546200424,1.15068531818688,472.980233678967,263.98304937426,6.61779128015041,2.86190045271069],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  236 <\/br>Relative freq.:  0.5413 <\/br>Relative Rep. freq.:  0.0207","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  193 <\/br>Relative freq.:  0.4427 <\/br>Relative Rep. freq.:  0.0169","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0069 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0046 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0046 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  268 <\/br>Relative freq.:  0.7016 <\/br>Relative Rep. freq.:  0.0713","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  114 <\/br>Relative freq.:  0.2984 <\/br>Relative Rep. freq.:  0.0303","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  1069 <\/br>Relative freq.:  0.9789 <\/br>Relative Rep. freq.:  0.0952","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  19 <\/br>Relative freq.:  0.0174 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  1303 <\/br>Relative freq.:  0.9931 <\/br>Relative Rep. freq.:  0.086","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  05 <\/br># assignments:  5 <\/br>Relative freq.:  0.0038 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  241 <\/br>Relative freq.:  0.9718 <\/br>Relative Rep. freq.:  0.046","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  05 <\/br># assignments:  3 <\/br>Relative freq.:  0.0121 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0081 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  306 <\/br>Relative freq.:  0.4984 <\/br>Relative Rep. freq.:  0.0179","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  185 <\/br>Relative freq.:  0.3013 <\/br>Relative Rep. freq.:  0.0108","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  03 <\/br># assignments:  123 <\/br>Relative freq.:  0.2003 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  879 <\/br>Relative freq.:  0.9788 <\/br>Relative Rep. freq.:  0.0739","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  8 <\/br>Relative freq.:  0.0089 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  05 <\/br># assignments:  8 <\/br>Relative freq.:  0.0089 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  1573 <\/br>Relative freq.:  0.9874 <\/br>Relative Rep. freq.:  0.0815","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  14 <\/br>Relative freq.:  0.0088 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  484 <\/br>Relative freq.:  0.9837 <\/br>Relative Rep. freq.:  0.0677","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  5 <\/br>Relative freq.:  0.0102 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0041 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  838 <\/br>Relative freq.:  0.5832 <\/br>Relative Rep. freq.:  0.0504","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  592 <\/br>Relative freq.:  0.412 <\/br>Relative Rep. freq.:  0.0356","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  05 <\/br># assignments:  7 <\/br>Relative freq.:  0.0049 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  833 <\/br>Relative freq.:  0.9917 <\/br>Relative Rep. freq.:  0.0568","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  5 <\/br>Relative freq.:  0.006 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  451 <\/br>Relative freq.:  0.6928 <\/br>Relative Rep. freq.:  0.04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  195 <\/br>Relative freq.:  0.2995 <\/br>Relative Rep. freq.:  0.0173","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  05 <\/br># assignments:  4 <\/br>Relative freq.:  0.0061 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  513 <\/br>Relative freq.:  0.5262 <\/br>Relative Rep. freq.:  0.0288","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  447 <\/br>Relative freq.:  0.4585 <\/br>Relative Rep. freq.:  0.0251","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  05 <\/br># assignments:  15 <\/br>Relative freq.:  0.0154 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  327 <\/br>Relative freq.:  0.9879 <\/br>Relative Rep. freq.:  0.0839","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.0121 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  642 <\/br>Relative freq.:  0.9907 <\/br>Relative Rep. freq.:  0.0957","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  05 <\/br># assignments:  5 <\/br>Relative freq.:  0.0077 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  421 <\/br>Relative freq.:  0.9929 <\/br>Relative Rep. freq.:  0.0458","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  396 <\/br>Relative freq.:  0.4323 <\/br>Relative Rep. freq.:  0.0298","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  393 <\/br>Relative freq.:  0.429 <\/br>Relative Rep. freq.:  0.0296","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  03 <\/br># assignments:  124 <\/br>Relative freq.:  0.1354 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  05 <\/br># assignments:  3 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  1236 <\/br>Relative freq.:  0.9841 <\/br>Relative Rep. freq.:  0.0817","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  13 <\/br>Relative freq.:  0.0104 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  5 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  2627 <\/br>Relative freq.:  0.9947 <\/br>Relative Rep. freq.:  0.1401","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  9 <\/br>Relative freq.:  0.0034 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  05 <\/br># assignments:  5 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  2288 <\/br>Relative freq.:  0.9965 <\/br>Relative Rep. freq.:  0.0914","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  05 <\/br># assignments:  5 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  2178 <\/br>Relative freq.:  0.9918 <\/br>Relative Rep. freq.:  0.0975","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  05 <\/br># assignments:  8 <\/br>Relative freq.:  0.0036 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  7 <\/br>Relative freq.:  0.0032 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  1676 <\/br>Relative freq.:  0.9813 <\/br>Relative Rep. freq.:  0.1076","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  17 <\/br>Relative freq.:  0.01 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  03 <\/br># assignments:  8 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  05 <\/br># assignments:  5 <\/br>Relative freq.:  0.0029 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  1287 <\/br>Relative freq.:  0.9892 <\/br>Relative Rep. freq.:  0.1092","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  10 <\/br>Relative freq.:  0.0077 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  05 <\/br># assignments:  4 <\/br>Relative freq.:  0.0031 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  346 <\/br>Relative freq.:  0.983 <\/br>Relative Rep. freq.:  0.0503","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.0114 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  205 <\/br>Relative freq.:  0.5791 <\/br>Relative Rep. freq.:  0.0238","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  148 <\/br>Relative freq.:  0.4181 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  752 <\/br>Relative freq.:  0.6002 <\/br>Relative Rep. freq.:  0.0509","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  491 <\/br>Relative freq.:  0.3919 <\/br>Relative Rep. freq.:  0.0332","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  05 <\/br># assignments:  6 <\/br>Relative freq.:  0.0048 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  543 <\/br>Relative freq.:  0.9731 <\/br>Relative Rep. freq.:  0.066","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  7 <\/br>Relative freq.:  0.0125 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  05 <\/br># assignments:  7 <\/br>Relative freq.:  0.0125 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  473 <\/br>Relative freq.:  0.6332 <\/br>Relative Rep. freq.:  0.0422","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  264 <\/br>Relative freq.:  0.3534 <\/br>Relative Rep. freq.:  0.0236","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  05 <\/br># assignments:  7 <\/br>Relative freq.:  0.0094 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  3e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(86,63,80,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(86,63,80,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":181.253632212536,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["V3-23*02","V3-23*03","V3-23*04","V3-23*05","V3-23D*01/V3-23*01"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["V3-23*02","V3-23*03","V3-23*04","V3-23*05","V3-23D*01/V3-23*01"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-130.693089018837,2758.46923562929],"tickmode":"array","ticktext":["0","1000","2000"],"tickvals":[0,1000,2000],"categoryorder":"array","categoryarray":["0","1000","2000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"56a7661f5e4ca":{"x":{},"y":{},"text":{},"type":"box"},"56a765165c1af":{"x":{},"y":{},"text":{}}},"cur_data":"56a7661f5e4ca","visdat":{"56a7661f5e4ca":["function (y) ","x"],"56a765165c1af":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-16" class="section level2" number="20.4">
<h2><span class="header-section-number">20.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 7 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV3-23G19%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:900px;" cellspacing="0">
</iframe>
</div>
<div id="observations-18" class="section level2" number="20.5">
<h2><span class="header-section-number">20.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/b2i3zc4291fby45jhu7jl/IGHV3-23G19.docx?dl=0&amp;rlkey=pjh29xcrfcwtnmkpozgdxwslf" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV3-23G19" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-18" class="section level2" number="20.6">
<h2><span class="header-section-number">20.6</span> Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="18-G18.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="20-G20.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/19-G19.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
