<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV3-48 - G23 | Functional groups</title>
  <meta name="description" content="IGHV3-48 - G23 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV3-48 - G23 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV3-48 - G23 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-15" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="22-G22.html"/>
<link rel="next" href="24-G24.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-checklist"><i class="fa fa-check"></i>Group Checklist</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv3-48---g23" class="section level1">
<h1>IGHV3-48 - G23</h1>
<div id="allele-appearnce-22" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of IGHV3-48 includes 4 alleles, 4 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-208-1.png" width="672" /></p>
</div>
<div id="group-alignment-19" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-209-1.png" width="1728" /><img src="reference-book_files/figure-html/unnamed-chunk-209-2.png" width="1728" /></p>
</div>
<div id="sequence-depth-22" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-039db9700b2704d122cb" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-039db9700b2704d122cb">{"x":{"visdat":{"9640955e2f802":["function () ","plotlyVisDat"]},"cur_data":"9640955e2f802","attrs":{"9640955e2f802":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatter","x":{},"y":{},"text":{},"marker":{"color":"#FAAB18","size":8,"line":{"width":1,"color":"gray"}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":"text","inherit":true},"9640955e2f802.1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"marker":{"color":"#FAAB18"},"type":"box","hoverinfo":"none","showlegend":false,"fillcolor":"transparent","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","xaxis":{"domain":[0,1],"automargin":true,"title":"Alleles","autotick":false,"tickmode":"array","tickvals":[1,2,3,4],"ticktext":["V3-48*01","V3-48*02","V3-48*03","V3-48*04"]},"yaxis":{"domain":[0,1],"automargin":true,"title":"# Sequences"},"annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}],"showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"type":"scatter","x":[3.08737492281944,1.88365771407261,2.91070913290605,2.08953713113442,0.816238622087985,1.89336216636002,2.96423583664,4.16772076934576,2.00744890738279,1.97300090584904,2.1683085183613,1.10820445762947,4.19475577948615,2.80820504231378,2.80256758863106,1.00888164211065,3.07530582388863,1.81577718807384,3.81202500825748,1.11584429498762,1.14937997115776,2.14865056434646,2.99031963674352,1.9357149428688,1.96577411713079,3.91227246010676,2.12852700175717,3.04780936604366,1.0716654474847,0.874053234327585,3.99687259569764,0.915431458130479,3.97143975729123,2.91806161291897,1.80234141042456,2.18846426270902,1.0539426099509,2.85342698199675,4.08435269994661,4.13222765214741,0.898322628252208,1.80125766722485,4.17031502248719,3.16487707644701,1.88050688114017,1.19126744261011,2.08960635848343,0.995545266382396,1.9120083944872,2.00779618592933,0.989402778912336,3.0437631001696,2.13232242763042,2.13674089340493,2.94959034351632,0.83049162440002,2.19086644714698,0.976443687174469,1.01805651849136,1.82838222105056,3.16876778835431,1.88995538447052,3.02352691711858,1.99531359905377,0.930632481537759,1.80153811927885,0.876923205144703,3.02425461784005,2.06304265148938,3.08726925039664,2.1060145335272,3.19475997304544,1.18752354066819,3.93223333479837,2.09897783454508,2.94125398956239,2.13522826153785,2.80531426426023,1.00210366696119,2.16807979037985,3.172509483248,1.03730473527685,1.87754729250446,1.11087507195771,1.98072011061013,2.17963355546817,0.849035008531064,1.90107057662681,1.02799592381343,3.9534954868257,2.0765842766501,0.923669583629817,1.0003173914738,2.06407928653061,4.17950030956417,3.09858955182135,1.16276986505836,2.08223558869213,2.17463034726679,1.19966730633751,1.87402116134763,1.16762312687933,2.10817935839295,1.14228017032146,3.9828203426674,0.94003071282059,2.10688937474042,2.92622289415449,2.09896270465106,2.01807398973033,3.1187789301388,2.08164880247787,1.82987226909027,0.971795879025012,3.06639291476458,1.18972374266014,1.96300964877009,0.883971489593387,2.09004275761545,2.0814260003157,3.01371321426705,2.10932156937197,2.93222977118567,1.00861679837108,2.8314105566591,2.16837743874639,2.86773035703227,1.99605898596346,1.89626810327172,1.03551184413955,3.017965719942,2.885850242991,4.0116944430396,2.19326279638335,2.90372312646359,1.96418630843982,0.814207051508129,2.19986074119806,2.09014603989199,0.972601706068963,1.94270077925175,3.1891313938424,1.13779943753034,2.01305518969893,2.14191180318594,0.888937502447516,1.06897327192128,0.992416362185031,3.90282944822684,0.810631398670375,2.16023536883295,4.1468952848576,3.94011997478083,1.90463184369728,3.10050517879426,0.980067002773285,1.87441315818578,0.967315608542412,1.80012668948621,1.13574399696663,1.04496343284845,2.06568336561322,3.95150643708184,1.94853882947937,1.10382555155084,1.96029933458194,0.903539481852204,1.17604951774701,0.906215666607022,2.07097846455872,0.828196296747774,3.81847402527928,2.1938305016607,2.14726143125445],"y":[145,116,1016,350,125,114,767,567,663,669,155,134,1,601,180,51,212,102,472,319,239,213,383,185,55,387,214,311,150,483,3,227,222,553,170,243,225,111,59,409,328,169,688,472,536,4,1,36,23,251,158,176,71,148,351,78,161,59,272,203,772,416,351,224,348,212,140,306,130,150,79,336,262,57,35,535,244,190,52,743,136,75,1,674,226,242,211,587,510,1,261,222,237,180,1,198,122,552,411,338,616,33,216,190,1,319,258,336,171,368,369,180,204,168,1317,127,66,464,1,604,1143,685,534,297,495,354,462,315,777,423,1289,541,306,884,842,400,203,178,395,212,170,675,437,1,77,57,348,674,1,502,383,1,373,336,393,159,324,312,249,233,151,149,1,377,304,173,166,311,18,16,444,1,508,58],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  145 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.0122","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  116 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  1016 <\/br>Relative freq.:  0.7438 <\/br>Relative Rep. freq.:  0.0447","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  350 <\/br>Relative freq.:  0.2562 <\/br>Relative Rep. freq.:  0.0154","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  125 <\/br>Relative freq.:  0.523 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  114 <\/br>Relative freq.:  0.477 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  767 <\/br>Relative freq.:  0.575 <\/br>Relative Rep. freq.:  0.0327","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  567 <\/br>Relative freq.:  0.425 <\/br>Relative Rep. freq.:  0.0242","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  663 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0284","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  669 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0278","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  155 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0264","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  134 <\/br>Relative freq.:  0.9926 <\/br>Relative Rep. freq.:  0.023","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0074 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  601 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0432","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  180 <\/br>Relative freq.:  0.7792 <\/br>Relative Rep. freq.:  0.0327","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  51 <\/br>Relative freq.:  0.2208 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  212 <\/br>Relative freq.:  0.6752 <\/br>Relative Rep. freq.:  0.0218","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  102 <\/br>Relative freq.:  0.3248 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  472 <\/br>Relative freq.:  0.5967 <\/br>Relative Rep. freq.:  0.0217","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  319 <\/br>Relative freq.:  0.4033 <\/br>Relative Rep. freq.:  0.0147","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  239 <\/br>Relative freq.:  0.5288 <\/br>Relative Rep. freq.:  0.0161","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  213 <\/br>Relative freq.:  0.4712 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  383 <\/br>Relative freq.:  0.6743 <\/br>Relative Rep. freq.:  0.0285","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  185 <\/br>Relative freq.:  0.3257 <\/br>Relative Rep. freq.:  0.0138","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  55 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0239","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  387 <\/br>Relative freq.:  0.6439 <\/br>Relative Rep. freq.:  0.0148","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  214 <\/br>Relative freq.:  0.3561 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  311 <\/br>Relative freq.:  0.6746 <\/br>Relative Rep. freq.:  0.018","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  150 <\/br>Relative freq.:  0.3254 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  483 <\/br>Relative freq.:  0.9938 <\/br>Relative Rep. freq.:  0.0219","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  3 <\/br>Relative freq.:  0.0062 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  227 <\/br>Relative freq.:  0.5056 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  222 <\/br>Relative freq.:  0.4944 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  553 <\/br>Relative freq.:  0.7649 <\/br>Relative Rep. freq.:  0.0546","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  170 <\/br>Relative freq.:  0.2351 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  243 <\/br>Relative freq.:  0.5192 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  225 <\/br>Relative freq.:  0.4808 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  111 <\/br>Relative freq.:  0.6529 <\/br>Relative Rep. freq.:  0.0289","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  59 <\/br>Relative freq.:  0.3471 <\/br>Relative Rep. freq.:  0.0154","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  409 <\/br>Relative freq.:  0.555 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  328 <\/br>Relative freq.:  0.445 <\/br>Relative Rep. freq.:  0.015","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  169 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  688 <\/br>Relative freq.:  0.5931 <\/br>Relative Rep. freq.:  0.0343","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  472 <\/br>Relative freq.:  0.4069 <\/br>Relative Rep. freq.:  0.0236","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  536 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.022","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  4 <\/br>Relative freq.:  0.8 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  1 <\/br>Relative freq.:  0.2 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  36 <\/br>Relative freq.:  0.6102 <\/br>Relative Rep. freq.:  0.0206","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  23 <\/br>Relative freq.:  0.3898 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  251 <\/br>Relative freq.:  0.6137 <\/br>Relative Rep. freq.:  0.0123","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  158 <\/br>Relative freq.:  0.3863 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  176 <\/br>Relative freq.:  0.7126 <\/br>Relative Rep. freq.:  0.0347","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  71 <\/br>Relative freq.:  0.2874 <\/br>Relative Rep. freq.:  0.014","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  148 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0208","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  351 <\/br>Relative freq.:  0.8182 <\/br>Relative Rep. freq.:  0.0313","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  78 <\/br>Relative freq.:  0.1818 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  161 <\/br>Relative freq.:  0.7318 <\/br>Relative Rep. freq.:  0.0201","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  59 <\/br>Relative freq.:  0.2682 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  272 <\/br>Relative freq.:  0.5726 <\/br>Relative Rep. freq.:  0.0155","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  203 <\/br>Relative freq.:  0.4274 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  772 <\/br>Relative freq.:  0.6498 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  416 <\/br>Relative freq.:  0.3502 <\/br>Relative Rep. freq.:  0.0149","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  351 <\/br>Relative freq.:  0.6104 <\/br>Relative Rep. freq.:  0.0147","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  224 <\/br>Relative freq.:  0.3896 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  348 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0206","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  212 <\/br>Relative freq.:  0.6023 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  140 <\/br>Relative freq.:  0.3977 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  306 <\/br>Relative freq.:  0.7018 <\/br>Relative Rep. freq.:  0.0271","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  130 <\/br>Relative freq.:  0.2982 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  150 <\/br>Relative freq.:  0.655 <\/br>Relative Rep. freq.:  0.0251","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  79 <\/br>Relative freq.:  0.345 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  336 <\/br>Relative freq.:  0.5619 <\/br>Relative Rep. freq.:  0.023","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  262 <\/br>Relative freq.:  0.4381 <\/br>Relative Rep. freq.:  0.018","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  57 <\/br>Relative freq.:  0.6196 <\/br>Relative Rep. freq.:  0.018","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  35 <\/br>Relative freq.:  0.3804 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  535 <\/br>Relative freq.:  0.6868 <\/br>Relative Rep. freq.:  0.0247","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  244 <\/br>Relative freq.:  0.3132 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  190 <\/br>Relative freq.:  0.7851 <\/br>Relative Rep. freq.:  0.0261","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  52 <\/br>Relative freq.:  0.2149 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  743 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0205","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  136 <\/br>Relative freq.:  0.6415 <\/br>Relative Rep. freq.:  0.022","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  75 <\/br>Relative freq.:  0.3538 <\/br>Relative Rep. freq.:  0.0121","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  674 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.025","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  226 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0251","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  242 <\/br>Relative freq.:  0.5342 <\/br>Relative Rep. freq.:  0.0111","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  211 <\/br>Relative freq.:  0.4658 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  587 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0277","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  510 <\/br>Relative freq.:  0.998 <\/br>Relative Rep. freq.:  0.0281","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  261 <\/br>Relative freq.:  0.5404 <\/br>Relative Rep. freq.:  0.0137","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  222 <\/br>Relative freq.:  0.4596 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  237 <\/br>Relative freq.:  0.567 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  180 <\/br>Relative freq.:  0.4306 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  198 <\/br>Relative freq.:  0.6188 <\/br>Relative Rep. freq.:  0.0267","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  122 <\/br>Relative freq.:  0.3812 <\/br>Relative Rep. freq.:  0.0165","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  552 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0205","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  411 <\/br>Relative freq.:  0.5487 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  338 <\/br>Relative freq.:  0.4513 <\/br>Relative Rep. freq.:  0.0108","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  616 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0193","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  33 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0252","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  216 <\/br>Relative freq.:  0.5307 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  190 <\/br>Relative freq.:  0.4668 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0025 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  319 <\/br>Relative freq.:  0.5529 <\/br>Relative Rep. freq.:  0.0147","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  258 <\/br>Relative freq.:  0.4471 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  336 <\/br>Relative freq.:  0.6627 <\/br>Relative Rep. freq.:  0.0181","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  171 <\/br>Relative freq.:  0.3373 <\/br>Relative Rep. freq.:  0.0092","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  368 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  369 <\/br>Relative freq.:  0.6721 <\/br>Relative Rep. freq.:  0.0189","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  180 <\/br>Relative freq.:  0.3279 <\/br>Relative Rep. freq.:  0.0092","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  204 <\/br>Relative freq.:  0.5484 <\/br>Relative Rep. freq.:  0.0175","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  168 <\/br>Relative freq.:  0.4516 <\/br>Relative Rep. freq.:  0.0144","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  1317 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0402","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  127 <\/br>Relative freq.:  0.658 <\/br>Relative Rep. freq.:  0.0171","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  66 <\/br>Relative freq.:  0.342 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  464 <\/br>Relative freq.:  0.9978 <\/br>Relative Rep. freq.:  0.0205","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  604 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0229","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  1143 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0353","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  685 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0221","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  534 <\/br>Relative freq.:  0.6426 <\/br>Relative Rep. freq.:  0.0301","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  297 <\/br>Relative freq.:  0.3574 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  495 <\/br>Relative freq.:  0.583 <\/br>Relative Rep. freq.:  0.02","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  354 <\/br>Relative freq.:  0.417 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  462 <\/br>Relative freq.:  0.5946 <\/br>Relative Rep. freq.:  0.0271","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  315 <\/br>Relative freq.:  0.4054 <\/br>Relative Rep. freq.:  0.0185","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  777 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0333","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  423 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0248","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  1289 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0601","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  541 <\/br>Relative freq.:  0.6387 <\/br>Relative Rep. freq.:  0.0311","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  306 <\/br>Relative freq.:  0.3613 <\/br>Relative Rep. freq.:  0.0176","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  884 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0275","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  842 <\/br>Relative freq.:  0.6779 <\/br>Relative Rep. freq.:  0.0289","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  400 <\/br>Relative freq.:  0.3221 <\/br>Relative Rep. freq.:  0.0137","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  203 <\/br>Relative freq.:  0.5328 <\/br>Relative Rep. freq.:  0.0092","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  178 <\/br>Relative freq.:  0.4672 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  395 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0322","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  212 <\/br>Relative freq.:  0.555 <\/br>Relative Rep. freq.:  0.0153","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  170 <\/br>Relative freq.:  0.445 <\/br>Relative Rep. freq.:  0.0123","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  675 <\/br>Relative freq.:  0.6065 <\/br>Relative Rep. freq.:  0.029","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  437 <\/br>Relative freq.:  0.3926 <\/br>Relative Rep. freq.:  0.0188","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  77 <\/br>Relative freq.:  0.5746 <\/br>Relative Rep. freq.:  0.0085","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  57 <\/br>Relative freq.:  0.4254 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  348 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0188","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  674 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0263","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  502 <\/br>Relative freq.:  0.5666 <\/br>Relative Rep. freq.:  0.0157","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  383 <\/br>Relative freq.:  0.4323 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  373 <\/br>Relative freq.:  0.5261 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  336 <\/br>Relative freq.:  0.4739 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  393 <\/br>Relative freq.:  0.712 <\/br>Relative Rep. freq.:  0.0249","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  159 <\/br>Relative freq.:  0.288 <\/br>Relative Rep. freq.:  0.0101","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  324 <\/br>Relative freq.:  0.5094 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  312 <\/br>Relative freq.:  0.4906 <\/br>Relative Rep. freq.:  0.0101","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  249 <\/br>Relative freq.:  0.5166 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  233 <\/br>Relative freq.:  0.4834 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  151 <\/br>Relative freq.:  0.5017 <\/br>Relative Rep. freq.:  0.0165","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  149 <\/br>Relative freq.:  0.495 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  377 <\/br>Relative freq.:  0.5536 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  304 <\/br>Relative freq.:  0.4464 <\/br>Relative Rep. freq.:  0.0138","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  173 <\/br>Relative freq.:  0.5103 <\/br>Relative Rep. freq.:  0.0142","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  166 <\/br>Relative freq.:  0.4897 <\/br>Relative Rep. freq.:  0.0136","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  311 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0254","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  18 <\/br>Relative freq.:  0.5294 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  16 <\/br>Relative freq.:  0.4706 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  444 <\/br>Relative freq.:  0.9978 <\/br>Relative Rep. freq.:  0.0204","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  508 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0312","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  58 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0314"],"marker":{"color":"#FAAB18","size":8,"line":{"color":"gray","width":1}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"transparent","x":[3,2,3,2,1,2,3,4,2,2,2,1,4,3,3,1,3,2,4,1,1,2,3,2,2,4,2,3,1,1,4,1,4,3,2,2,1,3,4,4,1,2,4,3,2,1,2,1,2,2,1,3,2,2,3,1,2,1,1,2,3,2,3,2,1,2,1,3,2,3,2,3,1,4,2,3,2,3,1,2,3,1,2,1,2,2,1,2,1,4,2,1,1,2,4,3,1,2,2,1,2,1,2,1,4,1,2,3,2,2,3,2,2,1,3,1,2,1,2,2,3,2,3,1,3,2,3,2,2,1,3,3,4,2,3,2,1,2,2,1,2,3,1,2,2,1,1,1,4,1,2,4,4,2,3,1,2,1,2,1,1,2,4,2,1,2,1,1,1,2,1,4,2,2],"y":[145,116,1016,350,125,114,767,567,663,669,155,134,1,601,180,51,212,102,472,319,239,213,383,185,55,387,214,311,150,483,3,227,222,553,170,243,225,111,59,409,328,169,688,472,536,4,1,36,23,251,158,176,71,148,351,78,161,59,272,203,772,416,351,224,348,212,140,306,130,150,79,336,262,57,35,535,244,190,52,743,136,75,1,674,226,242,211,587,510,1,261,222,237,180,1,198,122,552,411,338,616,33,216,190,1,319,258,336,171,368,369,180,204,168,1317,127,66,464,1,604,1143,685,534,297,495,354,462,315,777,423,1289,541,306,884,842,400,203,178,395,212,170,675,437,1,77,57,348,674,1,502,383,1,373,336,393,159,324,312,249,233,151,149,1,377,304,173,166,311,18,16,444,1,508,58],"marker":{"color":"#FAAB18","line":{"color":"rgba(255,127,14,1)"}},"type":"box","hoverinfo":["none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none"],"showlegend":false,"name":"transparent","line":{"color":"rgba(255,127,14,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-8a2285f9a8a78c4fedb7" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-8a2285f9a8a78c4fedb7">{"x":{"visdat":{"9640946a11969":["function () ","plotlyVisDat"]},"cur_data":"9640946a11969","attrs":{"9640946a11969":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatter","x":{},"y":{},"text":{},"marker":{"color":"#588300","size":8,"line":{"width":1,"color":"gray"}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":"text","inherit":true},"9640946a11969.1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"marker":{"color":"#588300"},"type":"box","hoverinfo":"none","showlegend":false,"fillcolor":"transparent","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","xaxis":{"domain":[0,1],"automargin":true,"title":"Alleles","autotick":false,"tickmode":"array","tickvals":[1,2,3,4],"ticktext":["V3-48*01","V3-48*02","V3-48*03","V3-48*04"]},"yaxis":{"domain":[0,1],"automargin":true,"title":"# Sequences"},"annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}],"showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"type":"scatter","x":[2.12429554816335,0.814944382756948,1.93544145459309,0.872335468418896,3.90388008048758,1.83055062685162,1.92814557952806,1.16258991109207,2.16191260879859,2.90477472878993,1.8442533957772,2.91663790438324,1.95983262993395,2.91364424591884,2.10155160371214,0.859819287993014,4.14839136553928,2.89345743088052,3.90339464014396,4.05395348863676,1.84110229788348,1.95014856401831,1.02071490818635,3.08316242359579,0.849063844792545,4.03234945582226,1.94365464057773,4.00530003989115,1.81587638389319,2.89820017814636,1.83431372214109,2.98351645227522,1.90369242047891,2.02591009037569,3.17208998985589,4.10611747754738,4.01041584135965,2.17637300072238,0.971350035164505,0.944961327407509,3.07590902373195,1.95601899959147,2.04863080801442,3.9686558756046,2.86243706513196,0.852192181441933,0.955940227396786,1.90757738361135,2.85599775006995,1.09657650897279,0.830747232213616,2.19576571155339,1.00587237579748,3.07840328058228,1.16317733097821,2.93198111904785,2.04815106531605,1.15131612727419,1.88907100437209,3.16251501832157,3.99794029202312,4.04753900235519,1.00226410990581,4.04522733120248,1.07508122874424,2.98357252469286,2.14835803220049,1.09138615997508,4.18533161999658,0.876962374709547,4.01639126315713,1.88943137098104,2.05957966381684,2.01913937488571,0.910675746574998,1.8995292593725,0.950173223763704,3.92028638422489,4.08858080869541,2.05576565498486,0.934442412666976,3.15414040340111,0.921284863445908],"y":[26,23,16,9,1,32,25,34,23,86,46,39,25,68,30,2,1,155,69,26,25,71,46,70,34,77,50,57,40,57,48,124,62,24,46,28,69,42,1,78,85,25,32,21,103,44,95,51,107,42,124,43,16,50,16,90,37,60,51,48,37,81,76,18,10,58,18,268,43,37,7,3,18,65,62,22,21,1,105,21,9,84,30],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  26 <\/br>Relative freq.:  0.5306 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  23 <\/br>Relative freq.:  0.4694 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  16 <\/br>Relative freq.:  0.6154 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  9 <\/br>Relative freq.:  0.3462 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0385 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  32 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  25 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0297","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  34 <\/br>Relative freq.:  0.5965 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  23 <\/br>Relative freq.:  0.4035 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  86 <\/br>Relative freq.:  0.6515 <\/br>Relative Rep. freq.:  0.0148","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  46 <\/br>Relative freq.:  0.3485 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  39 <\/br>Relative freq.:  0.6094 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  25 <\/br>Relative freq.:  0.3906 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  68 <\/br>Relative freq.:  0.6733 <\/br>Relative Rep. freq.:  0.0086","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  30 <\/br>Relative freq.:  0.297 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  2 <\/br>Relative freq.:  0.0198 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0099 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  155 <\/br>Relative freq.:  0.692 <\/br>Relative Rep. freq.:  0.0284","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  69 <\/br>Relative freq.:  0.308 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  26 <\/br>Relative freq.:  0.5098 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  25 <\/br>Relative freq.:  0.4902 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  71 <\/br>Relative freq.:  0.6068 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  46 <\/br>Relative freq.:  0.3932 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  70 <\/br>Relative freq.:  0.6731 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  34 <\/br>Relative freq.:  0.3269 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  77 <\/br>Relative freq.:  0.6063 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  50 <\/br>Relative freq.:  0.3937 <\/br>Relative Rep. freq.:  0.0085","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  57 <\/br>Relative freq.:  0.5876 <\/br>Relative Rep. freq.:  0.0147","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  40 <\/br>Relative freq.:  0.4124 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  57 <\/br>Relative freq.:  0.5429 <\/br>Relative Rep. freq.:  0.0137","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  48 <\/br>Relative freq.:  0.4571 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  124 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  62 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  24 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  46 <\/br>Relative freq.:  0.6216 <\/br>Relative Rep. freq.:  0.0211","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  28 <\/br>Relative freq.:  0.3784 <\/br>Relative Rep. freq.:  0.0128","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  69 <\/br>Relative freq.:  0.6161 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  42 <\/br>Relative freq.:  0.375 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0089 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  78 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0226","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  85 <\/br>Relative freq.:  0.7727 <\/br>Relative Rep. freq.:  0.0124","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  25 <\/br>Relative freq.:  0.2273 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  32 <\/br>Relative freq.:  0.6038 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  21 <\/br>Relative freq.:  0.3962 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  103 <\/br>Relative freq.:  0.7007 <\/br>Relative Rep. freq.:  0.0138","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  44 <\/br>Relative freq.:  0.2993 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  95 <\/br>Relative freq.:  0.6507 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  51 <\/br>Relative freq.:  0.3493 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  107 <\/br>Relative freq.:  0.7181 <\/br>Relative Rep. freq.:  0.0123","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  42 <\/br>Relative freq.:  0.2819 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  124 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  43 <\/br>Relative freq.:  0.7288 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  16 <\/br>Relative freq.:  0.2712 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  50 <\/br>Relative freq.:  0.7576 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  16 <\/br>Relative freq.:  0.2424 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  90 <\/br>Relative freq.:  0.7087 <\/br>Relative Rep. freq.:  0.0165","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  37 <\/br>Relative freq.:  0.2913 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  60 <\/br>Relative freq.:  0.5405 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  51 <\/br>Relative freq.:  0.4595 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  48 <\/br>Relative freq.:  0.5647 <\/br>Relative Rep. freq.:  0.0125","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  37 <\/br>Relative freq.:  0.4353 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  81 <\/br>Relative freq.:  0.5159 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  76 <\/br>Relative freq.:  0.4841 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  18 <\/br>Relative freq.:  0.6429 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  10 <\/br>Relative freq.:  0.3571 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  58 <\/br>Relative freq.:  0.7632 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  18 <\/br>Relative freq.:  0.2368 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  268 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0175","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  43 <\/br>Relative freq.:  0.5375 <\/br>Relative Rep. freq.:  0.0101","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  37 <\/br>Relative freq.:  0.4625 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  7 <\/br>Relative freq.:  0.7 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  3 <\/br>Relative freq.:  0.3 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  18 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  65 <\/br>Relative freq.:  0.5118 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  62 <\/br>Relative freq.:  0.4882 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  22 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  21 <\/br>Relative freq.:  0.4773 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0227 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  V3-48*04 <\/br># assignments:  105 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0188","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  V3-48*02 <\/br># assignments:  21 <\/br>Relative freq.:  0.7 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  9 <\/br>Relative freq.:  0.3 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  V3-48*03 <\/br># assignments:  84 <\/br>Relative freq.:  0.7368 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  V3-48*01 <\/br># assignments:  30 <\/br>Relative freq.:  0.2632 <\/br>Relative Rep. freq.:  0.0041"],"marker":{"color":"#588300","size":8,"line":{"color":"gray","width":1}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"transparent","x":[2,1,2,1,4,2,2,1,2,3,2,3,2,3,2,1,4,3,4,4,2,2,1,3,1,4,2,4,2,3,2,3,2,2,3,4,4,2,1,1,3,2,2,4,3,1,1,2,3,1,1,2,1,3,1,3,2,1,2,3,4,4,1,4,1,3,2,1,4,1,4,2,2,2,1,2,1,4,4,2,1,3,1],"y":[26,23,16,9,1,32,25,34,23,86,46,39,25,68,30,2,1,155,69,26,25,71,46,70,34,77,50,57,40,57,48,124,62,24,46,28,69,42,1,78,85,25,32,21,103,44,95,51,107,42,124,43,16,50,16,90,37,60,51,48,37,81,76,18,10,58,18,268,43,37,7,3,18,65,62,22,21,1,105,21,9,84,30],"marker":{"color":"#588300","line":{"color":"rgba(255,127,14,1)"}},"type":"box","hoverinfo":["none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none"],"showlegend":false,"name":"transparent","line":{"color":"rgba(255,127,14,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-19" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 4 zygousity states.</p>
<style>
  
#scroll-box {
  overflow-y: scroll;
  overflow-x: scroll !important;
}
</style>
<div class="container">
<iframe id="scroll-box" src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV3-48G23%22" scrolling="yes" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:800px;overflow-x: scroll !important;overflow: scroll;" cellspacing="0">
</iframe>
</div>
</div>
<div id="observations-22" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/rvxxlurjsg7eqj5lzvetz/IGHV3-48G23.docx?dl=0&amp;rlkey=92vd0zzpzfvet6px96wkpbdvi" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV3-48G23" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-22" class="section level2">
<h2>Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="22-G22.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="24-G24.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/23-G23.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
