<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>25 IGHV3-49 - G24 | Functional groups</title>
  <meta name="description" content="25 IGHV3-49 - G24 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="25 IGHV3-49 - G24 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="25 IGHV3-49 - G24 | Functional groups" />
  
  
  



<meta name="date" content="2021-11-30" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="23-G23.html"/>
<link rel="next" href="25-G25.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-2-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 2% for all except 0.5% for allele <strong>05</strong></a></li>
<li class="chapter" data-level="3.6.2" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.2</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./"> Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv3-49---g24" class="section level1" number="25">
<h1><span class="header-section-number">25</span> IGHV3-49 - G24</h1>
<div id="allele-appearnce-23" class="section level2" number="25.1">
<h2><span class="header-section-number">25.1</span> Allele appearnce</h2>
<p>The group of IGHV3-49 includes 5 alleles, 5 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-215-1.png" width="672" /></p>
</div>
<div id="group-alignment-20" class="section level2" number="25.2">
<h2><span class="header-section-number">25.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-216-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-23" class="section level2" number="25.3">
<h2><span class="header-section-number">25.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-92e700961b3d8c768e6b" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-92e700961b3d8c768e6b">{"x":{"data":[{"x":[3,2,2,3,1,3,2,1,2,3,2,3,3,1,2,2,1,2,3,1,1,3,2,3,1,3,1,3,2,1,1,1,2,3,1,3,1,2,1,3,1,2,3,3,1,3,3,1,2,3,3,2,1,3,1,1,3,2,3,3,2,1,1,3,2,1,3,2,2,1,1,3,2,3,2,1,3,2,1,1,3,1,3,3,1,1,3,1,3,2,1,3,1,3,3,1,3,1,1,3,1,3,3,2,3,1,3,2,1,3,1,2,3,2,3,2,1,2,3,2,3,2,3,1,2,2,2,1,3,2,3,1,1,3,1,3,2,1,3,1,1,1,1,3,3,1,2,1,1,3,1,3,1,3,3,1,2,1,3,1,3,1,1,3,2,3],"y":[34,30,250,100,100,55,220,209,135,120,144,127,69,62,225,58,27,77,61,281,80,58,109,63,1,25,164,86,97,58,235,242,151,65,123,67,38,31,339,83,238,115,236,2,17,17,135,119,40,15,62,62,54,39,22,85,84,298,217,171,127,162,118,88,97,71,35,34,129,104,34,15,205,110,56,22,338,49,42,241,133,92,62,247,212,134,125,149,117,67,56,208,185,164,298,11,126,116,112,109,67,59,152,110,93,72,64,355,58,25,257,154,116,246,260,166,140,172,164,129,105,244,132,222,1,453,140,134,423,447,87,64,61,42,104,50,176,157,29,23,146,281,224,135,116,108,85,82,122,107,111,93,62,61,149,145,1,84,45,176,11,8,227,146,19,18],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[3.09662800407968,2.09695141622797,2.0154042142909,3.04020441477187,0.920876196632162,3.08870504330844,2.05328492731787,1.03248503981158,1.91021433630958,2.97395919985138,1.96928998646326,2.9506606916897,2.91647709677927,0.923829951602966,1.95356030790135,2.0420759386383,0.979601809103042,2.03610383491032,2.96561542311683,0.973068988742307,1.04056933168322,3.02753431536257,2.08297234829515,2.93287010993809,1.01054629604332,2.90296939322725,1.02645695842803,2.94961555073969,1.99779387726448,0.983528448082507,0.987629919312894,0.90254180226475,1.92107007401064,3.01593212564476,0.91700161062181,2.91302123311907,0.926941368030384,2.07984106098302,1.09665225301869,2.94511827095412,0.996072041615844,2.0852700649295,2.99643629924394,3.07441084566526,0.939961032848805,2.9104423191864,2.92629726785235,1.02510418356396,2.04040254098363,3.06020508804359,2.9269081489183,2.08602166217752,1.03233533897437,2.99417894845828,0.961840216815472,1.03052967498079,3.04313930263743,2.00899536269717,3.091236292338,2.98061902322806,2.00262109395117,0.994798299949616,0.969521872885525,3.08954100282863,2.03246147427708,0.925066689727828,2.95338180018589,2.04980490193702,1.97504160352983,0.95633617346175,0.964227454178035,3.03459634436294,2.07685770574026,3.04335488425568,1.95303030516952,1.08742446624674,3.01682854793034,1.93044981383719,1.06476326109841,0.940263046370819,2.978394255694,1.00903131300583,3.04928233283572,3.08870510510169,0.937362864566967,0.955046582687646,2.95115133128129,0.93794171162881,2.99866079157218,1.900269333506,0.996719930926338,2.93267328171059,1.09639597549103,3.01321196751669,3.06953239846043,1.00374420653097,3.08179174908437,0.941566622490063,0.996780112804845,3.0520922773052,1.09134838539176,2.90386274927296,2.99497141917236,1.92194858817384,2.95274032987654,1.00918053649366,2.94928461862728,1.91061737830751,0.907570592919365,3.00800849013031,0.951539488648996,1.90072297300212,3.03378493967466,1.98545714560896,2.93837455352768,1.96444401317276,1.009477414703,1.95580491684377,2.98353223544545,2.08488184357993,3.09858846920542,2.05318003706634,3.00092051578686,0.939389400510117,2.00995769812725,2.03728317902423,1.93951558461413,0.954947255551815,3.03913588486612,1.95904103969224,3.00672155311331,1.0513402339071,0.991903854999691,2.93050721045583,1.02648114394397,2.9161615521647,2.02376463627443,0.96219991943799,2.98226854563691,1.05958675984293,0.928000432252884,1.02431231262162,0.962141991406679,3.0709401906468,3.00196204162203,0.90049147233367,1.9721756754443,0.923144117789343,0.992265719780698,2.97582937553525,1.03746460718103,3.02082229098305,1.01524337045848,3.0866274304688,2.96024176510982,1.05789012657478,1.95279726488516,1.05878155007958,3.03915587542579,0.920674471883103,2.96661465810612,0.905980367446318,1.0216098042205,3.03682104782201,2.05279242889956,3.0944895205088],"y":[33.764322783798,30.2387834882364,250.212869821675,100.214293287881,100.122483995929,54.7953248783946,220.225295980833,208.716414565407,135.367512948625,119.675624259934,143.953991094418,126.729651599377,69.1067296070978,61.8109845478088,225.221328638121,58.3379863463342,27.1234144385904,77.211755085364,61.2598379710689,281.205041546002,79.676889256388,57.6152479607612,109.087208497524,62.9505240522325,1.29128942657262,24.905402447097,163.861795835569,85.8754504997283,97.2536034977064,58.2793820120394,234.762495215237,242.168753399327,150.649919984117,64.9055878227577,123.30687717665,67.1293057437986,38.2926525140181,30.8231816148385,338.749314452335,83.355004741624,237.776476911828,115.271001453698,236.270280886814,1.88668062705547,17.2325394012034,17.2544088417664,134.75432325881,118.990898555145,39.8131954886019,15.2313836520538,61.6793378867209,62.0601368244737,54.0694059912115,39.3276432730257,21.9562234804034,84.6553297095001,84.1418528953567,298.165668636933,217.38803920038,170.733112556301,126.67362944074,162.391498769633,117.755016747303,88.2937578137964,96.9888934768736,70.8856777748093,34.6538046471775,34.0481170903891,128.719330150634,104.184576582164,33.7005362600088,14.7171299304813,205.186796753295,109.876790787093,56.3971568483859,21.8241194760427,338.039909241907,49.0475756440312,42.2711124910042,240.659314650111,132.999690238759,91.6522121243179,61.9060008393601,247.313478779793,211.978218168207,134.36337216571,125.047331107967,148.968872020021,117.030018732324,67.2857786107808,56.1910823652521,208.329620893486,184.793981464207,164.210009384155,298.114706650935,10.839731294103,126.392042466626,115.612123765424,111.999678028375,109.135572726279,66.8246269352734,58.8086852030829,151.918212531693,109.794319043122,92.8210285689682,72.3219414960593,63.6903026716784,354.885400561243,57.8813865071163,24.7842481175438,257.183601440303,154.216775886156,115.924677062035,245.617109612934,259.686943202093,165.747236013599,140.332267318666,171.747836606205,163.63806799911,128.850320275873,104.772685103118,243.987843982875,132.132798910886,221.995084783994,1.19573782775551,452.612138212845,140.374447873794,134.10177155491,422.728390507773,447.124117915705,86.7891398932785,64.045984306559,61.1026255536824,41.7146900188178,103.954508473538,50.0818009817973,176.311510856822,157.38622619193,28.6029840765521,23.0024576826021,145.964399303682,281.15663737189,223.957279012725,134.803326084279,116.348803382367,108.34148263298,85.1731523631141,81.9273148989305,121.624399829283,107.121893265657,110.608135344833,92.6752939188853,61.9096291562542,60.7115025376901,149.067674983293,144.624806056544,1.24055958259851,83.892139701359,44.966638476029,176.150964069366,11.1258676316589,7.62381795160472,226.694457006827,145.648838852346,19.3009501641616,17.7198170302436],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  05 <\/br># assignments:  34 <\/br>Relative freq.:  0.5312 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  30 <\/br>Relative freq.:  0.4688 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  250 <\/br>Relative freq.:  0.7143 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  05 <\/br># assignments:  100 <\/br>Relative freq.:  0.2857 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  03 <\/br># assignments:  100 <\/br>Relative freq.:  0.6452 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  05 <\/br># assignments:  55 <\/br>Relative freq.:  0.3548 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  220 <\/br>Relative freq.:  0.5128 <\/br>Relative Rep. freq.:  0.0092","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  03 <\/br># assignments:  209 <\/br>Relative freq.:  0.4872 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  135 <\/br>Relative freq.:  0.5294 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  05 <\/br># assignments:  120 <\/br>Relative freq.:  0.4706 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  144 <\/br>Relative freq.:  0.5314 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  05 <\/br># assignments:  127 <\/br>Relative freq.:  0.4686 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  05 <\/br># assignments:  69 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  03 <\/br># assignments:  62 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  225 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0158","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  58 <\/br>Relative freq.:  0.6824 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  03 <\/br># assignments:  27 <\/br>Relative freq.:  0.3176 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  77 <\/br>Relative freq.:  0.558 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  05 <\/br># assignments:  61 <\/br>Relative freq.:  0.442 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  03 <\/br># assignments:  281 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  03 <\/br># assignments:  80 <\/br>Relative freq.:  0.5797 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  05 <\/br># assignments:  58 <\/br>Relative freq.:  0.4203 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  04 <\/br># assignments:  109 <\/br>Relative freq.:  0.6301 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  05 <\/br># assignments:  63 <\/br>Relative freq.:  0.3642 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0058 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  05 <\/br># assignments:  25 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0104","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  03 <\/br># assignments:  164 <\/br>Relative freq.:  0.656 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  05 <\/br># assignments:  86 <\/br>Relative freq.:  0.344 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  97 <\/br>Relative freq.:  0.6258 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  03 <\/br># assignments:  58 <\/br>Relative freq.:  0.3742 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  03 <\/br># assignments:  235 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  03 <\/br># assignments:  242 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0104","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  151 <\/br>Relative freq.:  0.6991 <\/br>Relative Rep. freq.:  0.0144","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  65 <\/br>Relative freq.:  0.3009 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  03 <\/br># assignments:  123 <\/br>Relative freq.:  0.6474 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  05 <\/br># assignments:  67 <\/br>Relative freq.:  0.3526 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  03 <\/br># assignments:  38 <\/br>Relative freq.:  0.5507 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  31 <\/br>Relative freq.:  0.4493 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  03 <\/br># assignments:  339 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  05 <\/br># assignments:  83 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  03 <\/br># assignments:  238 <\/br>Relative freq.:  0.6742 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  115 <\/br>Relative freq.:  0.3258 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  05 <\/br># assignments:  236 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  03 <\/br># assignments:  17 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  05 <\/br># assignments:  17 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  05 <\/br># assignments:  135 <\/br>Relative freq.:  0.5315 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  03 <\/br># assignments:  119 <\/br>Relative freq.:  0.4685 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  04 <\/br># assignments:  40 <\/br>Relative freq.:  0.7273 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  05 <\/br># assignments:  15 <\/br>Relative freq.:  0.2727 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  05 <\/br># assignments:  62 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  04 <\/br># assignments:  62 <\/br>Relative freq.:  0.5345 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  03 <\/br># assignments:  54 <\/br>Relative freq.:  0.4655 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  05 <\/br># assignments:  39 <\/br>Relative freq.:  0.6393 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  03 <\/br># assignments:  22 <\/br>Relative freq.:  0.3607 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  03 <\/br># assignments:  85 <\/br>Relative freq.:  0.503 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  05 <\/br># assignments:  84 <\/br>Relative freq.:  0.497 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  04 <\/br># assignments:  298 <\/br>Relative freq.:  0.5786 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  05 <\/br># assignments:  217 <\/br>Relative freq.:  0.4214 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  05 <\/br># assignments:  171 <\/br>Relative freq.:  0.5738 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  04 <\/br># assignments:  127 <\/br>Relative freq.:  0.4262 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  03 <\/br># assignments:  162 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  03 <\/br># assignments:  118 <\/br>Relative freq.:  0.5728 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  05 <\/br># assignments:  88 <\/br>Relative freq.:  0.4272 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  04 <\/br># assignments:  97 <\/br>Relative freq.:  0.5774 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  03 <\/br># assignments:  71 <\/br>Relative freq.:  0.4226 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  05 <\/br># assignments:  35 <\/br>Relative freq.:  0.5072 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  04 <\/br># assignments:  34 <\/br>Relative freq.:  0.4928 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  04 <\/br># assignments:  129 <\/br>Relative freq.:  0.5536 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  03 <\/br># assignments:  104 <\/br>Relative freq.:  0.4464 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  03 <\/br># assignments:  34 <\/br>Relative freq.:  0.6939 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  05 <\/br># assignments:  15 <\/br>Relative freq.:  0.3061 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  04 <\/br># assignments:  205 <\/br>Relative freq.:  0.6508 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  05 <\/br># assignments:  110 <\/br>Relative freq.:  0.3492 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  04 <\/br># assignments:  56 <\/br>Relative freq.:  0.7179 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  03 <\/br># assignments:  22 <\/br>Relative freq.:  0.2821 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  05 <\/br># assignments:  338 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0092","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  04 <\/br># assignments:  49 <\/br>Relative freq.:  0.5385 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  03 <\/br># assignments:  42 <\/br>Relative freq.:  0.4615 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  03 <\/br># assignments:  241 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  05 <\/br># assignments:  133 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  03 <\/br># assignments:  92 <\/br>Relative freq.:  0.5974 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  05 <\/br># assignments:  62 <\/br>Relative freq.:  0.4026 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  05 <\/br># assignments:  247 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  03 <\/br># assignments:  212 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  03 <\/br># assignments:  134 <\/br>Relative freq.:  0.5174 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  05 <\/br># assignments:  125 <\/br>Relative freq.:  0.4826 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  03 <\/br># assignments:  149 <\/br>Relative freq.:  0.5602 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  05 <\/br># assignments:  117 <\/br>Relative freq.:  0.4398 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  04 <\/br># assignments:  67 <\/br>Relative freq.:  0.5447 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  03 <\/br># assignments:  56 <\/br>Relative freq.:  0.4553 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  05 <\/br># assignments:  208 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  03 <\/br># assignments:  185 <\/br>Relative freq.:  0.5301 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  05 <\/br># assignments:  164 <\/br>Relative freq.:  0.4699 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  05 <\/br># assignments:  298 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  03 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  05 <\/br># assignments:  126 <\/br>Relative freq.:  0.5207 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  03 <\/br># assignments:  116 <\/br>Relative freq.:  0.4793 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  03 <\/br># assignments:  112 <\/br>Relative freq.:  0.5068 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  05 <\/br># assignments:  109 <\/br>Relative freq.:  0.4932 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  03 <\/br># assignments:  67 <\/br>Relative freq.:  0.5317 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  05 <\/br># assignments:  59 <\/br>Relative freq.:  0.4683 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  05 <\/br># assignments:  152 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  04 <\/br># assignments:  110 <\/br>Relative freq.:  0.5419 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  05 <\/br># assignments:  93 <\/br>Relative freq.:  0.4581 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  03 <\/br># assignments:  72 <\/br>Relative freq.:  0.5294 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  05 <\/br># assignments:  64 <\/br>Relative freq.:  0.4706 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  04 <\/br># assignments:  355 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  03 <\/br># assignments:  58 <\/br>Relative freq.:  0.6988 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  05 <\/br># assignments:  25 <\/br>Relative freq.:  0.3012 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  03 <\/br># assignments:  257 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0109","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  04 <\/br># assignments:  154 <\/br>Relative freq.:  0.5704 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  05 <\/br># assignments:  116 <\/br>Relative freq.:  0.4296 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  04 <\/br># assignments:  246 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  05 <\/br># assignments:  260 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  04 <\/br># assignments:  166 <\/br>Relative freq.:  0.5425 <\/br>Relative Rep. freq.:  0.0092","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  03 <\/br># assignments:  140 <\/br>Relative freq.:  0.4575 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  04 <\/br># assignments:  172 <\/br>Relative freq.:  0.5119 <\/br>Relative Rep. freq.:  0.0069","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  05 <\/br># assignments:  164 <\/br>Relative freq.:  0.4881 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  04 <\/br># assignments:  129 <\/br>Relative freq.:  0.5513 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  05 <\/br># assignments:  105 <\/br>Relative freq.:  0.4487 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  04 <\/br># assignments:  244 <\/br>Relative freq.:  0.6489 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  05 <\/br># assignments:  132 <\/br>Relative freq.:  0.3511 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  03 <\/br># assignments:  222 <\/br>Relative freq.:  0.9955 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0045 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  04 <\/br># assignments:  453 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0209","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  04 <\/br># assignments:  140 <\/br>Relative freq.:  0.5109 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  03 <\/br># assignments:  134 <\/br>Relative freq.:  0.4891 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  05 <\/br># assignments:  423 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0129","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  04 <\/br># assignments:  447 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0149","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  05 <\/br># assignments:  87 <\/br>Relative freq.:  0.5762 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  03 <\/br># assignments:  64 <\/br>Relative freq.:  0.4238 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  03 <\/br># assignments:  61 <\/br>Relative freq.:  0.5922 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  05 <\/br># assignments:  42 <\/br>Relative freq.:  0.4078 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  03 <\/br># assignments:  104 <\/br>Relative freq.:  0.6753 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  05 <\/br># assignments:  50 <\/br>Relative freq.:  0.3247 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  04 <\/br># assignments:  176 <\/br>Relative freq.:  0.5285 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  03 <\/br># assignments:  157 <\/br>Relative freq.:  0.4715 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  05 <\/br># assignments:  29 <\/br>Relative freq.:  0.5577 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  03 <\/br># assignments:  23 <\/br>Relative freq.:  0.4423 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  03 <\/br># assignments:  146 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  03 <\/br># assignments:  281 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  03 <\/br># assignments:  224 <\/br>Relative freq.:  0.624 <\/br>Relative Rep. freq.:  0.0069","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  05 <\/br># assignments:  135 <\/br>Relative freq.:  0.376 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  05 <\/br># assignments:  116 <\/br>Relative freq.:  0.5179 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  03 <\/br># assignments:  108 <\/br>Relative freq.:  0.4821 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  85 <\/br>Relative freq.:  0.509 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  03 <\/br># assignments:  82 <\/br>Relative freq.:  0.491 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  03 <\/br># assignments:  122 <\/br>Relative freq.:  0.5328 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  05 <\/br># assignments:  107 <\/br>Relative freq.:  0.4672 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  03 <\/br># assignments:  111 <\/br>Relative freq.:  0.5441 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  05 <\/br># assignments:  93 <\/br>Relative freq.:  0.4559 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  03 <\/br># assignments:  62 <\/br>Relative freq.:  0.5041 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  05 <\/br># assignments:  61 <\/br>Relative freq.:  0.4959 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  05 <\/br># assignments:  149 <\/br>Relative freq.:  0.5051 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  03 <\/br># assignments:  145 <\/br>Relative freq.:  0.4915 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0034 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  03 <\/br># assignments:  84 <\/br>Relative freq.:  0.6512 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  05 <\/br># assignments:  45 <\/br>Relative freq.:  0.3488 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  03 <\/br># assignments:  176 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0139","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  05 <\/br># assignments:  11 <\/br>Relative freq.:  0.5789 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  03 <\/br># assignments:  8 <\/br>Relative freq.:  0.4211 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  03 <\/br># assignments:  227 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  05 <\/br># assignments:  146 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0085","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  04 <\/br># assignments:  19 <\/br>Relative freq.:  0.5135 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  05 <\/br># assignments:  18 <\/br>Relative freq.:  0.4865 <\/br>Relative Rep. freq.:  0.0092"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V3-49*03","V3-49*04","V3-49*05"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V3-49*03","V3-49*04","V3-49*05"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-21.6,475.6],"tickmode":"array","ticktext":["0","100","200","300","400"],"tickvals":[0,100,200,300,400],"categoryorder":"array","categoryarray":["0","100","200","300","400"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"319c857f8e41d":{"x":{},"y":{},"text":{},"type":"box"},"319c86277a1c6":{"x":{},"y":{},"text":{}}},"cur_data":"319c857f8e41d","visdat":{"319c857f8e41d":["function (y) ","x"],"319c86277a1c6":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-58bbdbf5bb581e8b3aab" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-58bbdbf5bb581e8b3aab">{"x":{"data":[{"x":[1,3,3,1,3,3,1,3,2,3,2,3,2,3,1,2,1,3,1,3,1,2,1,3,1,3,2,2,3,3,2,1,1,3,2,1,3,2,2,1,1,2,1,2,2,1,1,3,1,2,1,3,2,1,2,2,1,1,1,2,3,1,1,1,3,3,3,1,2,3,1,1,3,1,2,1],"y":[28,15,7,5,21,5,28,20,26,23,11,9,26,22,48,32,12,4,25,18,15,15,42,38,31,14,57,19,13,10,16,15,55,27,1,35,9,5,18,6,33,26,40,33,36,28,64,17,14,16,11,25,18,31,31,26,23,75,33,14,9,120,43,3,3,21,37,32,1,19,15,25,29,17,21,13],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(86,63,80,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.0282902400475,3.05500758672133,3.0008058488369,0.972186992783099,3.05187386786565,3.0781894936692,1.02887658388354,3.01626657368615,2.04858639207669,2.96554221734405,1.93783597769216,3.00053591048345,1.91047826008871,3.039391444996,0.925147103331983,2.08517181864008,0.903674007067457,3.01926758289337,1.00174183566123,3.07549323742278,1.00271080620587,1.94537886469625,0.949672218877822,3.09903763616458,0.985282890172675,3.02372472928837,2.03798251370899,1.92041859347373,2.93723209844902,2.91317703118548,1.90374866602942,0.966685399645939,0.930421163002029,2.95333773153834,1.93466376299039,0.974755764333531,3.04759438047186,1.9102545004338,1.92041948344558,1.01993194594979,1.04416763382033,1.93088975097053,0.954470737930387,1.92165467119776,2.05468447315507,1.04909799816087,1.07437076033093,2.94135913522914,1.02368612550199,2.05875573772937,1.04371844418347,3.06647616839036,2.07909790845588,1.01723967595026,1.98322598519735,1.9727461787872,1.05667724991217,0.904563898732886,1.0033401359804,2.04247007505037,2.92225962271914,1.01450171065517,0.923102206690237,0.910599492676556,3.0965427483432,2.98685504794121,3.03837331356481,1.03452991927043,1.99507452216931,3.04103567688726,0.928026270167902,1.0732775893528,2.9538116381038,1.09192474815063,1.97287948932499,1.09605064527132],"y":[28.3073537489399,15.2689670242369,7.00261163022369,4.68273684438318,21.0418491464108,4.97720310781151,28.1095294956118,19.8211374022067,26.1945745678619,22.7749564372003,11.1789838707075,9.25281815119088,25.6291392397135,22.2748336879537,47.9346544316038,32.3514333315194,11.6673059813678,4.28839714564383,25.344816984795,17.7670424044132,14.9414126820862,15.151624128595,42.3466881159693,37.6376614393666,31.2908323001117,14.2070762913674,56.7180411720648,18.9073270391673,13.208994820714,9.73106879256666,16.0755361843854,14.9520494418219,54.8577993953601,27.2045250479132,0.913826300948858,34.9005405390635,8.69174012355506,5.3655232232064,18.1335412144661,5.91433562450111,32.661178932339,26.3827702634037,40.2917825562879,32.7590435389429,35.8810002932325,27.7389604346827,64.3429111896083,16.7174330407754,14.255585482344,15.6497478257865,11.3318939637393,24.9632514057681,18.3844891160727,31.0479445885867,30.7208699120209,26.3618429647759,22.7821828082204,74.7552120558917,33.1167665218934,14.1888994447887,9.31513425800949,120.158792628348,42.7115894339979,2.99949546251446,2.89158791266382,20.7857052948326,36.7214469069615,32.3646416269243,1.25882728043944,18.8243752339855,14.8788573900238,25.0844791160896,29.0261534880847,16.6060324160382,21.0476334180683,12.9109161799774],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  03 <\/br># assignments:  28 <\/br>Relative freq.:  0.6512 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  05 <\/br># assignments:  15 <\/br>Relative freq.:  0.3488 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  05 <\/br># assignments:  7 <\/br>Relative freq.:  0.5833 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  03 <\/br># assignments:  5 <\/br>Relative freq.:  0.4167 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  05 <\/br># assignments:  21 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  05 <\/br># assignments:  5 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  03 <\/br># assignments:  28 <\/br>Relative freq.:  0.5833 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  05 <\/br># assignments:  20 <\/br>Relative freq.:  0.4167 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  26 <\/br>Relative freq.:  0.5306 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  05 <\/br># assignments:  23 <\/br>Relative freq.:  0.4694 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  04 <\/br># assignments:  11 <\/br>Relative freq.:  0.55 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  05 <\/br># assignments:  9 <\/br>Relative freq.:  0.45 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  26 <\/br>Relative freq.:  0.5417 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  05 <\/br># assignments:  22 <\/br>Relative freq.:  0.4583 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  03 <\/br># assignments:  48 <\/br>Relative freq.:  0.6 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  32 <\/br>Relative freq.:  0.4 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  03 <\/br># assignments:  12 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  05 <\/br># assignments:  4 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  03 <\/br># assignments:  25 <\/br>Relative freq.:  0.5814 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  05 <\/br># assignments:  18 <\/br>Relative freq.:  0.4186 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  03 <\/br># assignments:  15 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  04 <\/br># assignments:  15 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  03 <\/br># assignments:  42 <\/br>Relative freq.:  0.525 <\/br>Relative Rep. freq.:  0.0069","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  05 <\/br># assignments:  38 <\/br>Relative freq.:  0.475 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  03 <\/br># assignments:  31 <\/br>Relative freq.:  0.6889 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  05 <\/br># assignments:  14 <\/br>Relative freq.:  0.3111 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  57 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  19 <\/br>Relative freq.:  0.5938 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  05 <\/br># assignments:  13 <\/br>Relative freq.:  0.4062 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  05 <\/br># assignments:  10 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  16 <\/br>Relative freq.:  0.5161 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  03 <\/br># assignments:  15 <\/br>Relative freq.:  0.4839 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  03 <\/br># assignments:  55 <\/br>Relative freq.:  0.6627 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  27 <\/br>Relative freq.:  0.3253 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.012 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  03 <\/br># assignments:  35 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  05 <\/br># assignments:  9 <\/br>Relative freq.:  0.6429 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  04 <\/br># assignments:  5 <\/br>Relative freq.:  0.3571 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  18 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  03 <\/br># assignments:  6 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  03 <\/br># assignments:  33 <\/br>Relative freq.:  0.5593 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  04 <\/br># assignments:  26 <\/br>Relative freq.:  0.4407 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  03 <\/br># assignments:  40 <\/br>Relative freq.:  0.5479 <\/br>Relative Rep. freq.:  0.0069","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  33 <\/br>Relative freq.:  0.4521 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  36 <\/br>Relative freq.:  0.5625 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  03 <\/br># assignments:  28 <\/br>Relative freq.:  0.4375 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  03 <\/br># assignments:  64 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  05 <\/br># assignments:  17 <\/br>Relative freq.:  0.5484 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  03 <\/br># assignments:  14 <\/br>Relative freq.:  0.4516 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  04 <\/br># assignments:  16 <\/br>Relative freq.:  0.5926 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  03 <\/br># assignments:  11 <\/br>Relative freq.:  0.4074 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  05 <\/br># assignments:  25 <\/br>Relative freq.:  0.5814 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  04 <\/br># assignments:  18 <\/br>Relative freq.:  0.4186 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  03 <\/br># assignments:  31 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  04 <\/br># assignments:  31 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  26 <\/br>Relative freq.:  0.5306 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  03 <\/br># assignments:  23 <\/br>Relative freq.:  0.4694 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  03 <\/br># assignments:  75 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  03 <\/br># assignments:  33 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  04 <\/br># assignments:  14 <\/br>Relative freq.:  0.6087 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  05 <\/br># assignments:  9 <\/br>Relative freq.:  0.3913 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  03 <\/br># assignments:  120 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  03 <\/br># assignments:  43 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  05 <\/br># assignments:  3 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  05 <\/br># assignments:  21 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  05 <\/br># assignments:  37 <\/br>Relative freq.:  0.5286 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  03 <\/br># assignments:  32 <\/br>Relative freq.:  0.4571 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0143 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  05 <\/br># assignments:  19 <\/br>Relative freq.:  0.5588 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  03 <\/br># assignments:  15 <\/br>Relative freq.:  0.4412 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  03 <\/br># assignments:  25 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  05 <\/br># assignments:  29 <\/br>Relative freq.:  0.6304 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  03 <\/br># assignments:  17 <\/br>Relative freq.:  0.3696 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.6176 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  03 <\/br># assignments:  13 <\/br>Relative freq.:  0.3824 <\/br>Relative Rep. freq.:  0.0018"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(86,63,80,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(86,63,80,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V3-49*03","V3-49*04","V3-49*05"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V3-49*03","V3-49*04","V3-49*05"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-5.04842201542109,126.121040944718],"tickmode":"array","ticktext":["0","25","50","75","100","125"],"tickvals":[0,25,50,75,100,125],"categoryorder":"array","categoryarray":["0","25","50","75","100","125"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"319c865359eb5":{"x":{},"y":{},"text":{},"type":"box"},"319c85a547681":{"x":{},"y":{},"text":{}}},"cur_data":"319c865359eb5","visdat":{"319c865359eb5":["function (y) ","x"],"319c85a547681":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-b5c3d22ca87ac48dc202" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-b5c3d22ca87ac48dc202">{"x":{"data":[{"x":[2,1,2,3,2,1,1,3,2,2,3,1,2,1,3,3,1,1,3,2,2,3,2,3,1,2,1,3,1,1,3,2,1,2,2,1,3,1,2,3,1,1,3,3,1,1,2,3,1,2,1,3,3,1,1,2,3,2,1,2,2,1],"y":[62,19,21,17,63,58,97,76,3,24,4,1,177,76,1,71,2,59,44,2,28,20,90,55,3,41,40,40,4,132,46,3,86,92,56,21,160,30,85,37,1,81,61,199,2,221,1,1,111,1,74,28,13,2,21,19,2,76,75,129,74,30],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.95472445944324,0.957397292973474,1.97602966241539,3.06202737651765,2.0670362656936,1.09833761332557,1.04244356821291,3.06110635651275,2.09299063291401,1.92312259562314,2.90844473610632,0.930433689802885,1.94513584999368,0.918304887646809,3.06579234218225,2.94360201219097,0.95756165413186,0.970348554151133,2.92287682550959,1.92263331855647,1.97057059137151,2.91531718256883,2.01977490214631,2.92401761398651,1.05695045203902,2.0920663212426,0.97279791906476,2.95113175921142,1.04028057721443,0.941850934829563,3.08052390688099,2.02273261989467,0.904679147945717,2.03594074705616,1.9645096974913,1.00769102559425,2.97821121313609,0.935878342529759,2.02520827343687,3.04338343762793,0.944016236253083,1.08661915571429,3.02060863212682,2.97321162885055,1.01665781005286,0.900208911532536,2.09674834241159,2.94071431858465,0.992110446002334,2.06535991462879,0.941801556153223,3.01044610668905,3.05736922812648,1.0014052790124,0.998376898420975,1.90271190418862,2.99439119710587,1.93027742309496,0.948344893334433,1.96363058271818,2.04323476231657,1.07293170522898],"y":[61.8535669799894,19.2707054866478,20.8022589217871,17.0503808075562,63.2690161567181,58.0826940247789,97.0379832694307,75.6647959630936,3.1390366461128,24.0298859499395,4.31773775108159,0.623155754990876,177.281050623581,76.3928324058652,1.29689500294626,71.0124737799168,2.37161764595658,59.0458030618727,43.6261808693409,2.10326482448727,27.7770781062543,19.9573068406433,90.0035576190799,55.3602586679161,2.91026477217674,41.1908408747986,39.7483365591615,40.2968526415527,3.98465591091663,131.687417298742,46.2865922439843,3.20212342236191,86.1130779476836,92.3220946829766,56.380547118932,21.0105656545609,160.335140843503,30.3801838135347,85.3179648138583,37.2698641756549,1.05219891928136,80.8519157713279,60.9047867994756,199.102342192642,2.18477567341179,220.919716596045,0.987079972960055,1.27892157752067,111.331163035706,1.11702123042196,74.3314161432907,27.8077364150435,13.0937358757481,1.60664646886289,20.8904200604185,18.7907205013558,1.92119119614363,76.2758632367477,74.7699108595029,129.159213409014,73.7379448210821,30.1754756301641],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  62 <\/br>Relative freq.:  0.7654 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  03 <\/br># assignments:  19 <\/br>Relative freq.:  0.2346 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  04 <\/br># assignments:  21 <\/br>Relative freq.:  0.5526 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  05 <\/br># assignments:  17 <\/br>Relative freq.:  0.4474 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  63 <\/br>Relative freq.:  0.5207 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  03 <\/br># assignments:  58 <\/br>Relative freq.:  0.4793 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  03 <\/br># assignments:  97 <\/br>Relative freq.:  0.5511 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  05 <\/br># assignments:  76 <\/br>Relative freq.:  0.4318 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.017 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  04 <\/br># assignments:  24 <\/br>Relative freq.:  0.8276 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  05 <\/br># assignments:  4 <\/br>Relative freq.:  0.1379 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0345 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  177 <\/br>Relative freq.:  0.6969 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  03 <\/br># assignments:  76 <\/br>Relative freq.:  0.2992 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0039 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  05 <\/br># assignments:  71 <\/br>Relative freq.:  0.9726 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0274 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  03 <\/br># assignments:  59 <\/br>Relative freq.:  0.5619 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  05 <\/br># assignments:  44 <\/br>Relative freq.:  0.419 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.019 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  04 <\/br># assignments:  28 <\/br>Relative freq.:  0.5833 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  05 <\/br># assignments:  20 <\/br>Relative freq.:  0.4167 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  04 <\/br># assignments:  90 <\/br>Relative freq.:  0.6081 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  05 <\/br># assignments:  55 <\/br>Relative freq.:  0.3716 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.0203 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  04 <\/br># assignments:  41 <\/br>Relative freq.:  0.5062 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  03 <\/br># assignments:  40 <\/br>Relative freq.:  0.4938 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  05 <\/br># assignments:  40 <\/br>Relative freq.:  0.9091 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  03 <\/br># assignments:  4 <\/br>Relative freq.:  0.0909 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  03 <\/br># assignments:  132 <\/br>Relative freq.:  0.7293 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  05 <\/br># assignments:  46 <\/br>Relative freq.:  0.2541 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.0166 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  03 <\/br># assignments:  86 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0221","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  04 <\/br># assignments:  92 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0137","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  04 <\/br># assignments:  56 <\/br>Relative freq.:  0.7273 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  03 <\/br># assignments:  21 <\/br>Relative freq.:  0.2727 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  05 <\/br># assignments:  160 <\/br>Relative freq.:  0.8421 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  03 <\/br># assignments:  30 <\/br>Relative freq.:  0.1579 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  04 <\/br># assignments:  85 <\/br>Relative freq.:  0.6911 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  37 <\/br>Relative freq.:  0.3008 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0081 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  03 <\/br># assignments:  81 <\/br>Relative freq.:  0.5704 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  05 <\/br># assignments:  61 <\/br>Relative freq.:  0.4296 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  05 <\/br># assignments:  199 <\/br>Relative freq.:  0.99 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.01 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  03 <\/br># assignments:  221 <\/br>Relative freq.:  0.991 <\/br>Relative Rep. freq.:  0.0099","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0045 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0045 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  03 <\/br># assignments:  111 <\/br>Relative freq.:  0.9911 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0089 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  03 <\/br># assignments:  74 <\/br>Relative freq.:  0.7255 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  05 <\/br># assignments:  28 <\/br>Relative freq.:  0.2745 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  05 <\/br># assignments:  13 <\/br>Relative freq.:  0.8667 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.1333 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  03 <\/br># assignments:  21 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  19 <\/br>Relative freq.:  0.4524 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  05 <\/br># assignments:  2 <\/br>Relative freq.:  0.0476 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  76 <\/br>Relative freq.:  0.5033 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  03 <\/br># assignments:  75 <\/br>Relative freq.:  0.4967 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  04 <\/br># assignments:  129 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0157","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  74 <\/br>Relative freq.:  0.7115 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  03 <\/br># assignments:  30 <\/br>Relative freq.:  0.2885 <\/br>Relative Rep. freq.:  0.0027"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V3-49*03","V3-49*04","V3-49*05"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V3-49*03","V3-49*04","V3-49*05"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-10.3956864572596,232.01884221225],"tickmode":"array","ticktext":["0","50","100","150","200"],"tickvals":[0,50,100,150,200],"categoryorder":"array","categoryarray":["0","50","100","150","200"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"319c84c899321":{"x":{},"y":{},"text":{},"type":"box"},"319c87358f2c8":{"x":{},"y":{},"text":{}}},"cur_data":"319c84c899321","visdat":{"319c84c899321":["function (y) ","x"],"319c87358f2c8":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-20" class="section level2" number="25.4">
<h2><span class="header-section-number">25.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 6 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV3-49G24%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:1100px;" cellspacing="0">
</iframe>
</div>
<div id="observations-23" class="section level2" number="25.5">
<h2><span class="header-section-number">25.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/9vmbwupbyctmgu6sgy0at/IGHV3-49G24.docx?dl=0&amp;rlkey=p6nqpd246pnppaaqdb566qu76">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV3-49G24" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-23" class="section level2" number="25.6">
<h2><span class="header-section-number">25.6</span> Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="23-G23.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="25-G25.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/24-G24.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
