<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>28 IGHV3-64/IGHV3-64D - G27 | Functional groups</title>
  <meta name="description" content="28 IGHV3-64/IGHV3-64D - G27 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="28 IGHV3-64/IGHV3-64D - G27 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="28 IGHV3-64/IGHV3-64D - G27 | Functional groups" />
  
  
  



<meta name="date" content="2021-11-30" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="26-G26.html"/>
<link rel="next" href="28-G28.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-2-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 2% for all except 0.5% for allele <strong>05</strong></a></li>
<li class="chapter" data-level="3.6.2" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.2</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./"> Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv3-64ighv3-64d---g27" class="section level1" number="28">
<h1><span class="header-section-number">28</span> IGHV3-64/IGHV3-64D - G27</h1>
<div id="allele-appearnce-26" class="section level2" number="28.1">
<h2><span class="header-section-number">28.1</span> Allele appearnce</h2>
<p>The group of IGHV3-64/IGHV3-64D includes 9 alleles, 9 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-237-1.png" width="672" /></p>
</div>
<div id="group-alignment-22" class="section level2" number="28.2">
<h2><span class="header-section-number">28.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-238-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-25" class="section level2" number="28.3">
<h2><span class="header-section-number">28.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-b015121cf54a6f4817cb" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-b015121cf54a6f4817cb">{"x":{"data":[{"x":[4,2,4,1,1,1,4,6,1,2,4,1,2,4,2,4,1,6,2,4,2,1,6,1,2,4,2,5,1,2,5,2,2,1,2,1,2,4,1,2,4,1,2,6,4,1,4,1,3,5,1,2,1,4,2,1,4,1,4,1,3,4,1,1,2,5,1,2,2,4,1,2,3,4,2,4,1,2,1,2,1,2,4,2,1,4,1,2,4,1,2,1,3,6,2,4,1,4,1,2,4,2,4,1,2,2,4,5,1,2,4,5,1,2,4,2,4,1,4,2,4,1,2,5,1,2,3,1,4,1,2,4,1,2,4,1,2,2,5,5,2,1,2,3,5,1,2,5,2,6,5,1,1,2,5,2,6,4,1,2,4,1,2,4,1,2,5,4,5,2,4,1,1,5,2,4,2,5,4,2,4,1,2,6,2,6,1,2,4,1,2,1,4,6,3,1,2,2,1,2,6,4,1,2,1,3,5,2,4,1,2,4,1,2,4,1,3,4,1,2,4,1,4,1,4,2,6,1,3,6,2,1],"y":[79,4,156,116,20,186,100,122,29,8,138,89,4,38,3,46,28,1,4,40,2,42,168,39,7,40,10,1,20,3,14,15,7,34,4,43,4,58,30,3,366,55,7,1,22,5,275,94,1,1,17,1,187,139,12,2,20,3,205,77,1,33,14,11,1,50,11,2,1,123,53,4,1,165,15,190,27,7,31,3,41,4,89,2,7,83,25,4,26,3,10,57,1,173,7,57,25,174,65,4,45,1,166,46,1,5,89,89,28,3,130,109,31,4,161,3,59,39,163,7,205,29,3,1,66,3,3,7,6,44,1,84,30,1,85,28,2,8,106,74,6,81,6,1,89,9,2,92,5,1,167,112,66,3,128,5,131,109,49,8,165,85,6,133,47,2,1,208,75,14,116,49,171,130,6,120,12,1,187,10,316,62,4,172,10,98,33,3,89,27,3,148,146,138,2,12,5,5,60,7,205,180,110,7,102,1,70,5,161,40,4,128,38,6,53,37,1,179,60,5,35,20,92,61,14,1,106,66,1,89,10,3],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[3.90092905643396,2.00312111694366,4.0940785610117,1.01316870423034,0.935169193567708,1.05648945756257,3.96078962166794,5.95277401767671,0.986190795060247,2.04229107634164,3.92752045257948,0.97128713298589,1.99725168622099,3.97224393719807,2.0683654434979,4.08119752253406,0.935889764362946,6.08526496351697,2.0348370670341,3.92840067385696,1.96892911684699,0.941916332254186,6.02741938717663,1.02364664454944,1.99570883377455,4.03455416359939,2.0226713991724,4.97903001816012,0.955312704574317,1.95578279113397,5.00259962095879,2.09912478695624,2.09684184612706,1.00685618794523,1.99084119582549,0.960339157935232,2.07316070403904,4.00001525473781,0.981864548288286,1.91421516081318,3.9920583339408,1.04615347203799,1.99600578253157,6.02233205344528,3.92209438453428,1.09304411718622,4.06156846820377,1.0158450325951,3.03067247630097,4.94569680760615,1.06051687519066,1.96065587284975,1.00728399967775,4.06139653865248,2.01392857129686,1.0458028703928,4.00025187525898,0.927830749331042,4.0644127803389,1.00264131966978,2.92566126124002,4.03837986546569,1.06047841189429,0.920129939680919,2.0302477824036,4.94715592688881,0.972586234798655,1.93405385604128,1.95634387577884,4.05010600890964,1.08802114916034,1.9320406679064,3.01454981234856,4.00787336095236,2.01365915774368,3.98936127936468,1.09824577905238,1.9499428967014,0.913735390407965,2.00437957928516,1.09690086450428,2.09717232077382,3.90373823964037,1.91172422231175,0.971090478636324,4.039269622555,0.950197396567091,2.03713595457375,3.99502713307738,0.92391770076938,1.91504267887212,1.0828905583825,3.05828797961585,5.95175013113767,1.90390172735788,3.9982369965408,1.09107025871053,3.96340407230891,1.03816266790964,1.91202386501245,3.94634460150264,2.07108217920177,4.04043947281316,1.03425647369586,2.00853847395629,1.93349161772057,4.0488443359267,4.95471440055408,1.04478443083353,2.0056208728347,3.95970764807425,5.07310678106733,0.923155047325417,1.96181090250611,3.99906982202083,2.01237326147966,4.0237722559832,1.03570566731505,3.96613741330802,2.0481842759531,4.01179661061615,1.06294610747136,1.98179761231877,5.05810714010149,0.962843083776534,2.03905988098122,3.02357298322022,1.07266303221695,3.9816689198371,1.08687461386435,1.91306094708852,3.93460328970104,0.986158059490845,1.96405935646035,4.09257946163416,0.983236446883529,1.90472063221969,1.98369917366654,5.07085395287722,5.08669992168434,2.05180230634287,0.99808973246254,2.06522378134541,3.04594292831607,4.91986764571629,1.03275873400271,1.93750074882992,4.9720507632941,2.00639886199497,5.94244666337036,4.94716339707375,1.04313699174672,1.09844534960575,2.05251922565512,4.92125040050596,1.9112402736675,6.08918546126224,4.05432860343717,0.924589507980272,1.9692990809679,3.96703799692914,0.946876736171544,2.06757207587361,3.90633013402112,1.05139381615445,2.01750415754505,4.93858464299701,3.96325860219076,4.90558654516935,1.93818097659387,3.96210784534924,0.992382223065942,0.976855098642409,5.05323746381328,1.95525391260162,4.09860892770812,1.94936127201654,5.02434258968569,4.09240223313682,2.07170392721891,3.9409855148755,0.915782115980983,2.06266553597525,6.05438742386177,2.03385823303834,5.90265153138898,0.909471303550527,2.05197289520875,3.9392110702116,1.05761814066209,2.0645714727696,0.927469287253916,3.98743272144347,5.97035480854101,2.91951762060635,0.9200713169761,1.95405941125937,2.0662192851305,0.949987399764359,2.0982907618396,5.95982640604489,3.94214879251085,1.07111124102958,1.94513816018589,1.07192699392326,3.08004723223858,5.09869523607194,2.03306451234967,3.94363649804145,1.01594961429946,2.00142159522511,3.90975473038852,1.05483993375674,1.90349270328879,3.94125117212534,0.9641449439805,3.06864476953633,4.04031226332299,0.986577104963362,1.94562910813838,3.96719449283555,1.0215542091988,4.01629610294476,0.917510752379894,4.0650324115064,1.97400202439167,5.92140100067481,0.901241637207568,2.98628177470528,6.07418853119016,2.01723826401867,0.979144260054454],"y":[79.3060161856934,3.90817438159138,156.202822864428,115.682568671554,19.9768537852913,186.00310795065,100.304137528688,122.185453891009,28.683381173946,7.75125771984458,137.651050338335,88.6629353415221,4.04558785781264,37.6870974507183,2.88849365785718,46.1286817420274,27.9812961954623,0.643456782400608,3.65155875887722,39.974530380778,2.31476775277406,42.1672829432413,167.877786003426,38.6361295932904,6.81166810188442,40.3722285170108,9.81260475944728,1.18595979791135,19.8221990603954,2.82120701223612,14.0492830723524,14.8444115795195,7.18101040050387,34.0716168746352,4.05120122618973,42.7583509504795,4.13571745101362,57.7517952488735,30.1843877943233,2.65030398052186,365.778994185664,55.2672730438411,6.68824376855046,0.729221957921982,21.8133291844279,4.9352989083156,275.298735796474,93.8068785091862,1.10880238842219,1.14446596149355,17.271855664812,0.889737379550934,187.353297372162,139.315719340928,12.208533233963,2.27998116761446,19.6383345669135,2.86442437563092,204.882251943648,77.1113809799776,0.904141031205654,32.931658703275,14.1692582320422,11.0055466627702,0.938763042166829,50.2324497802183,10.8819295534864,1.98677920009941,1.24100406058133,122.893405145966,53.151369214803,3.9719626981765,1.35067832749337,164.782194222137,14.8112284166738,190.283582659811,26.7894927039742,6.68687020242214,31.0812463853508,3.23084850776941,40.9535228114575,3.68942381031811,89.0190546141937,1.63486412838101,6.66639097612351,83.0751299995929,25.0660962020978,4.32097776625305,25.727515688166,3.21650204900652,10.2535114821047,56.6827769285068,0.720901514403522,172.788267462328,6.71637489143759,57.1448046263307,25.0938815308735,173.870588500984,64.7192875057459,4.16001915950328,44.8366523673758,0.871414454095066,166.173616395891,45.6565020905808,1.24179812222719,5.07792223524302,89.0174877764657,89.2430775295943,27.6881807388738,2.73879292942584,129.631223848835,108.878183414415,30.6074681455269,4.01711369324476,161.130974861421,2.62197713926435,58.9299056040123,39.167908744514,163.11587775629,6.89969911873341,204.843314957619,28.7614063566551,3.25670486912131,0.623327847383916,65.7340176871046,3.29163278583437,3.12175213973969,7.09447145462036,6.21356869284064,43.6805576348677,0.810167546570301,84.140288537927,30.3091818736866,0.683641478791833,85.3620671637356,28.2796038253233,2.05656383857131,8.35497266221791,106.127259122021,74.3511259468272,6.16708487197757,80.8622869223356,6.20501627884805,1.07890279516578,89.2877308744937,8.76339077427983,2.10485222712159,92.3407096307725,4.86607087031007,0.717554031871259,166.694813307375,111.917201698758,65.7239581955597,3.2821513498202,128.18886175938,5.29209379144013,131.078726300225,108.801501013711,49.0455634800717,8.17848743256181,164.683586467803,85.3413094149902,6.23057955466211,132.616011413559,47.2533522700891,2.36290970537812,0.811847081407905,207.889029444754,75.1998842306435,14.3342430602759,115.923549283855,48.6592372216284,170.699269225076,130.084274238162,6.09971664622426,119.742795029283,12.3412276564166,0.68855768814683,186.808068593219,10.3686294145882,316.306152939796,61.782834312506,3.94673869777471,172.304159142077,9.8315447602421,98.0005041820928,33.2918679255992,3.02625048886985,89.278815753758,27.0410969166085,3.31963782645762,148.143102978542,145.690988566913,137.934612840042,2.08348906412721,12.3561709154397,5.06043875999749,4.73495654165745,59.6441738318652,7.12063357811421,204.713985810243,180.210027511045,109.720148829743,6.98994150534272,101.628443247452,1.0874157583341,69.8893275266513,5.36242067124695,160.773716581799,39.7197405621409,3.68483355324715,127.647512538917,37.83465669211,6.1453319132328,52.6243907554075,37.1557334212586,1.15827079638839,178.963518302701,59.6837994910777,5.37564735654742,35.0906325994059,19.6819759935141,92.1756859835237,60.9966015020385,14.0313917001709,1.29701351169497,105.946195169538,66.3931705433875,1.20269530452788,88.8255480252206,9.89794500675052,3.26323922239244],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  07 <\/br># assignments:  79 <\/br>Relative freq.:  0.9518 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0482 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  07 <\/br># assignments:  156 <\/br>Relative freq.:  0.5735 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  116 <\/br>Relative freq.:  0.4265 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  186 <\/br>Relative freq.:  0.6503 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  07 <\/br># assignments:  100 <\/br>Relative freq.:  0.3497 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  09 <\/br># assignments:  122 <\/br>Relative freq.:  0.7673 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.1824 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.0503 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  07 <\/br># assignments:  138 <\/br>Relative freq.:  0.5974 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  89 <\/br>Relative freq.:  0.3853 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0173 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  07 <\/br># assignments:  38 <\/br>Relative freq.:  0.9268 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0732 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  07 <\/br># assignments:  46 <\/br>Relative freq.:  0.6133 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.3733 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0133 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  07 <\/br># assignments:  40 <\/br>Relative freq.:  0.9524 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0476 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  09 <\/br># assignments:  168 <\/br>Relative freq.:  0.785 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  0.1822 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.0327 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  07 <\/br># assignments:  40 <\/br>Relative freq.:  0.7843 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.1961 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  08 <\/br># assignments:  1 <\/br>Relative freq.:  0.0196 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.8696 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.1304 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  08 <\/br># assignments:  14 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  15 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  0.8947 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.1053 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  43 <\/br>Relative freq.:  0.9149 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0851 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  07 <\/br># assignments:  58 <\/br>Relative freq.:  0.6374 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  0.3297 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.033 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  07 <\/br># assignments:  366 <\/br>Relative freq.:  0.8531 <\/br>Relative Rep. freq.:  0.0142","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  55 <\/br>Relative freq.:  0.1282 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.0163 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  07 <\/br># assignments:  22 <\/br>Relative freq.:  0.8148 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  0.1852 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  07 <\/br># assignments:  275 <\/br>Relative freq.:  0.7412 <\/br>Relative Rep. freq.:  0.0122","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  94 <\/br>Relative freq.:  0.2534 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  08 <\/br># assignments:  1 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  17 <\/br>Relative freq.:  0.9444 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0556 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  187 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  07 <\/br># assignments:  139 <\/br>Relative freq.:  0.9205 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  0.0795 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  07 <\/br># assignments:  20 <\/br>Relative freq.:  0.8696 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  0.1304 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  07 <\/br># assignments:  205 <\/br>Relative freq.:  0.7244 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  77 <\/br>Relative freq.:  0.2721 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0035 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  07 <\/br># assignments:  33 <\/br>Relative freq.:  0.7021 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  0.2979 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.9167 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0833 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  08 <\/br># assignments:  50 <\/br>Relative freq.:  0.7937 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.1746 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0317 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  07 <\/br># assignments:  123 <\/br>Relative freq.:  0.6796 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  53 <\/br>Relative freq.:  0.2928 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0221 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0055 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  07 <\/br># assignments:  165 <\/br>Relative freq.:  0.9167 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  02 <\/br># assignments:  15 <\/br>Relative freq.:  0.0833 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  07 <\/br># assignments:  190 <\/br>Relative freq.:  0.8482 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.1205 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.0312 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  31 <\/br>Relative freq.:  0.9118 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0882 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  0.9111 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0889 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  07 <\/br># assignments:  89 <\/br>Relative freq.:  0.978 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.022 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  07 <\/br># assignments:  83 <\/br>Relative freq.:  0.7411 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  25 <\/br>Relative freq.:  0.2232 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0357 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  07 <\/br># assignments:  26 <\/br>Relative freq.:  0.8966 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  0.1034 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  0.9828 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0172 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  09 <\/br># assignments:  173 <\/br>Relative freq.:  0.9611 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.0389 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  07 <\/br># assignments:  57 <\/br>Relative freq.:  0.6951 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  25 <\/br>Relative freq.:  0.3049 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  07 <\/br># assignments:  174 <\/br>Relative freq.:  0.716 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  65 <\/br>Relative freq.:  0.2675 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0165 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  07 <\/br># assignments:  45 <\/br>Relative freq.:  0.9783 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0217 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  07 <\/br># assignments:  166 <\/br>Relative freq.:  0.7793 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.216 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  07 <\/br># assignments:  89 <\/br>Relative freq.:  0.4258 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  08 <\/br># assignments:  89 <\/br>Relative freq.:  0.4258 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.134 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0144 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  07 <\/br># assignments:  130 <\/br>Relative freq.:  0.4745 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  08 <\/br># assignments:  109 <\/br>Relative freq.:  0.3978 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  31 <\/br>Relative freq.:  0.1131 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0146 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  07 <\/br># assignments:  161 <\/br>Relative freq.:  0.9817 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0183 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  07 <\/br># assignments:  59 <\/br>Relative freq.:  0.602 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  0.398 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  07 <\/br># assignments:  163 <\/br>Relative freq.:  0.9588 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.0412 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  07 <\/br># assignments:  205 <\/br>Relative freq.:  0.8613 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.1218 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0126 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  08 <\/br># assignments:  1 <\/br>Relative freq.:  0.0042 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  66 <\/br>Relative freq.:  0.9167 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0417 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.0417 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.5385 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  07 <\/br># assignments:  6 <\/br>Relative freq.:  0.4615 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  44 <\/br>Relative freq.:  0.9778 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0222 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  07 <\/br># assignments:  84 <\/br>Relative freq.:  0.7304 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  0.2609 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0087 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  07 <\/br># assignments:  85 <\/br>Relative freq.:  0.7391 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.2435 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0174 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  08 <\/br># assignments:  106 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  08 <\/br># assignments:  74 <\/br>Relative freq.:  0.925 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.075 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  81 <\/br>Relative freq.:  0.9205 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0682 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0114 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  08 <\/br># assignments:  89 <\/br>Relative freq.:  0.89 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  0.09 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.02 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  08 <\/br># assignments:  92 <\/br>Relative freq.:  0.9388 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.051 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0102 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  08 <\/br># assignments:  167 <\/br>Relative freq.:  0.5986 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  112 <\/br>Relative freq.:  0.4014 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  66 <\/br>Relative freq.:  0.9565 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0435 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  08 <\/br># assignments:  128 <\/br>Relative freq.:  0.9624 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0376 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  09 <\/br># assignments:  131 <\/br>Relative freq.:  0.4411 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  07 <\/br># assignments:  109 <\/br>Relative freq.:  0.367 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.165 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.0269 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  07 <\/br># assignments:  165 <\/br>Relative freq.:  0.6445 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  85 <\/br>Relative freq.:  0.332 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0234 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  07 <\/br># assignments:  133 <\/br>Relative freq.:  0.7268 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  47 <\/br>Relative freq.:  0.2568 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0109 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  08 <\/br># assignments:  1 <\/br>Relative freq.:  0.0055 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  07 <\/br># assignments:  208 <\/br>Relative freq.:  0.7003 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  08 <\/br># assignments:  75 <\/br>Relative freq.:  0.2525 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  02 <\/br># assignments:  14 <\/br>Relative freq.:  0.0471 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  07 <\/br># assignments:  116 <\/br>Relative freq.:  0.703 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.297 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  171 <\/br>Relative freq.:  0.557 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  08 <\/br># assignments:  130 <\/br>Relative freq.:  0.4235 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0195 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  07 <\/br># assignments:  120 <\/br>Relative freq.:  0.9023 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  0.0902 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  08 <\/br># assignments:  1 <\/br>Relative freq.:  0.0075 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  07 <\/br># assignments:  187 <\/br>Relative freq.:  0.9492 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.0508 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  07 <\/br># assignments:  316 <\/br>Relative freq.:  0.8272 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  01 <\/br># assignments:  62 <\/br>Relative freq.:  0.1623 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0105 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  09 <\/br># assignments:  172 <\/br>Relative freq.:  0.9451 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.0549 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  09 <\/br># assignments:  98 <\/br>Relative freq.:  0.7313 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  0.2463 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0224 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  07 <\/br># assignments:  89 <\/br>Relative freq.:  0.7479 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.2269 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0252 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  01 <\/br># assignments:  148 <\/br>Relative freq.:  0.341 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  07 <\/br># assignments:  146 <\/br>Relative freq.:  0.3364 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  09 <\/br># assignments:  138 <\/br>Relative freq.:  0.318 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.0046 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.7059 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.2941 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.8955 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.1045 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  09 <\/br># assignments:  205 <\/br>Relative freq.:  0.4084 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  07 <\/br># assignments:  180 <\/br>Relative freq.:  0.3586 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  110 <\/br>Relative freq.:  0.2191 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.0139 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  01 <\/br># assignments:  102 <\/br>Relative freq.:  0.9903 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0097 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  08 <\/br># assignments:  70 <\/br>Relative freq.:  0.9333 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0667 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  07 <\/br># assignments:  161 <\/br>Relative freq.:  0.7854 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  40 <\/br>Relative freq.:  0.1951 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0195 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  07 <\/br># assignments:  128 <\/br>Relative freq.:  0.7442 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  38 <\/br>Relative freq.:  0.2209 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0349 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  07 <\/br># assignments:  53 <\/br>Relative freq.:  0.5824 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  37 <\/br>Relative freq.:  0.4066 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.011 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  07 <\/br># assignments:  179 <\/br>Relative freq.:  0.7336 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.2459 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0205 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  07 <\/br># assignments:  35 <\/br>Relative freq.:  0.6364 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.3636 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  07 <\/br># assignments:  92 <\/br>Relative freq.:  0.6013 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  61 <\/br>Relative freq.:  0.3987 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  07 <\/br># assignments:  14 <\/br>Relative freq.:  0.9333 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0667 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  09 <\/br># assignments:  106 <\/br>Relative freq.:  0.6127 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  66 <\/br>Relative freq.:  0.3815 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0058 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  09 <\/br># assignments:  89 <\/br>Relative freq.:  0.899 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.101 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0015"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":97.5674553756745,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["V3-64*01","V3-64*02","V3-64*07","V3-64D*06","V3-64D*08","V3-64D*09"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["V3-64*01","V3-64*02","V3-64*07","V3-64D*06","V3-64D*08","V3-64D*09"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-17.6455057602469,384.268833607631],"tickmode":"array","ticktext":["0","100","200","300"],"tickvals":[0,100,200,300],"categoryorder":"array","categoryarray":["0","100","200","300"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"319c879fc6c66":{"x":{},"y":{},"text":{},"type":"box"},"319c878636851":{"x":{},"y":{},"text":{}}},"cur_data":"319c879fc6c66","visdat":{"319c879fc6c66":["function (y) ","x"],"319c878636851":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-f37c6093d9f393f76033" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-f37c6093d9f393f76033">{"x":{"data":[{"x":[6,1,8,6,8,6,8,6,2,6,2,8,6,1,6,7,1,2,6,1,6,2,8,1,6,1,2,8,1,2,6,1,2,2,8,1,6,8,1,2,4,5,6,1,6,1,1,6,2,7,8,6,1,6,1,8,1,2,1,6,6,8,1,1,1,1,8,6,1,2,6,1,1,6,6,8,2,3,6,8,2,6,8,1,8,2],"y":[14,8,5,1,11,9,7,6,1,33,1,37,12,4,33,28,15,1,5,1,25,1,17,2,27,10,1,65,7,1,18,6,2,1,15,1,51,46,5,1,1,1,14,4,9,3,6,65,1,1,39,19,10,28,5,36,4,1,2,14,18,20,6,6,6,3,9,8,5,2,76,12,1,1,5,45,1,1,16,14,1,19,19,8,13,1],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(151,152,82,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[5.99529544757679,1.04675551285036,7.90246594422497,5.96910048904829,7.9207138102036,6.06477361894213,8.09114816170186,6.00109303425998,2.02019113367423,6.04266356220469,1.91845461153425,8.01668988205492,5.96791184414178,0.938760808575898,5.92751855337992,7.04563243021257,0.911553270602599,2.09426302802749,5.95512337610126,0.988155328854919,5.93642257465981,1.95130769261159,8.02654366130009,0.962716652546078,5.91542690154165,1.09787070667371,2.04926931015216,8.09696746864356,0.961384261632338,1.97872639857233,6.02942434707657,0.944362420449033,1.91465611108579,1.95566861848347,7.98987676696852,1.0703210350126,5.97764540840872,8.07353691970929,0.916407794784755,1.97353255264461,4.02629714007489,4.97367870719172,5.96237015984952,0.95777624421753,6.0086667584721,0.938997000223026,1.06938680205494,5.90397038063966,2.09268666598946,7.04256457621232,8.03213551854715,6.03910690536723,1.01338113793172,6.04743630955927,0.958383090887219,8.00330285513774,0.903995613567531,2.08752468205057,0.934053754713386,5.9841163686011,5.97407678454183,7.92231854721904,1.07352549685165,1.01608865731396,0.929819670831785,0.902277385583147,7.95207231603563,6.05078617022373,0.975050654541701,1.98749750950374,6.09139814726077,1.00533909937367,1.02898033908568,5.9684451890178,6.09515796150081,7.99858992439695,1.95646726321429,2.99864405840635,5.9819507541135,8.06380418310873,1.96952401585877,5.94396569612436,7.96108969585039,0.980369388312101,7.98467810065486,2.00850965003483],"y":[14.3278948584571,8.05705840177834,5.32970390319824,0.888296001963317,10.6220570353791,8.64425776638091,7.08127851039171,6.00301935859025,0.699000093899667,33.2666658788919,1.11062893886119,37.0386120738462,12.2619825931266,3.91834899559617,33.3796362070367,27.8942014005035,14.8499883919954,1.17954031135887,4.91095236819237,0.833821828849614,24.8882481381297,0.825403994321823,16.9429455297068,2.1706315740943,27.0096862893552,9.60212239772081,0.810294160060585,64.7337853081524,6.81088249459863,0.983923063986003,17.70298414547,6.26878899484873,2.09437749590725,0.641823734715581,15.1058485506102,0.937886337190866,50.9607766484842,45.6284398900345,4.8425732228905,0.910134946741164,1.09785608742386,0.980615059658885,13.7474600249901,3.93238227386028,9.13026616796851,2.93367738258094,5.60124291051179,65.3229589389637,1.0534013453871,0.91907470151782,39.3661583980545,18.616975719668,9.8701160358265,28.3043404338881,5.03667231276631,35.6360787944868,3.77813376244158,0.752858365699649,2.29169571977109,14.0652825251222,18.2081222690642,20.1445152537897,6.27092671599239,6.0395311107859,6.10462297797203,3.14176628384739,8.89371049869806,8.31213538143784,4.81003234088421,1.84753145724535,76.0335416259244,12.022114504315,1.2537854488939,0.826255640201271,4.84553662464023,44.6728935427964,1.16668060421944,1.25487216766924,15.7669734640047,14.3102596458048,0.689271920546889,19.2396169705316,19.1538398860022,7.65708882380277,13.3646193446592,0.833153044246137],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  07 <\/br># assignments:  14 <\/br>Relative freq.:  0.6364 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.3636 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  09 <\/br># assignments:  5 <\/br>Relative freq.:  0.8333 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  09 <\/br># assignments:  11 <\/br>Relative freq.:  0.55 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  07 <\/br># assignments:  9 <\/br>Relative freq.:  0.45 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  09 <\/br># assignments:  7 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  07 <\/br># assignments:  6 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0714 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  07 <\/br># assignments:  33 <\/br>Relative freq.:  0.9706 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0294 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  09 <\/br># assignments:  37 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0062","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  07 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  07 <\/br># assignments:  33 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  08 <\/br># assignments:  28 <\/br>Relative freq.:  0.3636 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  15 <\/br>Relative freq.:  0.1948 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.013 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  07 <\/br># assignments:  5 <\/br>Relative freq.:  0.8333 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  07 <\/br># assignments:  25 <\/br>Relative freq.:  0.9615 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0385 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  09 <\/br># assignments:  17 <\/br>Relative freq.:  0.8947 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.1053 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  07 <\/br># assignments:  27 <\/br>Relative freq.:  0.7105 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  10 <\/br>Relative freq.:  0.2632 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0263 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  09 <\/br># assignments:  65 <\/br>Relative freq.:  0.8904 <\/br>Relative Rep. freq.:  0.0164","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.0959 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0137 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  07 <\/br># assignments:  18 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  09 <\/br># assignments:  15 <\/br>Relative freq.:  0.9375 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0625 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  07 <\/br># assignments:  51 <\/br>Relative freq.:  0.4857 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  09 <\/br># assignments:  46 <\/br>Relative freq.:  0.4381 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  0.0476 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0095 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  05 <\/br># assignments:  1 <\/br>Relative freq.:  0.0095 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0095 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  07 <\/br># assignments:  14 <\/br>Relative freq.:  0.7778 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  0.2222 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  07 <\/br># assignments:  9 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  07 <\/br># assignments:  65 <\/br>Relative freq.:  0.9701 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0149 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  08 <\/br># assignments:  1 <\/br>Relative freq.:  0.0149 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  09 <\/br># assignments:  39 <\/br>Relative freq.:  0.5735 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  07 <\/br># assignments:  19 <\/br>Relative freq.:  0.2794 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  10 <\/br>Relative freq.:  0.1471 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  07 <\/br># assignments:  28 <\/br>Relative freq.:  0.8485 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  0.1515 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  09 <\/br># assignments:  36 <\/br>Relative freq.:  0.878 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  0.0976 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0244 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  07 <\/br># assignments:  14 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  07 <\/br># assignments:  18 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  09 <\/br># assignments:  20 <\/br>Relative freq.:  0.7692 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  0.2308 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  09 <\/br># assignments:  9 <\/br>Relative freq.:  0.5294 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  07 <\/br># assignments:  8 <\/br>Relative freq.:  0.4706 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  0.7143 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.2857 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  07 <\/br># assignments:  76 <\/br>Relative freq.:  0.8636 <\/br>Relative Rep. freq.:  0.0174","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.1364 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  07 <\/br># assignments:  5 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  09 <\/br># assignments:  45 <\/br>Relative freq.:  0.9574 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0213 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0213 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  07 <\/br># assignments:  16 <\/br>Relative freq.:  0.5161 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  09 <\/br># assignments:  14 <\/br>Relative freq.:  0.4516 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0323 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  07 <\/br># assignments:  19 <\/br>Relative freq.:  0.413 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  09 <\/br># assignments:  19 <\/br>Relative freq.:  0.413 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.1739 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  09 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  1e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(151,152,82,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(151,152,82,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":97.5674553756745,"l":34.8692403486924},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,8.6],"tickmode":"array","ticktext":["V3-64*01","V3-64*02","V3-64*04","V3-64*05","V3-64*07","V3-64D*06","V3-64D*08","V3-64D*09"],"tickvals":[1,2,3,4,5,6,7,8],"categoryorder":"array","categoryarray":["V3-64*01","V3-64*02","V3-64*04","V3-64*05","V3-64*07","V3-64D*06","V3-64D*08","V3-64D*09"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-3.12776215984486,79.8031275204848],"tickmode":"array","ticktext":["0","20","40","60"],"tickvals":[0,20,40,60],"categoryorder":"array","categoryarray":["0","20","40","60"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"319c822b6422a":{"x":{},"y":{},"text":{},"type":"box"},"319c86b5f96f8":{"x":{},"y":{},"text":{}}},"cur_data":"319c822b6422a","visdat":{"319c822b6422a":["function (y) ","x"],"319c86b5f96f8":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-c516d86fe67470fa2dee" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-c516d86fe67470fa2dee">{"x":{"data":[{"x":[2,4,6,1,3,2,5,4,1,2,3,2,1,4,5,3,6,1,5,4,1,2,3,2,2,2,5,4,2,1,3,2,1,2,3,1,4,3,1,3,2,2,1,3,4,1,3,6,4,2,5,1,3,6,5,1,3,2,4,2,1,3,1,4,3,5,1,2,3,1,4,3],"y":[3,15,82,25,2,1,71,47,35,4,3,1,62,55,8,3,72,18,101,44,14,2,2,3,4,5,44,35,1,34,20,2,8,2,2,33,25,5,16,2,2,8,83,5,123,53,4,1,172,11,3,49,7,63,56,41,9,5,2,6,19,3,70,53,3,2,19,1,1,39,36,2],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(52,134,141,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.95996370040812,3.93911820426583,6.06553312470205,0.914162028860301,3.09521371894516,2.09777656211518,5.08770179129206,4.09408252844587,1.03603138583712,1.97872513541952,3.07231199191883,1.99471631911583,1.09011086048558,4.01828544614837,4.934976983862,3.04336499604397,6.09266015696339,0.956785346427932,4.94343628166243,4.06729380534962,0.907881254050881,1.97341947839595,3.00345494216308,1.91829783613794,2.03251319550909,2.09400771423243,4.96539071192965,4.09285150291398,1.95148220355622,1.0192206135951,3.02368242316879,2.05073093925603,0.924384643509984,1.95366771784611,2.95075429519638,0.905976366763934,3.93223318629898,2.97962507777847,1.06908449479379,2.99676825753413,2.08904192149639,1.92181123718619,1.08805867498741,2.95657099313103,3.98136814525351,0.90038873758167,3.00361968046054,5.94607794578187,4.01363174878061,2.02786381854676,5.06265086396597,0.923709830548614,2.93470136774704,6.09344774642959,4.98738308856264,1.04192265388556,2.90669192504138,2.07325261626393,3.94993496187963,1.97233189432882,1.09854148076847,3.05632393835112,1.04395185424946,3.97633969625458,2.93839329155162,5.04231773396023,0.981265158159658,1.9027340780478,2.9265141276177,0.909971350105479,3.92158151119947,2.96763624297455],"y":[2.63354017306119,15.3526459837332,81.7097137184814,25.1589939398691,1.70948574617505,0.754980381205678,71.2612594371662,46.7176920643076,34.8667956329882,4.36054646652192,3.19545953776687,1.39329650849104,62.0050969809294,55.3313862953335,7.78956110216677,3.02992348242551,72.0330856295302,17.9575588768348,101.162444361486,44.0517763918266,14.0588275752962,2.12570949774235,2.17270183973014,3.22689313795418,4.12487256806344,4.68429989013821,44.0209447307512,35.3175536712632,0.8789445662871,34.3213525228202,19.7267937544733,2.1454065695405,7.93670906927437,1.75228450261056,2.29971077311784,33.1460497200489,25.3778414499015,5.0110926900059,16.1936730323359,1.78543533682823,1.87195347156376,7.89270422980189,82.8805973449722,4.9079562516883,123.106928943284,53.3763831619173,3.97362684011459,0.710369717516005,171.752705006115,11.2944315930828,3.29340471588075,49.1734898941591,7.31306336894631,63.1983829701319,56.2207536865026,41.2099908646196,9.12663233168423,5.05640262067318,1.67976982444525,6.09898003544658,18.7000916428864,3.29760804437101,69.9410621752963,53.2466440809891,2.66456502471119,1.96892341915518,19.1421373261139,1.25467929914594,1.08876069411635,38.7157805437222,36.0972942808643,2.1210113350302],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  07 <\/br># assignments:  15 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  09 <\/br># assignments:  82 <\/br>Relative freq.:  0.7455 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  25 <\/br>Relative freq.:  0.2273 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.0182 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0091 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  08 <\/br># assignments:  71 <\/br>Relative freq.:  0.4438 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  07 <\/br># assignments:  47 <\/br>Relative freq.:  0.2938 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  35 <\/br>Relative freq.:  0.2188 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.025 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.0187 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  62 <\/br>Relative freq.:  0.4844 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  07 <\/br># assignments:  55 <\/br>Relative freq.:  0.4297 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  08 <\/br># assignments:  8 <\/br>Relative freq.:  0.0625 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.0234 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  09 <\/br># assignments:  72 <\/br>Relative freq.:  0.8 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  18 <\/br>Relative freq.:  0.2 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  08 <\/br># assignments:  101 <\/br>Relative freq.:  0.6196 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  07 <\/br># assignments:  44 <\/br>Relative freq.:  0.2699 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  0.0859 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0123 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.0123 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  08 <\/br># assignments:  44 <\/br>Relative freq.:  0.55 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  07 <\/br># assignments:  35 <\/br>Relative freq.:  0.4375 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0125 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  0.6071 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  06 <\/br># assignments:  20 <\/br>Relative freq.:  0.3571 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0357 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  0.5238 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  07 <\/br># assignments:  25 <\/br>Relative freq.:  0.3968 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  06 <\/br># assignments:  5 <\/br>Relative freq.:  0.0794 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  0.8889 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.1111 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  83 <\/br>Relative freq.:  0.9432 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  06 <\/br># assignments:  5 <\/br>Relative freq.:  0.0568 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  07 <\/br># assignments:  123 <\/br>Relative freq.:  0.6796 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  53 <\/br>Relative freq.:  0.2928 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  06 <\/br># assignments:  4 <\/br>Relative freq.:  0.0221 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0055 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  07 <\/br># assignments:  172 <\/br>Relative freq.:  0.9247 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  02 <\/br># assignments:  11 <\/br>Relative freq.:  0.0591 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  08 <\/br># assignments:  3 <\/br>Relative freq.:  0.0161 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.875 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  06 <\/br># assignments:  7 <\/br>Relative freq.:  0.125 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  09 <\/br># assignments:  63 <\/br>Relative freq.:  0.358 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  08 <\/br># assignments:  56 <\/br>Relative freq.:  0.3182 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  0.233 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  06 <\/br># assignments:  9 <\/br>Relative freq.:  0.0511 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0284 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  07 <\/br># assignments:  2 <\/br>Relative freq.:  0.0114 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  0.8636 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.1364 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  70 <\/br>Relative freq.:  0.5469 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  07 <\/br># assignments:  53 <\/br>Relative freq.:  0.4141 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.0234 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  08 <\/br># assignments:  2 <\/br>Relative freq.:  0.0156 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  0.9048 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0476 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0476 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  39 <\/br>Relative freq.:  0.5065 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  07 <\/br># assignments:  36 <\/br>Relative freq.:  0.4675 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.026 <\/br>Relative Rep. freq.:  2e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(52,134,141,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(52,134,141,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":97.5674553756745,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["V3-64*01","V3-64*02","V3-64*07","V3-64D*06","V3-64D*08","V3-64D*09"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["V3-64*01","V3-64*02","V3-64*07","V3-64D*06","V3-64D*08","V3-64D*09"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-7.8541117966082,180.564481514124],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[-8.88178419700125e-16,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"319c86b8bb0e2":{"x":{},"y":{},"text":{},"type":"box"},"319c87fd756d7":{"x":{},"y":{},"text":{}}},"cur_data":"319c86b8bb0e2","visdat":{"319c86b8bb0e2":["function (y) ","x"],"319c87fd756d7":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-22" class="section level2" number="28.4">
<h2><span class="header-section-number">28.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 9 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV3-64G27%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:1100px;" cellspacing="0">
</iframe>
</div>
<div id="observations-25" class="section level2" number="28.5">
<h2><span class="header-section-number">28.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/7k230ukykprjh6pxj8wdr/IGHV3-64G27.docx?dl=0&amp;rlkey=9e8bqpbbk3vcf61fa9giq7rqh">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV3-64G27" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-26" class="section level2" number="28.6">
<h2><span class="header-section-number">28.6</span> Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="26-G26.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="28-G28.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/27-G27.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
