<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV3-73 - G30 | Functional groups</title>
  <meta name="description" content="IGHV3-73 - G30 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV3-73 - G30 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV3-73 - G30 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="29-G29.html"/>
<link rel="next" href="31-G31.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i>Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv3-73---g30" class="section level1">
<h1>IGHV3-73 - G30</h1>
<div id="allele-appearnce-29" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of IGHV3-73 includes 2 alleles, 2 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-263-1.png" width="672" /></p>
</div>
<div id="group-alignment-24" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-264-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-28" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-9a5f48378155ddcb2f6d" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-9a5f48378155ddcb2f6d">{"x":{"data":[{"x":[2,1,2,1,1,2,1,2,1,2,1,2,2,1,1,2,2,2,1,2,2,2,2,2,1,2,1,2,1,2,1,1,1,2,1,2,1,2,1,2,1,1,2,2,2,1,2,2,2,1,2,2,1,2,1,2,2,1,2,1,2,1,2,1,2,1,2,2,2,1,2,1,2,1,2,1,2,1,2,2,1,1,2,2,2,2,2,1,1,2,2,1,2,1,2,2,1,2,1,2,1,2,1,1,2,2,2,2,1,2,1,1,2,1,2,1,2,1,2,1,1,2,1,2,1,2,2,1,2,1,2,1,1,2,1,2,1,1,2],"y":[13,108,31,9,248,52,20,83,38,58,48,128,42,57,41,32,24,22,17,25,110,54,86,73,57,74,46,53,51,21,17,143,11,5,149,121,9,2,81,17,14,28,24,31,29,19,8,133,160,56,33,51,27,108,30,24,54,40,11,7,101,36,233,33,69,48,46,121,120,50,63,34,71,22,97,55,31,4,105,74,57,18,10,70,80,44,112,73,33,32,70,97,118,96,130,31,20,71,58,47,29,122,54,90,26,69,232,62,50,193,57,31,30,126,16,13,35,28,73,65,96,70,92,78,88,69,98,73,76,46,31,15,75,16,71,58,48,10,8],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[2.03572158501483,0.940381190180778,2.05999387945049,0.910442207567394,0.997757425578311,2.04255366851576,0.958955197501928,1.95726304305717,0.952687151869759,2.05946587403305,1.05463050110266,2.04611595706083,2.06945409048349,0.987711492646486,1.04767257715575,1.97653177678585,2.01712857023813,2.03940546140075,0.947902931412682,2.07684154035524,2.09120711232536,1.96761788437143,2.00039169797674,1.99691606331617,0.99321744251065,1.9111040382646,1.01316766231321,1.9140070468653,0.997300806688145,1.97820225232281,1.04303645235486,0.976433368399739,0.995390058448538,2.05884602833539,1.0848972260952,2.06863312404603,0.941005051229149,2.07805651058443,0.929405814316124,1.91481825271621,1.09908230751753,1.02125326092355,1.93275970239192,1.9678692156449,1.97462783847004,0.939529311144724,1.9097684704233,1.9145428717602,2.08350106864236,0.999094496760517,1.91611261148937,1.92602598108351,0.937167670531198,1.98873258051462,1.00265496042557,2.0518258664757,2.00661682006903,1.07151969280094,1.95312005514279,0.991298235720024,2.09006075900979,1.04407341633923,2.02442294326611,1.0158643248491,1.97514925571159,0.985264369146898,2.03417723756284,2.04598674220033,1.98438593130559,0.994080045679584,1.96991067244671,1.0626286374405,2.05060882857069,1.02304100808688,2.06330902986228,0.937695010378957,1.91512411693111,1.03752259127796,2.072834809497,1.92298433175311,0.943343137064949,0.97975081205368,1.93159837396815,1.91951793148182,2.04805197725072,2.00710510304198,2.02246683896519,0.916163421981037,0.913319361768663,2.04868543134071,1.95188352884725,0.918727262178436,2.05973552302457,1.0732785562519,1.9467058646027,2.09038475193083,0.971033935202286,1.98397562699392,0.918816733406857,1.90763209420256,0.90694911563769,1.97780343089253,0.975903065921739,0.956751079997048,2.01714310226962,1.98053129506297,2.03734469786286,2.09211834999733,1.00276890038513,1.94819936174899,1.09751503211446,0.986709393467754,2.04222915777937,0.917429937561974,2.05663440818898,1.0503979193978,2.02750731422566,0.930946096638218,1.98165745195001,0.978552955435589,0.913454511482269,1.99072026479989,0.982303178776056,2.09051912683062,1.00070028002374,1.98672196660191,1.94896407364868,0.966668828809634,1.97625779784285,0.990729559911415,1.97339283181354,0.963818536745384,1.06195068089291,1.94936747704633,1.05064985048957,1.99462031340227,1.05054307347164,0.972437536111102,2.08710668645799],"y":[12.864008163847,108.008834145591,30.8590662900358,9.27486216537654,248.00518345423,51.7334722254425,19.9879924617708,82.8750117246062,37.7734142767265,57.6529507888481,47.8271144537255,127.985640919395,42.0267710465938,56.643547064811,41.0824100725353,32.3716880077496,24.265070300363,21.7776665266603,17.3033835820854,25.0372725456953,110.313742136583,53.7255940914154,85.7865677254274,73.1620969798416,57.294861792773,73.624136194028,45.9497021401301,53.3529401320964,50.7861047657207,20.8591383321211,16.9758214136586,143.306144867837,11.2021128119901,4.85546602644026,149.057349479385,120.854353172146,8.85003602337092,2.36402179449797,81.0261076441035,17.0609571991488,14.305531125702,27.6769029289484,24.2989788098261,30.6088051643223,29.1040529873222,18.6819202128798,8.27319748848677,133.011806479096,159.980500090681,55.6618835231289,32.7617459559813,51.0584502775222,27.160257637687,108.316290754452,30.15197011251,24.0751423031092,54.1016554294154,40.0089903309941,10.7021656902507,6.91300613470376,101.033085053042,36.189580206573,232.969513400458,33.3027091117576,69.0660064600408,48.246248277463,45.8968209234998,120.848903828114,120.204266008548,50.0211197499186,63.005602857843,33.9353576146066,71.3252877457067,22.3671831544489,97.1792549438775,54.7449072420597,31.2964672034606,3.92025724612176,105.168306272477,73.9284785026684,57.0475056141615,18.0850710557774,10.2307887149975,69.7361951069906,79.6082942452282,44.1137400142849,111.814468409307,72.773429344967,33.0106553144753,32.1685953304172,69.7600001834333,96.8092143381014,118.080844538286,96.1773469623178,129.673535025679,30.6049475966021,20.1472574532032,71.1547644097358,57.7140787197277,47.2314343532547,28.9443356862292,121.607768325508,53.9312549384311,89.9969159180298,26.220081478171,69.3049415128306,232.001710540801,61.9736880574375,50.1649046743289,192.793865170889,56.834305931069,31.2847209310159,30.0333118349314,126.209864032827,15.9828737169504,13.1468258196488,35.1238921640441,28.178454923816,72.916257574968,65.3211147937924,96.0240962121636,69.9857011966407,91.6485948422924,78.2015719924122,87.6519698061049,69.3788477437571,97.977733066678,73.0912795733661,75.6309074966237,45.7146075187251,31.3921483185142,15.0510468997061,75.388430375047,16.0686590040103,70.7842736242339,58.1798761146143,48.1680181937292,9.94016689583659,8.11041790302843],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  108 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  02 <\/br># assignments:  31 <\/br>Relative freq.:  0.775 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  0.225 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  248 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  52 <\/br>Relative freq.:  0.7222 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.2778 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  83 <\/br>Relative freq.:  0.686 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  38 <\/br>Relative freq.:  0.314 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  58 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0099","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  48 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  128 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0092","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  0.5616 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  32 <\/br>Relative freq.:  0.4384 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  24 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  22 <\/br>Relative freq.:  0.5641 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  17 <\/br>Relative freq.:  0.4359 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  25 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0109","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  110 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  54 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  86 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  73 <\/br>Relative freq.:  0.5615 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  0.4385 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  74 <\/br>Relative freq.:  0.6167 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.3833 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  53 <\/br>Relative freq.:  0.5096 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  0.4904 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  21 <\/br>Relative freq.:  0.5526 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  17 <\/br>Relative freq.:  0.4474 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  143 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.6875 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.3125 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  149 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  121 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  0.8182 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.1818 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  81 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  02 <\/br># assignments:  17 <\/br>Relative freq.:  0.5484 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  0.4516 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.5385 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  24 <\/br>Relative freq.:  0.4615 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  31 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  29 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  0.7037 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.2963 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  02 <\/br># assignments:  133 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  160 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  56 <\/br>Relative freq.:  0.6292 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  02 <\/br># assignments:  33 <\/br>Relative freq.:  0.3708 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  02 <\/br># assignments:  51 <\/br>Relative freq.:  0.6538 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.3462 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  02 <\/br># assignments:  108 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  02 <\/br># assignments:  24 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  02 <\/br># assignments:  54 <\/br>Relative freq.:  0.5745 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  40 <\/br>Relative freq.:  0.4255 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  02 <\/br># assignments:  11 <\/br>Relative freq.:  0.6111 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.3889 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  02 <\/br># assignments:  101 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  233 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  02 <\/br># assignments:  69 <\/br>Relative freq.:  0.5897 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  48 <\/br>Relative freq.:  0.4103 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  02 <\/br># assignments:  46 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  02 <\/br># assignments:  121 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  02 <\/br># assignments:  120 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  02 <\/br># assignments:  63 <\/br>Relative freq.:  0.6495 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  0.3505 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  02 <\/br># assignments:  71 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  22 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  02 <\/br># assignments:  97 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  55 <\/br>Relative freq.:  0.6395 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  02 <\/br># assignments:  31 <\/br>Relative freq.:  0.3605 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  02 <\/br># assignments:  105 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  02 <\/br># assignments:  74 <\/br>Relative freq.:  0.5649 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  0.4351 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  18 <\/br>Relative freq.:  0.6429 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.3571 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  02 <\/br># assignments:  70 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  02 <\/br># assignments:  80 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  02 <\/br># assignments:  44 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  02 <\/br># assignments:  112 <\/br>Relative freq.:  0.6054 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  73 <\/br>Relative freq.:  0.3946 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  0.5077 <\/br>Relative Rep. freq.:  0.0044","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  02 <\/br># assignments:  32 <\/br>Relative freq.:  0.4923 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  70 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  97 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  02 <\/br># assignments:  118 <\/br>Relative freq.:  0.5514 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  96 <\/br>Relative freq.:  0.4486 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  02 <\/br># assignments:  130 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  02 <\/br># assignments:  31 <\/br>Relative freq.:  0.6078 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.3922 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  02 <\/br># assignments:  71 <\/br>Relative freq.:  0.5504 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  58 <\/br>Relative freq.:  0.4496 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  02 <\/br># assignments:  47 <\/br>Relative freq.:  0.6184 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.3816 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  02 <\/br># assignments:  122 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  54 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  90 <\/br>Relative freq.:  0.7759 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  02 <\/br># assignments:  26 <\/br>Relative freq.:  0.2241 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  02 <\/br># assignments:  69 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  232 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  02 <\/br># assignments:  62 <\/br>Relative freq.:  0.5536 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  0.4464 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  02 <\/br># assignments:  193 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  31 <\/br>Relative freq.:  0.5082 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  02 <\/br># assignments:  30 <\/br>Relative freq.:  0.4918 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  01 <\/br># assignments:  126 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  02 <\/br># assignments:  16 <\/br>Relative freq.:  0.5517 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  0.4483 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  02 <\/br># assignments:  35 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  02 <\/br># assignments:  73 <\/br>Relative freq.:  0.529 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  65 <\/br>Relative freq.:  0.471 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  96 <\/br>Relative freq.:  0.5783 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  02 <\/br># assignments:  70 <\/br>Relative freq.:  0.4217 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  01 <\/br># assignments:  92 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  78 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  88 <\/br>Relative freq.:  0.5605 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  02 <\/br># assignments:  69 <\/br>Relative freq.:  0.4395 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  02 <\/br># assignments:  98 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  73 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.008","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  02 <\/br># assignments:  76 <\/br>Relative freq.:  0.623 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  46 <\/br>Relative freq.:  0.377 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  02 <\/br># assignments:  31 <\/br>Relative freq.:  0.6739 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  15 <\/br>Relative freq.:  0.3261 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  75 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  02 <\/br># assignments:  16 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  02 <\/br># assignments:  58 <\/br>Relative freq.:  0.5472 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  48 <\/br>Relative freq.:  0.4528 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  10 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  0.0043"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["V3-73*01","V3-73*02"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["V3-73*01","V3-73*02"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-10.3002591727115,260.305442626942],"tickmode":"array","ticktext":["0","50","100","150","200","250"],"tickvals":[0,50,100,150,200,250],"categoryorder":"array","categoryarray":["0","50","100","150","200","250"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"754052992d2fd":{"x":{},"y":{},"text":{},"type":"box"},"7540516721d20":{"x":{},"y":{},"text":{}}},"cur_data":"754052992d2fd","visdat":{"754052992d2fd":["function (y) ","x"],"7540516721d20":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-c704a7b336530eb1e046" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-c704a7b336530eb1e046">{"x":{"data":[{"x":[1,2,2,2,1,2,2,2,1,2,2,1,1,2,2,2,1,2,1,1,2,1,2,2,1,2,1,2,1,2,1,2,1,2,1,1,2,1,1,2,2,2,1,1,1,1,2,2,1,2,1,2,2,2,1,2,2],"y":[20,2,4,2,7,6,21,2,12,9,11,7,5,4,19,7,4,21,18,11,6,13,20,2,8,5,11,8,14,8,13,2,5,38,20,11,13,11,7,4,16,15,13,15,28,2,1,8,16,6,20,9,21,14,14,11,12],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.05867364788428,1.93363445973955,1.95038109091111,1.98924518739805,0.942774866707623,1.92918150923215,1.99013938079588,2.05939976307563,1.07293767412193,2.05436855615117,2.04527380149811,0.918051036354154,1.02925802120008,1.95848760590889,1.91061704773456,2.02445741263218,0.933833357552066,1.97909573582001,1.05137249310501,1.09584671244957,2.09273872417398,0.988653197651729,2.03282204819843,1.91147967311554,1.09954789169133,2.09228451028466,1.00777771649882,1.97644429183565,0.972566596791148,2.06199358603917,0.961322015011683,1.98617157004774,1.09698285316117,1.98485347726382,0.968365703662857,1.04647992430255,1.91086737359874,1.0464535290841,0.975135964667425,2.03916413825937,2.08130929740146,2.06613277336583,1.02497470723465,0.905316549399868,1.0866627804935,1.04191533806734,1.97076771659777,1.94132765070535,0.929513132432476,2.05866124844179,1.00897811017931,1.99896651399322,1.93591041299514,2.02328973407857,0.987906744517386,1.93089964063838,2.06209740745835],"y":[19.9546577360481,1.74439116604626,4.3172989891842,1.63692124467343,6.69196038264781,5.78657889254391,20.7904285820201,1.92497854512185,12.3846410827711,8.72711652182043,10.6580284720287,6.66877129096538,4.97240210026503,3.82656520437449,18.9163034824654,6.9234447998926,4.344032673724,20.6090056823567,18.2730681862682,11.2968417761847,5.60613458696753,13.1615889854729,20.2576590482146,2.30538947042078,8.33543445132673,4.6126914082095,11.2406550401822,7.703349115327,13.6173332212493,8.05499228984118,13.1099701061845,2.38158058580011,5.0399929035455,38.0791311167181,20.2898340372369,10.9258171366528,12.6063450455666,11.2137359382585,6.83816982321441,3.77790786940604,16.0876624194905,15.2966756332666,13.3935967354104,14.6285623263568,28.0389729946852,2.02704362347722,0.99726804215461,8.29775695241988,16.3940776817501,6.19358727559447,20.3484284350649,8.71365377437323,20.8146258594468,14.1608448607847,13.923565518856,11.0656734872609,11.7466887505725],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  9e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.5385 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.4615 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  21 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.5714 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  0.4286 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  11 <\/br>Relative freq.:  0.6111 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.3889 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  19 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.6364 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  0.3636 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  21 <\/br>Relative freq.:  0.5385 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  18 <\/br>Relative freq.:  0.4615 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.6471 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.3529 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  20 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.6154 <\/br>Relative Rep. freq.:  0.0037","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.3846 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.5789 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.4211 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  0.6364 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.3636 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  0.8667 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.1333 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  38 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  02 <\/br># assignments:  13 <\/br>Relative freq.:  0.5417 <\/br>Relative Rep. freq.:  0.0012","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.4583 <\/br>Relative Rep. freq.:  0.0011","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.6364 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.3636 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  02 <\/br># assignments:  16 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  02 <\/br># assignments:  15 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  15 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0016","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  0.7273 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.2727 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  21 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  02 <\/br># assignments:  14 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0017"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":34.8692403486924},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["V3-73*01","V3-73*02"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["V3-73*01","V3-73*02"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.856825111573562,39.9332242704462],"tickmode":"array","ticktext":["0","10","20","30"],"tickvals":[0,10,20,30],"categoryorder":"array","categoryarray":["0","10","20","30"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"754053ba424e8":{"x":{},"y":{},"text":{},"type":"box"},"754051a8c6407":{"x":{},"y":{},"text":{}}},"cur_data":"754053ba424e8","visdat":{"754053ba424e8":["function (y) ","x"],"754051a8c6407":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-24" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 2 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV3-73G30%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:800px;" cellspacing="0">
</iframe>
</div>
<div id="observations-28" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/5shua0bnn9nh1e8kbfkmv/IGHV3-73G30.docx?dl=0&amp;rlkey=fe4gpinczbc1xqqi2rnzbqfw4" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV3-73G30" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-29" class="section level2">
<h2>Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="29-G29.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="31-G31.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/30-G30.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
