<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>33 IGHV3-9 - G32 | Functional groups</title>
  <meta name="description" content="33 IGHV3-9 - G32 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="33 IGHV3-9 - G32 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="33 IGHV3-9 - G32 | Functional groups" />
  
  
  



<meta name="date" content="2021-11-23" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="31-G31.html"/>
<link rel="next" href="33-G33.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-2-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 2% for all except 0.5% for allele <strong>05</strong></a></li>
<li class="chapter" data-level="3.6.2" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.2</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv3-9---g32" class="section level1" number="33">
<h1><span class="header-section-number">33</span> IGHV3-9 - G32</h1>
<div id="allele-appearnce-31" class="section level2" number="33.1">
<h2><span class="header-section-number">33.1</span> Allele appearnce</h2>
<p>The group of IGHV3-9 includes 3 alleles, 3 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-278-1.png" width="672" /></p>
</div>
<div id="group-alignment-26" class="section level2" number="33.2">
<h2><span class="header-section-number">33.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-279-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-30" class="section level2" number="33.3">
<h2><span class="header-section-number">33.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-e549f2b9d19a0f438b3a" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-e549f2b9d19a0f438b3a">{"x":{"data":[{"x":[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,3,1,1,1,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,3,1,1,1,1,2,1,1,2,1,2,1,1,1,3,1,1,3,1,3,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,3,1,1,1,2,3,1,1,1,1,1,1,1],"y":[131,55,151,1,65,177,71,87,29,119,43,11,81,15,86,1,136,59,158,36,30,19,60,146,36,1,8,172,2,53,145,63,52,144,80,1,27,75,1,217,660,677,630,241,242,854,169,614,1,780,329,824,121,1297,777,1309,1,1747,140,459,667,711,16,189,485,465,603,1090,1088,1118,396,400,592,142,1221,564,1,1069,231,864,418,1,1049,704,1,264,2,621,827,1892,1,30,587,142,607,100,768,484,597,1848,1,522,514,1566,876,789,549,754,645,857,616,454,395,312,638,1433,1460,1,510,931,601,1,1,281,355,339,57,736,448,86],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.05676846429706,0.95176845099777,0.999807848874479,0.978237107908353,0.991963473288342,1.04016229044646,0.969932250864804,0.958660580031574,1.09709056434222,1.03998469556682,1.03252810188569,0.94877052907832,0.963063821010292,0.992790158325806,1.01222176062874,0.937758676195517,0.931135445088148,0.938366356585175,1.03815444954671,0.923175974749029,1.09503642017953,0.934671629173681,0.972223473293707,1.08287921953015,0.98892793757841,2.00160389747471,0.991162134474143,1.01321330727078,0.92252981280908,1.05587523016147,1.07226627320051,1.08675347808748,1.09463074211963,1.07287795371376,1.09464109805413,1.0363059557043,1.02021589414217,1.09263655371033,1.00305467126891,1.0448742125649,0.956298452522606,0.994644752377644,0.936766527686268,0.92989080324769,0.915748175000772,1.01309671900235,1.00930918222293,1.08329859687947,2.90581378596835,1.04043693412095,1.0345062637236,1.06759416949935,1.09730291450396,0.913816701760516,1.04614605591632,1.02984065422788,2.0211456888821,0.981465514190495,0.931486089294776,0.925722759077325,0.942813534103334,1.00849331719801,1.06851295176893,1.03889929400757,1.0007305928506,1.00108599532396,0.935190447885543,1.04965876052156,0.973098491830751,0.9129889592994,1.04243591772392,0.986373897595331,0.980164294084534,0.979446217417717,1.04256524834782,0.994841354899108,3.09622251992114,1.0868026219774,1.0585624774918,1.083433691971,0.982770774979144,2.00813942137174,0.947398439841345,1.05248595224693,1.98276988612488,0.915407419065014,1.93929510251619,0.98163124890998,0.964878167770803,1.09922492611222,3.00933156358078,0.997917425073683,0.919070051424205,2.92813611170277,0.907608639914542,3.04399680322967,0.918646981427446,1.0300356445834,0.940544626675546,1.07010872690007,2.0815637732856,0.913743225391954,0.927049864595756,1.0597351893317,0.989649643097073,0.971933822566643,0.992283742502332,1.08801199672744,1.03659246228635,0.957621291046962,0.994545729272068,0.982717661885545,0.97706123185344,0.973000441491604,0.904160662321374,1.00839183647186,0.924696070468053,2.9618914002087,1.02885909271426,0.99681431367062,1.09710894431919,1.94216508483514,3.07671649060212,0.900425091618672,1.07235843189992,1.07294637821615,0.950061504403129,1.04133059633896,0.991129101533443,1.05389358857647],"y":[130.873529023118,55.0377071727067,150.819330304675,0.969773126207292,65.3803003713489,176.992670374736,71.0778590535745,87.2843172779307,28.6769857726991,119.396516855806,43.2053490333259,11.3569741001353,80.9590889699757,14.7414050232619,86.3380472885445,0.871263994649053,135.952792787924,58.7570000199601,158.370515561476,35.7427383286878,29.6316269764677,18.7137973528355,60.3676239248365,145.786264472827,35.7313195947558,0.995135130360723,8.0973656244576,171.667279896326,1.81218791715801,52.8426606047899,144.84404759407,62.9618758317083,51.6299698337913,143.625417765789,80.2706073572859,1.23066685181111,26.6384378500283,75.1205871183425,1.39398564454168,217.196822333336,659.635460725427,677.033048197441,630.138325270638,240.915215160139,241.790973754786,853.779496758059,169.235526294261,613.67451928053,1.31473181974143,779.987589587085,329.258841526322,823.837659080513,121.033310235851,1297.12287703399,777.160827184841,1309.26567677744,1.05251398570836,1747.04716291223,140.143200672604,459.001544192247,667.362169178575,710.87308722157,15.8187662679702,189.14991634991,485.085099678673,464.764299915358,603.065706583485,1089.74990347382,1088.06546904035,1117.94791750479,395.738712284341,399.861102100648,591.84556806609,141.656548630074,1221.39613514617,563.814981457405,1.28611942082644,1069.12162485253,231.212195563316,864.326968785189,417.882384919189,0.968032034486532,1049.16453747507,703.88726750724,1.2189404970035,263.809618170001,1.88804444409907,621.253518863767,827.23899543006,1892.14389292859,1.21527462694794,30.2237886583433,586.811057799868,142.033336963132,606.706349329092,99.6020982829854,767.737229423225,483.608161119558,597.012748432532,1847.9914037481,0.932059281505644,521.880575106479,514.238170076534,1566.14214745536,876.39227551613,788.994281102344,548.884735901467,753.94227934517,644.796182607487,856.817364275828,616.053912956454,453.793219650723,395.153509526327,312.109463476017,638.352391807362,1433.30903498791,1460.2988299435,1.26147687844932,510.28319106251,930.637863292545,601.291018477827,0.832847171835601,1.27465770374984,281.329152347893,355.188673786633,338.76171061229,56.8067026397213,736.265977833234,448.115199454129,85.7856461176649],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  131 <\/br>Relative freq.:  0.7043 <\/br>Relative Rep. freq.:  0.0122","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  55 <\/br>Relative freq.:  0.2957 <\/br>Relative Rep. freq.:  0.0051","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  151 <\/br>Relative freq.:  0.9934 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0066 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  65 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  177 <\/br>Relative freq.:  0.7137 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.2863 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  87 <\/br>Relative freq.:  0.75 <\/br>Relative Rep. freq.:  0.005","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  0.25 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  119 <\/br>Relative freq.:  0.7346 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  43 <\/br>Relative freq.:  0.2654 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0313","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  81 <\/br>Relative freq.:  0.8438 <\/br>Relative Rep. freq.:  0.0099","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  15 <\/br>Relative freq.:  0.1562 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  86 <\/br>Relative freq.:  0.9885 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0115 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  136 <\/br>Relative freq.:  0.6974 <\/br>Relative Rep. freq.:  0.0069","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  59 <\/br>Relative freq.:  0.3026 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  158 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0109","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  0.6122 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  0.3878 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  146 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  36 <\/br>Relative freq.:  0.973 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.027 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  172 <\/br>Relative freq.:  0.9885 <\/br>Relative Rep. freq.:  0.0139","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.0115 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  53 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  145 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  63 <\/br>Relative freq.:  0.5478 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  52 <\/br>Relative freq.:  0.4522 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  144 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  80 <\/br>Relative freq.:  0.9877 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0123 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  75 <\/br>Relative freq.:  0.9868 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0132 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  217 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0159","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  660 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0615","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  677 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0283","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  630 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0256","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  241 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0399","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  242 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0389","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  854 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.06","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  169 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0299","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  614 <\/br>Relative freq.:  0.9984 <\/br>Relative Rep. freq.:  0.0608","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  780 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0351","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  329 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  824 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.06","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  121 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0506","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  1297 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0481","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  777 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0444","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  1309 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.0583","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  1747 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0748","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  140 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0354","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  459 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0366","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  667 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0326","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  711 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0285","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  16 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0456","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  189 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0343","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  485 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0639","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  465 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0408","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  603 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0735","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  1090 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0384","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  1088 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0624","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  1118 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0568","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  396 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.034","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  400 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0651","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  592 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0398","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  142 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0429","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  1221 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0554","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  564 <\/br>Relative freq.:  0.9982 <\/br>Relative Rep. freq.:  0.0555","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  1069 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.029","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  231 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0365","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  864 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0316","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  418 <\/br>Relative freq.:  0.9976 <\/br>Relative Rep. freq.:  0.0455","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  1049 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0489","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  704 <\/br>Relative freq.:  0.9986 <\/br>Relative Rep. freq.:  0.029","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  264 <\/br>Relative freq.:  0.9925 <\/br>Relative Rep. freq.:  0.0349","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0075 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  621 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0227","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  827 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0259","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  1892 <\/br>Relative freq.:  0.9995 <\/br>Relative Rep. freq.:  0.0555","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  5e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0222","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  587 <\/br>Relative freq.:  0.8052 <\/br>Relative Rep. freq.:  0.0273","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  03 <\/br># assignments:  142 <\/br>Relative freq.:  0.1948 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  607 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0265","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  03 <\/br># assignments:  100 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  768 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0409","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  484 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0246","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  597 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0483","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  1848 <\/br>Relative freq.:  0.9995 <\/br>Relative Rep. freq.:  0.0547","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  5e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  522 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0221","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  514 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.019","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  1566 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0477","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  876 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0273","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  789 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0316","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  549 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0314","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  754 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0405","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  645 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0348","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  857 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0262","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  616 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0268","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  454 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0328","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  395 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0273","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  312 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0336","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  638 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.034","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  1433 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0502","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  01 <\/br># assignments:  1460 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.0623","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  510 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0318","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  931 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0298","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  601 <\/br>Relative freq.:  0.9967 <\/br>Relative Rep. freq.:  0.0254","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  281 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0287","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  355 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0282","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  339 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0267","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0436","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  736 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0312","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  448 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.026","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  86 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0441"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":78.9705271897053,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V3-9*01","V3-9*02","V3-9*03"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V3-9*01","V3-9*02","V3-9*03"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-93.7327051160019,1986.70944521642],"tickmode":"array","ticktext":["0","500","1000","1500"],"tickvals":[0,500,1000,1500],"categoryorder":"array","categoryarray":["0","500","1000","1500"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5a2d26019f367":{"x":{},"y":{},"text":{},"type":"box"},"5a2d25fcd8932":{"x":{},"y":{},"text":{}}},"cur_data":"5a2d26019f367","visdat":{"5a2d26019f367":["function (y) ","x"],"5a2d25fcd8932":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-96788bce9507f4093eba" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-96788bce9507f4093eba">{"x":{"data":[{"x":[1,1,1,1,1,1,1,1,1,1,1,1,1,2,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,1,1,2,1,2,1,1,1,1,1],"y":[3,25,12,5,10,27,13,1,19,6,33,13,1,1,34,3,2,9,25,2,80,80,29,229,32,127,64,97,63,380,157,74,72,94,153,184,214,191,140,114,51,24,292,60,6,232,46,11,49,133,8,142],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(86,63,80,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.07624523760751,0.971957467170432,0.971353437798098,1.05559198996052,0.965435103606433,1.00980734908953,1.01557348864153,1.00892370133661,0.969258981477469,1.03869902272709,0.908429851010442,1.03319480526261,0.910380182415247,1.91952978651971,1.07139927065,2.05431749499403,0.951284092338756,1.04666629433632,1.08984207557514,1.01918801264837,1.08367787334137,0.917207789607346,0.97589986291714,1.05335452705622,0.909259174298495,1.06252253046259,1.00469800089486,1.09048675438389,0.939841046417132,0.903902507852763,1.08611442409456,0.912644861498848,0.918638222338632,1.07573799933307,1.09537331867032,1.06327217677608,0.950661617424339,1.09016706747934,1.01582972332835,1.02936518099159,0.934582907147706,2.08115303120576,1.00194817176089,1.06309613389894,1.90265321414918,0.992313792090863,1.91724736671895,1.0371496105101,1.00999283296987,1.03789440765977,0.934587941737846,0.999433230422437],"y":[3.16793784778565,25.2753466280177,12.2205413602293,4.71761086061597,9.78242578972131,26.8506466766819,12.6177298558876,0.639377245120704,18.6355608249083,5.87070434149355,32.86183047425,12.6362963417545,1.08546893894672,0.970777101814747,33.924746789597,2.77101543098688,1.68639663793147,9.1736657705158,25.0438335239887,1.76684148646891,79.903382274136,80.2878245007247,29.2132117323577,228.682709602639,32.3817728443071,127.30615234971,63.7862163871527,97.0877495748922,63.1927723014727,379.787237450667,157.186310028657,73.8382739726454,71.9166116435081,93.6191630559042,153.335237850994,183.721913829446,213.933371306397,190.605018584989,139.960952701978,114.213808152452,51.0962084149942,24.2887895401567,291.669658002816,59.7500418817624,6.23578870855272,231.843651243672,45.8558575043455,10.8739442335442,48.8176660282537,132.810427483171,8.39992980360985,141.675925338641],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  25 <\/br>Relative freq.:  0.6757 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  0.3243 <\/br>Relative Rep. freq.:  0.0014","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  5 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  10 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  0.9286 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0714 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  0.76 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  0.24 <\/br>Relative Rep. freq.:  0.0013","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  13 <\/br>Relative freq.:  0.8667 <\/br>Relative Rep. freq.:  0.0042","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0667 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0667 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  0.8718 <\/br>Relative Rep. freq.:  0.0022","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.0769 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  0.0513 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0029","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  25 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  2 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  80 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0136","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  80 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0133","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  29 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0056","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  229 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  32 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  127 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0149","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  64 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0125","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  97 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.016","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  63 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  380 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0482","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  157 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0309","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  74 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0327","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  72 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0201","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  94 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  153 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0336","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  184 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0206","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  214 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0202","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  191 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0342","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  140 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.024","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  01 <\/br># assignments:  114 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0212","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  0.68 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  03 <\/br># assignments:  24 <\/br>Relative freq.:  0.32 <\/br>Relative Rep. freq.:  0.0047","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  292 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0315","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  60 <\/br>Relative freq.:  0.9091 <\/br>Relative Rep. freq.:  0.0193","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  03 <\/br># assignments:  6 <\/br>Relative freq.:  0.0909 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  232 <\/br>Relative freq.:  0.8345 <\/br>Relative Rep. freq.:  0.015","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  03 <\/br># assignments:  46 <\/br>Relative freq.:  0.1655 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0158","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  133 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0157","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0015","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  142 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0192"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(86,63,80,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(86,63,80,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":78.9705271897053,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["V3-9*01","V3-9*03"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["V3-9*01","V3-9*03"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-18.3286538926233,398.968031137744],"tickmode":"array","ticktext":["0","100","200","300"],"tickvals":[0,100,200,300],"categoryorder":"array","categoryarray":["0","100","200","300"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5a2d2375aa3bb":{"x":{},"y":{},"text":{},"type":"box"},"5a2d24781b956":{"x":{},"y":{},"text":{}}},"cur_data":"5a2d2375aa3bb","visdat":{"5a2d2375aa3bb":["function (y) ","x"],"5a2d24781b956":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-cddd2f89394648bcf115" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-cddd2f89394648bcf115">{"x":{"data":[{"x":[1,1,1,2,1,1,3,1,1,2,1,1,1,1,2,1,1,2,1,2,1,2,1,1,2,1,2,1,2,1,3,1,2,1,3,1,2,1,2,1,2,1,2,1,2,1,1,2,3,1,2,1,2,1,1,2,1,3],"y":[53,23,14,4,12,1,1,33,28,4,108,65,22,9,3,55,204,6,141,10,94,9,286,268,1,265,7,905,5,173,3,679,12,86,23,7,2,216,1,766,10,664,2,990,1,543,477,82,37,136,2,170,3,323,207,5,216,2],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.03905070750043,1.07053094943985,1.05923389848322,2.0083079718519,0.972591700684279,1.08362303683534,3.04462162349373,0.995532039785758,1.02066239565611,2.03529384951107,0.917562762694433,0.980742884706706,0.915155256493017,0.921724416222423,1.99635111708194,1.05113493278623,1.02509170775302,1.97056799023412,1.09397070026025,2.05222879997455,0.938176269130781,2.09565135864541,0.917120691016316,1.09187568244524,1.98411779045127,1.03714779019356,2.04222228173167,0.931271754577756,1.96815168494359,1.06049524447881,2.98443905091845,1.07993451626971,2.07084294855595,1.027166327741,2.90058980803005,0.923380575189367,2.06858996483497,1.04164884877391,2.07608573045582,0.967816819623113,2.02675615935586,1.05513008618727,2.03497953931801,0.908483296865597,2.07242003176361,0.93834029790014,0.953307734709233,2.06266850717366,2.9937138278503,1.04837756813504,1.97065890100785,0.901022494863719,1.94449494774453,1.0004033472389,1.00922565422952,1.94531520777382,0.915040850220248,3.02319199484773],"y":[52.8847115274519,23.1413277272135,13.7798447363079,3.73835611473769,11.8482874033973,0.764224297739565,0.666562397405505,33.0951989941299,28.0431088948622,4.16454386245459,108.166441619955,65.1629243329167,21.7466677835211,8.65832410808653,2.67568767834455,55.3088081205264,204.102717546932,6.05258554257452,140.881633247435,10.3111255809665,94.091411682032,8.81412801072002,286.077717835829,267.782705179974,1.05393658038229,264.818255332112,7.27186506781727,904.813950632326,4.89032051693648,173.147093087248,2.61238979678601,678.873675507307,12.3105516452342,85.8194931764156,23.3756487643346,7.23494508080184,1.64358534794301,216.119044876099,0.640587053075433,765.914930374734,9.7701417081058,663.686954104528,1.8011798363179,990.263163170591,1.14845680259168,543.181132303551,476.828871971928,81.6953441916034,37.0709432659671,136.086058380827,1.9771031241864,169.608657040074,3.10913564283401,323.260279811919,206.779585648328,5.27398388404399,216.329646076448,1.73113413751125],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  53 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  0.561 <\/br>Relative Rep. freq.:  0.0032","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  0.3415 <\/br>Relative Rep. freq.:  0.002","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0976 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  12 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  1 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  0.5077 <\/br>Relative Rep. freq.:  0.0025","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  28 <\/br>Relative freq.:  0.4308 <\/br>Relative Rep. freq.:  0.0021","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0615 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  108 <\/br>Relative freq.:  0.6243 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  65 <\/br>Relative freq.:  0.3757 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  22 <\/br>Relative freq.:  0.6471 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  9 <\/br>Relative freq.:  0.2647 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0882 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  55 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0049","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  204 <\/br>Relative freq.:  0.9714 <\/br>Relative Rep. freq.:  0.0179","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0286 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  141 <\/br>Relative freq.:  0.9338 <\/br>Relative Rep. freq.:  0.0375","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.0662 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  94 <\/br>Relative freq.:  0.9126 <\/br>Relative Rep. freq.:  0.0179","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  0.0874 <\/br>Relative Rep. freq.:  0.0017","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  286 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  268 <\/br>Relative freq.:  0.9963 <\/br>Relative Rep. freq.:  0.0225","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0037 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  265 <\/br>Relative freq.:  0.9743 <\/br>Relative Rep. freq.:  0.037","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  7 <\/br>Relative freq.:  0.0257 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  905 <\/br>Relative freq.:  0.9945 <\/br>Relative Rep. freq.:  0.0544","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0055 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  173 <\/br>Relative freq.:  0.983 <\/br>Relative Rep. freq.:  0.0118","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  03 <\/br># assignments:  3 <\/br>Relative freq.:  0.017 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  679 <\/br>Relative freq.:  0.9826 <\/br>Relative Rep. freq.:  0.0381","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  02 <\/br># assignments:  12 <\/br>Relative freq.:  0.0174 <\/br>Relative Rep. freq.:  7e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  86 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0221","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  03 <\/br># assignments:  23 <\/br>Relative freq.:  0.7188 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  7 <\/br>Relative freq.:  0.2188 <\/br>Relative Rep. freq.:  0.001","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0625 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  216 <\/br>Relative freq.:  0.9954 <\/br>Relative Rep. freq.:  0.0235","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0046 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  766 <\/br>Relative freq.:  0.9871 <\/br>Relative Rep. freq.:  0.0576","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  10 <\/br>Relative freq.:  0.0129 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  664 <\/br>Relative freq.:  0.997 <\/br>Relative Rep. freq.:  0.0439","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.003 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  990 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.0528","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  543 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0217","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  477 <\/br>Relative freq.:  0.8003 <\/br>Relative Rep. freq.:  0.0306","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  02 <\/br># assignments:  82 <\/br>Relative freq.:  0.1376 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  03 <\/br># assignments:  37 <\/br>Relative freq.:  0.0621 <\/br>Relative Rep. freq.:  0.0024","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  136 <\/br>Relative freq.:  0.9855 <\/br>Relative Rep. freq.:  0.0198","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0145 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  170 <\/br>Relative freq.:  0.9827 <\/br>Relative Rep. freq.:  0.0197","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0173 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  323 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0219","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  207 <\/br>Relative freq.:  0.9764 <\/br>Relative Rep. freq.:  0.0252","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0236 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  216 <\/br>Relative freq.:  0.9908 <\/br>Relative Rep. freq.:  0.0193","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0092 <\/br>Relative Rep. freq.:  2e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":78.9705271897053,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V3-9*01","V3-9*02","V3-9*03"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V3-9*01","V3-9*02","V3-9*03"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-48.8405417528003,1039.74429197647],"tickmode":"array","ticktext":["0","250","500","750","1000"],"tickvals":[0,250,500,750,1000],"categoryorder":"array","categoryarray":["0","250","500","750","1000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5a2d276471915":{"x":{},"y":{},"text":{},"type":"box"},"5a2d268d6ebe4":{"x":{},"y":{},"text":{}}},"cur_data":"5a2d276471915","visdat":{"5a2d276471915":["function (y) ","x"],"5a2d268d6ebe4":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-26" class="section level2" number="33.4">
<h2><span class="header-section-number">33.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 5 zygousity states.</p>
<iframe width="900" height="600" src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV3-9G32%22&amp;allele_thresh=%220.5%22frameborder=&quot;0&quot;" allow="accelerometer; autoplay; encrypted-media;
  gyroscope; picture-in-picture">
</iframe>
</div>
<div id="observations-30" class="section level2" number="33.5">
<h2><span class="header-section-number">33.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/8febwwy92a64xc8znuh1f/IGHV3-9G32.docx?dl=0&amp;rlkey=qdqk2sydh5zxebgfqbt0k7d6b">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV3-9G32" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-31" class="section level2" number="33.6">
<h2><span class="header-section-number">33.6</span> Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="31-G31.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="33-G33.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/32-G32.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
