<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV4-30-4/IGHV4-31 - G35 | Functional groups</title>
  <meta name="description" content="IGHV4-30-4/IGHV4-31 - G35 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV4-30-4/IGHV4-31 - G35 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV4-30-4/IGHV4-31 - G35 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="34-G34.html"/>
<link rel="next" href="36-G36.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i>Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-40"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-46"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-40"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-47"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-48"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv4-30-4ighv4-31---g35" class="section level1">
<h1>IGHV4-30-4/IGHV4-31 - G35</h1>
<div id="allele-appearnce-34" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of IGHV4-30-4/IGHV4-31 includes 20 alleles, 20 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-308-1.png" width="672" /></p>
</div>
<div id="group-alignment-29" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-309-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-33" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-75001329d36d8072c66d" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-75001329d36d8072c66d">{"x":{"data":[{"x":[6,1,8,2,6,1,3,8,6,1,4,6,1,6,1,6,1,6,6,1,6,1,6,1,4,6,1,6,6,1,8,6,1,6,1,6,8,6,4,6,1,1,6,6,6,6,6,1,6,1,6,8,6,1,6,1,8,6,1,6,1,6,6,1,4,3,6,1,6,6,4,6,2,6,1,8,6,1,4,6,1,6,8,6,1,6,6,2,6,1,8,6,1,6,6,1,6,1,6,6,6,1,6,1,8,6,1,6,1,6,1,6,8,6,6,1,3,8,6,6,3,1,6,1,6,1,4,6,1,6,1,5,8,1,6,1,6,1,6,1,6,1,4,4,6,1,6,6,6,1,6,1,1,6,1,4,6,1,3,6,1,6,1,6,5,6,1,6,1,3,7,6,8,5,6,1,6,1,6,6,1,6,1,6,6,8,6,1,6,1,5,8,6,1,6,8,4],"y":[331,103,1,1,829,500,1,1,179,67,32,350,205,311,229,519,130,165,188,92,348,80,102,38,21,228,167,263,444,138,1,421,104,73,44,347,1,422,57,509,324,263,240,147,604,55,300,73,191,131,1168,1,7,6,69,19,1,567,354,66,45,131,145,116,45,1,277,37,543,473,121,365,272,679,136,1,862,234,41,151,91,283,2,49,20,487,213,70,1078,319,2,99,72,653,219,143,1051,416,1,433,849,619,896,373,1,72,51,443,325,1468,416,920,1,51,605,202,3,1,346,545,211,205,885,165,274,199,71,186,100,980,694,1,1,109,60,677,619,517,999,527,379,229,120,80,661,211,479,763,364,118,365,235,187,1045,363,258,864,294,1,471,105,167,116,678,1,672,108,614,130,1,1,1262,2,1,664,314,367,610,430,1567,242,873,392,241,710,1,538,102,296,114,2,1,29,4,352,1,5],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[5.99306533434428,0.937811703514308,8.04139096313156,2.02784843742847,6.05452376422472,0.907688425527886,3.02474705814384,8.01171615733765,6.01537392074242,0.969576634746045,4.03591618034989,5.94297872493044,0.917971628764644,5.91071594296955,0.981899664411321,6.06769564123824,1.09494884428568,5.98076903726906,5.95221504843794,1.0443451019004,6.09463722617365,0.920523153245449,5.981187835522,0.973352691344917,4.09809297560714,6.08197428742424,1.08431663108058,6.09155116840266,5.99522147602402,0.919088245648891,8.0115554026328,5.94303792407736,0.920108893699944,6.04286029646173,1.09695705878548,5.92140117436647,8.08155415961519,6.06870717597194,3.92529888418503,5.92531444136985,1.01050269193947,0.952104890113696,5.98246063166298,5.95873316149227,5.93778767148033,6.04451529965736,5.98584868377075,0.9175546746701,6.09419091469608,1.08990229531191,5.96645589242689,7.9045518389903,5.99408351830207,1.07739474978298,6.09730899692513,0.920306593272835,7.98223520647734,5.91258191270754,1.06487811277621,5.91747476584278,1.0770301415585,5.91231229430996,5.94335019406863,1.05342124002054,4.08557711658068,3.09981760377996,6.04837633934803,1.09520557648502,5.99000372667797,5.9310440917965,3.9218227695208,6.00875186659396,2.06903701527044,6.09085683068261,0.953388252062723,8.03127528000623,6.06148259723559,1.07938867863268,3.95258478680626,5.9475189127028,1.07060563247651,6.07584166666493,8.05096120359376,5.96554559282959,0.939473731443286,6.00808958439156,6.05345528256148,1.91052578771487,6.09192034103908,0.938444709032774,7.90416976925917,5.90658343937248,1.04838291020133,5.94587437761948,6.02105658459477,0.903156594000757,5.93251499077305,1.03668611785397,6.05363113293424,5.97125365114771,6.04828936588019,1.08134722197428,6.06551071046852,1.05122641013004,8.01231142338365,5.9644265956711,1.08286904934794,6.04812156753615,1.0684085062705,6.0639165929053,0.991626561200246,6.05461884238757,7.95344929341227,5.96813851376064,5.98059690082446,0.981591760180891,2.9872564998921,8.08719391482882,5.95822339444421,6.02038055532612,2.99671945073642,0.903063006978482,6.03044734932482,0.922929804958403,6.06773741045035,1.03267043204978,4.07463362622075,6.06692838622257,1.07792351655662,6.02145284111612,1.01237121522427,4.99567299671471,8.00729370070621,0.958139739558101,6.02560564172454,0.968276314483955,6.05589335481636,0.924239269411191,6.08859597584233,0.905224210349843,6.08193041621707,1.0207196299918,3.90962675372139,3.92390003534965,6.09615371883847,0.915974482521415,5.90499973837286,6.00274143721908,6.09866344337352,0.929433838836849,5.98105144733563,0.99045297568664,0.916343706846237,6.0596784720663,0.904300556238741,4.00659040058963,5.97929832041264,1.03526050131768,3.06705549992621,5.91431070202962,0.926851171487942,5.96142604025081,0.977229873323813,6.04850947731174,4.98850383264944,5.92879293146543,0.985738322278485,6.01360508338548,1.07879713489674,2.94158636643551,6.91803996097296,5.96508503728546,7.95023228344508,5.0452464572154,6.08396855881438,1.08126780800521,5.93598424820229,1.00584984417073,6.05982299870811,6.07471587308683,0.988706389162689,6.05949703161605,1.00934090679511,5.95011513205245,5.98150194715708,8.05802879189141,6.0672980518546,0.916896236129105,6.08349393103272,1.05129221179523,4.90472957720049,7.9044782510493,5.96227280655876,0.922996715176851,6.0055745956488,7.97719097291119,3.95334210223518],"y":[330.855024152249,103.118306108192,0.925256807543337,1.18998652659357,829.110447038338,500.141749583185,0.979991296306252,1.03313693553209,179.314126975276,67.2234707269818,32.0044484889135,349.717830116116,205.333630653098,310.720667455532,229.385198262706,519.298167723045,130.386898886971,165.346969299763,188.333487138525,91.6070066731423,348.087159997225,80.3928876543418,102.341539751552,37.7707309084013,20.7783411728218,227.936482526362,166.711183439009,263.168321788125,444.314821877703,137.612004867196,0.996650116704404,420.702645191737,104.216533336416,73.2627588381991,44.0661477610469,346.630345603824,0.833634708449244,422.158114311658,56.9688083812594,509.112369279563,324.285318169743,263.138342736661,240.095196001977,146.862789961323,604.30112325605,55.1692755497992,300.356143944897,73.0094199825078,190.884039484337,130.648400091752,1168.29986304156,1.04072791729122,6.86748452447355,5.86430830881,69.2311400437728,18.9206038301811,1.19449484869838,566.742679828778,353.93288900163,66.1796545272693,44.775009727478,130.765717945062,144.790465081111,116.077918895334,45.3681597257033,0.654275434277952,277.077920162864,37.2242359533906,542.664144495502,473.078069707006,120.7176541619,364.794749942049,271.810332672298,679.190478719957,135.722974411957,1.28854853175581,862.121029134281,233.637975828908,40.8433658834547,150.605544677936,90.8461109478027,282.991333299316,2.06616177652031,48.604120914638,19.8384618606418,487.136681617983,213.290111519769,69.6099934438244,1078.07268823907,318.631663507037,2.11100154146552,98.6010419197381,71.9080234399065,653.340445392765,219.390580419078,142.835968624242,1050.95863838792,415.810054493137,0.827092464827001,432.916800524294,849.133230370842,618.777971779369,895.969198822975,373.166839189827,1.02170813884586,71.643719448708,51.0976064575836,443.137607318722,324.797129419819,1467.9520836411,415.835317373462,920.240032747015,1.35719517804682,51.272810845077,605.278789845109,202.044748275727,3.20492450054735,1.28278369009495,346.163037212566,545.047742960229,210.662363938801,205.221141996048,885.184355584159,164.777998542041,273.863514014892,199.221067184024,71.2346097592264,186.308049988933,99.8833347568288,979.876311225444,694.242138284445,1.02404044158757,1.29168877862394,108.872275879048,59.8127623943612,677.070094322972,618.759849720076,516.913140392303,998.980383963324,527.116780241765,379.266958434693,228.626967637241,120.026809084602,79.9863296598196,660.67149703335,210.714660919085,479.293217061087,762.790099097043,364.294398081303,118.393037868477,364.858819431439,234.693260505609,187.385148506798,1044.88630823418,362.673183630966,258.348245488293,864.049506293982,293.8626809448,1.14901310559362,471.384424224868,105.306675579213,166.960289313272,116.387651740946,678.18922981713,1.09667703621089,671.643707254156,108.100404103287,614.346591054648,129.75303953737,0.74976603128016,0.838598096370697,1262.26991957333,2.2462889701128,0.711844315752387,664.148786680959,313.781090853363,366.770610731281,609.95114680212,429.907939201221,1567.35797223635,241.844640702009,873.135020877421,392.007611705363,241.305386519805,709.859875199012,1.34946953449398,537.826804565266,102.095965881646,295.72021593824,114.296688430384,1.84178639054298,0.702493105456233,28.9719722516835,4.18724319431931,352.041989909858,0.777451202087104,5.22420590464026],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  10 <\/br># assignments:  331 <\/br>Relative freq.:  0.7592 <\/br>Relative Rep. freq.:  0.0244","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  103 <\/br>Relative freq.:  0.2362 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  19 <\/br># assignments:  1 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  10 <\/br># assignments:  829 <\/br>Relative freq.:  0.6228 <\/br>Relative Rep. freq.:  0.0354","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  500 <\/br>Relative freq.:  0.3757 <\/br>Relative Rep. freq.:  0.0213","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  10 <\/br># assignments:  179 <\/br>Relative freq.:  0.6439 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  67 <\/br>Relative freq.:  0.241 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  08 <\/br># assignments:  32 <\/br>Relative freq.:  0.1151 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  10 <\/br># assignments:  350 <\/br>Relative freq.:  0.6306 <\/br>Relative Rep. freq.:  0.0148","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  205 <\/br>Relative freq.:  0.3694 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  10 <\/br># assignments:  311 <\/br>Relative freq.:  0.5759 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  229 <\/br>Relative freq.:  0.4241 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  10 <\/br># assignments:  519 <\/br>Relative freq.:  0.7997 <\/br>Relative Rep. freq.:  0.0213","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  130 <\/br>Relative freq.:  0.2003 <\/br>Relative Rep. freq.:  0.0053","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  10 <\/br># assignments:  165 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  10 <\/br># assignments:  188 <\/br>Relative freq.:  0.6714 <\/br>Relative Rep. freq.:  0.0304","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  92 <\/br>Relative freq.:  0.3286 <\/br>Relative Rep. freq.:  0.0149","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  10 <\/br># assignments:  348 <\/br>Relative freq.:  0.8131 <\/br>Relative Rep. freq.:  0.0246","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  80 <\/br>Relative freq.:  0.1869 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  10 <\/br># assignments:  102 <\/br>Relative freq.:  0.6335 <\/br>Relative Rep. freq.:  0.0182","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  38 <\/br>Relative freq.:  0.236 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  08 <\/br># assignments:  21 <\/br>Relative freq.:  0.1304 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  10 <\/br># assignments:  228 <\/br>Relative freq.:  0.5772 <\/br>Relative Rep. freq.:  0.0227","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  167 <\/br>Relative freq.:  0.4228 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  10 <\/br># assignments:  263 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  10 <\/br># assignments:  444 <\/br>Relative freq.:  0.7616 <\/br>Relative Rep. freq.:  0.0272","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  138 <\/br>Relative freq.:  0.2367 <\/br>Relative Rep. freq.:  0.0084","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  10 <\/br># assignments:  421 <\/br>Relative freq.:  0.8019 <\/br>Relative Rep. freq.:  0.0309","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  104 <\/br>Relative freq.:  0.1981 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  10 <\/br># assignments:  73 <\/br>Relative freq.:  0.6239 <\/br>Relative Rep. freq.:  0.0308","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  44 <\/br>Relative freq.:  0.3761 <\/br>Relative Rep. freq.:  0.0186","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  10 <\/br># assignments:  347 <\/br>Relative freq.:  0.9971 <\/br>Relative Rep. freq.:  0.013","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0029 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  10 <\/br># assignments:  422 <\/br>Relative freq.:  0.881 <\/br>Relative Rep. freq.:  0.0243","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  08 <\/br># assignments:  57 <\/br>Relative freq.:  0.119 <\/br>Relative Rep. freq.:  0.0033","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  10 <\/br># assignments:  509 <\/br>Relative freq.:  0.611 <\/br>Relative Rep. freq.:  0.0228","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  324 <\/br>Relative freq.:  0.389 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  263 <\/br>Relative freq.:  0.5229 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  10 <\/br># assignments:  240 <\/br>Relative freq.:  0.4771 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  10 <\/br># assignments:  147 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  10 <\/br># assignments:  604 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0236","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  10 <\/br># assignments:  55 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  10 <\/br># assignments:  300 <\/br>Relative freq.:  0.8043 <\/br>Relative Rep. freq.:  0.0242","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  73 <\/br>Relative freq.:  0.1957 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  10 <\/br># assignments:  191 <\/br>Relative freq.:  0.5932 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  131 <\/br>Relative freq.:  0.4068 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  10 <\/br># assignments:  1168 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.0472","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  10 <\/br># assignments:  7 <\/br>Relative freq.:  0.5385 <\/br>Relative Rep. freq.:  0.0201","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  6 <\/br>Relative freq.:  0.4615 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  10 <\/br># assignments:  69 <\/br>Relative freq.:  0.7753 <\/br>Relative Rep. freq.:  0.0387","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  0.2135 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0112 <\/br>Relative Rep. freq.:  6e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  10 <\/br># assignments:  567 <\/br>Relative freq.:  0.6156 <\/br>Relative Rep. freq.:  0.0273","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  354 <\/br>Relative freq.:  0.3844 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  10 <\/br># assignments:  66 <\/br>Relative freq.:  0.5946 <\/br>Relative Rep. freq.:  0.0121","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  45 <\/br>Relative freq.:  0.4054 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  10 <\/br># assignments:  131 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0174","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  10 <\/br># assignments:  145 <\/br>Relative freq.:  0.4723 <\/br>Relative Rep. freq.:  0.0128","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  116 <\/br>Relative freq.:  0.3779 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  08 <\/br># assignments:  45 <\/br>Relative freq.:  0.1466 <\/br>Relative Rep. freq.:  0.004","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  10 <\/br># assignments:  277 <\/br>Relative freq.:  0.8822 <\/br>Relative Rep. freq.:  0.034","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  37 <\/br>Relative freq.:  0.1178 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  10 <\/br># assignments:  543 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0297","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  10 <\/br># assignments:  473 <\/br>Relative freq.:  0.7963 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  08 <\/br># assignments:  121 <\/br>Relative freq.:  0.2037 <\/br>Relative Rep. freq.:  0.0043","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  10 <\/br># assignments:  365 <\/br>Relative freq.:  0.573 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  19 <\/br># assignments:  272 <\/br>Relative freq.:  0.427 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  10 <\/br># assignments:  679 <\/br>Relative freq.:  0.8321 <\/br>Relative Rep. freq.:  0.0392","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  136 <\/br>Relative freq.:  0.1667 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  10 <\/br># assignments:  862 <\/br>Relative freq.:  0.7865 <\/br>Relative Rep. freq.:  0.0441","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  234 <\/br>Relative freq.:  0.2135 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  08 <\/br># assignments:  41 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0035","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  10 <\/br># assignments:  151 <\/br>Relative freq.:  0.624 <\/br>Relative Rep. freq.:  0.0247","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  91 <\/br>Relative freq.:  0.376 <\/br>Relative Rep. freq.:  0.0149","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  10 <\/br># assignments:  283 <\/br>Relative freq.:  0.993 <\/br>Relative Rep. freq.:  0.0192","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  18 <\/br># assignments:  2 <\/br>Relative freq.:  0.007 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  10 <\/br># assignments:  49 <\/br>Relative freq.:  0.7101 <\/br>Relative Rep. freq.:  0.0146","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.2899 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  10 <\/br># assignments:  487 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0223","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  10 <\/br># assignments:  213 <\/br>Relative freq.:  0.7527 <\/br>Relative Rep. freq.:  0.0212","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  19 <\/br># assignments:  70 <\/br>Relative freq.:  0.2473 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  10 <\/br># assignments:  1078 <\/br>Relative freq.:  0.7706 <\/br>Relative Rep. freq.:  0.0294","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  319 <\/br>Relative freq.:  0.228 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  18 <\/br># assignments:  2 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  10 <\/br># assignments:  99 <\/br>Relative freq.:  0.5789 <\/br>Relative Rep. freq.:  0.0157","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  72 <\/br>Relative freq.:  0.4211 <\/br>Relative Rep. freq.:  0.0115","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  10 <\/br># assignments:  653 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  10 <\/br># assignments:  219 <\/br>Relative freq.:  0.605 <\/br>Relative Rep. freq.:  0.024","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  143 <\/br>Relative freq.:  0.395 <\/br>Relative Rep. freq.:  0.0157","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  10 <\/br># assignments:  1051 <\/br>Relative freq.:  0.7164 <\/br>Relative Rep. freq.:  0.0473","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  416 <\/br>Relative freq.:  0.2836 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  10 <\/br># assignments:  1 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  10 <\/br># assignments:  433 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0236","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  10 <\/br># assignments:  849 <\/br>Relative freq.:  0.5783 <\/br>Relative Rep. freq.:  0.0435","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  619 <\/br>Relative freq.:  0.4217 <\/br>Relative Rep. freq.:  0.0317","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  10 <\/br># assignments:  896 <\/br>Relative freq.:  0.7055 <\/br>Relative Rep. freq.:  0.0372","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  373 <\/br>Relative freq.:  0.2937 <\/br>Relative Rep. freq.:  0.0155","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  10 <\/br># assignments:  72 <\/br>Relative freq.:  0.5854 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  0.4146 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  10 <\/br># assignments:  443 <\/br>Relative freq.:  0.5768 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  325 <\/br>Relative freq.:  0.4232 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  10 <\/br># assignments:  1468 <\/br>Relative freq.:  0.7792 <\/br>Relative Rep. freq.:  0.0463","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  416 <\/br>Relative freq.:  0.2208 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  10 <\/br># assignments:  920 <\/br>Relative freq.:  0.9989 <\/br>Relative Rep. freq.:  0.0272","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  10 <\/br># assignments:  51 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.038","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  10 <\/br># assignments:  605 <\/br>Relative freq.:  0.746 <\/br>Relative Rep. freq.:  0.0284","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  202 <\/br>Relative freq.:  0.2491 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  07 <\/br># assignments:  3 <\/br>Relative freq.:  0.0037 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  10 <\/br># assignments:  346 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0152","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  10 <\/br># assignments:  545 <\/br>Relative freq.:  0.5671 <\/br>Relative Rep. freq.:  0.0285","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  07 <\/br># assignments:  211 <\/br>Relative freq.:  0.2196 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  205 <\/br>Relative freq.:  0.2133 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  10 <\/br># assignments:  885 <\/br>Relative freq.:  0.8429 <\/br>Relative Rep. freq.:  0.0475","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  165 <\/br>Relative freq.:  0.1571 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  10 <\/br># assignments:  274 <\/br>Relative freq.:  0.5037 <\/br>Relative Rep. freq.:  0.014","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  199 <\/br>Relative freq.:  0.3658 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  08 <\/br># assignments:  71 <\/br>Relative freq.:  0.1305 <\/br>Relative Rep. freq.:  0.0036","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  10 <\/br># assignments:  186 <\/br>Relative freq.:  0.6503 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  100 <\/br>Relative freq.:  0.3497 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  10 <\/br># assignments:  980 <\/br>Relative freq.:  0.5847 <\/br>Relative Rep. freq.:  0.0292","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  694 <\/br>Relative freq.:  0.4141 <\/br>Relative Rep. freq.:  0.0207","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  109 <\/br>Relative freq.:  0.645 <\/br>Relative Rep. freq.:  0.0144","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  10 <\/br># assignments:  60 <\/br>Relative freq.:  0.355 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  677 <\/br>Relative freq.:  0.5224 <\/br>Relative Rep. freq.:  0.0289","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  10 <\/br># assignments:  619 <\/br>Relative freq.:  0.4776 <\/br>Relative Rep. freq.:  0.0264","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  517 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0192","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  10 <\/br># assignments:  999 <\/br>Relative freq.:  0.6547 <\/br>Relative Rep. freq.:  0.0306","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  527 <\/br>Relative freq.:  0.3453 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  10 <\/br># assignments:  379 <\/br>Relative freq.:  0.5206 <\/br>Relative Rep. freq.:  0.0119","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  229 <\/br>Relative freq.:  0.3146 <\/br>Relative Rep. freq.:  0.0072","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  08 <\/br># assignments:  120 <\/br>Relative freq.:  0.1648 <\/br>Relative Rep. freq.:  0.0038","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  08 <\/br># assignments:  80 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  10 <\/br># assignments:  661 <\/br>Relative freq.:  0.758 <\/br>Relative Rep. freq.:  0.0266","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  211 <\/br>Relative freq.:  0.242 <\/br>Relative Rep. freq.:  0.0085","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  10 <\/br># assignments:  479 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  10 <\/br># assignments:  763 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0322","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  10 <\/br># assignments:  364 <\/br>Relative freq.:  0.7552 <\/br>Relative Rep. freq.:  0.0197","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  118 <\/br>Relative freq.:  0.2448 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  10 <\/br># assignments:  365 <\/br>Relative freq.:  0.6083 <\/br>Relative Rep. freq.:  0.0169","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  235 <\/br>Relative freq.:  0.3917 <\/br>Relative Rep. freq.:  0.0109","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  187 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  10 <\/br># assignments:  1045 <\/br>Relative freq.:  0.7422 <\/br>Relative Rep. freq.:  0.0321","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  363 <\/br>Relative freq.:  0.2578 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  08 <\/br># assignments:  258 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  10 <\/br># assignments:  864 <\/br>Relative freq.:  0.7455 <\/br>Relative Rep. freq.:  0.0378","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  294 <\/br>Relative freq.:  0.2537 <\/br>Relative Rep. freq.:  0.0129","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  10 <\/br># assignments:  471 <\/br>Relative freq.:  0.8177 <\/br>Relative Rep. freq.:  0.0344","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  105 <\/br>Relative freq.:  0.1823 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  10 <\/br># assignments:  167 <\/br>Relative freq.:  0.5901 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  116 <\/br>Relative freq.:  0.4099 <\/br>Relative Rep. freq.:  0.0081","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  10 <\/br># assignments:  678 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0287","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  10 <\/br># assignments:  672 <\/br>Relative freq.:  0.8615 <\/br>Relative Rep. freq.:  0.0728","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  108 <\/br>Relative freq.:  0.1385 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  10 <\/br># assignments:  614 <\/br>Relative freq.:  0.8231 <\/br>Relative Rep. freq.:  0.0329","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  130 <\/br>Relative freq.:  0.1743 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  10 <\/br># assignments:  1262 <\/br>Relative freq.:  0.9976 <\/br>Relative Rep. freq.:  0.0446","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  18 <\/br># assignments:  2 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  10 <\/br># assignments:  664 <\/br>Relative freq.:  0.6789 <\/br>Relative Rep. freq.:  0.0205","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  314 <\/br>Relative freq.:  0.3211 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  10 <\/br># assignments:  367 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0158","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  610 <\/br>Relative freq.:  0.5865 <\/br>Relative Rep. freq.:  0.0383","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  10 <\/br># assignments:  430 <\/br>Relative freq.:  0.4135 <\/br>Relative Rep. freq.:  0.027","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  10 <\/br># assignments:  1567 <\/br>Relative freq.:  0.8662 <\/br>Relative Rep. freq.:  0.0505","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  242 <\/br>Relative freq.:  0.1338 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  10 <\/br># assignments:  873 <\/br>Relative freq.:  0.6901 <\/br>Relative Rep. freq.:  0.0372","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  392 <\/br>Relative freq.:  0.3099 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  10 <\/br># assignments:  241 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0248","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  10 <\/br># assignments:  710 <\/br>Relative freq.:  0.9986 <\/br>Relative Rep. freq.:  0.0306","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  10 <\/br># assignments:  538 <\/br>Relative freq.:  0.8406 <\/br>Relative Rep. freq.:  0.043","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  102 <\/br>Relative freq.:  0.1594 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  10 <\/br># assignments:  296 <\/br>Relative freq.:  0.7167 <\/br>Relative Rep. freq.:  0.0235","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  114 <\/br>Relative freq.:  0.276 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  09 <\/br># assignments:  2 <\/br>Relative freq.:  0.0048 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  10 <\/br># assignments:  29 <\/br>Relative freq.:  0.8788 <\/br>Relative Rep. freq.:  0.0224","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  4 <\/br>Relative freq.:  0.1212 <\/br>Relative Rep. freq.:  0.0031","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  10 <\/br># assignments:  352 <\/br>Relative freq.:  0.9972 <\/br>Relative Rep. freq.:  0.015","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  18 <\/br># assignments:  1 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  08 <\/br># assignments:  5 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0026"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":209.14902449149,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,8.6],"tickmode":"array","ticktext":["V4-30-4*01","V4-30-4*01_A70G_A107G","V4-30-4*08","V4-31*01","V4-31*02","V4-31*03","V4-31*05","V4-31*11"],"tickvals":[1,2,3,4,5,6,7,8],"categoryorder":"array","categoryarray":["V4-30-4*01","V4-30-4*01_A70G_A107G","V4-30-4*08","V4-31*01","V4-31*02","V4-31*03","V4-31*05","V4-31*11"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-77.6809094058257,1645.69315707645],"tickmode":"array","ticktext":["0","500","1000","1500"],"tickvals":[0,500,1000,1500],"categoryorder":"array","categoryarray":["0","500","1000","1500"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5f3116e51bde9":{"x":{},"y":{},"text":{},"type":"box"},"5f3116c349f6b":{"x":{},"y":{},"text":{}}},"cur_data":"5f3116e51bde9","visdat":{"5f3116e51bde9":["function (y) ","x"],"5f3116c349f6b":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-2b0f4417b86aa673df08" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-2b0f4417b86aa673df08">{"x":{"data":[{"x":[5,1,1,5,5,1,5,1,5,1,5,5,1,4,5,1,3,5,4,5,1,5,1,4,1,5,1,1,5,1,5,2,5,1,1,5,5,3,1,5,1,2,5,1,4,5,1,5,1,5,1,3,2,5,3,5,1,2,1,5,1,5,5,1,5,3,5,1,1,5,1,5,1,6,2,5,3],"y":[191,27,70,56,109,41,19,3,192,100,68,191,25,1,64,34,25,47,1,153,52,185,38,1,42,30,27,125,80,78,59,1,26,20,114,87,74,19,57,72,40,1,227,71,1,320,86,128,30,50,33,13,1,72,16,33,23,1,80,72,61,39,19,11,90,7,163,50,51,1,163,217,75,57,1,146,20],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[5.06911702761427,0.936365298787132,1.09212488308549,4.99792888811789,4.91353033874184,1.07728581400588,5.06834935653023,0.921398603497073,4.98741268473677,0.941223845072091,5.09589960686862,4.94619174120016,1.05092510511167,3.95724547035061,5.04437317978591,0.92347875861451,2.95592778762802,4.94144465145655,3.97327408147976,4.90571100572124,1.07225171453319,4.99755259612575,1.09999728240073,4.04543714881875,1.07188466046937,4.99071325631812,1.0131834352389,1.02501606442966,4.96823978582397,0.963832249352709,5.01698365258053,1.91333809308708,5.05903841005638,1.09370477069169,1.08430735622533,4.98359657358378,5.03076830576174,3.00263440283015,0.942796810762957,4.92296184496954,1.09038944710046,2.01993172476068,4.99241687050089,0.937675140518695,3.9131919328589,5.00068476684392,1.07948460476473,4.99574335296638,0.986810000333935,4.94520433666185,0.982727424940094,2.95862255454995,2.02970106247813,4.95542296711355,2.93766158842482,5.01528087593615,0.956654561357573,1.98731746529229,0.933236336940899,5.06146514946595,1.05441062566824,4.95741082234308,5.08364421557635,1.03763687061146,5.09114124667831,2.92343689179979,4.93749899542891,0.998564663389698,0.916104460228235,4.97096342546865,1.07493749661371,4.95358180189505,1.04593767500483,5.92231711298227,1.97604132932611,5.01122360262088,2.91476356848143],"y":[190.755385923386,27.1281544419006,70.1402446070686,56.3095786850899,108.947866689973,41.1947526022792,19.2212487870827,2.64910440053791,192.168921293132,99.6889908995479,68.2564562637359,191.109629411437,25.0018953917548,0.901006395742297,64.3551614234224,34.2381033511832,25.1186189102009,47.0753046926111,1.31585674192756,152.971284065768,52.1664054412395,184.870127798803,37.6994433905929,1.33720402363688,41.7021292092279,29.6204738013446,26.999892299436,125.067106859013,80.1422180978581,78.2614264383912,59.2095912085846,1.10307460967451,26.169860695675,20.296084167622,113.616204079427,86.8081449586898,73.7396430086344,18.6560466710478,57.1487735372037,71.9660980269313,39.9014570139349,0.772179481945932,227.173600926064,70.6030508296564,1.3153582893312,320.237617192976,85.7004099002108,127.6047003638,29.7500815566629,49.8170670038089,32.7531587529927,12.8272126732394,1.00984147191048,71.8881927410141,15.9126111282036,32.8573137326166,22.8760587582365,0.85147608295083,80.3729164926335,71.7848012354225,61.1439887624234,39.2527542158961,18.9451292548329,11.2500232320279,90.2147761184722,7.14617353640497,162.690829627216,50.3886283557862,51.1640076946467,1.1493330674246,163.230574547686,216.666642266139,74.6758551361039,57.1162369104102,1.268027937226,145.969986444525,20.1749848719686],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  10 <\/br># assignments:  191 <\/br>Relative freq.:  0.8761 <\/br>Relative Rep. freq.:  0.0328","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.1239 <\/br>Relative Rep. freq.:  0.0046","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  70 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.0178","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  10 <\/br># assignments:  56 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  0.0142","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  10 <\/br># assignments:  109 <\/br>Relative freq.:  0.7267 <\/br>Relative Rep. freq.:  0.024","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  0.2733 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  10 <\/br># assignments:  19 <\/br>Relative freq.:  0.8636 <\/br>Relative Rep. freq.:  0.0216","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  3 <\/br>Relative freq.:  0.1364 <\/br>Relative Rep. freq.:  0.0034","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  10 <\/br># assignments:  192 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.026","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  100 <\/br>Relative freq.:  0.5952 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  10 <\/br># assignments:  68 <\/br>Relative freq.:  0.4048 <\/br>Relative Rep. freq.:  0.0114","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  10 <\/br># assignments:  191 <\/br>Relative freq.:  0.8802 <\/br>Relative Rep. freq.:  0.037","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  25 <\/br>Relative freq.:  0.1152 <\/br>Relative Rep. freq.:  0.0048","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0046 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  10 <\/br># assignments:  64 <\/br>Relative freq.:  0.5203 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  34 <\/br>Relative freq.:  0.2764 <\/br>Relative Rep. freq.:  0.0041","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  08 <\/br># assignments:  25 <\/br>Relative freq.:  0.2033 <\/br>Relative Rep. freq.:  0.003","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  10 <\/br># assignments:  47 <\/br>Relative freq.:  0.9792 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0208 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  10 <\/br># assignments:  153 <\/br>Relative freq.:  0.7463 <\/br>Relative Rep. freq.:  0.018","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  52 <\/br>Relative freq.:  0.2537 <\/br>Relative Rep. freq.:  0.0061","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  10 <\/br># assignments:  185 <\/br>Relative freq.:  0.8259 <\/br>Relative Rep. freq.:  0.0364","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  38 <\/br>Relative freq.:  0.1696 <\/br>Relative Rep. freq.:  0.0075","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0045 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  10 <\/br># assignments:  30 <\/br>Relative freq.:  0.5263 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  27 <\/br>Relative freq.:  0.4737 <\/br>Relative Rep. freq.:  0.0063","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  125 <\/br>Relative freq.:  0.6098 <\/br>Relative Rep. freq.:  0.0159","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  10 <\/br># assignments:  80 <\/br>Relative freq.:  0.3902 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  78 <\/br>Relative freq.:  0.5652 <\/br>Relative Rep. freq.:  0.0154","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  10 <\/br># assignments:  59 <\/br>Relative freq.:  0.4275 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0072 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  10 <\/br># assignments:  26 <\/br>Relative freq.:  0.5652 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  20 <\/br>Relative freq.:  0.4348 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  114 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  10 <\/br># assignments:  87 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0244","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  10 <\/br># assignments:  74 <\/br>Relative freq.:  0.7957 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  08 <\/br># assignments:  19 <\/br>Relative freq.:  0.2043 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  57 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0126","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  10 <\/br># assignments:  72 <\/br>Relative freq.:  0.6372 <\/br>Relative Rep. freq.:  0.0094","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  40 <\/br>Relative freq.:  0.354 <\/br>Relative Rep. freq.:  0.0052","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0088 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  10 <\/br># assignments:  227 <\/br>Relative freq.:  0.7592 <\/br>Relative Rep. freq.:  0.0256","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  71 <\/br>Relative freq.:  0.2375 <\/br>Relative Rep. freq.:  0.008","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  09 <\/br># assignments:  1 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  10 <\/br># assignments:  320 <\/br>Relative freq.:  0.7882 <\/br>Relative Rep. freq.:  0.0304","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  86 <\/br>Relative freq.:  0.2118 <\/br>Relative Rep. freq.:  0.0082","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  10 <\/br># assignments:  128 <\/br>Relative freq.:  0.8101 <\/br>Relative Rep. freq.:  0.0231","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  30 <\/br>Relative freq.:  0.1899 <\/br>Relative Rep. freq.:  0.0054","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  10 <\/br># assignments:  50 <\/br>Relative freq.:  0.5155 <\/br>Relative Rep. freq.:  0.0105","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  0.3402 <\/br>Relative Rep. freq.:  0.007","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  08 <\/br># assignments:  13 <\/br>Relative freq.:  0.134 <\/br>Relative Rep. freq.:  0.0027","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0103 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  10 <\/br># assignments:  72 <\/br>Relative freq.:  0.8182 <\/br>Relative Rep. freq.:  0.0124","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  08 <\/br># assignments:  16 <\/br>Relative freq.:  0.1818 <\/br>Relative Rep. freq.:  0.0028","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  10 <\/br># assignments:  33 <\/br>Relative freq.:  0.5789 <\/br>Relative Rep. freq.:  0.0065","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  0.4035 <\/br>Relative Rep. freq.:  0.0045","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0175 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  80 <\/br>Relative freq.:  0.5263 <\/br>Relative Rep. freq.:  0.0155","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  10 <\/br># assignments:  72 <\/br>Relative freq.:  0.4737 <\/br>Relative Rep. freq.:  0.014","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  61 <\/br>Relative freq.:  0.61 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  10 <\/br># assignments:  39 <\/br>Relative freq.:  0.39 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  10 <\/br># assignments:  19 <\/br>Relative freq.:  0.6333 <\/br>Relative Rep. freq.:  0.0169","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  11 <\/br>Relative freq.:  0.3667 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  10 <\/br># assignments:  90 <\/br>Relative freq.:  0.9278 <\/br>Relative Rep. freq.:  0.0292","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  08 <\/br># assignments:  7 <\/br>Relative freq.:  0.0722 <\/br>Relative Rep. freq.:  0.0023","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  10 <\/br># assignments:  163 <\/br>Relative freq.:  0.7653 <\/br>Relative Rep. freq.:  0.0193","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  0.2347 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  51 <\/br>Relative freq.:  0.9808 <\/br>Relative Rep. freq.:  0.0091","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  10 <\/br># assignments:  1 <\/br>Relative freq.:  0.0192 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  163 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.027","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  10 <\/br># assignments:  217 <\/br>Relative freq.:  0.62 <\/br>Relative Rep. freq.:  0.0406","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  75 <\/br>Relative freq.:  0.2143 <\/br>Relative Rep. freq.:  0.014","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  18 <\/br># assignments:  57 <\/br>Relative freq.:  0.1629 <\/br>Relative Rep. freq.:  0.0107","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0029 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  10 <\/br># assignments:  146 <\/br>Relative freq.:  0.8795 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  08 <\/br># assignments:  20 <\/br>Relative freq.:  0.1205 <\/br>Relative Rep. freq.:  0.0027"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":106.865919468659,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["V4-30-4*01","V4-30-4*08","V4-31*01","V4-31*02","V4-31*03","V4-31*11"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["V4-30-4*01","V4-30-4*08","V4-31*01","V4-31*02","V4-31*03","V4-31*11"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-15.2010924036056,336.210889078528],"tickmode":"array","ticktext":["0","100","200","300"],"tickvals":[0,100,200,300],"categoryorder":"array","categoryarray":["0","100","200","300"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"5f31178ca0574":{"x":{},"y":{},"text":{},"type":"box"},"5f3113abe042":{"x":{},"y":{},"text":{}}},"cur_data":"5f31178ca0574","visdat":{"5f31178ca0574":["function (y) ","x"],"5f3113abe042":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-29" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 4 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV4-30-4G35%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:800px;" cellspacing="0">
</iframe>
</div>
<div id="observations-33" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/v8rjoooj6dgo4fwk42cwu/IGHV4-30-4G35.docx?dl=0&amp;rlkey=pza9p8z08dydjgtzgv0cry1v8" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV4-30-4G35" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-34" class="section level2">
<h2>Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="34-G34.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="36-G36.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/35-G35.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
