<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV4-34 - G36 | Functional groups</title>
  <meta name="description" content="IGHV4-34 - G36 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV4-34 - G36 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV4-34 - G36 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="35-G35.html"/>
<link rel="next" href="37-G37.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-checklist"><i class="fa fa-check"></i>Group Checklist</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./">© Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv4-34---g36" class="section level1">
<h1>IGHV4-34 - G36</h1>
<div id="allele-appearnce-35" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of IGHV4-34 includes 10 alleles, 10 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-317-1.png" width="672" /></p>
</div>
<div id="group-alignment-30" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-318-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-34" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-33539f9004e45d25c669" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-33539f9004e45d25c669">{"x":{"data":[{"x":[1,3,1,2,1,1,2,1,2,1,2,1,1,2,1,2,1,2,1,1,2,1,2,1,1,1,2,3,1,2,1,2,1,1,3,1,2,1,1,2,1,2,3,1,2,1,2,1,1,1,1,2,1,1,2,1,1,2,1,1,2,1,1,1,1,1,2,1,2,1,2,1,2,3,1,1,1,1,1,2,1,3,1,2,3,1,1,2,1,2,1,1,2,3,1,2,3,1,1,2,3,1,2,1,3,1,1,2,1,2,1,2,1,1,3,2,1,2,1,1,2,1,2,3,1,2,1,1,2,1,2,1,2,1,2,1,2,1,2,1,1,2,1,2,1,2,1,1,2,1,2,3,1,2,1,2,3,1,2,1,1,2,1,1,2,1,2,1,1,2,1,2,3,1,2,1],"y":[1252,2,2156,4,995,2312,1,2250,1,2221,2,314,526,1,1060,2,349,2,448,1090,1,1571,3,668,125,3587,3,1,2633,4,2437,1,691,1091,2,2298,4,301,2036,3,1998,5,1,1371,1,2543,1,19,83,1230,346,4,720,610,1,354,2352,1,2092,3103,3,881,1176,583,339,998,1,214,1,1006,2,1258,14,1,2927,353,3086,447,1960,2,1373,1,2295,1,1,1152,874,1,469,1,2325,3766,2,1,3354,2,1,59,1049,6,5,1211,2,1517,1,1657,3063,2,865,1,2722,5,510,1478,2,1,3459,3,3006,2426,5,1102,1,1,1042,1,1248,857,3,1196,3,1668,1,1345,2,1645,1,2685,4,868,607,2,1098,1,3057,2,391,1369,2,1763,9,1,2520,2,1320,2,1,937,2,2060,1631,2,518,2308,8,991,1,808,89,2,1238,2,1,808,3,74],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[0.956239038426429,3.09632200682536,0.914755492797121,2.08569079199806,1.03586768168025,0.943498548585922,1.95134888053872,1.03354066764005,1.95329538793303,1.00854872255586,2.09003660064191,1.03625120255165,0.950977671239525,1.94664415470324,0.92705171960406,1.97126098517328,0.957413680339232,1.91513577341102,1.02424535597675,1.030032009352,2.056704253098,0.992293273890391,1.94526398140006,1.02673024730757,0.905369052197784,0.960940913064405,1.99598205117509,3.07542273513973,0.994268151419237,1.92446545702405,0.973496935330331,2.03410983979702,1.06830565417185,1.01046847235411,2.96044846670702,1.02464924496599,1.90799846723676,1.0215656586457,0.944491886300966,1.92468695468269,1.03440642599016,2.07866701856256,3.08038410795853,1.02175195794553,1.98486152738333,0.938735817559063,2.09973894981667,0.976279684714973,0.902341024205089,0.909542918857187,1.0106502817478,1.93928424464539,0.926384509447962,1.0768613376189,2.06342085441574,1.03178868568502,0.916024791775271,1.90007631201297,0.916818400099874,0.991327946307138,1.90225956388749,0.961645243875682,0.931863382644951,0.990391984768212,0.959144068788737,1.07428727969527,1.99407375180162,1.03920432543382,2.05127957114018,1.08642674931325,1.91563148847781,0.990950132301077,2.03435109830461,3.00232505626045,1.08472353662364,0.982972141588107,1.04594793524593,0.948096343269572,0.980377504788339,2.07776322606951,1.09255148628727,2.98001437913626,0.918978928076103,2.04261164763011,2.94760069795884,1.07262288858183,0.932200798159465,2.09634084720165,0.927533092908561,1.90287475599907,1.01900969506241,0.90395610909909,1.97928326339461,3.08107880037278,0.941826332453638,1.97928685299121,2.99357111430727,0.976000999845564,1.04777152324095,2.00321973739192,2.91308727138676,1.03107424876653,2.09877949845977,1.05224033505656,2.90017681522295,0.988242262043059,0.924531788798049,2.01733893444762,0.945954270055518,1.93393649710342,0.958476371271536,1.91310131684877,0.986046711402014,0.942345020035282,3.01448308886029,1.96376691092737,1.07296745157801,1.92447015331127,1.0114696632605,0.908546005561948,2.09358891327865,1.01306582689285,1.96588517944328,3.04495124719106,1.08238518023863,1.95490320925601,1.03700858904049,0.971532493736595,1.94897175822407,1.02903074650094,2.04969625188969,0.97408539946191,2.01789534096606,0.972540622251108,1.94747545961291,1.0747215654701,1.97428608173504,1.03016524165869,2.02306438237429,1.00552301974967,0.995271091116592,2.08206343464553,0.943251244025305,2.03362326137722,0.946251227101311,2.06704632854089,0.999416480306536,0.945726745389402,1.96552194831893,0.968017848907039,1.99999578315765,2.91490658489056,0.93340331800282,2.01527798944153,1.04362678849138,1.93364561158232,3.03503194418736,1.06507436181419,2.00649286876433,1.02156000621617,0.934274518862367,1.90298551204614,0.925769673055038,1.07347449874505,1.93157309032977,0.97981671593152,2.04370974325575,1.07809329605661,0.953218685276806,2.0219724833034,1.00851375535131,1.9685276941862,3.06548143988475,0.969139378936961,2.03319262959994,0.918841353803873],"y":[1252.27330766059,2.05616667680442,2155.87520887479,4.15461091585457,995.225659858435,2311.65672083851,0.650366690568626,2250.10784744285,0.864398846775293,2221.36175568867,1.63640148136765,314.282099360041,525.741352478042,1.08001551628113,1059.74129068255,1.73035253547132,348.973060550727,2.31961701624095,448.241268729418,1090.39258096796,0.912425816804171,1571.26053577065,2.69734175447375,668.253156574443,124.814135946892,3587.39195280634,2.96462417002767,0.944907439500093,2632.84830149394,4.1202550617978,2437.2460343875,0.739951926097274,691.116684843227,1090.78840623833,2.0261740475893,2298.30561133046,4.38317701872438,300.651782924123,2036.32130480129,2.85777867343277,1998.31828502938,4.69958094134927,0.831397590972483,1370.68892876413,0.663910532742739,2543.00789666027,1.37865022104234,19.2318189768121,82.6563936196268,1230.22336177807,345.775559506938,4.21890380326658,720.030517643318,609.712282477133,1.39045628011227,354.36135729868,2352.33327519801,0.784432974271476,2091.99323670473,3103.35678119529,2.62062309570611,881.245685573667,1176.26479729116,583.182780061848,338.755942192115,997.940333962254,0.711820939555764,213.737131441571,0.612900294736028,1006.00330823101,2.02950099576265,1257.81961713508,14.1286379063502,1.24508137330413,2927.01766807921,353.254785114154,3085.90763618797,446.844313531555,1959.83984631598,1.88099099062383,1373.29643524587,0.609734750166535,2294.79693830255,0.865860923193395,1.35131625477225,1151.88934268728,873.701502598636,0.957324914820492,469.169034464471,0.789228836819529,2324.87522217929,3765.88563061468,1.97843843623996,0.845844912715256,3354.05794866942,2.39939096234739,1.3172167211771,58.7021173037589,1049.03372871112,5.60362555999309,5.04314096607268,1211.32134129591,2.33464207202196,1517.2199940782,0.770075124129653,1656.67867511548,3062.83509035781,2.14049345180392,865.095214651711,1.12861400954425,2722.1978636736,4.77876013312489,510.137399750575,1478.24628386665,1.88095961939543,0.974210863746703,3459.38155504856,2.80141150746495,3005.88376711272,2426.29366433397,5.2767888167873,1102.28547481596,1.32586813755333,1.18991156928241,1042.21554682571,0.847879314795136,1247.66479501426,857.30639791023,3.2491485664621,1196.0101673061,3.09357971716672,1668.04721744955,1.10668716132641,1345.39641428236,1.69395458959043,1644.93344271537,1.13739759419113,2684.95293499734,3.94548851214349,868.258964608796,606.685179760121,1.84442382752895,1097.87614329308,1.23420134410262,3057.31219940297,2.23242699820548,391.326891163178,1369.09151389133,2.26216911748052,1762.7580190111,9.37298815008253,0.928181075491011,2520.2763231108,2.2002615230158,1320.02873677015,1.77902872432023,1.04137170594186,936.809431851842,1.83054285831749,2059.80807458069,1631.06455052998,2.29518490880728,518.190623901598,2307.79299737532,7.74378698412329,991.313156234473,1.11838080324233,808.012945131026,88.8288729596883,2.05318182222545,1237.90255920365,1.6113932127133,1.13328020833433,807.671055463515,2.68481287453324,73.9815902272239],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  1252 <\/br>Relative freq.:  0.9984 <\/br>Relative Rep. freq.:  0.1056","<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  12 <\/br># assignments:  2 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  2156 <\/br>Relative freq.:  0.9981 <\/br>Relative Rep. freq.:  0.0948","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  995 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0958","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  2312 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.0985","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  2250 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.0964","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  2221 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.0923","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  314 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0535","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  526 <\/br>Relative freq.:  0.9981 <\/br>Relative Rep. freq.:  0.0904","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  1060 <\/br>Relative freq.:  0.9981 <\/br>Relative Rep. freq.:  0.0762","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  349 <\/br>Relative freq.:  0.9943 <\/br>Relative Rep. freq.:  0.0633","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0057 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  448 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.046","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  1090 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.0501","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  1571 <\/br>Relative freq.:  0.9981 <\/br>Relative Rep. freq.:  0.1057","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  668 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0497","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  125 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0544","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  3587 <\/br>Relative freq.:  0.9989 <\/br>Relative Rep. freq.:  0.1371","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  3e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  2633 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.1528","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  2437 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.1105","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  691 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0304","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  1091 <\/br>Relative freq.:  0.9982 <\/br>Relative Rep. freq.:  0.1078","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  12 <\/br># assignments:  2 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  2298 <\/br>Relative freq.:  0.9983 <\/br>Relative Rep. freq.:  0.0928","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  301 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0784","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  2036 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0933","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  1998 <\/br>Relative freq.:  0.997 <\/br>Relative Rep. freq.:  0.1923","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0025 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  5e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  1371 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.0684","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  2543 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.1045","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  19 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0572","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  83 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0475","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  1230 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0601","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  346 <\/br>Relative freq.:  0.9886 <\/br>Relative Rep. freq.:  0.0683","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0114 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  720 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1011","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  610 <\/br>Relative freq.:  0.9984 <\/br>Relative Rep. freq.:  0.0545","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  354 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0442","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  2352 <\/br>Relative freq.:  0.9996 <\/br>Relative Rep. freq.:  0.1337","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  2092 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0748","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  3103 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.1297","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  881 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0522","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  1176 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0609","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  583 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0517","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  339 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0566","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  998 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.0684","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  214 <\/br>Relative freq.:  0.9953 <\/br>Relative Rep. freq.:  0.0675","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  1006 <\/br>Relative freq.:  0.998 <\/br>Relative Rep. freq.:  0.0465","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  1258 <\/br>Relative freq.:  0.9882 <\/br>Relative Rep. freq.:  0.1729","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  02 <\/br># assignments:  14 <\/br>Relative freq.:  0.011 <\/br>Relative Rep. freq.:  0.0019","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  2927 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0809","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  353 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.057","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  3086 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1145","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  447 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0497","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  1960 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.0897","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  1373 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.0648","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  01 <\/br># assignments:  2295 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.1266","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  4e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  1152 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0603","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  874 <\/br>Relative freq.:  0.9989 <\/br>Relative Rep. freq.:  0.0381","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  469 <\/br>Relative freq.:  0.9979 <\/br>Relative Rep. freq.:  0.0633","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  2325 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0861","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  3766 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.1205","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  5e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  3e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  3354 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.1049","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  3e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  59 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.045","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  1049 <\/br>Relative freq.:  0.9896 <\/br>Relative Rep. freq.:  0.0551","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  02 <\/br># assignments:  6 <\/br>Relative freq.:  0.0057 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  12 <\/br># assignments:  5 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  1211 <\/br>Relative freq.:  0.9984 <\/br>Relative Rep. freq.:  0.0559","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  1517 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.0817","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  1657 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0895","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  3063 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.157","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  865 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.074","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  2722 <\/br>Relative freq.:  0.9982 <\/br>Relative Rep. freq.:  0.0831","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  510 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0686","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  1478 <\/br>Relative freq.:  0.998 <\/br>Relative Rep. freq.:  0.0652","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  12 <\/br># assignments:  2 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  3459 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.131","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  3006 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0929","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  2426 <\/br>Relative freq.:  0.9979 <\/br>Relative Rep. freq.:  0.0783","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  02 <\/br># assignments:  5 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  1102 <\/br>Relative freq.:  0.9982 <\/br>Relative Rep. freq.:  0.0621","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  1042 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.0421","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  1248 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0732","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  01 <\/br># assignments:  857 <\/br>Relative freq.:  0.9965 <\/br>Relative Rep. freq.:  0.0367","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0035 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  1196 <\/br>Relative freq.:  0.9975 <\/br>Relative Rep. freq.:  0.0702","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0025 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  1668 <\/br>Relative freq.:  0.9994 <\/br>Relative Rep. freq.:  0.0777","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  1345 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0773","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  1645 <\/br>Relative freq.:  0.9994 <\/br>Relative Rep. freq.:  0.0511","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  01 <\/br># assignments:  2685 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.092","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  02 <\/br># assignments:  4 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  868 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0393","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  607 <\/br>Relative freq.:  0.9967 <\/br>Relative Rep. freq.:  0.0495","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0033 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  1098 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.0791","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  01 <\/br># assignments:  3057 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.1314","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  391 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0433","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  1369 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.074","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  1763 <\/br>Relative freq.:  0.9944 <\/br>Relative Rep. freq.:  0.0689","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  02 <\/br># assignments:  9 <\/br>Relative freq.:  0.0051 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  6e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  2520 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.0789","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  01 <\/br># assignments:  1320 <\/br>Relative freq.:  0.9977 <\/br>Relative Rep. freq.:  0.0595","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  937 <\/br>Relative freq.:  0.9979 <\/br>Relative Rep. freq.:  0.0595","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  2060 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.067","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  1631 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0723","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  518 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0565","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  2308 <\/br>Relative freq.:  0.9965 <\/br>Relative Rep. freq.:  0.1051","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  02 <\/br># assignments:  8 <\/br>Relative freq.:  0.0035 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  991 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.0812","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  808 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0661","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  89 <\/br>Relative freq.:  0.978 <\/br>Relative Rep. freq.:  0.0808","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.022 <\/br>Relative Rep. freq.:  0.0018","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  1238 <\/br>Relative freq.:  0.9976 <\/br>Relative Rep. freq.:  0.0569","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  808 <\/br>Relative freq.:  0.9963 <\/br>Relative Rep. freq.:  0.0496","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0037 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  74 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0401"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V4-34*01","V4-34*02","V4-34*12"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V4-34*01","V4-34*02","V4-34*12"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-187.659778512325,3954.26951326249],"tickmode":"array","ticktext":["0","1000","2000","3000"],"tickvals":[0,1000,2000,3000],"categoryorder":"array","categoryarray":["0","1000","2000","3000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"801a95198b74f":{"x":{},"y":{},"text":{},"type":"box"},"801a91c6e64c1":{"x":{},"y":{},"text":{}}},"cur_data":"801a95198b74f","visdat":{"801a95198b74f":["function (y) ","x"],"801a91c6e64c1":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-fa0cd4d5184bff1584e0" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-fa0cd4d5184bff1584e0">{"x":{"data":[{"x":[1,1,1,1,1,1,2,1,1,2,1,1,1,3,1,1,2,1,1,1,2,1,1,2,1,1,2,1,1,2,1,1,1,2,1,2,1,1,1,2,1,1,2,3,1,2,1,1,1,1,1,1,1,2,1,2,1,2,1,2,1],"y":[383,263,395,33,1034,486,2,577,937,1,861,1664,1000,1,160,837,1,436,354,564,1,230,226,1,507,343,1,809,267,1,593,516,855,1,620,1,646,493,745,2,525,841,3,2,650,1,417,534,2044,220,50,323,840,2,726,1,639,3,372,1,1409],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(88,131,0,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[0.981755381915718,0.945109533146024,0.95250059813261,0.924692439427599,1.09688830133528,0.934891319321469,1.94857268924825,0.979252053471282,0.950010926602408,1.98883973169141,1.07077380041592,1.00096588479355,0.978104374418035,2.94158062851056,0.986695278668776,0.914935408858582,2.04953579036519,1.06662195050158,1.03984509832226,0.919786822237074,1.93538645338267,0.957146799424663,0.915323439147323,1.9067191068083,1.0144294637721,1.00661794277839,2.00189131074585,0.911781244259328,1.0471015653573,1.94525489271618,1.02494675372727,1.05390506312251,1.03882606904954,2.05543303526938,1.04776461995207,2.00887562190182,0.993499608524144,0.965961494483054,1.02727515441366,2.07257246575318,1.02610704810359,1.09476136951707,2.01365675046109,3.05948865623213,0.986438393034041,1.96470221397467,1.03941406011581,0.980013655964285,0.924858797388151,1.02020324175246,1.06716964459047,0.918168034683913,0.950320719648153,1.92153395111673,0.932398035703227,2.07510264818557,0.91996800894849,2.07049187771045,0.996403452800587,2.05039136237465,0.901871731737629],"y":[383.390790827386,262.915880651586,395.233134176023,32.7127536876127,1034.29932659287,485.735780513473,2.21277914401144,577.146188259311,936.62187567614,1.24663829505444,860.870025300421,1664.16546275355,999.636138107255,1.3510334206745,160.340857398883,837.002609383129,1.29246918242425,435.974095244147,354.272303165123,563.731743651815,1.3438276514411,230.211913991719,225.610169291496,1.26498016174883,506.907303017005,342.849035268091,1.3194947110489,808.88887711782,267.11570776049,1.11984507963061,592.754423816688,516.155201283097,854.695249983855,1.37689482513815,619.668640403077,1.34129022359848,645.785345231183,493.000698071532,745.266203676723,1.62353159114718,524.618409095146,841.151015857235,3.0163946967572,2.02427703943104,650.134456744604,1.34995368979871,416.809413675405,534.215416851826,2043.73511477485,220.182304613292,50.1648195294663,323.365757846087,839.641806131415,2.35704692490399,726.051223721355,0.615411964245141,638.980969489366,2.70630569010973,371.756798441708,1.19158381056041,1408.69658524282],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  383 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0675","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  263 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0695","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  395 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0887","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  33 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0392","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  1034 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1432","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  486 <\/br>Relative freq.:  0.9959 <\/br>Relative Rep. freq.:  0.0834","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0041 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  577 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1138","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  937 <\/br>Relative freq.:  0.9989 <\/br>Relative Rep. freq.:  0.1188","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  861 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.158","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  1664 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.2745","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  1000 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.1227","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  12 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  160 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0321","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  837 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.1431","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  436 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1127","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  354 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.085","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  564 <\/br>Relative freq.:  0.9982 <\/br>Relative Rep. freq.:  0.0764","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  230 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0462","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  226 <\/br>Relative freq.:  0.9956 <\/br>Relative Rep. freq.:  0.1034","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0044 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  507 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0853","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  343 <\/br>Relative freq.:  0.9971 <\/br>Relative Rep. freq.:  0.0993","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0029 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  809 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1181","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  267 <\/br>Relative freq.:  0.9963 <\/br>Relative Rep. freq.:  0.0606","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0037 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  593 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0794","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  516 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0904","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  855 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0985","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  620 <\/br>Relative freq.:  0.9984 <\/br>Relative Rep. freq.:  0.0594","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  646 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.126","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  493 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1114","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  745 <\/br>Relative freq.:  0.9973 <\/br>Relative Rep. freq.:  0.1362","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  01 <\/br># assignments:  525 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1006","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  841 <\/br>Relative freq.:  0.9941 <\/br>Relative Rep. freq.:  0.2184","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0035 <\/br>Relative Rep. freq.:  8e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  12 <\/br># assignments:  2 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  650 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0714","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  417 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1369","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  534 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1075","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  2044 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1333","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  220 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0517","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  50 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0447","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  323 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1088","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  840 <\/br>Relative freq.:  0.9976 <\/br>Relative Rep. freq.:  0.1044","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  02 <\/br># assignments:  2 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  726 <\/br>Relative freq.:  0.9986 <\/br>Relative Rep. freq.:  0.134","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  639 <\/br>Relative freq.:  0.9953 <\/br>Relative Rep. freq.:  0.1142","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  02 <\/br># assignments:  3 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  5e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  372 <\/br>Relative freq.:  0.9973 <\/br>Relative Rep. freq.:  0.0727","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  1409 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1939"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(88,131,0,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(88,131,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V4-34*01","V4-34*02","V4-34*12"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V4-34*01","V4-34*02","V4-34*12"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-101.553817437543,2146.16922940179],"tickmode":"array","ticktext":["0","500","1000","1500","2000"],"tickvals":[0,500,1000,1500,2000],"categoryorder":"array","categoryarray":["0","500","1000","1500","2000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"801a943f9192b":{"x":{},"y":{},"text":{},"type":"box"},"801a94aa57410":{"x":{},"y":{},"text":{}}},"cur_data":"801a943f9192b","visdat":{"801a943f9192b":["function (y) ","x"],"801a94aa57410":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-30" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 2 zygousity states.</p>
<style>
  
#scroll-box {
  overflow-y: scroll;
  overflow-x: scroll !important;
}
</style>
<div class="container">
<iframe id="scroll-box" src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV4-34G36%22" scrolling="yes" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:800px;overflow-x: scroll !important;overflow: scroll;" cellspacing="0">
</iframe>
</div>
</div>
<div id="observations-34" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/p6swnkdifhmjlbicc6kfx/IGHV4-34G36.docx?dl=0&amp;rlkey=v81zkdky34gijpj0gv8wpws81" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV4-34G36" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-35" class="section level2">
<h2>Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="35-G35.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="37-G37.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/36-G36.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
