<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>47 IGHV5-51 - G46 | Functional groups</title>
  <meta name="description" content="47 IGHV5-51 - G46 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="47 IGHV5-51 - G46 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="47 IGHV5-51 - G46 | Functional groups" />
  
  
  



<meta name="date" content="2021-11-30" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="45-G45.html"/>
<link rel="next" href="47-G47.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Functional groups</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i><b>1.1</b> Sequence pre-processing</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i><b>1.2</b> Group devision by 95% similarity</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#projects-sequence-depth"><i class="fa fa-check"></i><b>1.3</b> Projects sequence depth</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i><b>2</b> IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="2.1" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i><b>2.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="2.2" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i><b>2.2</b> Group alignment</a></li>
<li class="chapter" data-level="2.3" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i><b>2.3</b> Sequence depth</a></li>
<li class="chapter" data-level="2.4" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i><b>2.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="2.5" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i><b>2.5</b> Observations</a></li>
<li class="chapter" data-level="2.6" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i><b>3</b> IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="3.1" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i><b>3.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="3.2" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i><b>3.2</b> Group alignment</a></li>
<li class="chapter" data-level="3.3" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i><b>3.3</b> Sequence depth</a></li>
<li class="chapter" data-level="3.4" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i><b>3.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="3.5" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i><b>3.5</b> Observations</a></li>
<li class="chapter" data-level="3.6" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i><b>3.6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="02-G2.html"><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05"><i class="fa fa-check"></i><b>3.6.1</b> Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i><b>4</b> IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="4.1" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i><b>4.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="4.2" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i><b>4.2</b> Sequence depth</a></li>
<li class="chapter" data-level="4.3" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i><b>4.3</b> Observations</a></li>
<li class="chapter" data-level="4.4" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i><b>4.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i><b>5</b> IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="5.1" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i><b>5.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="5.2" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i><b>5.2</b> Group alignment</a></li>
<li class="chapter" data-level="5.3" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i><b>5.3</b> Sequence depth</a></li>
<li class="chapter" data-level="5.4" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i><b>5.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="5.5" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i><b>5.5</b> Observations</a></li>
<li class="chapter" data-level="5.6" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i><b>5.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i><b>6</b> IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="6.1" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i><b>6.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="6.2" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i><b>6.2</b> Group alignment</a></li>
<li class="chapter" data-level="6.3" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i><b>6.3</b> Sequence depth</a></li>
<li class="chapter" data-level="6.4" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i><b>6.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="6.5" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i><b>6.5</b> Observations</a></li>
<li class="chapter" data-level="6.6" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i><b>6.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i><b>7</b> IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="7.1" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i><b>7.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="7.2" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i><b>7.2</b> Group alignment</a></li>
<li class="chapter" data-level="7.3" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i><b>7.3</b> Sequence depth</a></li>
<li class="chapter" data-level="7.4" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i><b>7.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="7.5" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i><b>7.5</b> Observations</a></li>
<li class="chapter" data-level="7.6" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i><b>7.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i><b>8</b> IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="8.1" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i><b>8.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="8.2" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i><b>8.2</b> Group alignment</a></li>
<li class="chapter" data-level="8.3" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i><b>8.3</b> Sequence depth</a></li>
<li class="chapter" data-level="8.4" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i><b>8.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="8.5" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i><b>8.5</b> Observations</a></li>
<li class="chapter" data-level="8.6" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i><b>8.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i><b>9</b> IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="9.1" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i><b>9.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="9.2" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i><b>9.2</b> Group alignment</a></li>
<li class="chapter" data-level="9.3" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i><b>9.3</b> Sequence depth</a></li>
<li class="chapter" data-level="9.4" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i><b>9.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="9.5" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i><b>9.5</b> Observations</a></li>
<li class="chapter" data-level="9.6" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i><b>9.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i><b>10</b> IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="10.1" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i><b>10.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="10.2" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i><b>10.2</b> Sequence depth</a></li>
<li class="chapter" data-level="10.3" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i><b>10.3</b> Observations</a></li>
<li class="chapter" data-level="10.4" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i><b>10.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i><b>11</b> IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="11.1" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i><b>11.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="11.2" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i><b>11.2</b> Group alignment</a></li>
<li class="chapter" data-level="11.3" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i><b>11.3</b> Sequence depth</a></li>
<li class="chapter" data-level="11.4" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i><b>11.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="11.5" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i><b>11.5</b> Observations</a></li>
<li class="chapter" data-level="11.6" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i><b>11.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i><b>12</b> IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="12.1" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i><b>12.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="12.2" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i><b>12.2</b> Group alignment</a></li>
<li class="chapter" data-level="12.3" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i><b>12.3</b> Sequence depth</a></li>
<li class="chapter" data-level="12.4" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i><b>12.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="12.5" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i><b>12.5</b> Observations</a></li>
<li class="chapter" data-level="12.6" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i><b>12.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i><b>13</b> IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="13.1" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i><b>13.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="13.2" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i><b>13.2</b> Group alignment</a></li>
<li class="chapter" data-level="13.3" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i><b>13.3</b> Sequence depth</a></li>
<li class="chapter" data-level="13.4" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i><b>13.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="13.5" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i><b>13.5</b> Observations</a></li>
<li class="chapter" data-level="13.6" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i><b>13.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i><b>14</b> IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="14.1" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i><b>14.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="14.2" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i><b>14.2</b> Group alignment</a></li>
<li class="chapter" data-level="14.3" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i><b>14.3</b> Sequence depth</a></li>
<li class="chapter" data-level="14.4" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i><b>14.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="14.5" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i><b>14.5</b> Observations</a></li>
<li class="chapter" data-level="14.6" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i><b>14.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i><b>15</b> IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="15.1" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i><b>15.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="15.2" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i><b>15.2</b> Group alignment</a></li>
<li class="chapter" data-level="15.3" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i><b>15.3</b> Sequence depth</a></li>
<li class="chapter" data-level="15.4" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i><b>15.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="15.5" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i><b>15.5</b> Observations</a></li>
<li class="chapter" data-level="15.6" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i><b>15.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i><b>16</b> IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="16.1" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i><b>16.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="16.2" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i><b>16.2</b> Group alignment</a></li>
<li class="chapter" data-level="16.3" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i><b>16.3</b> Sequence depth</a></li>
<li class="chapter" data-level="16.4" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i><b>16.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="16.5" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i><b>16.5</b> Observations</a></li>
<li class="chapter" data-level="16.6" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i><b>16.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i><b>17</b> IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="17.1" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i><b>17.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="17.2" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i><b>17.2</b> Group alignment</a></li>
<li class="chapter" data-level="17.3" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i><b>17.3</b> Sequence depth</a></li>
<li class="chapter" data-level="17.4" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i><b>17.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="17.5" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i><b>17.5</b> Observations</a></li>
<li class="chapter" data-level="17.6" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i><b>17.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i><b>18</b> IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="18.1" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i><b>18.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="18.2" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i><b>18.2</b> Group alignment</a></li>
<li class="chapter" data-level="18.3" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i><b>18.3</b> Sequence depth</a></li>
<li class="chapter" data-level="18.4" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i><b>18.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="18.5" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i><b>18.5</b> Observations</a></li>
<li class="chapter" data-level="18.6" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i><b>18.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i><b>19</b> IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="19.1" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i><b>19.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="19.2" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i><b>19.2</b> Group alignment</a></li>
<li class="chapter" data-level="19.3" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i><b>19.3</b> Sequence depth</a></li>
<li class="chapter" data-level="19.4" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i><b>19.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="19.5" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i><b>19.5</b> Observations</a></li>
<li class="chapter" data-level="19.6" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i><b>19.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i><b>20</b> IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="20.1" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i><b>20.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="20.2" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i><b>20.2</b> Group alignment</a></li>
<li class="chapter" data-level="20.3" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i><b>20.3</b> Sequence depth</a></li>
<li class="chapter" data-level="20.4" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i><b>20.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="20.5" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i><b>20.5</b> Observations</a></li>
<li class="chapter" data-level="20.6" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i><b>20.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i><b>21</b> IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="21.1" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i><b>21.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="21.2" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i><b>21.2</b> Group alignment</a></li>
<li class="chapter" data-level="21.3" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i><b>21.3</b> Sequence depth</a></li>
<li class="chapter" data-level="21.4" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i><b>21.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="21.5" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i><b>21.5</b> Observations</a></li>
<li class="chapter" data-level="21.6" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i><b>21.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="22" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i><b>22</b> IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="22.1" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i><b>22.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="22.2" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i><b>22.2</b> Sequence depth</a></li>
<li class="chapter" data-level="22.3" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i><b>22.3</b> Observations</a></li>
<li class="chapter" data-level="22.4" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i><b>22.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i><b>23</b> IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="23.1" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i><b>23.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="23.2" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i><b>23.2</b> Group alignment</a></li>
<li class="chapter" data-level="23.3" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i><b>23.3</b> Sequence depth</a></li>
<li class="chapter" data-level="23.4" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i><b>23.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="23.5" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i><b>23.5</b> Observations</a></li>
<li class="chapter" data-level="23.6" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i><b>23.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="24" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i><b>24</b> IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="24.1" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i><b>24.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="24.2" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i><b>24.2</b> Group alignment</a></li>
<li class="chapter" data-level="24.3" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i><b>24.3</b> Sequence depth</a></li>
<li class="chapter" data-level="24.4" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i><b>24.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="24.5" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i><b>24.5</b> Observations</a></li>
<li class="chapter" data-level="24.6" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i><b>24.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i><b>25</b> IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="25.1" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i><b>25.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="25.2" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i><b>25.2</b> Group alignment</a></li>
<li class="chapter" data-level="25.3" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i><b>25.3</b> Sequence depth</a></li>
<li class="chapter" data-level="25.4" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i><b>25.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="25.5" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i><b>25.5</b> Observations</a></li>
<li class="chapter" data-level="25.6" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i><b>25.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i><b>26</b> IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="26.1" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i><b>26.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="26.2" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i><b>26.2</b> Group alignment</a></li>
<li class="chapter" data-level="26.3" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i><b>26.3</b> Sequence depth</a></li>
<li class="chapter" data-level="26.4" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i><b>26.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="26.5" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i><b>26.5</b> Observations</a></li>
<li class="chapter" data-level="26.6" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i><b>26.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i><b>27</b> IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="27.1" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i><b>27.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="27.2" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i><b>27.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i><b>28</b> IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="28.1" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i><b>28.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="28.2" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i><b>28.2</b> Group alignment</a></li>
<li class="chapter" data-level="28.3" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i><b>28.3</b> Sequence depth</a></li>
<li class="chapter" data-level="28.4" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i><b>28.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="28.5" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i><b>28.5</b> Observations</a></li>
<li class="chapter" data-level="28.6" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i><b>28.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="29" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i><b>29</b> IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="29.1" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i><b>29.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="29.2" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i><b>29.2</b> Group alignment</a></li>
<li class="chapter" data-level="29.3" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i><b>29.3</b> Sequence depth</a></li>
<li class="chapter" data-level="29.4" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i><b>29.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="29.5" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i><b>29.5</b> Observations</a></li>
<li class="chapter" data-level="29.6" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i><b>29.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="30" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i><b>30</b> IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="30.1" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i><b>30.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="30.2" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i><b>30.2</b> Sequence depth</a></li>
<li class="chapter" data-level="30.3" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i><b>30.3</b> Observations</a></li>
<li class="chapter" data-level="30.4" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i><b>30.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="31" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i><b>31</b> IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="31.1" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i><b>31.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="31.2" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i><b>31.2</b> Group alignment</a></li>
<li class="chapter" data-level="31.3" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i><b>31.3</b> Sequence depth</a></li>
<li class="chapter" data-level="31.4" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i><b>31.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="31.5" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i><b>31.5</b> Observations</a></li>
<li class="chapter" data-level="31.6" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i><b>31.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="32" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i><b>32</b> IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="32.1" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i><b>32.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="32.2" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i><b>32.2</b> Group alignment</a></li>
<li class="chapter" data-level="32.3" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i><b>32.3</b> Sequence depth</a></li>
<li class="chapter" data-level="32.4" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i><b>32.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="32.5" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i><b>32.5</b> Observations</a></li>
<li class="chapter" data-level="32.6" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i><b>32.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="33" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i><b>33</b> IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="33.1" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i><b>33.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="33.2" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i><b>33.2</b> Group alignment</a></li>
<li class="chapter" data-level="33.3" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i><b>33.3</b> Sequence depth</a></li>
<li class="chapter" data-level="33.4" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i><b>33.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="33.5" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i><b>33.5</b> Observations</a></li>
<li class="chapter" data-level="33.6" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i><b>33.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="34" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i><b>34</b> IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="34.1" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i><b>34.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="34.2" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i><b>34.2</b> Group alignment</a></li>
<li class="chapter" data-level="34.3" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i><b>34.3</b> Sequence depth</a></li>
<li class="chapter" data-level="34.4" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i><b>34.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="34.5" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i><b>34.5</b> Observations</a></li>
<li class="chapter" data-level="34.6" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i><b>34.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="35" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i><b>35</b> IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="35.1" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i><b>35.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="35.2" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i><b>35.2</b> Group alignment</a></li>
<li class="chapter" data-level="35.3" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i><b>35.3</b> Sequence depth</a></li>
<li class="chapter" data-level="35.4" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i><b>35.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="35.5" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i><b>35.5</b> Observations</a></li>
<li class="chapter" data-level="35.6" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i><b>35.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="36" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i><b>36</b> IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="36.1" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i><b>36.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="36.2" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i><b>36.2</b> Group alignment</a></li>
<li class="chapter" data-level="36.3" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i><b>36.3</b> Sequence depth</a></li>
<li class="chapter" data-level="36.4" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i><b>36.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="36.5" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i><b>36.5</b> Observations</a></li>
<li class="chapter" data-level="36.6" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i><b>36.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="37" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i><b>37</b> IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="37.1" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i><b>37.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="37.2" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i><b>37.2</b> Group alignment</a></li>
<li class="chapter" data-level="37.3" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i><b>37.3</b> Sequence depth</a></li>
<li class="chapter" data-level="37.4" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i><b>37.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="37.5" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i><b>37.5</b> Observations</a></li>
<li class="chapter" data-level="37.6" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i><b>37.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="38" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i><b>38</b> IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="38.1" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i><b>38.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="38.2" data-path="37-G37.html"><a href="37-G37.html#group-alignment-31"><i class="fa fa-check"></i><b>38.2</b> Group alignment</a></li>
<li class="chapter" data-level="38.3" data-path="37-G37.html"><a href="37-G37.html#sequence-depth-35"><i class="fa fa-check"></i><b>38.3</b> Sequence depth</a></li>
<li class="chapter" data-level="38.4" data-path="37-G37.html"><a href="37-G37.html#cutoff-31"><i class="fa fa-check"></i><b>38.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="38.5" data-path="37-G37.html"><a href="37-G37.html#observations-35"><i class="fa fa-check"></i><b>38.5</b> Observations</a></li>
<li class="chapter" data-level="38.6" data-path="37-G37.html"><a href="37-G37.html#conclusions-36"><i class="fa fa-check"></i><b>38.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="39" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i><b>39</b> IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="39.1" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i><b>39.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="39.2" data-path="38-G38.html"><a href="38-G38.html#sequence-depth-36"><i class="fa fa-check"></i><b>39.2</b> Sequence depth</a></li>
<li class="chapter" data-level="39.3" data-path="38-G38.html"><a href="38-G38.html#observations-36"><i class="fa fa-check"></i><b>39.3</b> Observations</a></li>
<li class="chapter" data-level="39.4" data-path="38-G38.html"><a href="38-G38.html#conclusions-37"><i class="fa fa-check"></i><b>39.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="40" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i><b>40</b> IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="40.1" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i><b>40.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="40.2" data-path="39-G39.html"><a href="39-G39.html#group-alignment-32"><i class="fa fa-check"></i><b>40.2</b> Group alignment</a></li>
<li class="chapter" data-level="40.3" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-37"><i class="fa fa-check"></i><b>40.3</b> Sequence depth</a></li>
<li class="chapter" data-level="40.4" data-path="39-G39.html"><a href="39-G39.html#cutoff-32"><i class="fa fa-check"></i><b>40.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="40.5" data-path="39-G39.html"><a href="39-G39.html#observations-37"><i class="fa fa-check"></i><b>40.5</b> Observations</a></li>
<li class="chapter" data-level="40.6" data-path="39-G39.html"><a href="39-G39.html#conclusions-38"><i class="fa fa-check"></i><b>40.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="41" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i><b>41</b> IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="41.1" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i><b>41.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="41.2" data-path="40-G40.html"><a href="40-G40.html#group-alignment-33"><i class="fa fa-check"></i><b>41.2</b> Group alignment</a></li>
<li class="chapter" data-level="41.3" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-38"><i class="fa fa-check"></i><b>41.3</b> Sequence depth</a></li>
<li class="chapter" data-level="41.4" data-path="40-G40.html"><a href="40-G40.html#cutoff-33"><i class="fa fa-check"></i><b>41.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="41.5" data-path="40-G40.html"><a href="40-G40.html#observations-38"><i class="fa fa-check"></i><b>41.5</b> Observations</a></li>
<li class="chapter" data-level="41.6" data-path="40-G40.html"><a href="40-G40.html#conclusions-39"><i class="fa fa-check"></i><b>41.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="42" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i><b>42</b> IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="42.1" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i><b>42.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="42.2" data-path="41-G41.html"><a href="41-G41.html#group-alignment-34"><i class="fa fa-check"></i><b>42.2</b> Group alignment</a></li>
<li class="chapter" data-level="42.3" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-39"><i class="fa fa-check"></i><b>42.3</b> Sequence depth</a></li>
<li class="chapter" data-level="42.4" data-path="41-G41.html"><a href="41-G41.html#cutoff-34"><i class="fa fa-check"></i><b>42.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="42.5" data-path="41-G41.html"><a href="41-G41.html#observations-39"><i class="fa fa-check"></i><b>42.5</b> Observations</a></li>
<li class="chapter" data-level="42.6" data-path="41-G41.html"><a href="41-G41.html#conclusions-40"><i class="fa fa-check"></i><b>42.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="43" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i><b>43</b> IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="43.1" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i><b>43.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="43.2" data-path="42-G42.html"><a href="42-G42.html#observations-40"><i class="fa fa-check"></i><b>43.2</b> Observations</a></li>
<li class="chapter" data-level="43.3" data-path="42-G42.html"><a href="42-G42.html#conclusions-41"><i class="fa fa-check"></i><b>43.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="44" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i><b>44</b> IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="44.1" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i><b>44.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="44.2" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-40"><i class="fa fa-check"></i><b>44.2</b> Sequence depth</a></li>
<li class="chapter" data-level="44.3" data-path="43-G43.html"><a href="43-G43.html#observations-41"><i class="fa fa-check"></i><b>44.3</b> Observations</a></li>
<li class="chapter" data-level="44.4" data-path="43-G43.html"><a href="43-G43.html#conclusions-42"><i class="fa fa-check"></i><b>44.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="45" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i><b>45</b> IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="45.1" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i><b>45.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="45.2" data-path="44-G44.html"><a href="44-G44.html#group-alignment-35"><i class="fa fa-check"></i><b>45.2</b> Group alignment</a></li>
<li class="chapter" data-level="45.3" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-41"><i class="fa fa-check"></i><b>45.3</b> Sequence depth</a></li>
<li class="chapter" data-level="45.4" data-path="44-G44.html"><a href="44-G44.html#cutoff-35"><i class="fa fa-check"></i><b>45.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="45.5" data-path="44-G44.html"><a href="44-G44.html#observations-42"><i class="fa fa-check"></i><b>45.5</b> Observations</a></li>
<li class="chapter" data-level="45.6" data-path="44-G44.html"><a href="44-G44.html#conclusions-43"><i class="fa fa-check"></i><b>45.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="46" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i><b>46</b> IGHV5-10-1 - G45</a>
<ul>
<li class="chapter" data-level="46.1" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i><b>46.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="46.2" data-path="45-G45.html"><a href="45-G45.html#group-alignment-36"><i class="fa fa-check"></i><b>46.2</b> Group alignment</a></li>
<li class="chapter" data-level="46.3" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-42"><i class="fa fa-check"></i><b>46.3</b> Sequence depth</a></li>
<li class="chapter" data-level="46.4" data-path="45-G45.html"><a href="45-G45.html#cutoff-36"><i class="fa fa-check"></i><b>46.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="46.5" data-path="45-G45.html"><a href="45-G45.html#observations-43"><i class="fa fa-check"></i><b>46.5</b> Observations</a></li>
<li class="chapter" data-level="46.6" data-path="45-G45.html"><a href="45-G45.html#conclusions-44"><i class="fa fa-check"></i><b>46.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="47" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i><b>47</b> IGHV5-51 - G46</a>
<ul>
<li class="chapter" data-level="47.1" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i><b>47.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="47.2" data-path="46-G46.html"><a href="46-G46.html#group-alignment-37"><i class="fa fa-check"></i><b>47.2</b> Group alignment</a></li>
<li class="chapter" data-level="47.3" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-43"><i class="fa fa-check"></i><b>47.3</b> Sequence depth</a></li>
<li class="chapter" data-level="47.4" data-path="46-G46.html"><a href="46-G46.html#cutoff-37"><i class="fa fa-check"></i><b>47.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="47.5" data-path="46-G46.html"><a href="46-G46.html#observations-44"><i class="fa fa-check"></i><b>47.5</b> Observations</a></li>
<li class="chapter" data-level="47.6" data-path="46-G46.html"><a href="46-G46.html#conclusions-45"><i class="fa fa-check"></i><b>47.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="48" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i><b>48</b> IGHV6-1 - G47</a>
<ul>
<li class="chapter" data-level="48.1" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i><b>48.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="48.2" data-path="47-G47.html"><a href="47-G47.html#group-alignment-38"><i class="fa fa-check"></i><b>48.2</b> Group alignment</a></li>
<li class="chapter" data-level="48.3" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-44"><i class="fa fa-check"></i><b>48.3</b> Sequence depth</a></li>
<li class="chapter" data-level="48.4" data-path="47-G47.html"><a href="47-G47.html#cutoff-38"><i class="fa fa-check"></i><b>48.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="48.5" data-path="47-G47.html"><a href="47-G47.html#observations-45"><i class="fa fa-check"></i><b>48.5</b> Observations</a></li>
<li class="chapter" data-level="48.6" data-path="47-G47.html"><a href="47-G47.html#conclusions-46"><i class="fa fa-check"></i><b>48.6</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="49" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i><b>49</b> IGHV7-4-1 - G48</a>
<ul>
<li class="chapter" data-level="49.1" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i><b>49.1</b> Allele appearnce</a></li>
<li class="chapter" data-level="49.2" data-path="48-G48.html"><a href="48-G48.html#group-alignment-39"><i class="fa fa-check"></i><b>49.2</b> Group alignment</a></li>
<li class="chapter" data-level="49.3" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-45"><i class="fa fa-check"></i><b>49.3</b> Sequence depth</a></li>
<li class="chapter" data-level="49.4" data-path="48-G48.html"><a href="48-G48.html#cutoff-39"><i class="fa fa-check"></i><b>49.4</b> 0.5% cutoff</a></li>
<li class="chapter" data-level="49.5" data-path="48-G48.html"><a href="48-G48.html#observations-46"><i class="fa fa-check"></i><b>49.5</b> Observations</a></li>
<li class="chapter" data-level="49.6" data-path="48-G48.html"><a href="48-G48.html#conclusions-47"><i class="fa fa-check"></i><b>49.6</b> Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./"> Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv5-51---g46" class="section level1" number="47">
<h1><span class="header-section-number">47</span> IGHV5-51 - G46</h1>
<div id="allele-appearnce-45" class="section level2" number="47.1">
<h2><span class="header-section-number">47.1</span> Allele appearnce</h2>
<p>The group of IGHV5-51 includes 7 alleles, 7 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-390-1.png" width="672" /></p>
</div>
<div id="group-alignment-37" class="section level2" number="47.2">
<h2><span class="header-section-number">47.2</span> Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-391-1.png" width="1728" /></p>
</div>
<div id="sequence-depth-43" class="section level2" number="47.3">
<h2><span class="header-section-number">47.3</span> Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-0f2a4f36ac2e741b5c8f" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-0f2a4f36ac2e741b5c8f">{"x":{"data":[{"x":[1,1,1,3,4,1,1,1,3,1,3,1,1,1,3,1,1,3,1,1,3,1,1,1,3,1,3,5,1,3,4,1,6,1,3,4,1,1,3,4,1,1,6,1,1,3,1,3,1,3,1,1,1,1,3,1,1,1,6,3,1,4,1,3,1,3,2,1,1,1,1,3,1,1,6,1,3,1,3,1,1,1,1,3,1,3,1,1,3,1,1,3,1,3,1,1,1,3,5,1,3,4,1,1,3,1,1,1,1,3,4,1,3,1,1,1,3,1,1,1,3,1,1,6,1,6,1,1,1,3,1,3,1,3,6,1,3,4,1,3,3,1,3,1,1,3,4,6,1,1,1,3,6,1,1,3,6,1,1,3,3,1,1,3,1,4,5,1,1],"y":[473,982,155,82,1,987,645,679,1,186,104,41,471,97,37,257,451,223,541,337,1,83,801,321,119,411,252,1,502,275,1,551,1,441,188,1,153,585,294,2,434,739,1,759,8,2,23,21,514,180,269,371,446,208,32,688,1022,1177,1,290,101,1,378,176,348,147,1,212,418,135,603,174,56,1336,1,107,49,360,253,236,585,539,425,237,311,116,618,156,88,830,622,235,1282,28,14,698,588,332,1,447,215,1,449,661,1,300,795,295,658,226,1,734,1,714,1076,448,203,815,541,727,338,132,1053,1,808,1,1382,1183,448,179,225,109,241,130,1,390,208,1,156,81,184,49,538,170,529,353,1,1,762,313,530,341,1,638,272,106,1,815,318,166,245,105,49,663,291,1,1,771,95],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(250,171,24,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[0.941593377944082,0.908804418519139,0.94531091763638,2.93248348990455,4.0373780396767,0.922365180915222,1.03068155902438,0.923484652256593,3.02182737211697,1.01746273678727,3.02101804814301,1.08222375013866,1.06510128327645,1.06877450752072,2.9201833922416,1.04643637887202,0.990618328284472,3.09553452236578,1.07378275357187,0.993848130432889,2.97900491431355,0.999621364567429,0.970239769481122,0.952606612071395,3.05287897572853,0.971907540038228,3.04033619058318,5.05110899084248,0.905015754234046,2.90630932180211,3.9711532600224,1.02993300175294,5.99962091469206,1.00375596377999,3.01428817370906,4.06804517912678,1.08011658075266,1.05438875062391,3.09474291494116,3.93734631426632,1.00480256555602,0.956915771961212,6.04515667157248,1.02807755102403,1.03291090778075,3.01311016427353,1.08382945028134,2.95897882692516,1.06167013691738,2.94313513878733,1.0214401113335,0.938883111765608,0.964418258843943,0.907737658172846,3.03553605997004,0.969574047252536,1.08606129260734,0.97209726460278,5.93963733743876,3.04138798783533,0.918406931404024,4.02562820804305,0.921481394441798,3.0394381842576,0.994175529805943,2.93903888585046,1.90952897346579,1.06046209279448,0.904417280247435,0.963432979630306,0.928086392162368,2.95348399281502,0.951905973441899,0.995128023531288,5.96219181674533,1.05082239992917,3.03727431884035,1.04666155362502,2.91954962727614,0.932967661134899,1.05459660179913,1.00030246875249,0.986592467362061,2.93182423468679,1.0891881307587,2.9087785938289,0.95942833176814,1.04121360988356,3.02843661345541,1.05639520050026,1.05998917166144,3.05186697668396,0.979494651593268,2.90516700763255,1.06407466651872,0.942471174662933,0.956414381135255,2.92000991534442,4.99823560183868,0.916638828953728,3.03568445402198,4.06456478070468,0.932586803939193,0.931968836206943,2.95329149989411,0.943239944986999,1.09641097774729,1.04651535348967,1.00330939302221,3.0464089340996,3.94056690069847,1.00765200760216,3.01018743938766,1.04421732719056,0.909130398835987,0.942633919231594,2.92064745035022,0.998777311900631,1.07731183567084,0.924513665260747,2.94734787615016,1.09036052813753,0.940887942491099,5.91251800726168,1.02792860832997,5.9854069118388,0.936542783584446,0.934873676812276,1.0629818837624,3.08379454086535,1.0933105412405,2.93975443504751,1.01508797318675,2.93773028799333,5.9283375505358,0.908772920491174,2.91614247830585,3.90154308015481,1.0350703143049,2.98956170054153,3.01751090534963,1.00069699180312,2.9407390399836,0.910876174317673,0.989480987843126,2.92333986246958,3.95489149596542,6.02433943538927,1.05590048278682,1.05225755809806,0.915974021656439,3.06626390558667,5.97393351453356,0.960018477635458,1.00656171729788,2.95418639779091,5.91720107970759,0.925168762542307,1.07425187695771,3.08181976508349,3.09123350833543,1.09981226795353,0.972924388572574,2.91474954406731,0.954733124747872,4.00282870316878,4.9249207012821,1.04842632217333,1.06100674499758],"y":[472.772550744191,982.196475100145,155.251624589041,82.1001433510333,1.06046107597649,987.170893431082,644.711757156253,679.295380186476,0.838585679791868,185.780206280574,104.08420217447,40.6607975207269,470.798412305117,97.2565606096759,37.2962599694729,257.056348916516,450.608609724231,222.865370397083,541.308884065598,337.321172678843,0.780257721617818,82.8515209145844,801.297367131524,320.896687447093,118.752846504375,410.714855806716,252.229805556871,1.26603946834803,502.324810171314,274.696207942255,1.19371180459857,551.120121958666,1.15993913970888,441.234727399424,187.920965380222,1.1209693018347,153.05360618867,585.074606372416,293.837044297904,2.17343807034194,434.26127331201,738.854752205312,1.17698649577796,759.132358416915,8.12292366009206,1.87614369895309,22.8937270347029,20.9681615982205,513.882959639654,179.785958898067,269.024833412468,370.932293157093,445.796848201007,207.746943757124,31.9461051814258,688.096040449105,1021.95815430824,1176.79095536806,0.843624073453248,289.725586218387,100.957200361416,0.96523257419467,377.932462781854,176.053710653633,348.088997702487,146.89561123848,1.24158914498985,211.728082013316,417.875844492577,134.631706245989,602.707778459042,173.998059212044,56.372589433752,1336.00079775862,0.8077661447227,106.96579641141,49.1214938018471,360.215087460354,252.969242510572,235.910755249113,585.095413471386,539.149397657625,424.729985559545,237.298541650921,311.2007464692,116.389053475484,617.663749420457,156.089199635759,87.7435436151922,830.058500089496,621.756270381995,235.1366837807,1281.63736502323,27.9315953440964,13.6874291159213,698.317278022878,587.603283502162,332.368316931277,1.03157498966902,446.820047957078,214.793779836781,1.19913688655943,449.391664203256,660.773553735949,1.14554267767817,299.734565986507,794.793653762341,295.083897143975,657.734764974006,226.201116359606,0.676724383980036,733.740564698912,1.04842836614698,714.321605384909,1075.94017149974,447.945318023302,203.284746575914,815.023492855206,541.308229426481,726.915933172591,338.327941058762,132.39320640061,1053.15309572443,0.914219026453793,807.87503814958,1.26045201830566,1381.68696494438,1182.67413148712,448.094014842436,178.991038861312,224.669694809057,109.177104605362,241.139883877337,129.734363398328,0.809685965254903,389.877012760937,207.792173693329,1.06470890622586,155.883616692945,81.1596265641972,183.986778658815,49.2070173470303,537.978351498023,169.947261747159,528.735705989227,353.334698018804,0.708881370350718,1.15408465564251,762.20410448648,312.965536461584,529.943385079689,341.286650346406,0.860610396228731,637.613432999142,271.795003075339,106.064131652564,0.827710854448378,814.781902695447,318.234405563772,165.983036207408,245.014191865176,105.00409701746,48.9402918418869,662.658000551723,291.261041588709,1.3635747730732,1.27176863923669,770.962121562473,95.0638946115971],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  473 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0346","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  982 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0417","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  01 <\/br># assignments:  155 <\/br>Relative freq.:  0.6513 <\/br>Relative Rep. freq.:  0.0144","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  03 <\/br># assignments:  82 <\/br>Relative freq.:  0.3445 <\/br>Relative Rep. freq.:  0.0076","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0042 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  987 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0414","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  645 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.027","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  679 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  186 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0308","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  03 <\/br># assignments:  104 <\/br>Relative freq.:  0.7172 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  41 <\/br>Relative freq.:  0.2828 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  471 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0331","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  97 <\/br>Relative freq.:  0.7239 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  03 <\/br># assignments:  37 <\/br>Relative freq.:  0.2761 <\/br>Relative Rep. freq.:  0.0066","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  257 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0255","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  451 <\/br>Relative freq.:  0.6691 <\/br>Relative Rep. freq.:  0.0203","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  03 <\/br># assignments:  223 <\/br>Relative freq.:  0.3309 <\/br>Relative Rep. freq.:  0.01","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  541 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0328","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  337 <\/br>Relative freq.:  0.997 <\/br>Relative Rep. freq.:  0.0246","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.003 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  83 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0347","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  801 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0297","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  321 <\/br>Relative freq.:  0.7295 <\/br>Relative Rep. freq.:  0.0183","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  03 <\/br># assignments:  119 <\/br>Relative freq.:  0.2705 <\/br>Relative Rep. freq.:  0.0068","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  411 <\/br>Relative freq.:  0.619 <\/br>Relative Rep. freq.:  0.0183","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  03 <\/br># assignments:  252 <\/br>Relative freq.:  0.3795 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  502 <\/br>Relative freq.:  0.6452 <\/br>Relative Rep. freq.:  0.0215","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  03 <\/br># assignments:  275 <\/br>Relative freq.:  0.3535 <\/br>Relative Rep. freq.:  0.0118","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  551 <\/br>Relative freq.:  0.9982 <\/br>Relative Rep. freq.:  0.0525","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0018 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  441 <\/br>Relative freq.:  0.7 <\/br>Relative Rep. freq.:  0.0171","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  03 <\/br># assignments:  188 <\/br>Relative freq.:  0.2984 <\/br>Relative Rep. freq.:  0.0073","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  153 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0387","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  585 <\/br>Relative freq.:  0.664 <\/br>Relative Rep. freq.:  0.026","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  03 <\/br># assignments:  294 <\/br>Relative freq.:  0.3337 <\/br>Relative Rep. freq.:  0.0131","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  434 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0347","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  739 <\/br>Relative freq.:  0.9986 <\/br>Relative Rep. freq.:  0.0361","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  759 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0304","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  8 <\/br>Relative freq.:  0.8 <\/br>Relative Rep. freq.:  0.0228","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.2 <\/br>Relative Rep. freq.:  0.0057","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  23 <\/br>Relative freq.:  0.5227 <\/br>Relative Rep. freq.:  0.0128","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  03 <\/br># assignments:  21 <\/br>Relative freq.:  0.4773 <\/br>Relative Rep. freq.:  0.0117","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  514 <\/br>Relative freq.:  0.7406 <\/br>Relative Rep. freq.:  0.0246","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  03 <\/br># assignments:  180 <\/br>Relative freq.:  0.2594 <\/br>Relative Rep. freq.:  0.0086","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  269 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0487","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  371 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0489","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  446 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0392","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  208 <\/br>Relative freq.:  0.8667 <\/br>Relative Rep. freq.:  0.0254","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  03 <\/br># assignments:  32 <\/br>Relative freq.:  0.1333 <\/br>Relative Rep. freq.:  0.0039","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  688 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0374","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  1022 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.036","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  1177 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.0483","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  03 <\/br># assignments:  290 <\/br>Relative freq.:  0.7398 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  101 <\/br>Relative freq.:  0.2577 <\/br>Relative Rep. freq.:  0.0058","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0026 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  378 <\/br>Relative freq.:  0.6823 <\/br>Relative Rep. freq.:  0.0192","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  03 <\/br># assignments:  176 <\/br>Relative freq.:  0.3177 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  01 <\/br># assignments:  348 <\/br>Relative freq.:  0.7016 <\/br>Relative Rep. freq.:  0.0299","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  03 <\/br># assignments:  147 <\/br>Relative freq.:  0.2964 <\/br>Relative Rep. freq.:  0.0126","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  02 <\/br># assignments:  1 <\/br>Relative freq.:  0.002 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  01 <\/br># assignments:  212 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0345","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  01 <\/br># assignments:  418 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0281","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  01 <\/br># assignments:  135 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0407","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  01 <\/br># assignments:  603 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0274","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  03 <\/br># assignments:  174 <\/br>Relative freq.:  0.7565 <\/br>Relative Rep. freq.:  0.0171","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  01 <\/br># assignments:  56 <\/br>Relative freq.:  0.2435 <\/br>Relative Rep. freq.:  0.0055","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  1336 <\/br>Relative freq.:  0.9993 <\/br>Relative Rep. freq.:  0.0362","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  7e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  01 <\/br># assignments:  107 <\/br>Relative freq.:  0.6859 <\/br>Relative Rep. freq.:  0.0169","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  03 <\/br># assignments:  49 <\/br>Relative freq.:  0.3141 <\/br>Relative Rep. freq.:  0.0077","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  01 <\/br># assignments:  360 <\/br>Relative freq.:  0.5873 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  03 <\/br># assignments:  253 <\/br>Relative freq.:  0.4127 <\/br>Relative Rep. freq.:  0.0093","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  01 <\/br># assignments:  236 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0257","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  01 <\/br># assignments:  585 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0261","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  01 <\/br># assignments:  539 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0251","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  01 <\/br># assignments:  425 <\/br>Relative freq.:  0.642 <\/br>Relative Rep. freq.:  0.023","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  03 <\/br># assignments:  237 <\/br>Relative freq.:  0.358 <\/br>Relative Rep. freq.:  0.0128","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  01 <\/br># assignments:  311 <\/br>Relative freq.:  0.7283 <\/br>Relative Rep. freq.:  0.0158","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  03 <\/br># assignments:  116 <\/br>Relative freq.:  0.2717 <\/br>Relative Rep. freq.:  0.0059","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  01 <\/br># assignments:  618 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0255","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  01 <\/br># assignments:  156 <\/br>Relative freq.:  0.6393 <\/br>Relative Rep. freq.:  0.0206","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  03 <\/br># assignments:  88 <\/br>Relative freq.:  0.3607 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  01 <\/br># assignments:  830 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0304","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  622 <\/br>Relative freq.:  0.7258 <\/br>Relative Rep. freq.:  0.0195","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  03 <\/br># assignments:  235 <\/br>Relative freq.:  0.2742 <\/br>Relative Rep. freq.:  0.0074","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  01 <\/br># assignments:  1282 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0376","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  03 <\/br># assignments:  28 <\/br>Relative freq.:  0.6667 <\/br>Relative Rep. freq.:  0.0208","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  01 <\/br># assignments:  14 <\/br>Relative freq.:  0.3333 <\/br>Relative Rep. freq.:  0.0104","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  01 <\/br># assignments:  698 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0325","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  01 <\/br># assignments:  588 <\/br>Relative freq.:  0.6384 <\/br>Relative Rep. freq.:  0.0257","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  03 <\/br># assignments:  332 <\/br>Relative freq.:  0.3605 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  01 <\/br># assignments:  447 <\/br>Relative freq.:  0.6742 <\/br>Relative Rep. freq.:  0.0232","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  03 <\/br># assignments:  215 <\/br>Relative freq.:  0.3243 <\/br>Relative Rep. freq.:  0.0112","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  01 <\/br># assignments:  449 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0239","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  01 <\/br># assignments:  661 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0336","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  01 <\/br># assignments:  300 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0243","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  01 <\/br># assignments:  795 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0235","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  01 <\/br># assignments:  295 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0387","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  658 <\/br>Relative freq.:  0.7435 <\/br>Relative Rep. freq.:  0.0279","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  03 <\/br># assignments:  226 <\/br>Relative freq.:  0.2554 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  01 <\/br># assignments:  734 <\/br>Relative freq.:  0.9986 <\/br>Relative Rep. freq.:  0.0271","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  01 <\/br># assignments:  714 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0218","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  01 <\/br># assignments:  1076 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0336","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  01 <\/br># assignments:  448 <\/br>Relative freq.:  0.6882 <\/br>Relative Rep. freq.:  0.0249","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  03 <\/br># assignments:  203 <\/br>Relative freq.:  0.3118 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  01 <\/br># assignments:  815 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0326","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  01 <\/br># assignments:  541 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0309","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  01 <\/br># assignments:  727 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0305","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  03 <\/br># assignments:  338 <\/br>Relative freq.:  0.7191 <\/br>Relative Rep. freq.:  0.0182","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  01 <\/br># assignments:  132 <\/br>Relative freq.:  0.2809 <\/br>Relative Rep. freq.:  0.0071","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  01 <\/br># assignments:  1053 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.0485","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  01 <\/br># assignments:  808 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0436","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  1382 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0422","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  01 <\/br># assignments:  1183 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0395","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  01 <\/br># assignments:  448 <\/br>Relative freq.:  0.7145 <\/br>Relative Rep. freq.:  0.0195","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  03 <\/br># assignments:  179 <\/br>Relative freq.:  0.2855 <\/br>Relative Rep. freq.:  0.0078","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  01 <\/br># assignments:  225 <\/br>Relative freq.:  0.6737 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  03 <\/br># assignments:  109 <\/br>Relative freq.:  0.3263 <\/br>Relative Rep. freq.:  0.0079","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  01 <\/br># assignments:  241 <\/br>Relative freq.:  0.6478 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  03 <\/br># assignments:  130 <\/br>Relative freq.:  0.3495 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0027 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  01 <\/br># assignments:  390 <\/br>Relative freq.:  0.6511 <\/br>Relative Rep. freq.:  0.0164","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  03 <\/br># assignments:  208 <\/br>Relative freq.:  0.3472 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  01 <\/br># assignments:  156 <\/br>Relative freq.:  0.6582 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  03 <\/br># assignments:  81 <\/br>Relative freq.:  0.3418 <\/br>Relative Rep. freq.:  0.0087","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  03 <\/br># assignments:  184 <\/br>Relative freq.:  0.7897 <\/br>Relative Rep. freq.:  0.0098","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  0.2103 <\/br>Relative Rep. freq.:  0.0026","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  03 <\/br># assignments:  538 <\/br>Relative freq.:  0.7599 <\/br>Relative Rep. freq.:  0.0189","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  01 <\/br># assignments:  170 <\/br>Relative freq.:  0.2401 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  01 <\/br># assignments:  529 <\/br>Relative freq.:  0.5984 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  03 <\/br># assignments:  353 <\/br>Relative freq.:  0.3993 <\/br>Relative Rep. freq.:  0.0108","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  01 <\/br># assignments:  762 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0325","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  313 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0195","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  01 <\/br># assignments:  530 <\/br>Relative freq.:  0.6078 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  03 <\/br># assignments:  341 <\/br>Relative freq.:  0.3911 <\/br>Relative Rep. freq.:  0.0109","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  01 <\/br># assignments:  638 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.027","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  01 <\/br># assignments:  272 <\/br>Relative freq.:  0.7177 <\/br>Relative Rep. freq.:  0.0277","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  03 <\/br># assignments:  106 <\/br>Relative freq.:  0.2797 <\/br>Relative Rep. freq.:  0.0108","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0026 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  01 <\/br># assignments:  815 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0348","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  01 <\/br># assignments:  318 <\/br>Relative freq.:  0.657 <\/br>Relative Rep. freq.:  0.0253","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  03 <\/br># assignments:  166 <\/br>Relative freq.:  0.343 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  03 <\/br># assignments:  245 <\/br>Relative freq.:  0.7 <\/br>Relative Rep. freq.:  0.0193","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  01 <\/br># assignments:  105 <\/br>Relative freq.:  0.3 <\/br>Relative Rep. freq.:  0.0083","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  01 <\/br># assignments:  49 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0375","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  03 <\/br># assignments:  663 <\/br>Relative freq.:  0.6935 <\/br>Relative Rep. freq.:  0.0281","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  01 <\/br># assignments:  291 <\/br>Relative freq.:  0.3044 <\/br>Relative Rep. freq.:  0.0123","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  01 <\/br># assignments:  771 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0447","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  01 <\/br># assignments:  95 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0487"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(250,171,24,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(250,171,24,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["V5-51*01","V5-51*02","V5-51*03","V5-51*04","V5-51*06","V5-51*07"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["V5-51*01","V5-51*02","V5-51*03","V5-51*04","V5-51*06","V5-51*07"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-68.389439396821,1451.0661637808],"tickmode":"array","ticktext":["0","500","1000"],"tickvals":[0,500,1000],"categoryorder":"array","categoryarray":["0","500","1000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"3c32a61a819c6":{"x":{},"y":{},"text":{},"type":"box"},"3c32a50e498e6":{"x":{},"y":{},"text":{}}},"cur_data":"3c32a61a819c6","visdat":{"3c32a61a819c6":["function (y) ","x"],"3c32a50e498e6":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-cb414701829b836b2e49" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-cb414701829b836b2e49">{"x":{"data":[{"x":[1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,1,1,1,1,1,2,1,2,1,2,1,2,1,1,1,2,1,2,3,1,2,1,2,1,2,1,1,2,1,1,1,2,1,1,1,1,2],"y":[239,171,305,294,48,508,398,407,611,313,387,489,292,391,190,269,526,362,146,544,126,61,502,278,479,283,184,420,211,365,222,333,114,379,419,215,130,170,105,2,495,261,205,89,432,708,250,219,103,114,228,337,178,505,430,369,303,138],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(111,145,106,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[0.959511061757803,1.98299500634894,0.926536192558706,0.984641634393483,1.08001476898789,0.926601981138811,1.09872641833499,0.955330441845581,0.9642129980959,0.952112622139975,0.92921050931327,0.911196993244812,1.07713184654713,1.06364889247343,1.07792627913877,0.976170018315315,0.946993119688705,0.912200499558821,0.970778673514724,1.08718788865954,1.90194807960652,1.0259484176524,0.949339568987489,0.973858991404995,1.06683938987553,0.910154757974669,1.90491364104673,1.07812395617366,1.9584110327065,0.909586935583502,1.99219388999045,1.0783644865267,2.00426357802935,1.0823413961567,0.941506606154144,0.914107368979603,2.08605479695834,0.97067580325529,2.06068244199269,3.00267184758559,1.02939904867671,2.08316892199218,0.913413584884256,2.01668479889631,1.02789668096229,2.05758738992736,1.0415378249716,0.942295279400423,2.08736691321246,1.07719724434428,1.02684426507913,1.02667354913428,1.95661637387238,1.05663299513981,0.979719865182415,1.09660629611462,1.02804396478459,2.03597601735964],"y":[239.039909221604,170.932961102761,304.75993004255,294.175106058829,47.9242426943034,508.316669455543,398.258196724206,407.012362862192,611.349853954464,312.910965296999,387.169466475211,489.257721321844,292.139979107305,391.114273201115,189.863533956371,269.38627821859,525.816953761503,361.746288817562,145.81601994969,544.158317005262,126.037785466574,60.6272344015539,502.298535232991,277.90811260324,479.016383928061,283.038296878897,184.305425134487,420.284185324982,210.619120340794,364.715921923704,221.976094257459,332.689042595774,114.238682663441,379.043480901606,419.392771394923,214.782463477924,129.664680859447,170.312087768503,105.280435799621,1.71313616018742,494.972130715288,260.860295496881,205.035390143283,88.9826534226537,431.805634048395,708.314937917702,250.08830074314,218.914830133691,102.698754635826,114.125529215671,227.782464389317,336.826419723034,177.819760200195,504.602523003146,430.071615465544,368.991023589112,303.122868131287,138.289366425201],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  239 <\/br>Relative freq.:  0.5829 <\/br>Relative Rep. freq.:  0.0408","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  03 <\/br># assignments:  171 <\/br>Relative freq.:  0.4171 <\/br>Relative Rep. freq.:  0.0292","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  305 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0771","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  294 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0645","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  48 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0545","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  508 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0686","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  398 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0663","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  407 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0784","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  611 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0736","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  313 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.055","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  387 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0629","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  489 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0572","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  292 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0571","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  01 <\/br># assignments:  391 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0646","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  190 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0478","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  269 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0619","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  526 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0668","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  362 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0712","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  146 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0645","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  544 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0769","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  03 <\/br># assignments:  126 <\/br>Relative freq.:  0.6738 <\/br>Relative Rep. freq.:  0.0351","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  61 <\/br>Relative freq.:  0.3262 <\/br>Relative Rep. freq.:  0.017","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  502 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.072","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  278 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.061","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  01 <\/br># assignments:  479 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0621","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  283 <\/br>Relative freq.:  0.606 <\/br>Relative Rep. freq.:  0.0488","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  03 <\/br># assignments:  184 <\/br>Relative freq.:  0.394 <\/br>Relative Rep. freq.:  0.0318","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  420 <\/br>Relative freq.:  0.6656 <\/br>Relative Rep. freq.:  0.0471","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  03 <\/br># assignments:  211 <\/br>Relative freq.:  0.3344 <\/br>Relative Rep. freq.:  0.0236","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  01 <\/br># assignments:  365 <\/br>Relative freq.:  0.6218 <\/br>Relative Rep. freq.:  0.0344","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  03 <\/br># assignments:  222 <\/br>Relative freq.:  0.3782 <\/br>Relative Rep. freq.:  0.0209","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  01 <\/br># assignments:  333 <\/br>Relative freq.:  0.745 <\/br>Relative Rep. freq.:  0.0597","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  03 <\/br># assignments:  114 <\/br>Relative freq.:  0.255 <\/br>Relative Rep. freq.:  0.0204","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  01 <\/br># assignments:  379 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0792","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  01 <\/br># assignments:  419 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0719","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  01 <\/br># assignments:  215 <\/br>Relative freq.:  0.6232 <\/br>Relative Rep. freq.:  0.04","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  03 <\/br># assignments:  130 <\/br>Relative freq.:  0.3768 <\/br>Relative Rep. freq.:  0.0242","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  01 <\/br># assignments:  170 <\/br>Relative freq.:  0.6137 <\/br>Relative Rep. freq.:  0.0333","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  03 <\/br># assignments:  105 <\/br>Relative freq.:  0.3791 <\/br>Relative Rep. freq.:  0.0206","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0072 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  01 <\/br># assignments:  495 <\/br>Relative freq.:  0.6548 <\/br>Relative Rep. freq.:  0.0534","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  03 <\/br># assignments:  261 <\/br>Relative freq.:  0.3452 <\/br>Relative Rep. freq.:  0.0282","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  01 <\/br># assignments:  205 <\/br>Relative freq.:  0.6973 <\/br>Relative Rep. freq.:  0.066","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  03 <\/br># assignments:  89 <\/br>Relative freq.:  0.3027 <\/br>Relative Rep. freq.:  0.0287","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  432 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0836","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  03 <\/br># assignments:  708 <\/br>Relative freq.:  0.739 <\/br>Relative Rep. freq.:  0.0459","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  01 <\/br># assignments:  250 <\/br>Relative freq.:  0.261 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  01 <\/br># assignments:  219 <\/br>Relative freq.:  0.6801 <\/br>Relative Rep. freq.:  0.0501","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  03 <\/br># assignments:  103 <\/br>Relative freq.:  0.3199 <\/br>Relative Rep. freq.:  0.0235","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  01 <\/br># assignments:  114 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1012","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  01 <\/br># assignments:  228 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0736","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  337 <\/br>Relative freq.:  0.6544 <\/br>Relative Rep. freq.:  0.0398","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  03 <\/br># assignments:  178 <\/br>Relative freq.:  0.3456 <\/br>Relative Rep. freq.:  0.021","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  01 <\/br># assignments:  505 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0896","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  430 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0706","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  369 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0688","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  303 <\/br>Relative freq.:  0.6871 <\/br>Relative Rep. freq.:  0.041","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  03 <\/br># assignments:  138 <\/br>Relative freq.:  0.3129 <\/br>Relative Rep. freq.:  0.0187"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(111,145,106,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(111,145,106,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":46.8244084682441},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,3.6],"tickmode":"array","ticktext":["V5-51*01","V5-51*03","V5-51*04"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["V5-51*01","V5-51*03","V5-51*04"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-33.6169539276883,743.645028005578],"tickmode":"array","ticktext":["0","200","400","600"],"tickvals":[0,200,400,600],"categoryorder":"array","categoryarray":["0","200","400","600"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"3c32a7430365c":{"x":{},"y":{},"text":{},"type":"box"},"3c32a6d0c19c7":{"x":{},"y":{},"text":{}}},"cur_data":"3c32a7430365c","visdat":{"3c32a7430365c":["function (y) ","x"],"3c32a6d0c19c7":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-25acb8ddeeb1c26f4fe5" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-25acb8ddeeb1c26f4fe5">{"x":{"data":[{"x":[1,2,4,3,1,5,1,2,3,4,1,1,5,2,1,3,4,1,5,2,1,2,3,5,1,1,1,1,2,1,2,3,2,1,3,4,1,1,5,1,5,1,1,2,3,1,5,1,5,1,2,3,1,2,4,3,1,1,5,1,2,4,3,1,1,2,3,4],"y":[227,185,4,1,100,1,258,185,2,1,541,265,1,335,115,4,3,442,2,1,546,154,2,1,250,735,337,406,2,299,169,1,122,35,1,1,242,141,1,469,1,623,355,212,4,918,1,967,1,291,162,3,296,163,5,3,225,149,1,384,250,3,2,317,346,152,4,2],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,255,255,1)","marker":{"opacity":0},"line":{"color":"rgba(45,102,128,1)","width":1.88976377952756},"showlegend":false,"xaxis":"x","yaxis":"y","frame":null},{"x":[1.02891293438151,2.04768113587052,4.00759443519637,3.02402731324546,1.04950861614197,5.00467087468132,1.00937962038442,1.97647837991826,3.03518620948307,3.93126706285402,0.958095306623727,1.08748650173657,5.01546876700595,2.08443504581228,1.01690902346745,3.04562619845383,4.00838416018523,1.09767504455522,4.96916624852456,1.94569122996181,0.972361694043502,2.08442634195089,2.9419083816465,4.90677158455364,1.07930379114114,1.05219370927662,0.959807071322575,1.03703532619402,2.02080968166702,1.0205280798953,1.97883845367469,3.04822515794076,2.01652838937007,0.919168600346893,2.92378526288085,4.09318191353232,0.918193099135533,1.05452483752742,4.91598531422205,1.01033146539703,5.06457482031547,0.919214535830542,1.08819234566763,2.08578255721368,2.94965498787351,0.911580589646473,4.92819428774528,0.930425464874133,5.09688703650609,1.08939913315699,2.07901914729737,3.06560611906461,0.985484234848991,1.95631782077253,4.02107864539139,3.08885360057466,1.04322147504427,0.912990714004263,4.93694567824714,1.07204233366065,2.04483988736756,3.92990268454887,2.90282225538976,1.0904628214892,1.02518962696195,2.0828864257317,3.04270808594301,3.94465965763666],"y":[226.814531405084,185.261884831078,3.92093200627714,0.860969530791044,99.836420715414,1.20091715361923,258.219412912428,185.382509794831,1.65105813927948,1.19003818314523,541.218444464728,265.253291699477,0.996508058160543,335.044722134434,115.197933506966,4.18697423133999,3.34609671719372,442.017609237321,1.79824374113232,1.23023636620492,545.65940357279,153.694578447193,1.61428073048592,0.646301679499447,249.79971532505,735.066033910401,336.962331351824,406.279462537356,2.39079409372061,299.395324358717,168.71405963134,0.826736847124994,122.358301684074,34.9970797546208,0.980939885415137,1.34571857303381,241.757130458578,141.1916078059,1.34529685527086,469.298911975697,1.26340580340475,623.041078815982,355.104491308704,212.166217049211,3.63318901155144,917.915334328264,1.09002510756254,967.103269898146,1.15063972752541,290.668886527419,161.948250774294,3.2262559114024,295.793745090812,162.942927178927,5.26785826403648,2.81690203268081,225.167458085716,149.220044217072,1.13244660887867,383.69615446385,250.186316315457,3.12828328423202,2.32545982189477,317.034207872301,345.768123831786,152.244174653105,4.33443664126098,2.16411482542753],"text":["<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  01 <\/br># assignments:  227 <\/br>Relative freq.:  0.5444 <\/br>Relative Rep. freq.:  0.0199","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  03 <\/br># assignments:  185 <\/br>Relative freq.:  0.4436 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  06 <\/br># assignments:  4 <\/br>Relative freq.:  0.0096 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I1_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0024 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  01 <\/br># assignments:  100 <\/br>Relative freq.:  0.9901 <\/br>Relative Rep. freq.:  0.0266","<\/br>Project:  P4 <\/br>Subject:  P4_I10_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0099 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  01 <\/br># assignments:  258 <\/br>Relative freq.:  0.5785 <\/br>Relative Rep. freq.:  0.023","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  03 <\/br># assignments:  185 <\/br>Relative freq.:  0.4148 <\/br>Relative Rep. freq.:  0.0165","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0045 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I11_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I12_S1 <\/br>Alleles:  01 <\/br># assignments:  541 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0357","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  01 <\/br># assignments:  265 <\/br>Relative freq.:  0.9962 <\/br>Relative Rep. freq.:  0.0506","<\/br>Project:  P4 <\/br>Subject:  P4_I13_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0038 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  03 <\/br># assignments:  335 <\/br>Relative freq.:  0.733 <\/br>Relative Rep. freq.:  0.0196","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  01 <\/br># assignments:  115 <\/br>Relative freq.:  0.2516 <\/br>Relative Rep. freq.:  0.0067","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.0088 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I14_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.0066 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  01 <\/br># assignments:  442 <\/br>Relative freq.:  0.9933 <\/br>Relative Rep. freq.:  0.0372","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  07 <\/br># assignments:  2 <\/br>Relative freq.:  0.0045 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I15_S1 <\/br>Alleles:  03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0022 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  01 <\/br># assignments:  546 <\/br>Relative freq.:  0.7767 <\/br>Relative Rep. freq.:  0.0283","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  03 <\/br># assignments:  154 <\/br>Relative freq.:  0.2191 <\/br>Relative Rep. freq.:  0.008","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I16_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I17_S1 <\/br>Alleles:  01 <\/br># assignments:  250 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.035","<\/br>Project:  P4 <\/br>Subject:  P4_I18_S1 <\/br>Alleles:  01 <\/br># assignments:  735 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0442","<\/br>Project:  P4 <\/br>Subject:  P4_I19_S1 <\/br>Alleles:  01 <\/br># assignments:  337 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.023","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  01 <\/br># assignments:  406 <\/br>Relative freq.:  0.9951 <\/br>Relative Rep. freq.:  0.036","<\/br>Project:  P4 <\/br>Subject:  P4_I2_S1 <\/br>Alleles:  03 <\/br># assignments:  2 <\/br>Relative freq.:  0.0049 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  01 <\/br># assignments:  299 <\/br>Relative freq.:  0.6375 <\/br>Relative Rep. freq.:  0.0168","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  03 <\/br># assignments:  169 <\/br>Relative freq.:  0.3603 <\/br>Relative Rep. freq.:  0.0095","<\/br>Project:  P4 <\/br>Subject:  P4_I21_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  03 <\/br># assignments:  122 <\/br>Relative freq.:  0.7673 <\/br>Relative Rep. freq.:  0.0313","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  01 <\/br># assignments:  35 <\/br>Relative freq.:  0.2201 <\/br>Relative Rep. freq.:  0.009","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0063 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I22_S1 <\/br>Alleles:  06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0063 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I23_S1 <\/br>Alleles:  01 <\/br># assignments:  242 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0361","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  01 <\/br># assignments:  141 <\/br>Relative freq.:  0.993 <\/br>Relative Rep. freq.:  0.0153","<\/br>Project:  P4 <\/br>Subject:  P4_I24_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.007 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  01 <\/br># assignments:  469 <\/br>Relative freq.:  0.9979 <\/br>Relative Rep. freq.:  0.0353","<\/br>Project:  P4 <\/br>Subject:  P4_I25_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0021 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I26_S1 <\/br>Alleles:  01 <\/br># assignments:  623 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0412","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  01 <\/br># assignments:  355 <\/br>Relative freq.:  0.6217 <\/br>Relative Rep. freq.:  0.0189","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  03 <\/br># assignments:  212 <\/br>Relative freq.:  0.3713 <\/br>Relative Rep. freq.:  0.0113","<\/br>Project:  P4 <\/br>Subject:  P4_I27_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.007 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  01 <\/br># assignments:  918 <\/br>Relative freq.:  0.9989 <\/br>Relative Rep. freq.:  0.0367","<\/br>Project:  P4 <\/br>Subject:  P4_I28_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0011 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  01 <\/br># assignments:  967 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.0433","<\/br>Project:  P4 <\/br>Subject:  P4_I29_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  01 <\/br># assignments:  291 <\/br>Relative freq.:  0.6382 <\/br>Relative Rep. freq.:  0.0187","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  03 <\/br># assignments:  162 <\/br>Relative freq.:  0.3553 <\/br>Relative Rep. freq.:  0.0104","<\/br>Project:  P4 <\/br>Subject:  P4_I30_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.0066 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  01 <\/br># assignments:  296 <\/br>Relative freq.:  0.6338 <\/br>Relative Rep. freq.:  0.0251","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  03 <\/br># assignments:  163 <\/br>Relative freq.:  0.349 <\/br>Relative Rep. freq.:  0.0138","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  06 <\/br># assignments:  5 <\/br>Relative freq.:  0.0107 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I31_S1 <\/br>Alleles:  04 <\/br># assignments:  3 <\/br>Relative freq.:  0.0064 <\/br>Relative Rep. freq.:  3e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I4_S1 <\/br>Alleles:  01 <\/br># assignments:  225 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0327","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  01 <\/br># assignments:  149 <\/br>Relative freq.:  0.9933 <\/br>Relative Rep. freq.:  0.0173","<\/br>Project:  P4 <\/br>Subject:  P4_I5_S1 <\/br>Alleles:  07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0067 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  01 <\/br># assignments:  384 <\/br>Relative freq.:  0.6009 <\/br>Relative Rep. freq.:  0.026","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  03 <\/br># assignments:  250 <\/br>Relative freq.:  0.3912 <\/br>Relative Rep. freq.:  0.0169","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  06 <\/br># assignments:  3 <\/br>Relative freq.:  0.0047 <\/br>Relative Rep. freq.:  2e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I7_S1 <\/br>Alleles:  04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0031 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I8_S1 <\/br>Alleles:  01 <\/br># assignments:  317 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0385","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  01 <\/br># assignments:  346 <\/br>Relative freq.:  0.6865 <\/br>Relative Rep. freq.:  0.0309","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  03 <\/br># assignments:  152 <\/br>Relative freq.:  0.3016 <\/br>Relative Rep. freq.:  0.0136","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  04 <\/br># assignments:  4 <\/br>Relative freq.:  0.0079 <\/br>Relative Rep. freq.:  4e-04","<\/br>Project:  P4 <\/br>Subject:  P4_I9_S1 <\/br>Alleles:  06 <\/br># assignments:  2 <\/br>Relative freq.:  0.004 <\/br>Relative Rep. freq.:  2e-04"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(45,102,128,1)","opacity":1,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(45,102,128,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":88.2689912826899,"l":58.7795765877958},"paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["V5-51*01","V5-51*03","V5-51*04","V5-51*06","V5-51*07"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["V5-51*01","V5-51*03","V5-51*04","V5-51*06","V5-51*07"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":18.5969281859693},"tickangle":-90,"showline":false,"linecolor":null,"linewidth":0,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-47.6765467314329,1015.42611830908],"tickmode":"array","ticktext":["0","250","500","750","1000"],"tickvals":[0,250,500,750,1000],"categoryorder":"array","categoryarray":["0","250","500","750","1000"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(203,203,203,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(34,34,34,1)","family":"Helvetica","size":23.9103362391034},"title":{"text":"","font":{"color":null,"family":null,"size":0}}},"hovermode":"closest","barmode":"relative","annotations":[{"text":"P4","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}]},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"3c32a4c3fc123":{"x":{},"y":{},"text":{},"type":"box"},"3c32a5f185557":{"x":{},"y":{},"text":{}}},"cur_data":"3c32a4c3fc123","visdat":{"3c32a4c3fc123":["function (y) ","x"],"3c32a5f185557":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-37" class="section level2" number="47.4">
<h2><span class="header-section-number">47.4</span> 0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 8 zygousity states.</p>
<iframe src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV5-51G46%22" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 150%;height:900px;" cellspacing="0">
</iframe>
</div>
<div id="observations-44" class="section level2" number="47.5">
<h2><span class="header-section-number">47.5</span> Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="https://www.dropbox.com/scl/fi/xbl19317icff3d17e8ddg/IGHV5-51G46.docx?dl=0&amp;rlkey=39gzbbcst2aks4hxwfj9tygos">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV5-51G46" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-45" class="section level2" number="47.6">
<h2><span class="header-section-number">47.6</span> Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="45-G45.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="47-G47.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/46-G46.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
