<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IGHV5-51 - G47 | Functional groups</title>
  <meta name="description" content="IGHV5-51 - G47 | Functional groups" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="IGHV5-51 - G47 | Functional groups" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IGHV5-51 - G47 | Functional groups" />
  
  
  



<meta name="date" content="2021-12-15" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="46-G46.html"/>
<link rel="next" href="48-G48.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.20/datatables.js"></script>
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Functional groups referebce book</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Functional groups</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#sequence-pre-processing"><i class="fa fa-check"></i>Sequence pre-processing</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-devision-by-95-similarity"><i class="fa fa-check"></i>Group devision by 95% similarity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#group-checklist"><i class="fa fa-check"></i>Group Checklist</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html"><i class="fa fa-check"></i>IGHV1-18 - G1</a>
<ul>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#allele-appearnce"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#group-alignment"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#sequence-depth"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#cutoff"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#observations"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="01-G1.html"><a href="01-G1.html#conclusions"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="01-G1.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-03">Allele specific cutoff: 5% for all except 0.5% for allele <strong>03</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html"><i class="fa fa-check"></i>IGHV1-2 - G2</a>
<ul>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#allele-appearnce-1"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#group-alignment-1"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#sequence-depth-1"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#cutoff-1"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#observations-1"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="02-G2.html"><a href="02-G2.html#conclusions-1"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="02-G2.html#allele-specific-cutoff-5-for-all-except-0.5-for-allele-05">Allele specific cutoff: 5% for all except 0.5% for allele <strong>05</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html"><i class="fa fa-check"></i>IGHV1-24 - G3</a>
<ul>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#allele-appearnce-2"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#sequence-depth-2"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#observations-2"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="03-G3.html"><a href="03-G3.html#conclusions-2"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html"><i class="fa fa-check"></i>IGHV1-3 - G4</a>
<ul>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#allele-appearnce-3"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#group-alignment-2"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#sequence-depth-3"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#cutoff-2"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#observations-3"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="04-G4.html"><a href="04-G4.html#conclusions-3"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li><a href="04-G4.html#allele-specific-cutoff-5-for-alleles-0105-and-04-except-0.5-for-allele-02">Allele specific cutoff: 5% for alleles <strong>01,05, and 04</strong> except 0.5% for allele <strong>02</strong></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html"><i class="fa fa-check"></i>IGHV1-45 - G5</a>
<ul>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#allele-appearnce-4"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#group-alignment-3"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#sequence-depth-4"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#cutoff-3"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#observations-4"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="05-G5.html"><a href="05-G5.html#conclusions-4"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html"><i class="fa fa-check"></i>IGHV1-46 - G6</a>
<ul>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#allele-appearnce-5"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#group-alignment-4"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#sequence-depth-5"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#cutoff-4"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#observations-5"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="06-G6.html"><a href="06-G6.html#conclusions-5"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html"><i class="fa fa-check"></i>IGHV1-58 - G7</a>
<ul>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#allele-appearnce-6"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#group-alignment-5"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#sequence-depth-6"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#cutoff-5"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#observations-6"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="07-G7.html"><a href="07-G7.html#conclusions-6"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html"><i class="fa fa-check"></i>IGHV1-69/IGHV1-69D - G8</a>
<ul>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#allele-appearnce-7"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#group-alignment-6"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#sequence-depth-7"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#cutoff-6"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#observations-7"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="08-G8.html"><a href="08-G8.html#conclusions-7"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html"><i class="fa fa-check"></i>IGHV1-69-2 - G9</a>
<ul>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#allele-appearnce-8"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#sequence-depth-8"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#observations-8"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="09-G9.html"><a href="09-G9.html#conclusions-8"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html"><i class="fa fa-check"></i>IGHV1-8 - G10</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-appearnce-9"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#group-alignment-7"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#sequence-depth-9"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#cutoff-7"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#observations-9"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#conclusions-9"><i class="fa fa-check"></i>Conclusions</a>
<ul>
<li class="chapter" data-level="" data-path="10-G10.html"><a href="10-G10.html#allele-specific-cutoff-5-for-all-alleles"><i class="fa fa-check"></i>Allele specific cutoff: 5% for all alleles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html"><i class="fa fa-check"></i>IGHV2-26 - G11</a>
<ul>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#allele-appearnce-10"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#group-alignment-8"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#sequence-depth-10"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#cutoff-8"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#observations-10"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="11-G11.html"><a href="11-G11.html#conclusions-10"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html"><i class="fa fa-check"></i>IGHV2-5 - G12</a>
<ul>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#allele-appearnce-11"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#group-alignment-9"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#sequence-depth-11"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#cutoff-9"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#observations-11"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="12-G12.html"><a href="12-G12.html#conclusions-11"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html"><i class="fa fa-check"></i>IGHV2-70/IGHV2-70D - G13</a>
<ul>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#allele-appearnce-12"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#group-alignment-10"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#sequence-depth-12"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#cutoff-10"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#observations-12"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="13-G13.html"><a href="13-G13.html#conclusions-12"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html"><i class="fa fa-check"></i>IGHV3-11 - G14</a>
<ul>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#allele-appearnce-13"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#group-alignment-11"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#sequence-depth-13"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#cutoff-11"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#observations-13"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="14-G14.html"><a href="14-G14.html#conclusions-13"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html"><i class="fa fa-check"></i>IGHV3-13 - G15</a>
<ul>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#allele-appearnce-14"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#group-alignment-12"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#sequence-depth-14"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#cutoff-12"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#observations-14"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="15-G15.html"><a href="15-G15.html#conclusions-14"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html"><i class="fa fa-check"></i>IGHV3-15 - G16</a>
<ul>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#allele-appearnce-15"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#group-alignment-13"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#sequence-depth-15"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#cutoff-13"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#observations-15"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="16-G16.html"><a href="16-G16.html#conclusions-15"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html"><i class="fa fa-check"></i>IGHV3-20 - G17</a>
<ul>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#allele-appearnce-16"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#group-alignment-14"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#sequence-depth-16"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#cutoff-14"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#observations-16"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="17-G17.html"><a href="17-G17.html#conclusions-16"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html"><i class="fa fa-check"></i>IGHV3-21 - G18</a>
<ul>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#allele-appearnce-17"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#group-alignment-15"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#sequence-depth-17"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#cutoff-15"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#observations-17"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="18-G18.html"><a href="18-G18.html#conclusions-17"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html"><i class="fa fa-check"></i>IGHV3-23/IGHV3-23D - G19</a>
<ul>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#allele-appearnce-18"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#group-alignment-16"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#sequence-depth-18"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#cutoff-16"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#observations-18"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="19-G19.html"><a href="19-G19.html#conclusions-18"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html"><i class="fa fa-check"></i>IGHV3-30/IGHV3-30-3/IGHV3-30-5/IGHV3-33 - G20</a>
<ul>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#allele-appearnce-19"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#group-alignment-17"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#sequence-depth-19"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#cutoff-17"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#observations-19"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="20-G20.html"><a href="20-G20.html#conclusions-19"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html"><i class="fa fa-check"></i>IGHV3-35 - G21</a>
<ul>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#allele-appearnce-20"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#sequence-depth-20"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#observations-20"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="21-G21.html"><a href="21-G21.html#conclusions-20"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html"><i class="fa fa-check"></i>IGHV3-43/IGHV3-43D - G22</a>
<ul>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#allele-appearnce-21"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#group-alignment-18"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#sequence-depth-21"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#cutoff-18"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#observations-21"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="22-G22.html"><a href="22-G22.html#conclusions-21"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html"><i class="fa fa-check"></i>IGHV3-48 - G23</a>
<ul>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#allele-appearnce-22"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#group-alignment-19"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#sequence-depth-22"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#cutoff-19"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#observations-22"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="23-G23.html"><a href="23-G23.html#conclusions-22"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html"><i class="fa fa-check"></i>IGHV3-49 - G24</a>
<ul>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#allele-appearnce-23"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#group-alignment-20"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#sequence-depth-23"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#cutoff-20"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#observations-23"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="24-G24.html"><a href="24-G24.html#conclusions-23"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html"><i class="fa fa-check"></i>IGHV3-53/IGHV3-66 - G25</a>
<ul>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#allele-appearnce-24"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#group-alignment-21"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#sequence-depth-24"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#cutoff-21"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#observations-24"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="25-G25.html"><a href="25-G25.html#conclusions-24"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html"><i class="fa fa-check"></i>IGHV3-62 - G26</a>
<ul>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#allele-appearnce-25"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="26-G26.html"><a href="26-G26.html#conclusions-25"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html"><i class="fa fa-check"></i>IGHV3-64/IGHV3-64D - G27</a>
<ul>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#allele-appearnce-26"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#group-alignment-22"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#sequence-depth-25"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#cutoff-22"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#observations-25"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="27-G27.html"><a href="27-G27.html#conclusions-26"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html"><i class="fa fa-check"></i>IGHV3-7 - G28</a>
<ul>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#allele-appearnce-27"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#group-alignment-23"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#sequence-depth-26"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#cutoff-23"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#observations-26"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="28-G28.html"><a href="28-G28.html#conclusions-27"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html"><i class="fa fa-check"></i>IGHV3-72 - G29</a>
<ul>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#allele-appearnce-28"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#sequence-depth-27"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#observations-27"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="29-G29.html"><a href="29-G29.html#conclusions-28"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html"><i class="fa fa-check"></i>IGHV3-73 - G30</a>
<ul>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#allele-appearnce-29"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#group-alignment-24"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#sequence-depth-28"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#cutoff-24"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#observations-28"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="30-G30.html"><a href="30-G30.html#conclusions-29"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html"><i class="fa fa-check"></i>IGHV3-74 - G31</a>
<ul>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#allele-appearnce-30"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#group-alignment-25"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#sequence-depth-29"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#cutoff-25"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#observations-29"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="31-G31.html"><a href="31-G31.html#conclusions-30"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html"><i class="fa fa-check"></i>IGHV3-9 - G32</a>
<ul>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#allele-appearnce-31"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#group-alignment-26"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#sequence-depth-30"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#cutoff-26"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#observations-30"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="32-G32.html"><a href="32-G32.html#conclusions-31"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html"><i class="fa fa-check"></i>IGHV4-28 - G33</a>
<ul>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#allele-appearnce-32"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#group-alignment-27"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#sequence-depth-31"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#cutoff-27"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#observations-31"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="33-G33.html"><a href="33-G33.html#conclusions-32"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html"><i class="fa fa-check"></i>IGHV4-30-2/IGHV4-30-4 - G34</a>
<ul>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#allele-appearnce-33"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#group-alignment-28"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#sequence-depth-32"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#cutoff-28"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#observations-32"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="34-G34.html"><a href="34-G34.html#conclusions-33"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html"><i class="fa fa-check"></i>IGHV4-30-4/IGHV4-31 - G35</a>
<ul>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#allele-appearnce-34"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#group-alignment-29"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#sequence-depth-33"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#cutoff-29"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#observations-33"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="35-G35.html"><a href="35-G35.html#conclusions-34"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html"><i class="fa fa-check"></i>IGHV4-34 - G36</a>
<ul>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#allele-appearnce-35"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#group-alignment-30"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#sequence-depth-34"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#cutoff-30"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#observations-34"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="36-G36.html"><a href="36-G36.html#conclusions-35"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html"><i class="fa fa-check"></i>IGHV4-34 - G37</a>
<ul>
<li class="chapter" data-level="" data-path="37-G37.html"><a href="37-G37.html#allele-appearnce-36"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html"><i class="fa fa-check"></i>IGHV4-34 - G38</a>
<ul>
<li class="chapter" data-level="" data-path="38-G38.html"><a href="38-G38.html#allele-appearnce-37"><i class="fa fa-check"></i>Allele appearnce</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html"><i class="fa fa-check"></i>IGHV4-38-2/IGHV4-39 - G39</a>
<ul>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#allele-appearnce-38"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#group-alignment-31"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#sequence-depth-35"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#cutoff-31"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#observations-35"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="39-G39.html"><a href="39-G39.html#conclusions-36"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html"><i class="fa fa-check"></i>IGHV4-4 - G40</a>
<ul>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#allele-appearnce-39"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#group-alignment-32"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#sequence-depth-36"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#cutoff-32"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#observations-36"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="40-G40.html"><a href="40-G40.html#conclusions-37"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html"><i class="fa fa-check"></i>IGHV4-4/IGHV4-59 - G41</a>
<ul>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#allele-appearnce-40"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#group-alignment-33"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#sequence-depth-37"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#cutoff-33"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#observations-37"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="41-G41.html"><a href="41-G41.html#conclusions-38"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html"><i class="fa fa-check"></i>IGHV4-59 - G42</a>
<ul>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#allele-appearnce-41"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#observations-38"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="42-G42.html"><a href="42-G42.html#conclusions-39"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html"><i class="fa fa-check"></i>IGHV4-59 - G43</a>
<ul>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#allele-appearnce-42"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#sequence-depth-38"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#observations-39"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="43-G43.html"><a href="43-G43.html#conclusions-40"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html"><i class="fa fa-check"></i>IGHV4-61 - G44</a>
<ul>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#allele-appearnce-43"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#group-alignment-34"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#sequence-depth-39"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#cutoff-34"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#observations-40"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="44-G44.html"><a href="44-G44.html#conclusions-41"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html"><i class="fa fa-check"></i>IGHV4-61 - G45</a>
<ul>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#allele-appearnce-44"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#group-alignment-35"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#sequence-depth-40"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#cutoff-35"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#observations-41"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="45-G45.html"><a href="45-G45.html#conclusions-42"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html"><i class="fa fa-check"></i>IGHV5-10-1 - G46</a>
<ul>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#allele-appearnce-45"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#group-alignment-36"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#sequence-depth-41"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#cutoff-36"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#observations-42"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="46-G46.html"><a href="46-G46.html#conclusions-43"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html"><i class="fa fa-check"></i>IGHV5-51 - G47</a>
<ul>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#allele-appearnce-46"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#group-alignment-37"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#sequence-depth-42"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#cutoff-37"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#observations-43"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="47-G47.html"><a href="47-G47.html#conclusions-44"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html"><i class="fa fa-check"></i>IGHV6-1 - G48</a>
<ul>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#allele-appearnce-47"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#group-alignment-38"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#sequence-depth-43"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#cutoff-38"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#observations-44"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="48-G48.html"><a href="48-G48.html#conclusions-45"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html"><i class="fa fa-check"></i>IGHV7-4-1 - G49</a>
<ul>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#allele-appearnce-48"><i class="fa fa-check"></i>Allele appearnce</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#group-alignment-39"><i class="fa fa-check"></i>Group alignment</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#sequence-depth-44"><i class="fa fa-check"></i>Sequence depth</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#cutoff-39"><i class="fa fa-check"></i>0.5% cutoff</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#observations-45"><i class="fa fa-check"></i>Observations</a></li>
<li class="chapter" data-level="" data-path="49-G49.html"><a href="49-G49.html#conclusions-46"><i class="fa fa-check"></i>Conclusions</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="./"> Copyright 2021 [The Functional Group Authors] (https://github.com/ayeletperes/reference_book/master/AUTHORS) Software licensed under the [Apache License, v2.0] (https://www.apache.org/licenses/LICENSE-2.0).</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Functional groups</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ighv5-51---g47" class="section level1">
<h1>IGHV5-51 - G47</h1>
<div id="allele-appearnce-46" class="section level2">
<h2>Allele appearnce</h2>
<p>The group of includes 7 alleles, 7 out of the alleles are functional.</p>
<p>For each allele we counted the number of appearances across the population, any appearance was considered valid.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-396-1.png" width="672" /></p>
</div>
<div id="group-alignment-37" class="section level2">
<h2>Group alignment</h2>
<p>Based on the viewd alleles, we calculated the distance between the germline sequences.</p>
<p><img src="reference-book_files/figure-html/unnamed-chunk-397-1.png" width="1728" /><img src="reference-book_files/figure-html/unnamed-chunk-397-2.png" width="1728" /></p>
</div>
<div id="sequence-depth-42" class="section level2">
<h2>Sequence depth</h2>
<p>To examine the potential cutoff we observed the sequence depth for each allele</p>
<div id="htmlwidget-db7c6e9e213ce6418b3e" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-db7c6e9e213ce6418b3e">{"x":{"visdat":{"964091e415cbb":["function () ","plotlyVisDat"]},"cur_data":"964091e415cbb","attrs":{"964091e415cbb":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatter","x":{},"y":{},"text":{},"marker":{"color":"#FAAB18","size":8,"line":{"width":1,"color":"gray"}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":"text","inherit":true},"964091e415cbb.1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"marker":{"color":"#FAAB18"},"type":"box","hoverinfo":"none","showlegend":false,"fillcolor":"transparent","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","xaxis":{"domain":[0,1],"automargin":true,"title":"Alleles","autotick":false,"tickmode":"array","tickvals":[1,2,3,4,5],"ticktext":["V5-51*01","V5-51*03","V5-51*04","V5-51*06","V5-51*07"]},"yaxis":{"domain":[0,1],"automargin":true,"title":"# Sequences"},"annotations":[{"text":"P1","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}],"showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"type":"scatter","x":[1.16855919156224,0.997262399364263,2.18854355672374,0.930459686368704,3.0916390741244,0.985948314424604,1.04820051873103,0.938612278271467,2.15884354989976,0.810256960988045,1.84476528028026,0.810335669200867,1.08717733845115,1.88338537300006,0.869758107978851,1.12984487172216,1.9635209241882,1.07112226635218,0.888841087650508,2.00325038814917,1.1100100335665,1.01968540847301,1.04536067917943,1.82679184097797,1.88796387081966,1.04114073626697,3.9094715914689,1.05987734710798,2.02269841926172,2.89166310429573,0.8054403164424,4.87725671753287,1.11065816609189,2.1019479653798,2.87741757296026,0.949487637542188,0.951246477477252,1.86826728833839,3.14893880961463,0.95928249116987,1.05637759463862,1.03262581080198,1.01808921005577,1.96711736703292,1.94798740036786,0.90698045771569,0.969134446140379,1.83497603256255,0.870056940522045,1.13373474245891,0.937003421876579,0.861880333628505,2.05881083477288,1.19005711227655,1.19536392884329,1.09304905170575,5.03334520393983,1.91577791403979,3.1066679594107,0.951135329157114,2.17001838237047,0.800690558832139,2.14107931191102,1.03218748522922,0.943596305791289,0.993010414578021,0.909115422237665,1.95121675375849,1.19998202063143,4.89554120060056,1.00226771375164,1.89377045882866,2.13818856924772,0.910306993685663,1.09580788016319,1.06425953889266,1.11829354390502,1.8411987141706,1.195586165227,0.867847711965442,1.99934063227847,1.08799824900925,1.91088381418958,0.899840386770666,1.01591238565743,0.870361392572522,2.04266288010403,0.974781633261591,2.12259455639869,0.938741873390973,1.08195655848831,2.13014210425317,0.906060833390802,2.08731336956844,2.87004212476313,0.985177459940314,0.93468126328662,2.04734005173668,1.18858394594863,0.840477697830647,1.00791523195803,0.909684515185654,2.0998667595908,2.81865997258574,1.10758796390146,2.10513435387984,0.850159876793623,0.893022357299924,1.07892188159749,2.17885970221832,0.831400339771062,1.11401858460158,0.952280035056174,2.10287605207413,0.972356448695064,5.11469959709793,0.826677472610027,5.00990391271189,1.15700510842726,0.924328959174454,0.954408922977745,2.0827319172211,1.16111039249226,1.84256660537794,0.928711009491235,2.04635697687045,0.909018011763692,2.02742902003229,3.04094347497448,1.03861243762076,2.13522863509133,1.8845617341809,2.16867289124057,2.19550810838118,1.06399043975398,3.03671386539936,5.12725546145812,0.972059043589979,1.13897781260312,2.02942720772699,1.15068569313735,1.01825530091301,1.06022244142368,1.9674177069217,5.09222942087799,1.09046299830079,0.829117241501808,1.8693825350143,2.16917262785137,0.934772662445903,0.962395586632192,2.05555059481412,2.97655870849267,0.903715572878718,0.932933784741908],"y":[401,951,121,110,1,979,625,666,1,185,137,465,79,53,251,371,287,455,332,1,77,779,280,151,336,320,1,389,370,1,538,1,349,253,1,150,465,386,2,331,724,741,8,2,28,15,432,253,250,355,440,186,51,650,1005,1156,1,378,1,301,241,279,201,210,408,128,598,157,1313,1,93,60,338,272,232,576,533,328,321,240,170,574,121,118,823,520,322,1198,42,630,457,424,353,284,1,443,657,1,283,755,290,546,289,1,717,1,704,1043,360,288,807,526,714,433,1042,1,754,1,1365,1148,364,242,165,130,183,174,312,279,1,128,98,228,635,484,383,1,1,709,307,447,415,605,200,153,1,772,263,210,338,1,36,866,1,731,93],"text":["<\/br>Project:  P1 <\/br>Subject:  P1_I1_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  401 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0338","<\/br>Project:  P1 <\/br>Subject:  P1_I10_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  951 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0418","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  121 <\/br>Relative freq.:  0.5216 <\/br>Relative Rep. freq.:  0.0116","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  110 <\/br>Relative freq.:  0.4741 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P1 <\/br>Subject:  P1_I100_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0043 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I11_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  979 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0417","<\/br>Project:  P1 <\/br>Subject:  P1_I12_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  625 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0268","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  666 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0277","<\/br>Project:  P1 <\/br>Subject:  P1_I13_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I14_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  185 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0315","<\/br>Project:  P1 <\/br>Subject:  P1_I15_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  137 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0236","<\/br>Project:  P1 <\/br>Subject:  P1_I16_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  465 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0334","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  79 <\/br>Relative freq.:  0.5985 <\/br>Relative Rep. freq.:  0.0143","<\/br>Project:  P1 <\/br>Subject:  P1_I17_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  53 <\/br>Relative freq.:  0.4015 <\/br>Relative Rep. freq.:  0.0096","<\/br>Project:  P1 <\/br>Subject:  P1_I18_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  251 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0258","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  371 <\/br>Relative freq.:  0.5638 <\/br>Relative Rep. freq.:  0.0171","<\/br>Project:  P1 <\/br>Subject:  P1_I19_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  287 <\/br>Relative freq.:  0.4362 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P1 <\/br>Subject:  P1_I2_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  455 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0306","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  332 <\/br>Relative freq.:  0.997 <\/br>Relative Rep. freq.:  0.0247","<\/br>Project:  P1 <\/br>Subject:  P1_I20_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.003 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I21_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  77 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0335","<\/br>Project:  P1 <\/br>Subject:  P1_I22_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  779 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0298","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  280 <\/br>Relative freq.:  0.6497 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P1 <\/br>Subject:  P1_I23_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  151 <\/br>Relative freq.:  0.3503 <\/br>Relative Rep. freq.:  0.0088","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  336 <\/br>Relative freq.:  0.5114 <\/br>Relative Rep. freq.:  0.0152","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  320 <\/br>Relative freq.:  0.4871 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P1 <\/br>Subject:  P1_I24_S1 <\/br>Alleles:  V5-51*06 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  389 <\/br>Relative freq.:  0.5118 <\/br>Relative Rep. freq.:  0.0171","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  370 <\/br>Relative freq.:  0.4868 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I25_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  538 <\/br>Relative freq.:  0.9981 <\/br>Relative Rep. freq.:  0.0531","<\/br>Project:  P1 <\/br>Subject:  P1_I26_S1 <\/br>Alleles:  V5-51*07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0019 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  349 <\/br>Relative freq.:  0.5788 <\/br>Relative Rep. freq.:  0.0141","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  253 <\/br>Relative freq.:  0.4196 <\/br>Relative Rep. freq.:  0.0102","<\/br>Project:  P1 <\/br>Subject:  P1_I27_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I28_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  150 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0391","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  465 <\/br>Relative freq.:  0.5451 <\/br>Relative Rep. freq.:  0.0213","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  386 <\/br>Relative freq.:  0.4525 <\/br>Relative Rep. freq.:  0.0177","<\/br>Project:  P1 <\/br>Subject:  P1_I29_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  2 <\/br>Relative freq.:  0.0023 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I3_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  331 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0319","<\/br>Project:  P1 <\/br>Subject:  P1_I30_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  724 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0361","<\/br>Project:  P1 <\/br>Subject:  P1_I31_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  741 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0304","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  8 <\/br>Relative freq.:  0.8 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I32_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  2 <\/br>Relative freq.:  0.2 <\/br>Relative Rep. freq.:  0.006","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  28 <\/br>Relative freq.:  0.6512 <\/br>Relative Rep. freq.:  0.016","<\/br>Project:  P1 <\/br>Subject:  P1_I33_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  15 <\/br>Relative freq.:  0.3488 <\/br>Relative Rep. freq.:  0.0086","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  432 <\/br>Relative freq.:  0.6307 <\/br>Relative Rep. freq.:  0.0211","<\/br>Project:  P1 <\/br>Subject:  P1_I34_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  253 <\/br>Relative freq.:  0.3693 <\/br>Relative Rep. freq.:  0.0124","<\/br>Project:  P1 <\/br>Subject:  P1_I35_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  250 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0493","<\/br>Project:  P1 <\/br>Subject:  P1_I37_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  355 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0498","<\/br>Project:  P1 <\/br>Subject:  P1_I38_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  440 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0393","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  186 <\/br>Relative freq.:  0.7848 <\/br>Relative Rep. freq.:  0.0232","<\/br>Project:  P1 <\/br>Subject:  P1_I39_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  51 <\/br>Relative freq.:  0.2152 <\/br>Relative Rep. freq.:  0.0064","<\/br>Project:  P1 <\/br>Subject:  P1_I4_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  650 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0369","<\/br>Project:  P1 <\/br>Subject:  P1_I40_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1005 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0359","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1156 <\/br>Relative freq.:  0.9991 <\/br>Relative Rep. freq.:  0.0483","<\/br>Project:  P1 <\/br>Subject:  P1_I41_S1 <\/br>Alleles:  V5-51*07 <\/br># assignments:  1 <\/br>Relative freq.:  9e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  378 <\/br>Relative freq.:  0.9974 <\/br>Relative Rep. freq.:  0.0224","<\/br>Project:  P1 <\/br>Subject:  P1_I42_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0026 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  301 <\/br>Relative freq.:  0.5554 <\/br>Relative Rep. freq.:  0.0156","<\/br>Project:  P1 <\/br>Subject:  P1_I43_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  241 <\/br>Relative freq.:  0.4446 <\/br>Relative Rep. freq.:  0.0125","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  279 <\/br>Relative freq.:  0.5812 <\/br>Relative Rep. freq.:  0.0247","<\/br>Project:  P1 <\/br>Subject:  P1_I44_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  201 <\/br>Relative freq.:  0.4188 <\/br>Relative Rep. freq.:  0.0178","<\/br>Project:  P1 <\/br>Subject:  P1_I45_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  210 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0351","<\/br>Project:  P1 <\/br>Subject:  P1_I46_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  408 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.028","<\/br>Project:  P1 <\/br>Subject:  P1_I47_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  128 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0404","<\/br>Project:  P1 <\/br>Subject:  P1_I48_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  598 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P1 <\/br>Subject:  P1_I49_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  157 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0216","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1313 <\/br>Relative freq.:  0.9992 <\/br>Relative Rep. freq.:  0.0363","<\/br>Project:  P1 <\/br>Subject:  P1_I5_S1 <\/br>Alleles:  V5-51*07 <\/br># assignments:  1 <\/br>Relative freq.:  8e-04 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  93 <\/br>Relative freq.:  0.6078 <\/br>Relative Rep. freq.:  0.015","<\/br>Project:  P1 <\/br>Subject:  P1_I50_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  60 <\/br>Relative freq.:  0.3922 <\/br>Relative Rep. freq.:  0.0097","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  338 <\/br>Relative freq.:  0.5541 <\/br>Relative Rep. freq.:  0.0125","<\/br>Project:  P1 <\/br>Subject:  P1_I51_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  272 <\/br>Relative freq.:  0.4459 <\/br>Relative Rep. freq.:  0.0101","<\/br>Project:  P1 <\/br>Subject:  P1_I52_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  232 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0258","<\/br>Project:  P1 <\/br>Subject:  P1_I53_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  576 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0264","<\/br>Project:  P1 <\/br>Subject:  P1_I54_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  533 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0251","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  328 <\/br>Relative freq.:  0.5054 <\/br>Relative Rep. freq.:  0.0181","<\/br>Project:  P1 <\/br>Subject:  P1_I55_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  321 <\/br>Relative freq.:  0.4946 <\/br>Relative Rep. freq.:  0.0177","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  240 <\/br>Relative freq.:  0.5854 <\/br>Relative Rep. freq.:  0.0126","<\/br>Project:  P1 <\/br>Subject:  P1_I56_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  170 <\/br>Relative freq.:  0.4146 <\/br>Relative Rep. freq.:  0.0089","<\/br>Project:  P1 <\/br>Subject:  P1_I57_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  574 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.025","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  121 <\/br>Relative freq.:  0.5063 <\/br>Relative Rep. freq.:  0.0163","<\/br>Project:  P1 <\/br>Subject:  P1_I58_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  118 <\/br>Relative freq.:  0.4937 <\/br>Relative Rep. freq.:  0.0159","<\/br>Project:  P1 <\/br>Subject:  P1_I59_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  823 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0305","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  520 <\/br>Relative freq.:  0.6176 <\/br>Relative Rep. freq.:  0.0166","<\/br>Project:  P1 <\/br>Subject:  P1_I6_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  322 <\/br>Relative freq.:  0.3824 <\/br>Relative Rep. freq.:  0.0103","<\/br>Project:  P1 <\/br>Subject:  P1_I60_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1198 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0375","<\/br>Project:  P1 <\/br>Subject:  P1_I61_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  42 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0321","<\/br>Project:  P1 <\/br>Subject:  P1_I62_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  630 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0331","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  457 <\/br>Relative freq.:  0.5187 <\/br>Relative Rep. freq.:  0.0211","<\/br>Project:  P1 <\/br>Subject:  P1_I63_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  424 <\/br>Relative freq.:  0.4813 <\/br>Relative Rep. freq.:  0.0196","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  353 <\/br>Relative freq.:  0.5533 <\/br>Relative Rep. freq.:  0.019","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  284 <\/br>Relative freq.:  0.4451 <\/br>Relative Rep. freq.:  0.0153","<\/br>Project:  P1 <\/br>Subject:  P1_I64_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0016 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I65_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  443 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0239","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  657 <\/br>Relative freq.:  0.9985 <\/br>Relative Rep. freq.:  0.0337","<\/br>Project:  P1 <\/br>Subject:  P1_I66_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0015 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I67_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  283 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0242","<\/br>Project:  P1 <\/br>Subject:  P1_I68_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  755 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0231","<\/br>Project:  P1 <\/br>Subject:  P1_I69_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  290 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.039","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  546 <\/br>Relative freq.:  0.6531 <\/br>Relative Rep. freq.:  0.0241","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  289 <\/br>Relative freq.:  0.3457 <\/br>Relative Rep. freq.:  0.0127","<\/br>Project:  P1 <\/br>Subject:  P1_I7_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  717 <\/br>Relative freq.:  0.9986 <\/br>Relative Rep. freq.:  0.0272","<\/br>Project:  P1 <\/br>Subject:  P1_I70_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  1 <\/br>Relative freq.:  0.0014 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I71_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  704 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0218","<\/br>Project:  P1 <\/br>Subject:  P1_I72_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1043 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0337","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  360 <\/br>Relative freq.:  0.5556 <\/br>Relative Rep. freq.:  0.0203","<\/br>Project:  P1 <\/br>Subject:  P1_I73_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  288 <\/br>Relative freq.:  0.4444 <\/br>Relative Rep. freq.:  0.0162","<\/br>Project:  P1 <\/br>Subject:  P1_I74_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  807 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0326","<\/br>Project:  P1 <\/br>Subject:  P1_I75_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  526 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0309","<\/br>Project:  P1 <\/br>Subject:  P1_I76_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  714 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0306","<\/br>Project:  P1 <\/br>Subject:  P1_I77_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  433 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0254","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1042 <\/br>Relative freq.:  0.999 <\/br>Relative Rep. freq.:  0.0486","<\/br>Project:  P1 <\/br>Subject:  P1_I78_S1 <\/br>Alleles:  V5-51*07 <\/br># assignments:  1 <\/br>Relative freq.:  0.001 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  754 <\/br>Relative freq.:  0.9987 <\/br>Relative Rep. freq.:  0.0434","<\/br>Project:  P1 <\/br>Subject:  P1_I79_S1 <\/br>Alleles:  V5-51*07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0013 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I8_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1365 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0424","<\/br>Project:  P1 <\/br>Subject:  P1_I80_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1148 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0394","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  364 <\/br>Relative freq.:  0.6007 <\/br>Relative Rep. freq.:  0.0165","<\/br>Project:  P1 <\/br>Subject:  P1_I81_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  242 <\/br>Relative freq.:  0.3993 <\/br>Relative Rep. freq.:  0.011","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  165 <\/br>Relative freq.:  0.5593 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P1 <\/br>Subject:  P1_I82_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  130 <\/br>Relative freq.:  0.4407 <\/br>Relative Rep. freq.:  0.0106","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  183 <\/br>Relative freq.:  0.5126 <\/br>Relative Rep. freq.:  0.0132","<\/br>Project:  P1 <\/br>Subject:  P1_I83_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  174 <\/br>Relative freq.:  0.4874 <\/br>Relative Rep. freq.:  0.0125","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  312 <\/br>Relative freq.:  0.527 <\/br>Relative Rep. freq.:  0.0134","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  279 <\/br>Relative freq.:  0.4713 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P1 <\/br>Subject:  P1_I84_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0017 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  128 <\/br>Relative freq.:  0.5664 <\/br>Relative Rep. freq.:  0.0142","<\/br>Project:  P1 <\/br>Subject:  P1_I85_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  98 <\/br>Relative freq.:  0.4336 <\/br>Relative Rep. freq.:  0.0109","<\/br>Project:  P1 <\/br>Subject:  P1_I86_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  228 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0123","<\/br>Project:  P1 <\/br>Subject:  P1_I87_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  635 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0248","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  484 <\/br>Relative freq.:  0.557 <\/br>Relative Rep. freq.:  0.0151","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  383 <\/br>Relative freq.:  0.4407 <\/br>Relative Rep. freq.:  0.012","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I88_S1 <\/br>Alleles:  V5-51*07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I89_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  709 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.032","<\/br>Project:  P1 <\/br>Subject:  P1_I9_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  307 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0195","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  447 <\/br>Relative freq.:  0.5186 <\/br>Relative Rep. freq.:  0.0145","<\/br>Project:  P1 <\/br>Subject:  P1_I90_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  415 <\/br>Relative freq.:  0.4814 <\/br>Relative Rep. freq.:  0.0135","<\/br>Project:  P1 <\/br>Subject:  P1_I91_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  605 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0268","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  200 <\/br>Relative freq.:  0.565 <\/br>Relative Rep. freq.:  0.0218","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  153 <\/br>Relative freq.:  0.4322 <\/br>Relative Rep. freq.:  0.0167","<\/br>Project:  P1 <\/br>Subject:  P1_I92_S1 <\/br>Alleles:  V5-51*07 <\/br># assignments:  1 <\/br>Relative freq.:  0.0028 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I93_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  772 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0352","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  263 <\/br>Relative freq.:  0.556 <\/br>Relative Rep. freq.:  0.0215","<\/br>Project:  P1 <\/br>Subject:  P1_I94_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  210 <\/br>Relative freq.:  0.444 <\/br>Relative Rep. freq.:  0.0172","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  338 <\/br>Relative freq.:  0.9971 <\/br>Relative Rep. freq.:  0.0276","<\/br>Project:  P1 <\/br>Subject:  P1_I95_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  1 <\/br>Relative freq.:  0.0029 <\/br>Relative Rep. freq.:  1e-04","<\/br>Project:  P1 <\/br>Subject:  P1_I96_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  36 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0327","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  866 <\/br>Relative freq.:  0.9988 <\/br>Relative Rep. freq.:  0.0398","<\/br>Project:  P1 <\/br>Subject:  P1_I97_S1 <\/br>Alleles:  V5-51*04 <\/br># assignments:  1 <\/br>Relative freq.:  0.0012 <\/br>Relative Rep. freq.:  0","<\/br>Project:  P1 <\/br>Subject:  P1_I98_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  731 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0448","<\/br>Project:  P1 <\/br>Subject:  P1_I99_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  93 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0504"],"marker":{"color":"#FAAB18","size":8,"line":{"color":"gray","width":1}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"transparent","x":[1,1,2,1,3,1,1,1,2,1,2,1,1,2,1,1,2,1,1,2,1,1,1,2,2,1,4,1,2,3,1,5,1,2,3,1,1,2,3,1,1,1,1,2,2,1,1,2,1,1,1,1,2,1,1,1,5,2,3,1,2,1,2,1,1,1,1,2,1,5,1,2,2,1,1,1,1,2,1,1,2,1,2,1,1,1,2,1,2,1,1,2,1,2,3,1,1,2,1,1,1,1,2,3,1,2,1,1,1,2,1,1,1,2,1,5,1,5,1,1,1,2,1,2,1,2,1,2,3,1,2,2,2,2,1,3,5,1,1,2,1,1,1,2,5,1,1,2,2,1,1,2,3,1,1],"y":[401,951,121,110,1,979,625,666,1,185,137,465,79,53,251,371,287,455,332,1,77,779,280,151,336,320,1,389,370,1,538,1,349,253,1,150,465,386,2,331,724,741,8,2,28,15,432,253,250,355,440,186,51,650,1005,1156,1,378,1,301,241,279,201,210,408,128,598,157,1313,1,93,60,338,272,232,576,533,328,321,240,170,574,121,118,823,520,322,1198,42,630,457,424,353,284,1,443,657,1,283,755,290,546,289,1,717,1,704,1043,360,288,807,526,714,433,1042,1,754,1,1365,1148,364,242,165,130,183,174,312,279,1,128,98,228,635,484,383,1,1,709,307,447,415,605,200,153,1,772,263,210,338,1,36,866,1,731,93],"marker":{"color":"#FAAB18","line":{"color":"rgba(255,127,14,1)"}},"type":"box","hoverinfo":["none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none"],"showlegend":false,"name":"transparent","line":{"color":"rgba(255,127,14,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="htmlwidget-f13c030e3a52826ca4a5" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-f13c030e3a52826ca4a5">{"x":{"visdat":{"9640919014387":["function () ","plotlyVisDat"]},"cur_data":"9640919014387","attrs":{"9640919014387":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatter","x":{},"y":{},"text":{},"marker":{"color":"#588300","size":8,"line":{"width":1,"color":"gray"}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":"text","inherit":true},"9640919014387.1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"marker":{"color":"#588300"},"type":"box","hoverinfo":"none","showlegend":false,"fillcolor":"transparent","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","xaxis":{"domain":[0,1],"automargin":true,"title":"Alleles","autotick":false,"tickmode":"array","tickvals":[1,2,3,4,5],"ticktext":["V5-51*01","V5-51*03","V5-51*04","V5-51*06","V5-51*07"]},"yaxis":{"domain":[0,1],"automargin":true,"title":"# Sequences"},"annotations":[{"text":"P11","x":0.5,"y":1.1,"yref":"paper","xref":"paper","xanchor":"middle","yanchor":"top","showarrow":false,"font":{"size":15}}],"showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"type":"scatter","x":[2.06232370045036,1.18298760699108,1.07471630070359,1.0926977946423,1.05558796795085,0.950010646693409,0.847794506233186,1.19721481073648,1.02910557491705,0.875763555336744,1.08265316002071,0.830818195920438,1.02845052201301,0.885904369223863,1.06430769320577,1.17758979042992,1.19625714942813,1.13318961653858,1.127828447707,1.02363268043846,2.14768753852695,0.998356701247394,0.98062675781548,1.18353279810399,2.12432870250195,0.833624216169119,0.844071211479604,2.09032215345651,0.988373215310276,2.16479660095647,0.963686338346452,1.84548487970606,0.904474036581814,1.04016431570053,0.905298504885286,1.91077833678573,1.93300269171596,1.13860529446974,0.870496686827391,2.01311906222254,0.914205919671804,1.81955810496584,1.03292996007949,2.14598494805396,0.836678010597825,2.13319918848574,0.983289940934628,0.863971826247871,1.14710976611823,1.82944837678224,1.00850125523284,1.02843744726852,1.07794126821682,0.838586393371224,1.84237918052822],"y":[221,177,287,290,46,498,383,395,574,292,384,464,287,381,186,259,488,360,141,425,175,497,271,465,274,184,324,289,291,290,268,145,361,395,170,170,105,89,384,353,174,115,414,954,173,140,113,218,265,230,489,395,356,252,183],"text":["<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  221 <\/br>Relative freq.:  0.5553 <\/br>Relative Rep. freq.:  0.039","<\/br>Project:  P11 <\/br>Subject:  P11_I1_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  177 <\/br>Relative freq.:  0.4447 <\/br>Relative Rep. freq.:  0.0312","<\/br>Project:  P11 <\/br>Subject:  P11_I10_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  287 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0759","<\/br>Project:  P11 <\/br>Subject:  P11_I11_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  290 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0651","<\/br>Project:  P11 <\/br>Subject:  P11_I12_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  46 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0547","<\/br>Project:  P11 <\/br>Subject:  P11_I13_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  498 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.069","<\/br>Project:  P11 <\/br>Subject:  P11_I14_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  383 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0657","<\/br>Project:  P11 <\/br>Subject:  P11_I15_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  395 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0779","<\/br>Project:  P11 <\/br>Subject:  P11_I16_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  574 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0728","<\/br>Project:  P11 <\/br>Subject:  P11_I17_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  292 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0536","<\/br>Project:  P11 <\/br>Subject:  P11_I18_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  384 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0633","<\/br>Project:  P11 <\/br>Subject:  P11_I19_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  464 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0569","<\/br>Project:  P11 <\/br>Subject:  P11_I2_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  287 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0576","<\/br>Project:  P11 <\/br>Subject:  P11_I20_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  381 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0651","<\/br>Project:  P11 <\/br>Subject:  P11_I21_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  186 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0481","<\/br>Project:  P11 <\/br>Subject:  P11_I22_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  259 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0622","<\/br>Project:  P11 <\/br>Subject:  P11_I23_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  488 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0661","<\/br>Project:  P11 <\/br>Subject:  P11_I24_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  360 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0724","<\/br>Project:  P11 <\/br>Subject:  P11_I25_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  141 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0645","<\/br>Project:  P11 <\/br>Subject:  P11_I26_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  425 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0715","<\/br>Project:  P11 <\/br>Subject:  P11_I27_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  175 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0507","<\/br>Project:  P11 <\/br>Subject:  P11_I28_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  497 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0725","<\/br>Project:  P11 <\/br>Subject:  P11_I29_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  271 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0615","<\/br>Project:  P11 <\/br>Subject:  P11_I3_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  465 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0623","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  274 <\/br>Relative freq.:  0.5983 <\/br>Relative Rep. freq.:  0.048","<\/br>Project:  P11 <\/br>Subject:  P11_I30_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  184 <\/br>Relative freq.:  0.4017 <\/br>Relative Rep. freq.:  0.0323","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  324 <\/br>Relative freq.:  0.5285 <\/br>Relative Rep. freq.:  0.0373","<\/br>Project:  P11 <\/br>Subject:  P11_I31_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  289 <\/br>Relative freq.:  0.4715 <\/br>Relative Rep. freq.:  0.0333","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  291 <\/br>Relative freq.:  0.5009 <\/br>Relative Rep. freq.:  0.0279","<\/br>Project:  P11 <\/br>Subject:  P11_I32_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  290 <\/br>Relative freq.:  0.4991 <\/br>Relative Rep. freq.:  0.0278","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  268 <\/br>Relative freq.:  0.6489 <\/br>Relative Rep. freq.:  0.0523","<\/br>Project:  P11 <\/br>Subject:  P11_I33_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  145 <\/br>Relative freq.:  0.3511 <\/br>Relative Rep. freq.:  0.0283","<\/br>Project:  P11 <\/br>Subject:  P11_I34_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  361 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0816","<\/br>Project:  P11 <\/br>Subject:  P11_I35_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  395 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0722","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  170 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0326","<\/br>Project:  P11 <\/br>Subject:  P11_I36_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  170 <\/br>Relative freq.:  0.5 <\/br>Relative Rep. freq.:  0.0326","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  105 <\/br>Relative freq.:  0.5412 <\/br>Relative Rep. freq.:  0.0273","<\/br>Project:  P11 <\/br>Subject:  P11_I37_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  89 <\/br>Relative freq.:  0.4588 <\/br>Relative Rep. freq.:  0.0231","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  384 <\/br>Relative freq.:  0.521 <\/br>Relative Rep. freq.:  0.0422","<\/br>Project:  P11 <\/br>Subject:  P11_I38_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  353 <\/br>Relative freq.:  0.479 <\/br>Relative Rep. freq.:  0.0388","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  174 <\/br>Relative freq.:  0.6021 <\/br>Relative Rep. freq.:  0.0571","<\/br>Project:  P11 <\/br>Subject:  P11_I39_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  115 <\/br>Relative freq.:  0.3979 <\/br>Relative Rep. freq.:  0.0377","<\/br>Project:  P11 <\/br>Subject:  P11_I4_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  414 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0834","<\/br>Project:  P11 <\/br>Subject:  P11_I40_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  954 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0622","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  173 <\/br>Relative freq.:  0.5527 <\/br>Relative Rep. freq.:  0.0406","<\/br>Project:  P11 <\/br>Subject:  P11_I41_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  140 <\/br>Relative freq.:  0.4473 <\/br>Relative Rep. freq.:  0.0329","<\/br>Project:  P11 <\/br>Subject:  P11_I42_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  113 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.1011","<\/br>Project:  P11 <\/br>Subject:  P11_I43_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  218 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0734","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  265 <\/br>Relative freq.:  0.5354 <\/br>Relative Rep. freq.:  0.0329","<\/br>Project:  P11 <\/br>Subject:  P11_I5_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  230 <\/br>Relative freq.:  0.4646 <\/br>Relative Rep. freq.:  0.0286","<\/br>Project:  P11 <\/br>Subject:  P11_I6_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  489 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0903","<\/br>Project:  P11 <\/br>Subject:  P11_I7_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  395 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0706","<\/br>Project:  P11 <\/br>Subject:  P11_I8_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  356 <\/br>Relative freq.:  1 <\/br>Relative Rep. freq.:  0.0696","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  V5-51*01 <\/br># assignments:  252 <\/br>Relative freq.:  0.5793 <\/br>Relative Rep. freq.:  0.0347","<\/br>Project:  P11 <\/br>Subject:  P11_I9_S1 <\/br>Alleles:  V5-51*03 <\/br># assignments:  183 <\/br>Relative freq.:  0.4207 <\/br>Relative Rep. freq.:  0.0252"],"marker":{"color":"#588300","size":8,"line":{"color":"gray","width":1}},"mode":"markers","showlegend":false,"opacity":0.8,"hoverinfo":["text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text","text"],"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"transparent","x":[2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,1,1,1,2,1,1,2,1,2,1,2,1,1,1,2,2,1,1,2,1,2,1,2,1,2,1,1,1,2,1,1,1,1,2],"y":[221,177,287,290,46,498,383,395,574,292,384,464,287,381,186,259,488,360,141,425,175,497,271,465,274,184,324,289,291,290,268,145,361,395,170,170,105,89,384,353,174,115,414,954,173,140,113,218,265,230,489,395,356,252,183],"marker":{"color":"#588300","line":{"color":"rgba(255,127,14,1)"}},"type":"box","hoverinfo":["none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none","none"],"showlegend":false,"name":"transparent","line":{"color":"rgba(255,127,14,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="cutoff-37" class="section level2">
<h2>0.5% cutoff</h2>
<p>We set an initial cutoff of <span class="math inline">\(0.5\%\)</span> to determine the potential genotype priors. For this cutoff we examined the zygousity state, such as homozygousity, heterozygousity and so on.</p>
<p>With the selected cutoff we saw that there are 2 zygousity states.</p>
<style>
  
#scroll-box {
  overflow-y: scroll;
  overflow-x: scroll !important;
}
</style>
<div class="container">
<iframe id="scroll-box" src="https://peresay.shinyapps.io/relative_usage_haplo/?g_group=%22IGHV5-51G47%22" scrolling="yes" border="0" frameborder="0" style="border-style:none;box-shadow:0px 0px 2px 2px rgba(0,0,0,0.2); width: 100%;height:800px;overflow-x: scroll !important;overflow: scroll;" cellspacing="0">
</iframe>
</div>
</div>
<div id="observations-43" class="section level2">
<h2>Observations</h2>
<p>This section is editable by clicking on the edit button below. To refresh the section click on the refresh button</p>
<p>You can access the file also from <a href="" target="_blank">here</a></p>
<iframe src="https://peresay.shinyapps.io/G2_group/?group=IGHV5-51G47" width="672" height="400px" data-external="1">
</iframe>
</div>
<div id="conclusions-44" class="section level2">
<h2>Conclusions</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="46-G46.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="48-G48.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ayeletperes/reference_book/edit/master/47-G47.rmd",
"text": "Suggest an edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
